---
layout: page
title: "FHIR Chat  · Inclusion of Sanger/MLPA confirmation information · genomics / eMerge Pilot"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/index.html">genomics / eMerge Pilot</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html">Inclusion of Sanger/MLPA confirmation information</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="160154999"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160154999" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Mullai Murugan <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160154999">(Mar 07 2019 at 00:28)</a>:</h4>
<p>For our clinical genetic tests, we validate SNVs using Sanger and CNVs using MLPA.  Would the Described or Complex Variant be the appropriate resource to include validation info? If so, what would be an appropriate field to house this info?</p>
<p>As also detailed in a gforge tracking item <a href="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19829" target="_blank" title="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19829">#19829</a></p>



<a name="160202823"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160202823" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160202823">(Mar 07 2019 at 14:40)</a>:</h4>
<p>We really have not talked about this one.  At least I haven't.</p>
<p>My first question is - if you detected the variant with NGS, then confirmed with Sanger - it is still one variant to report, right?  I almost think of it as the same Variant, but with 2 methods?   Does that make sense to others?  (I say this noting that Observation.method is 0..1).</p>
<p>This might be a good question for the <a class="stream" data-stream-id="179256" href="/#narrow/stream/179256-Orders-and-Observation-WG">#Orders and Observation WG</a> stream - at least to see if they have conceptually thought about how to appropriately model confirmation testing results.</p>



<a name="160209391"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160209391" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160209391">(Mar 07 2019 at 15:52)</a>:</h4>
<p>I think it might be important to note which results are with each method.  Recommend different LOINCs for each to reflect those differences so folks/computers don't comingle the two....</p>



<a name="160210148"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160210148" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160210148">(Mar 07 2019 at 16:00)</a>:</h4>
<p>Yea, that is fair.  Lots of other metadata (who, when, ...) that would be different.  I do think it will be important to link the NGS Variant with the Sanger Variant though - something like a 'confirms' attribute from the Sanger Variant to the NGS Variant?</p>



<a name="160211843"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160211843" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160211843">(Mar 07 2019 at 16:15)</a>:</h4>
<p>Right.  Thinking like how LOINC models a screening test for X and confirmatory test for X.  It makes a difference for downstream end users from a positive predictive Value, AUC reporting, etc.</p>



<a name="160212724"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160212724" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160212724">(Mar 07 2019 at 16:23)</a>:</h4>
<p>Yet another area we need additional guidance and examples <span aria-label="frown" class="emoji emoji-1f641" role="img" title="frown">:frown:</span></p>



<a name="160218997"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160218997" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160218997">(Mar 07 2019 at 17:35)</a>:</h4>
<p>Sounds like the use case <span class="user-mention" data-user-id="209510">@Larry Babb</span>  shared would be a starting place. Perhaps others can ask their labs for more examples of screening vs more in depth testing (like whole genome sequencing).  How do we get more folks from the laboratories to share?  Can more folks join from these initiatives or is it a timing issue?</p>



<a name="160235242"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160235242" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160235242">(Mar 07 2019 at 20:36)</a>:</h4>
<p>My perspective and experience with labs reporting validated NGS data is that they write up a methodology section that indicates how and when they perform validation and with what technology.  They don't report or even necessarily capture the variant record from both assays independently. They ultimately are trying to get to the point of expressing their representation of the variant (a representation that all labs wish was standardized - so mostly HGVS at this point). In our messages we will provide a single representation of the variant reported (not one for each methodology) and we'd like to potentially annotate that the variant was confirmed at a minimum.  How it was confirmed would require a lookup of the methodology and presumption that the methodology section is accurate and has enough structure that a consuming system could derive what technology was used (if that was even necessary).</p>
<p>I believe the important fact that downstream folks would care about is merely "was the variant confirmed".</p>



<a name="160235628"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160235628" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160235628">(Mar 07 2019 at 20:41)</a>:</h4>
<p>Certainly, something to indicate the confirmation would be much easier to address in our model.</p>



<a name="160239446"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160239446" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160239446">(Mar 07 2019 at 21:30)</a>:</h4>
<p>Agree that the downstream folks only want to know variant.  Suspect due to US lab accreditation/CLIA requirements, the original screening results, methods, etc are captured in the internal laboratory information system (LIS).  Test system validation and other quality data would need to be available to CMS inspectors, etc.  Most all this is internal to the performing lab, and not released externally.  </p>
<p>Also would be easy to confirm by looking at test compendium/CLIA specimen collection manual information about the test orders as physicians need to know what they get with their order and if it meets their clinical need before they order and collect patient specimens.  (Realize we haven't discussed order info hardly at all on CG calls.)</p>



<a name="160243129"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160243129" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160243129">(Mar 07 2019 at 22:17)</a>:</h4>
<p>We could add a new Observation.component[] in our Variant profile called something like "Confirmation" - and that could be a simple boolean or perhaps we develop our own value set with some reasonable terms in it - like standard methods for confirming?</p>



<a name="160243278"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160243278" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160243278">(Mar 07 2019 at 22:19)</a>:</h4>
<p>Do we want/need to indicate other findings being confirmed?  I don't think so, but wanted to make sure I wasn't missing something.</p>



<a name="160293115"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160293115" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160293115">(Mar 08 2019 at 14:22)</a>:</h4>
<p>like the idea of a confirms link to an observation with methodology specified for specific results confirmation. BUT if it is routine for the test always be confirmed by such and such sequencing method then this seems more apart of the test methodology on the variant. <span class="user-mention" data-user-id="209510">@Larry Babb</span>  which is the case for your use case?</p>



<a name="160300989"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160300989" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160300989">(Mar 08 2019 at 15:57)</a>:</h4>
<p>Disclaimer: I am not a domain expert on methodology design and assay processing.<br>
I believe the methodology for the overall test typically describes the conditions under which they variant is confirmed by and orthogonal technology (and what that is). I believe that any "clinically significant" variant (path or Lik path) would be confirmed and typically be a given technology (like Sanger), but benign variants would not (necessarily).  I think each lab defines things there own way.  I believe some labs don't even confirm significant variants or use the same tech do confirm it. </p>
<p>In regards to having a "confirms" link on each variant finding observation that points to another observation with the methodology for the confirmation, I think that is an interesting proposal, but would need to be fleshed out. First, I would consider using the ObservationDefinition concept that <span class="user-mention" data-user-id="196271">@Andrea Pitkus, PhD, MLS(ASCP)CM, CSM</span> has enlightened me on related to the OO Catalog work that is underway. But even that seems a bit heavy. Maybe the best solution is a simple flag or coded Y/N/U field called "confirmationTested" on any ObsDescribedVariant or MolecularSequence item since they seem to be the lowest level of genetic findings (maybe it should be on ObsGenotype and ObsHap... and ObsComplex.. too?).  </p>
<p>It's messy because there's so many forms and profiles for representing variants (i think).  Anyway, sticking a "confirmationTested" field somewhere would be a good step in the right direction. </p>
<p>I do believe we will eventually find a path foward whereby we can model the variant definitions more formally with explicit resources and then use them in a clearer approach to passing Observations about the variantDefinitions found, associated to interpretations, contributing to overall interps, etc....</p>



<a name="160320830"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/Inclusion%20of%20Sanger/MLPA%20confirmation%20information/near/160320830" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/Inclusion.20of.20Sanger.2FMLPA.20confirmation.20information.html#160320830">(Mar 08 2019 at 20:06)</a>:</h4>
<p>Concur with your 1st paragraph.  Not all variants detected are reported.</p>
<p>be interested more in how the report is currently structured in pdf/paper format to report variants as labs structure them differently too.  I have seen some labs report (not for variants or genomics, but other testing) the result and method used to obtain the result in a separate line/field as described.  I wouldn't use Observation Definition as it's more the catalog (but would be used to describe the variants and all the test information shown for a random test order here:  <a href="https://www.bcm.edu/research/medical-genetics-labs/test_detail.cfm?testcode=22350" target="_blank" title="https://www.bcm.edu/research/medical-genetics-labs/test_detail.cfm?testcode=22350">https://www.bcm.edu/research/medical-genetics-labs/test_detail.cfm?testcode=22350</a>  )  May wish to start with simpler examples and work up to more complex.  With reflexive orders, it may list order than could be added on to the original specimen/order requisition/service request, assuming they are built separately so billing charges can be dropped as more testing (i.e. confirmatory) is added.  Depending on the type of confirmation, seems plausible that any of the items described might be confirmed.</p>
<p>Totally agree that there is quite a variety out there currently and can be messy.  Concur on path.  May have a few twists and turns in the meanwhile. ;)</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
