---
layout: page
title: "FHIR Chat  · relatedArtifact extension change request · genomics / eMerge Pilot"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/index.html">genomics / eMerge Pilot</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html">relatedArtifact extension change request</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="178391193"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/178391193" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#178391193">(Oct 17 2019 at 15:19)</a>:</h4>
<p>The emerge pilot would like to request expanding the use of the "relatedArtifact" extension from DR and Observation to include Observation.component as well.<br>
In the case of Medication Implication profiles, the component for "medication-assessed" often requires a supporting related artifact specific to a medication within the overall implication.  We thought we could get away with including all related artifacts at the Observation level but often times the related artifact only applies to one of the several assessed medications defined by the individual component entries.  Being able to explicitly link the correct related artifact to the assessed medication within a medication implication observation is needed. </p>
<p>We are going forward with our own extension for the time being, but we are hoping the above use case is persuasive enough to simply expand the ability to use the RelatedArtifact extension in components as well as DRs and OBSs.</p>
<p>Let us know if this is a possibility or if I should add a tracker for it.</p>



<a name="178395126"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/178395126" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#178395126">(Oct 17 2019 at 16:00)</a>:</h4>
<p>We have a pending change that may help you--we voted (<a href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=24879" target="_blank" title="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=24879">GF#24879</a>) to remove the restriction from using Observation.component.interpretation (CodeableConcept 0..*). It may not semantically be the best fit for supporting info but it does give you the option to supply text telling which relatedArtifact applies to the component.</p>



<a name="178410009"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/178410009" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#178410009">(Oct 17 2019 at 18:49)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@James Jones</span> are you suggesting that we would use the CodeableConcept.text property to describe the mappings between related artifacts captured in the Observation.RelatedArtifact extension with the Medication assessed Components?  I'd have to think about how that could potentially work in practice.  At first brush it seems like a much trickier association than simply enabling the relatedartifact extension to be used on the component. please clarify your proposal if I'm missing something.</p>



<a name="178412986"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/178412986" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#178412986">(Oct 17 2019 at 19:21)</a>:</h4>
<p>Yes, I was suggesting it would be possible to use component.interpretation to add additional info for each component as it would be technically compliant with the build version of the IG after applying that tracker.<br>
However, after looking at the way interpretation is presented on Observation, I agree that sending relatedArtifacts at that level is a much better solution. I hope that extension as a whole gets reworked in short order.</p>



<a name="178705381"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/178705381" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#178705381">(Oct 21 2019 at 22:31)</a>:</h4>
<p><span class="user-mention" data-user-id="209510">@Larry Babb</span>  what do you think of using a single Medication implication profile for each medication assessed? this would give you 1  related artifact extension in the observation per medication assessed? *which seems reasonable if  the  information in the related artifact is specific to a single medication.</p>



<a name="178866489"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/178866489" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#178866489">(Oct 23 2019 at 15:40)</a>:</h4>
<p>If we were building a definitional artifact here I would push for only sending one medication (or combination therapy) and one relatedartifact per resource, but since we are building Observations for patient-relevant results, reporting a summary of a query to a knowledge base in one observation may be appropriate. I am also wondering what other areas in the spec would benefit from permitting relatedartifact at the component level. It may be just the thing (if treated properly) to help convey much needed context and rigor for some concepts</p>



<a name="179051413"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/179051413" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#179051413">(Oct 25 2019 at 13:57)</a>:</h4>
<p>That is not how the information is arranged. Each medication has its own recommendation.  What you are describing would be a bundle of recommendations with some top level. This would be the diagnostic report, in my opinion.</p>



<a name="179051577"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/179051577" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#179051577">(Oct 25 2019 at 13:58)</a>:</h4>
<p>however, related artifact for an option to deliver specific additional knoweldge base support sounds good. Even at the component level. But for medication assessed, each recommendation is specific to the medication.</p>



<a name="179355401"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/179355401" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#179355401">(Oct 29 2019 at 17:54)</a>:</h4>
<p>Actually, I'm seeing examples where multiple medications are linked to a single recommendation (e.g. <a href="https://cpicpgx.org/guidelines/guideline-for-peg-interferon-alpha-based-regimens-and-ifnl3/" target="_blank" title="https://cpicpgx.org/guidelines/guideline-for-peg-interferon-alpha-based-regimens-and-ifnl3/">https://cpicpgx.org/guidelines/guideline-for-peg-interferon-alpha-based-regimens-and-ifnl3/</a> is linked to 3 medications peginterferon alfa-2a,  peginterferon alfa-2b and ribavirin). Some have only one, it seems to be more of a way of how they've (CPIC) crafted the recommendation than any kind of strict one-to-one with a given medication/drug.  I suppose these can be classes of drugs, but nevertheless the reporting is arbitrary and it appears to me as if the lab simply wants to cite a cpic recommendation for one or more drugs for a given pgx genotype or star allele finding and corresponding phenotype.<br>
Having the flexibility to add related artifacts to the medication assessed component of the med-implication profile allows the reporter to structure it the way they want.</p>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> In regards to your comment (2 above) about using a single med implication profile for each assessed medication... If we did this, then we'd be replicating the PGx finding/phenotype assessment for each medication assessed, when it seems like the grouping should be based on the core pairing of PGX finding/phenotype assessment.  The medications are a critical components but seem less like the subject or driver of the implication than the object or descriptor.  The recommendation is merely related or supporting info.</p>



<a name="181132104"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/181132104" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#181132104">(Nov 19 2019 at 17:03)</a>:</h4>
<p><span class="user-mention" data-user-id="209510">@Larry Babb</span> The recommendation is everything from a clinical point-of-view. Although ephemeral, it is the reason the the ordering provider ordered the test. The recommendation is not supporting information - it is ephemeral.  Your example is for similar medications - they are seen as the same thing. Each would  share the same recommendation - so in that case, putting all three together would be accurate. It is about accurately providing the recommendations.</p>



<a name="181238582"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/181238582" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#181238582">(Nov 20 2019 at 19:11)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> actually, the recommendation in this case is not everything. There's actually a lot of challenges going around on whether these CPIC recommendations should even be included in lab reports. The FDA seems to be laying down the law that these therapeutic interps and the associated drug recommendations should NOT be allowed on clinical reports. <br>
Be that as it may, the current state of PGx reporting as I understand it is that the labs want to minimally "CITE" the recommendation from CPIC (it's unclear if the FDA will even allow this at this time and for the near future).  So, I do think CITING a recommendation from a paper or secondary authority is much different than providing guidance or recommendations directly.</p>
<p>The related artifact is a piece of secondary information from the emerge PGx reporting perspective, so we are going to continue with our "relatedArtifact" extension off of the mediation assessment component of the MedicationImplication abstract profile. </p>
<p>If you still feel strongly that this is not the correct approach let me know an alternative way that I could reasonably structure the set of PGx results and their many-to-many set of relationships between the specific assessed medications and the cited CPIC recommendation documents.  I do feel that referencing the same paper in different PGx specific medication assessments is the only reasonable approach, but I could be missing something.</p>



<a name="181295884"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/181295884" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#181295884">(Nov 21 2019 at 11:33)</a>:</h4>
<p>Yes. I feel strongly that the recommendation that the lab is providing is different than citing a guideline. Many labs today include a long reference list which may or may not tie back to specific recommendations.. The recommendations are typically a mix and can be minimal.  For example, 'consider lowering dose.' Totally agree with you that this is pretty minimal. In our PGx examples,  I recall that we showed one way this might be communicated. Have you looked?</p>



<a name="181295907"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/181295907" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#181295907">(Nov 21 2019 at 11:33)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span>  can you provide the link to the examples you and I worked on for Larry? thanks!</p>



<a name="182793564"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182793564" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182793564">(Dec 06 2019 at 19:16)</a>:</h4>
<p>In terms of linking to a component, the general expectation is that each Observation should be a leaf-level statement.  If the recommendation is to reduce the dose on two drugs, that should be two Observations.  Do we have a sense of what the use-case is to link multiple medications?</p>



<a name="182794852"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182794852" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182794852">(Dec 06 2019 at 19:28)</a>:</h4>
<p>This was the example from <span class="user-mention" data-user-id="209510">@Larry Babb</span> earlier:</p>
<blockquote>
<p>Actually, I'm seeing examples where multiple medications are linked to a single recommendation (e.g. <a href="https://cpicpgx.org/guidelines/guideline-for-peg-interferon-alpha-based-regimens-and-ifnl3/" target="_blank" title="https://cpicpgx.org/guidelines/guideline-for-peg-interferon-alpha-based-regimens-and-ifnl3/">https://cpicpgx.org/guidelines/guideline-for-peg-interferon-alpha-based-regimens-and-ifnl3/</a> is linked to 3 medications peginterferon alfa-2a, peginterferon alfa-2b and ribavirin). Some have only one, it seems to be more of a way of how they've (CPIC) crafted the recommendation than any kind of strict one-to-one with a given medication/drug. I suppose these can be classes of drugs, but nevertheless the reporting is arbitrary and it appears to me as if the lab simply wants to cite a cpic recommendation for one or more drugs for a given pgx genotype or star allele finding and corresponding phenotype.<br>
Having the flexibility to add related artifacts to the medication assessed component of the med-implication profile allows the reporter to structure it the way they want.</p>
</blockquote>



<a name="182864345"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182864345" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182864345">(Dec 07 2019 at 22:08)</a>:</h4>
<p>uh, "allows the reporter to structure it the way they want" FHIR is not a Document. The user experience of consuming the report is through  consuming the FHIR resource and coding internally to your system.  If one wants the structure to look like the PDF report then one might use CCDA instead and design an XML template for the report. FHIR is not a markup language for viewing.</p>



<a name="182865859"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182865859" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182865859">(Dec 07 2019 at 22:55)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> You can define a FHIR document that has the same rendering capabilities that CDA has.  (With the benefit of the discrete data being represented in FHIR.)  So if you wanted to have tight control over the rendering of a genetic report you could throw in a Composition and package up the DiagnosticReport and all associated Observations into a FHIR document.</p>



<a name="182866634"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182866634" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182866634">(Dec 07 2019 at 23:21)</a>:</h4>
<p><span class="user-mention" data-user-id="191320">@Lloyd McKenzie</span>  sure it is possible. But out IG is not a FHIR document. As  you note, there is a mechanism. Thanks for the clarification</p>



<a name="182867967"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182867967" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182867967">(Dec 08 2019 at 00:04)</a>:</h4>
<p>Our IG is independent of whether an implementer chooses documents, REST, messaging or something else.</p>



<a name="182922316"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182922316" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182922316">(Dec 09 2019 at 03:47)</a>:</h4>
<blockquote>
<p>In terms of linking to a component, the general expectation is that each Observation should be a leaf-level statement.  If the recommendation is to reduce the dose on two drugs, that should be two Observations.  Do we have a sense of what the use-case is to link multiple medications?</p>
</blockquote>
<p><span class="user-mention" data-user-id="191320">@Lloyd McKenzie</span> <br>
the medications mentioned in a given observation are all associated with the same pgx  phenotype (eg extensive metabolizer). Additionally each medication may have zero to many cited recommendations from a professional society that describes in greater detail the phenotype and the research related to the recommendation. But the reporting lab is in NO way making a recommendation for the physician or the patient. They are merely referencing information that may be helpful to the physician in developing a treatment plan. Some of these cited guidelines in Pgx are for multiple drug phenotype genotype associations. So I have been structuring these cpic professional guideline references as related artifacts with the specific medication associated with the genotype phenotype observation.  It provides the best flexibility for being able to link these related artifacts to the correct medications within these observations.</p>



<a name="182926573"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/182926573" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#182926573">(Dec 09 2019 at 05:54)</a>:</h4>
<p>Right, but if we're wanting to point to a specific medication issue, that suggests each medication issue should be a separate statement (even if they're all tied to the same phenotype)</p>



<a name="183003574"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/183003574" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#183003574">(Dec 09 2019 at 21:44)</a>:</h4>
<p>I'm not sure where the requirement is that we have specific medication issues. The statement (as I understand it) is that "this PGx diplotype, haplotype, genotype, etc.. is associated with a PGX phenotype (e.g. extensive metabolism, responsive-efficacy, etc..)." The class of medications that fall under that pgx phenotype are all assessed together.  If any of them vary or are unique in any way from the others, then they would be stated separately.</p>
<p>I've yet to see a situation where the PGX variant genotype-phenotype was different for one or more drugs (for example var1-ext metabolizer=clipidogrel while var1-moderate metabolizer=drug 2?). In the end, the PGx Var1 is assessed to cause a certain class or set of drugs to be have a certain pgx phenotype.  I'm sure the use case suggested may happen, but these statements are collectively representing a set of drugs/drug classes. </p>
<p>I also think we put much more weight on these guidelines by referring to the associated drugs called out in these PGX genotype/phenotype statements as "medications".  While it is certainly reasonable to call these drugs medications, it should be clear that the lab is not in any way making a statement about patient treatment (at least not in the germline use cases). They are providing knowledge that may or may not be helpful to a given patient's case and one which would presumably provide helpful information to the treating physician to make a final decision on which medication treatment plan to prescribe if one is needed at all.</p>
<p>If you scan through the CPIC guidelines you will see that they refer to our term "medications" as drugs.  Again, this may be a silly point to harp on, but it is interesting why we didn't follow their lead in using their terminology. Especially since they are the standards group we are aiming to support.  Might it be possible that we have read more into this observation than what was needed at least for the CPIC guideline knowledge reporting use cases.</p>
<p>In the end, if you think it wise to replicate the genotype-phenotype observation for each "medication" (drug) that's the call of the group. If so, then let's simply change the cardinality on the component to be 1..1 (not 1..*). Otherwise we should clarify when multiple medications (drugs) would make sense. (maybe it they are prescribed together and assessed together? - that's one I've never seen in germline PGX guidelines - but it may be so)</p>



<a name="183023043"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/183023043" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#183023043">(Dec 10 2019 at 02:45)</a>:</h4>
<p><span class="user-mention" data-user-id="209510">@Larry Babb</span> I believe the suggestion is that you would be reusing the same variant in the link but a different medication implication for each medication generic. Multiple medications make sense if there is the same implication for multiple medications. but the medications have to have the same implication.</p>



<a name="183050379"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/183050379" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#183050379">(Dec 10 2019 at 11:17)</a>:</h4>
<p>correct. They do have the same implication. The implication is the phenotype. Several drugs are related to the same genotype/phenotype observation (or assessed knowledge) that is being summarized based on the CPIC study (guidance doc) that is provided as a basis for more detailed information about the assessment by CPIC and why this lab determined it was the phenotype they chose to share with the physician.</p>
<p>This is why the lab and cpic aggregate these genotype-phenotype assertions together and provide related artifacts detailing the associated drug-specific study background for supporting information.</p>
<p>In the end, it probably doesn't matter whether we share this data as independent observations or aggregations. If we had the ability to share actual asserted knowledge independent of patient data (like ClinVar or VICC or other knowledge repos) then it would be a stronger argument for medication specific assertions with evidence and supporting information. But at the clinical patient reporting level the labs will be reporting the aggregate assertion about the variant (genotype) finding in the context of the pgx phenotype and summarize the drugs that are implicated.</p>



<a name="183554421"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/183554421" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#183554421">(Dec 16 2019 at 14:41)</a>:</h4>
<p>related artifact can refer to ClinVar, VICC, or other knowledge repos to share the knowledge that is independent of patient data. regards same implication - so You would have the structure Variants X, Y and Z all pointing to the same implication if the medications were mentioned in the same CPIC guideline and the variants has the same inferred phenotype. Right <span class="user-mention" data-user-id="209510">@Larry Babb</span> ? you'll probably have separate Tasks for  increased or decreased dose.</p>



<a name="183591845"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/183591845" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#183591845">(Dec 16 2019 at 21:18)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> I don't usually infer more than what is provided by the report or the geneticist that is directing our work. I understand the desire to aggregate based on the cpic guideline, but I'm unsure that is what is really happening. I primarily observe the way they present the information then ask them what the intention is. To date, my understanding is that the labs are simply citing a reference to the guideline but taking responsibility for the assertion (using evidence providing by the guideline) of the phenotype to a given genotype. Since it is possible and has been observed that some supporting CPIC guideline reference citations differ based on the medication, we decided it was best to aggregate our observations based on the genotype-phenotype and then provide secondary references to the mediations and their supporting evidence (related artifact citations).</p>
<p>Labs do not (and should not - based on recent FDA warnings) be guiding dosage recommendations.  This is a hot button issue right now and one that we should not structure in a way that makes it appear as if the labs are providing actual or even proposed guidance. </p>
<p>It is my understanding that labs may have to omit all references to medications in general until the FDA can determine what the right way to share this information without potentially harming patients.</p>
<p>So we are pretty certain that the cpic guideline references are citations that are secondarily used to back the mediation that is a component of the genotype - phenotype (derivedFrom variant finding - code.value) in the medication implication profiles for all pgx information in the emerge RoR.</p>



<a name="183662865"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/189875-genomics%20/%20eMerge%20Pilot/topic/relatedArtifact%20extension%20change%20request/near/183662865" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/189875-genomics-/-eMerge-Pilot/topic/relatedArtifact.20extension.20change.20request.html#183662865">(Dec 17 2019 at 16:09)</a>:</h4>
<p>right. many clinical labs are moving to reporting inferred phenotype ('resistant' etc...) only, but pair their services with a clinical system (such as Coriell) that Health care systems contract with to deliver recommendations. The output from these 'clinical systems' is often delivered as the the report to the Health care system. It is a very dynamic and divergent space. It will be interesting to see how you put it all together. Looking forward to it!</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
