---
layout: page
title: "FHIR Chat  · affected genes/exons · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/affected.20genes.2Fexons.html">affected genes/exons</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="167467203"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/affected%20genes/exons/near/167467203" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/affected.20genes.2Fexons.html#167467203">(Jun 06 2019 at 09:54)</a>:</h4>
<p>Our lab reports the affected genes and/or exons for CNVs an SVs, do we see a need to include this to the IG?</p>



<a name="167526700"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/affected%20genes/exons/near/167526700" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/affected.20genes.2Fexons.html#167526700">(Jun 06 2019 at 21:57)</a>:</h4>
<p>I'd say most important would be to represent the range as accurately as possible, so that folks with a knowledgebase can resolve on their own. From there, I can see how it could be helpful to report affected genes and/or exons - but it could be a long list...</p>



<a name="167613550"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/affected%20genes/exons/near/167613550" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/affected.20genes.2Fexons.html#167613550">(Jun 07 2019 at 21:02)</a>:</h4>
<p>I vaguely remember some free text component that could be used, but I can’t see it on the IG now (looking from my phone so might be missing it)</p>



<a name="171063623"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/affected%20genes/exons/near/171063623" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/affected.20genes.2Fexons.html#171063623">(Jul 17 2019 at 09:55)</a>:</h4>
<blockquote>
<p>I'd say most important would be to represent the range as accurately as possible</p>
</blockquote>
<p>i agree, but for reports in a clinical environment you want to transport the affected genes explicitly as hospitals don't have the possibility to lookup these affected genes.<br>
To me this would be a component in obs-variant.<br>
Just putting this into gene studied ID would be semantically wrong.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
