---
layout: page
title: "FHIR Chat  · another question on Variant1 example · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html">another question on Variant1 example</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="263048202"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263048202" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Liz Amos <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263048202">(Nov 29 2021 at 18:22)</a>:</h4>
<p>In regards to the Variant1 example: <a href="https://build.fhir.org/ig/HL7/genomics-reporting/branches/variant-component-groups/Observation-VariantExample1.html">https://build.fhir.org/ig/HL7/genomics-reporting/branches/variant-component-groups/Observation-VariantExample1.html</a>, what is meant by Observation.interpretation: Positive ? I thought this field was more of an abnormal flag - and it's kind of confusing with the value as Present.</p>



<a name="263049154"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263049154" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263049154">(Nov 29 2021 at 18:29)</a>:</h4>
<p>Present - means the variant was observed. <br>
You'll see text in PDF reports things like 'present, benign variant found'</p>



<a name="263049405"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263049405" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263049405">(Nov 29 2021 at 18:31)</a>:</h4>
<p>More of an FYI, but this example is aligned with how mCODE is modeled as well. value as <code>Present</code> or <code>Not Present</code> and interpretation as <code>POS</code> or <code>NEG</code></p>



<a name="263049986"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263049986" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Liz Amos <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263049986">(Nov 29 2021 at 18:36)</a>:</h4>
<p>Right - this example has both Present (as the value) and Positive (as the interpretation). Why both?</p>



<a name="263049990"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263049990" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263049990">(Nov 29 2021 at 18:36)</a>:</h4>
<p>Given Liz's question. <span class="user-mention" data-user-id="192302">@May Terry</span> when would you have value=Present and interpretation=Negative?</p>



<a name="263050026"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263050026" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263050026">(Nov 29 2021 at 18:36)</a>:</h4>
<p>My understanding was that mCODE took that approach (specifically for cancer variants) because they didn't adopt the separate "Implication" observations</p>



<a name="263050110"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263050110" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263050110">(Nov 29 2021 at 18:37)</a>:</h4>
<p>In our model we prefer having interpretations done through a separate observation, as the context is not always cancer</p>



<a name="263050265"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263050265" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263050265">(Nov 29 2021 at 18:38)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> - that was partially the reason. honestly, what is being done with implications, like clinical significance is overly complex and no one reference lab had adopted it and still hasn't so our approach was to keep it as simple as possible until otherwise adopted.</p>



<a name="263050349"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263050349" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263050349">(Nov 29 2021 at 18:39)</a>:</h4>
<p>So <span class="user-mention" data-user-id="192302">@May Terry</span> in mcode would interpretation=Negative be good or bad?</p>



<a name="263050800"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263050800" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263050800">(Nov 29 2021 at 18:43)</a>:</h4>
<p>re: interpretation. If it was observed and the result was negative then that's what's populated in interpretation. it's no different with an answer list in LOINC that states positive or negative.</p>



<a name="263050835"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263050835" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263050835">(Nov 29 2021 at 18:43)</a>:</h4>
<p>Interpretation has 0...* That means one could refrain from populating it. Additionally, like Liz said in other contexts it is used for High, low etc... By keeping it permissible to use in variant we open up to interesting options. However, it does allow mcode to use it and still be largely consistent with the CG IG.</p>



<a name="263051258"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051258" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051258">(Nov 29 2021 at 18:46)</a>:</h4>
<p>I do think we ought to pull the use of interp from our examples though, unless we want to specifically mention it as an expected pattern</p>



<a name="263051293"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051293" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051293">(Nov 29 2021 at 18:47)</a>:</h4>
<p><span class="user-mention silent" data-user-id="192302">May Terry</span> <a href="#narrow/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example/near/263050800">said</a>:</p>
<blockquote>
<p>re: interpretation. If it was observed and the result was negative then that's what's populated in interpretation. it's no different with an answer list in LOINC that states positive or negative.</p>
</blockquote>
<p>Sorry, maybe I am confused. I get Observation.value being Present or Not Present, but I do not understand Observation.interpretation being Positive or Negative? Pos or Neg for what exactly?</p>



<a name="263051303"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051303" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051303">(Nov 29 2021 at 18:47)</a>:</h4>
<p>Since we are still trying to sell folks on testing the Implications</p>



<a name="263051328"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051328" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Liz Amos <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051328">(Nov 29 2021 at 18:47)</a>:</h4>
<p>it seems duplicative in this case... unless I'm missing something?</p>



<a name="263051546"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051546" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051546">(Nov 29 2021 at 18:48)</a>:</h4>
<p>k. then there is a deviation with mCODE. we need to be able to address something like "ALK positive" without having all this overly complicated diagnostic implications. show me a reference lab that loves this complexity and we'll reconsider.</p>



<a name="263051623"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051623" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051623">(Nov 29 2021 at 18:48)</a>:</h4>
<p>Yes, this ties back to the biomarker discussion. I think using interpretation here could make sense for a more refined observation.code</p>



<a name="263051832"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051832" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051832">(Nov 29 2021 at 18:49)</a>:</h4>
<p><span class="user-mention" data-user-id="192302">@May Terry</span> I am curious about 'answer list in LOINC that states positive or negative' I understood that to be used in the observation.value, typically. Does Observation.value=Present with Observation.interpretation=NEG in the mcode context mean that the variation was found and that the variation has no bearing on the Tumor's potential for causing pathology? Or is understood based on what the specific test performed was? would you mind providing an example (just text - not look for a JSON object : ^ ) or two?</p>



<a name="263051883"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051883" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051883">(Nov 29 2021 at 18:49)</a>:</h4>
<p>Alk positive - you get a LOINC code and value.<br>
if it is sequencing then you would add a variant profile with value present.</p>
<p>The source of information matters with biomarkers. If it is not a sequencing test then variant profile is not appropriate.</p>



<a name="263051902"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051902" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051902">(Nov 29 2021 at 18:49)</a>:</h4>
<p>yes ALK positive. a LOINC code and a value.</p>



<a name="263051907"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263051907" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263051907">(Nov 29 2021 at 18:49)</a>:</h4>
<p><span class="user-mention silent" data-user-id="192302">May Terry</span> <a href="#narrow/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example/near/263051546">said</a>:</p>
<blockquote>
<p>k. then there is a deviation with mCODE. we need to be able to address something like "ALK positive" without having all this overly complicated diagnostic implications. show me a reference lab that loves this complexity and we'll reconsider.</p>
</blockquote>
<p>Sorry, dummy here, but does 'ALK positive' mean "we found this one variant and that means we want to tell you they are ALK positive" ?</p>



<a name="263052052"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263052052" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263052052">(Nov 29 2021 at 18:51)</a>:</h4>
<p>The source of information matters with biomarkers. If it is not a sequencing test then variant profile is not appropriate, in my opinion.</p>



<a name="263052171"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263052171" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263052171">(Nov 29 2021 at 18:52)</a>:</h4>
<p>Probe hybridization like  microarrays and PCR based methodologies are related to the sequence. <br>
Whereas antibody tests should be described by their target</p>



<a name="263052230"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263052230" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263052230">(Nov 29 2021 at 18:52)</a>:</h4>
<p>it get's really messy though and needs thought</p>



<a name="263052318"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263052318" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263052318">(Nov 29 2021 at 18:53)</a>:</h4>
<p>So is ALK positive really more of a 'biomarker' then?</p>



<a name="263052443"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263052443" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263052443">(Nov 29 2021 at 18:54)</a>:</h4>
<p>hmm...based on this dialogue, I suppose this will be revisited with a biomarkers conversation. in all honesty, the CG Reporting IG is moving in a direction that really only applies to sequencing test results and is forcing complexity in tests where we care about DNA-based biomarkers where we do get results like ERBB2 positive and not necessarily including an actual variant.</p>



<a name="263052647"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263052647" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263052647">(Nov 29 2021 at 18:56)</a>:</h4>
<p>Yea, if the test doesn't actually determine if there is specific "variation" that we want to describe and instead measures some higher lever thing, that feels like what we were calling a 'biomarker' to me?</p>



<a name="263052745"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263052745" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263052745">(Nov 29 2021 at 18:57)</a>:</h4>
<p>(and I may not have any idea what I am talking about here <span aria-label="smile" class="emoji emoji-1f642" role="img" title="smile">:smile:</span> )</p>



<a name="263053107"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263053107" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263053107">(Nov 29 2021 at 19:00)</a>:</h4>
<p>I think mCODE's main usage of variant may be as a biomarker. It would align more semantically if mCODE used "diagnostic-implication" as Obs.code to indicate ALK positive and we loosened the restriction on our Implication profile to allow pointing to just an HGVS string for derivedFrom rather than a whole observation if they aren't sending those.</p>



<a name="263053481"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263053481" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263053481">(Nov 29 2021 at 19:03)</a>:</h4>
<p>ugh...I can push back again in CodeX, but give me a reference lab that will adopt use of implications and let's work with them. Can you give me a name? Otherwise, current CodeX stakeholders do not want the complexity.</p>



<a name="263053606"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263053606" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263053606">(Nov 29 2021 at 19:04)</a>:</h4>
<p>I think in this case the definition of the 'genetic' piosition would be in the test definition and the standard Observation used for the biomarker.</p>



<a name="263053770"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263053770" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263053770">(Nov 29 2021 at 19:05)</a>:</h4>
<p>I would not propose you change anything until we work out what biomarker looks like.</p>



<a name="263054195"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263054195" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263054195">(Nov 29 2021 at 19:09)</a>:</h4>
<p>all I was just saying is that this <a href="https://build.fhir.org/ig/HL7/genomics-reporting/branches/variant-component-groups/Observation-VariantExample1.html">https://build.fhir.org/ig/HL7/genomics-reporting/branches/variant-component-groups/Observation-VariantExample1.html</a> example is aligned with mCODE. It fits your IG and our simplified case where we had something like BRCA1 or BRCA2  positive or negative, with similar references here: <a href="https://www.cancer.gov/about-cancer/causes-prevention/genetics/brca-fact-sheet">https://www.cancer.gov/about-cancer/causes-prevention/genetics/brca-fact-sheet</a></p>



<a name="263054297"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263054297" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263054297">(Nov 29 2021 at 19:10)</a>:</h4>
<p>Biomarker - specific 'well-defined' entity, not provided as a sequencing result. The methodology contains a full description of the molecular target, assay type (activity or detection). </p>
<p>Problem is, one could define a NGS panel along the same lines and we would be stuck with where genetic data is today. For tumors, this is perhaps not a problem. For germline tests it obfuscates data that could be important outside of the indication for the specific test.</p>



<a name="263054385"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263054385" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263054385">(Nov 29 2021 at 19:11)</a>:</h4>
<p>I think we can just add the HGVS component to the VariantExample1 and it will be aligned with our new variant guidance documentation.</p>



<a name="263054728"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263054728" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263054728">(Nov 29 2021 at 19:14)</a>:</h4>
<p>we had this example in mCODE that adds the HGVS component: <a href="http://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/master/Observation-genomic-variant-somatic-single-nucleotide.html">http://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/master/Observation-genomic-variant-somatic-single-nucleotide.html</a></p>



<a name="263056494"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263056494" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263056494">(Nov 29 2021 at 19:28)</a>:</h4>
<p>Looks ok to me. <span class="user-mention" data-user-id="192302">@May Terry</span> - In case you haven't been following other conversations and calls, we are working on some new 'Variant Guidance' (previously known as IG lite), and one of the keys is that we are driving towards one of two ways to define each variant:<br>
<a href="https://build.fhir.org/ig/HL7/genomics-reporting/branches/variant-component-groups/sequencing.html#DefiningVariants">https://build.fhir.org/ig/HL7/genomics-reporting/branches/variant-component-groups/sequencing.html#DefiningVariants</a></p>



<a name="263058561"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263058561" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263058561">(Nov 29 2021 at 19:44)</a>:</h4>
<p>might be pretty simple, with a biomarker, for a lab to have a pre-populated template for a biomarker where dates, specimen and patient are filled in. The receiving system, if it is getting non-biomarker genomic data, will already know what to do with the profiles. So for the recipient there's a really good consistency and reliability by a lab using the profiles - and NO extra work. I know that to a person looking at it on paper it seems complicated, but it will be the computer doing the work. Consistency and reliability are keys.</p>



<a name="263058684"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/another%20question%20on%20Variant1%20example/near/263058684" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/another.20question.20on.20Variant1.20example.html#263058684">(Nov 29 2021 at 19:45)</a>:</h4>
<p>(with the use of the profiles you will be able to easily tell that two different biomarker tests which used different methods (i.e. had different LOINC codes) actually report on the same biomarker.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
