---
layout: page
title: "FHIR Chat  · Cancer Interoperability Chat · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html">Cancer Interoperability Chat</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="159928413"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159928413" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159928413">(Mar 04 2019 at 16:39)</a>:</h4>
<p>This stream is a place for folks to discuss and implementation of FHIR in Cancer genetics as it relates to the items such as the Cancer Interoperability Council use cases.</p>



<a name="159929471"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159929471" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159929471">(Mar 04 2019 at 16:51)</a>:</h4>
<p>Thanks for setting this up, Bret!  I'll have questions as I continue work on the minimal Common Oncology Data Elements (mCODE) genomics profiles.</p>



<a name="159929652"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159929652" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159929652">(Mar 04 2019 at 16:54)</a>:</h4>
<p>Awesome! <span class="user-mention" data-user-id="192302">@May Terry</span>  if you have an URLs to point folks to or explanatory material please feel free to add them.</p>



<a name="159929741"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159929741" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159929741">(Mar 04 2019 at 16:54)</a>:</h4>
<p><span class="user-mention" data-user-id="196271">@Andrea Pitkus, PhD, MLS(ASCP)CM, CSM</span>  did you have any comments from Today you wanted to add?</p>



<a name="159932521"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159932521" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159932521">(Mar 04 2019 at 17:33)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="192123">@Bret H</span>  !   I think it would be helpful for CG to hear/understand some of the downstream CIC/Mitre use cases, so resources can be harmonized (if they aren't already). IMO, lab/path results should be modeled up front , structured, and flow freely to downstream entities for use cases such as CDS, Precision Medicine, including PGx and Cancer Reporting, discussed today.</p>



<a name="159933677"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159933677" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159933677">(Mar 04 2019 at 17:46)</a>:</h4>
<p>not today but likely next week sometime.</p>



<a name="159934021"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159934021" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159934021">(Mar 04 2019 at 17:50)</a>:</h4>
<p>We could add this as a topic on next Tuesday call (March 12, 11 AM US Eastern) if you are available <span class="user-mention" data-user-id="192302">@May Terry</span>  ?</p>



<a name="159934219"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159934219" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159934219">(Mar 04 2019 at 17:53)</a>:</h4>
<p>I'm available.  I'll see if <span class="user-mention" data-user-id="191447">@Mark Kramer</span> can also join to provide additional mCODE context for our genomics use case.</p>



<a name="159934832"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/159934832" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#159934832">(Mar 04 2019 at 18:01)</a>:</h4>
<p>Sounds great, I will add you to the agenda for next Tuesday!</p>



<a name="160015229"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160015229" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160015229">(Mar 05 2019 at 16:07)</a>:</h4>
<p>Should include <span class="user-mention" data-user-id="191718">@Richard Esmond</span>  He's organizing CIC</p>



<a name="160031534"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160031534" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Richard Esmond <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160031534">(Mar 05 2019 at 19:16)</a>:</h4>
<p>Hey guys - </p>
<p>I am certainly one of the people, along with a bunch from MITRE, and some others spread out across RSNA / ACR and also Intermountain (obviously</p>
<p>Now that we are all aware of each others work I suspect that we will spend some time between the May and Sept ballots figuring out how to better consolidate our work.  With less than three weeks to go on the May ballot, not much will be happening  until next cycle.</p>
<p>But I'm looking forward to it!</p>
<p>Richard</p>



<a name="160034752"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160034752" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160034752">(Mar 05 2019 at 19:56)</a>:</h4>
<p><span aria-label="+1" class="emoji emoji-1f44d" role="img" title="+1">:+1:</span></p>



<a name="160035149"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160035149" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160035149">(Mar 05 2019 at 20:01)</a>:</h4>
<p>Sounds great.  Call details here: <a href="http://www.hl7.org/concalls/CallDetails.aspx?concall=43004" target="_blank" title="http://www.hl7.org/concalls/CallDetails.aspx?concall=43004">http://www.hl7.org/concalls/CallDetails.aspx?concall=43004</a></p>



<a name="160050456"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160050456" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160050456">(Mar 05 2019 at 23:16)</a>:</h4>
<p>Also, I was reviewing the mCODE <a href="https://confluence.hl7.org/pages/viewpage.action?pageId=40737853" target="_blank" title="https://confluence.hl7.org/pages/viewpage.action?pageId=40737853">PSS</a> - and had some questions. </p>
<p>I see you list the CG DIM as a dependency (HL7 Clinical Genomics Domain Information Model(s) (Project ID 1050))?  Honestly, it is more than a bit dated, and we are completely revamping it (CC <span class="user-mention" data-user-id="192198">@Bob Freimuth</span>)  For now, would it make more sense to reference our <a href="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=479" target="_blank" title="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=479">DAM</a>? (CC <span class="user-mention" data-user-id="191719">@Gil Alterovitz</span> )</p>
<p>And this might be pre-mature, but should we consider mCODE list our <a href="http://build.fhir.org/ig/HL7/genomics-reporting/index.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/index.html">CG FHIR IG</a> as a dependency?  Considering the vote on the PSS is this week (before you join our call next week), it likely is pre-mature, but I did want to ask the question.</p>



<a name="160051266"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160051266" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160051266">(Mar 05 2019 at 23:29)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="191913">@Kevin Power</span> - yeh, both the DIM reference in the PSS and our initial mCODE IG containing genomics profiles is a bit outdated to the DSTU2 version, which is why I'm spending time this week figuring out how to update it to the more recent CG IG and artifacts.  I'm looking forward to some additional guidance from the CGWG at the meeting next week.  Also, as an FYI, the work that Richard is doing with RSNA/ACR is not related to the mCODE efforts so I really don't want to be conflating both at this time and hope to keep whatever is presented completely separate.</p>



<a name="160054001"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160054001" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160054001">(Mar 06 2019 at 00:10)</a>:</h4>
<p>Ah, OK - sorry for conflating.  Can you provide a quick summary of the differences?</p>



<a name="160061676"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160061676" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160061676">(Mar 06 2019 at 02:51)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> -  mCODE is an ASCO/CancerLinQ led project focusing on a minimal set of Common Oncology Data Elements, primarly intended for EHRs: <a href="https://www.asco.org/practice-guidelines/cancer-care-initiatives/mcode-creating-set-standard-data-elements-oncology-ehrs" target="_blank" title="https://www.asco.org/practice-guidelines/cancer-care-initiatives/mcode-creating-set-standard-data-elements-oncology-ehrs">https://www.asco.org/practice-guidelines/cancer-care-initiatives/mcode-creating-set-standard-data-elements-oncology-ehrs</a>.  MITRE is supporting ASCO in this endeavor by translating their defined minimal list of elements into a FHIR logical model and profiles, supporting an ASCO-defined use case for an EHR which includes processing somatic and germline mutation results.  The definition and prioritization of model elements are driven by the ASCO and CancerLinQ.  Alternatively, Richard's work with RSNA/ACR is focused on radiology elements and although the HL7 CI team has started some dialogue trying to introduce genomics, there is no clear use case on how it would be used.  We hope some of the mCODE model will be merged holistically with other HL7 Cancer Interoperability work at some point, however given our separate stakeholders, use cases, and IG delivery timelines for mCODE, we don't anticipate an IG convergence between both projects anytime soon.  cc'ing <span class="user-mention" data-user-id="191447">@Mark Kramer</span> to further comment if needed.</p>



<a name="160107755"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160107755" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160107755">(Mar 06 2019 at 15:56)</a>:</h4>
<p>fyi: <a href="#narrow/stream/189875-genomics-.2F.20eMerge.20Pilot" title="#narrow/stream/189875-genomics-.2F.20eMerge.20Pilot">https://chat.fhir.org/#narrow/stream/189875-genomics-.2F.20eMerge.20Pilot</a> is a place for PGx examples with eMerge. might be some overlap in terms of questions...as we get moving along</p>



<a name="160111467"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160111467" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160111467">(Mar 06 2019 at 16:35)</a>:</h4>
<p>Thanks, <span class="user-mention" data-user-id="191913">@Kevin Power</span> . I agree that the DAM would be a better reference.</p>
<p><span class="user-mention" data-user-id="192302">@May Terry</span> : It seems mCODE is concerned more with the generation of a <em>set</em> of CDEs, rather than with the creation of those CDEs themselves.  Is that correct?  Secondly, how might this relate to the content in the NCI caDSR?</p>



<a name="160132139"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160132139" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160132139">(Mar 06 2019 at 20:36)</a>:</h4>
<p><span class="user-mention" data-user-id="192198">@Bob Freimuth</span>  - to start, yes. On the second question, honestly, I haven't given its relation to NCI caDSR much thought since our focus was on a couple of specific ASCO-identified use cases.  But good idea to look into it.  I've got some homework to do before next Tuesday I suppose. :-)</p>



<a name="160515738"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/160515738" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#160515738">(Mar 11 2019 at 20:37)</a>:</h4>
<p>Hi all - This is on the agenda for tomorrow:</p>
<p>HL7 Clinical Genomics Weekly Call - 12 Mar 2019 11:00 AM (US Eastern)<br>
Minutes<br>
<a href="http://tinyurl.com/HL7CGGroupCall" target="_blank" title="http://tinyurl.com/HL7CGGroupCall">http://tinyurl.com/HL7CGGroupCall</a><br>
<a href="https://docs.google.com/document/d/12-uBrMmav71a3_c9h_FXQteJo_I5Kt72NEBYXZuwhFg/edit" target="_blank" title="https://docs.google.com/document/d/12-uBrMmav71a3_c9h_FXQteJo_I5Kt72NEBYXZuwhFg/edit">https://docs.google.com/document/d/12-uBrMmav71a3_c9h_FXQteJo_I5Kt72NEBYXZuwhFg/edit</a></p>
<p>Attending the meeting<br>
Join the online meeting (VoIP available with this):<br>
Online Meeting Link:<br>
<a href="https://join.freeconferencecall.com/clingenomics" target="_blank" title="https://join.freeconferencecall.com/clingenomics">https://join.freeconferencecall.com/clingenomics</a><br>
Meeting ID:     clingenomics<br>
Dial into the conference:<br>
Dial-in Number:<br>
(515) 604-9708 - United States<br>
Access Code:     289092<br>
International Dial-in Numbers:<br>
<a href="https://www.freeconferencecall.com/wall/clingenomics/#international" target="_blank" title="https://www.freeconferencecall.com/wall/clingenomics/#international">https://www.freeconferencecall.com/wall/clingenomics/#international</a></p>



<a name="161106060"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/161106060" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#161106060">(Mar 18 2019 at 23:27)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  - attached are the mCODE slides that were presented last Tuesday at the CG WG weekly meeting. <a href="/user_uploads/10155/HPKOwHFcgW2k_9_GjituqcPr/MCODE_CancerBiomarker_Notes_ext.pptx" target="_blank" title="MCODE_CancerBiomarker_Notes_ext.pptx">MCODE_CancerBiomarker_Notes_ext.pptx</a>   As a status, we're currently working on updating our mCODE Cancer Genomics Report and observation profiles and hope to have a preliminary version out in the next 2 weeks.  I'll post the link here when we're ready to share it and we look forward to your feedback and guidance then.  Thanks!  cc: <span class="user-mention" data-user-id="191447">@Mark Kramer</span></p>



<a name="161680898"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/161680898" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#161680898">(Mar 25 2019 at 19:48)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="192123">@Bret H</span> or <span class="user-mention" data-user-id="191913">@Kevin Power</span> - as I'm working through some of these mCODE modeling updates for variants, would you recommend  <a href="http://build.fhir.org/ig/HL7/genomics-reporting/obs-variant.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/obs-variant.html">http://build.fhir.org/ig/HL7/genomics-reporting/obs-variant.html</a> as a reference? I just want to make sure that our elements and bindings are aligned to the current thinking.  Thanks!</p>



<a name="161684199"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/161684199" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#161684199">(Mar 25 2019 at 20:26)</a>:</h4>
<p>Hi May, yes that is the current profile. After consideration at the WGM, we combined complex variant, copy number change, and described variant into one (only slightly larger) profile and named it Variant.</p>



<a name="161685657"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/161685657" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#161685657">(Mar 25 2019 at 20:44)</a>:</h4>
<p>ah! Thanks <span class="user-mention" data-user-id="196553">@James Jones</span> !  The context is super helpful.</p>



<a name="162456528"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/162456528" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#162456528">(Apr 03 2019 at 18:43)</a>:</h4>
<p>Hi CGWG team!  Some questions from your friendly neighborhood genomics newbie as I'm reviewing obs-region-studied profile (<a href="http://build.fhir.org/ig/HL7/genomics-reporting/obs-region-studied.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/obs-region-studied.html">http://build.fhir.org/ig/HL7/genomics-reporting/obs-region-studied.html</a>): 1) Do we have a suitable Observation.code? Currently it's TBD. 2) There's an Observation.valueCodeableConcept that is "Positive|Negative|Unknown".  Is this applicable or should it be zero'd out? same question with Observation.referenceRange on whether we should zero this out, 3) are we to think of this profile as a finding with a result or just as a profile that describes the characteristics of the genetic region studied?  4) do you have a FHIR example for the obs-region-studied profile that you can share?  Thanks!</p>



<a name="162456917"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/162456917" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#162456917">(Apr 03 2019 at 18:47)</a>:</h4>
<p>Hi May, region-studied is possibly our youngest profile, introduced just before the last ballot. There are trackers in the system currently to update the code and clarify the value list (current tracker suggests binding it to LOINC answer list LL1971-2 'present/absent/indeterminate' --same answer list as is used for Variant)</p>



<a name="162457571"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/162457571" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#162457571">(Apr 03 2019 at 18:53)</a>:</h4>
<p>The profile was originally introduced to allow a lab specifying what genomic regions the report covers (and potentially a way to clarify regions that were not studied closely or may have been too poor quality to report, etc), as something that would hang off the report as its own 'result'. I recall <span class="user-mention" data-user-id="192123">@Bret H</span> used the underlying Observation's <code>hasmember</code> field to reference specific results housed in a "GACS" server that carried variant information for the named region, but I haven't seen any push for that functionality to be included as part of the IG yet.</p>



<a name="162457935"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/162457935" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#162457935">(Apr 03 2019 at 18:56)</a>:</h4>
<p>The comment on 'referenceRange' is interesting, it doesn't appear that we've given any guidance on how (or how not) to use that on any of our Observation profiles... someone more senior than me will have to weigh in on that decision (or indecision). It appears to currently be 0..*and not required to be supported, so you can likely ignore it without issue.</p>



<a name="162458613"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/162458613" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#162458613">(Apr 03 2019 at 19:03)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="192302">@May Terry</span> , we're using the profile to indicate whether or not a region has been studied, and summarize whether or not variants were found in the region. We also use the profile to indicate that a region was not callable. </p>
<p>Here are some outstanding ballot comments that overlap your comments: <a href="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19839" target="_blank" title="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19839">https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19839</a>; <a href="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19838" target="_blank" title="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19838">https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19838</a>; <a href="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19845" target="_blank" title="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19845">https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19845</a>.</p>
<p>All the examples on <a href="https://github.com/openelimu/VCF-2-FHIR/tree/master/public" target="_blank" title="https://github.com/openelimu/VCF-2-FHIR/tree/master/public">https://github.com/openelimu/VCF-2-FHIR/tree/master/public</a> include the profile.</p>



<a name="162482237"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/162482237" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#162482237">(Apr 04 2019 at 00:18)</a>:</h4>
<p>thanks <span class="user-mention" data-user-id="196553">@James Jones</span> and <span class="user-mention" data-user-id="193751">@Bob Dolin</span> ! As usual, this team is incredibly responsive and helpful . So appreciative of your input and guidance.</p>



<a name="163147216"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163147216" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163147216">(Apr 11 2019 at 23:47)</a>:</h4>
<p>Question on the obs-region-studied (<a href="http://build.fhir.org/ig/HL7/genomics-reporting/obs-region-studied.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/obs-region-studied.html">http://build.fhir.org/ig/HL7/genomics-reporting/obs-region-studied.html</a>) profile. I noticed that Observation.code specifies a Required Pattern {"system":"http://loinc.org","code":"TBD-RegionsStudied"}.  Is this how you would prefer our equivalent mCODE profile to represent its code?  Also wondering, is it a problem from a validation perspective if we do this, keeping <a href="http://loinc.org" target="_blank" title="http://loinc.org">loinc.org</a> as the coding system?  Thanks!</p>



<a name="163154775"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163154775" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163154775">(Apr 12 2019 at 02:32)</a>:</h4>
<p>Yes, the validator will yell at it being a non-proper code.  As it should.  This needs to be fixed before we publish it as "implementable".</p>



<a name="163191498"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163191498" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163191498">(Apr 12 2019 at 13:52)</a>:</h4>
<p>Thanks Lloyd.  Any guidance from the CGWG team on what we could use as a codeableConcept from LOINC or other coding system?</p>



<a name="163222620"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163222620" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163222620">(Apr 12 2019 at 20:09)</a>:</h4>
<p>Bob Dolin's suggestion to use answers from the list LL1971-2 'present/absent/indeterminate'  seems like a good approach for now, we'll hopefully address this tracker this coming Monday or next. You could use LOINC code 69548-6 in the meantime but we will end up getting a new code for it before publication, however (several left to finish proposing/requesting).</p>



<a name="163230425"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163230425" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163230425">(Apr 12 2019 at 21:59)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="196553">@James Jones</span> .  I realize that some of this work is an evolving WIP, so this is at least helpful for us to keep abreast of changes and annotate in our IG when we can.  Our first shareable draft of the mCODE IG (including our minimally cherry-picked elements from the CG R4 Genomics Reporting IG) should be available for review mid-to-late next week so I hope to get some time on the calendar to go over it with the team during one of the CGWG weekly calls or perhaps during one of the CG quarters at the HL7 WGM in Montreal?  I plan on attending so perhaps that might be a good venue.</p>



<a name="163280958"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163280958" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163280958">(Apr 13 2019 at 19:58)</a>:</h4>
<p>Happy to review in an upcoming call and/or WGM.  If you want to discuss in a call, let us know when it is ready. We will be discussing the WGM agenda this Tuesday so we can look to find a time slot that works if you are available.</p>



<a name="163606058"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163606058" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163606058">(Apr 17 2019 at 22:21)</a>:</h4>
<p><span class="user-mention" data-user-id="192302">@May Terry</span> Hi May, sorry if I missed this, but is there a place we can see the mCode data set, along with current FHIR mapping?</p>



<a name="163665286"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/163665286" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#163665286">(Apr 18 2019 at 16:09)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span>  We do have a posting online but I first want to go over the rationale in our decisions and a high level walkthrough with CGWG rather than to address questions prior to this. Hence the request to schedule a review first.</p>



<a name="166728251"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/166728251" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#166728251">(May 28 2019 at 15:36)</a>:</h4>
<p>Attached are the slides presented to the CGWG at the HL7 Montreal May2019 WGM in the Wednesday Q4 session. <a href="/user_uploads/10155/LFTqrqjvw6JtDI_idrjywTDr/20190508_mCODE_CGWGPreso.pptx" target="_blank" title="20190508_mCODE_CGWGPreso.pptx">20190508_mCODE_CGWGPreso.pptx</a></p>



<a name="171117826"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/171117826" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#171117826">(Jul 17 2019 at 21:10)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="192302">@May Terry</span>. I finally had a chance to review the FHIR mCode IG, and I think it is quite good, and very intuitive. Just out of curiosity, what is the relationship to the FHIR breast cancer IG? It seems there are a number of similar but not quite the same profiles. Are there plans to reconcile?</p>



<a name="171435347"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cancer%20Interoperability%20Chat/near/171435347" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cancer.20Interoperability.20Chat.html#171435347">(Jul 22 2019 at 14:30)</a>:</h4>
<p>Wondering if comparing/converting the genomics-relevant profiles between our IG, mCode, and the breast cancer IG may be a useful exercise for the upcoming connectathon... I feel like for the most part they are compatible but a deep dive may be in order</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
