---
layout: page
title: "FHIR Chat  · Clinvar Submission API · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html">Clinvar Submission API</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="253429797"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/253429797" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#253429797">(Sep 15 2021 at 15:06)</a>:</h4>
<p><span class="user-mention" data-user-id="194730">@Liz Amos</span> pointed at this on the call yesterday this:<br>
<a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/">https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/</a></p>
<p>Might be good to review the "Submission Schema" and "Submission description" and compare to how we have defined Variant.</p>



<a name="256142022"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/256142022" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#256142022">(Oct 04 2021 at 22:15)</a>:</h4>
<p>can we say a little more about what the comparison is for? Do you want a spreadsheet that shows a mapping between the two? Or a paragraph about the contrast?</p>



<a name="256238623"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/256238623" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#256238623">(Oct 05 2021 at 12:48)</a>:</h4>
<p>We didn’t discuss in great detail, but a mapping between the two would be interesting.</p>



<a name="256756553"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/256756553" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#256756553">(Oct 08 2021 at 15:24)</a>:</h4>
<p>looking at it now. will append report later. However, first quick blush - there are several items that I would categorize as important in the research space (and knowledge repository space. e.g. details on a research study subject demographic) that will not be in our profiles. BRIDG is a group focused on Biomedical research and if the elements specific to research/studies are to be included, I'd recommend a collaboration with that group or moving those items to that group to give guidance on. PS Here's info on BRIDG: <a href="https://bridgmodel.nci.nih.gov/hl7-fhir">https://bridgmodel.nci.nih.gov/hl7-fhir</a></p>



<a name="256791456"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/256791456" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#256791456">(Oct 08 2021 at 19:17)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="192123">@Bret H</span> - I think calling out items they have that do not map well to our IG would be important, as well as flagging those that feel more research versus clinically oriented. I appreciate your review!</p>
<p>I will add that I think <span class="user-mention" data-user-id="194730">@Liz Amos</span> was going to identify a contact for the api in case we have questions or clarifications?</p>



<a name="256794475"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/256794475" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Liz Amos <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#256794475">(Oct 08 2021 at 19:40)</a>:</h4>
<p>yep, I do, planning on trying to arrange something for an upcoming call. however, my original intent was to focus on the variant description for the Variant Lite work</p>



<a name="256796800"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/256796800" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#256796800">(Oct 08 2021 at 19:56)</a>:</h4>
<p>Sounds like the right priority to me, thanks <span class="user-mention" data-user-id="194730">@Liz Amos</span></p>



<a name="257068367"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257068367" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257068367">(Oct 11 2021 at 14:00)</a>:</h4>
<p><span class="user-mention" data-user-id="194730">@Liz Amos</span> <span class="user-mention" data-user-id="191913">@Kevin Power</span>  I have been through the API model and our FHIR models. My analysis is available for comment at: <a href="https://docs.google.com/spreadsheets/d/1fLNIj5QXRbfBQqvUoolC7m6AvyYg9x-dCkvKqMsEkKM/edit?usp=sharing">https://docs.google.com/spreadsheets/d/1fLNIj5QXRbfBQqvUoolC7m6AvyYg9x-dCkvKqMsEkKM/edit?usp=sharing</a> Here's what I saw: IMPORTANT THIS IS AS OF Oct 11, 2021. Both groups are actively evolving<br>
ClinVarAPI: <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/">https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/</a><br>
CG IG: <a href="http://build.fhir.org/ig/HL7/genomics-reporting/">http://build.fhir.org/ig/HL7/genomics-reporting/</a><br>
Generally, ClinVar submission is meant to be a self-contained record of information on a variant-disease pair (potentially including results from multiple individuals) for deposit in a knowledge repository, a variant database. The CG IG profiles are meant to communicate the clinically relevant information on a genetic result for a patient. The EMR provides additional details on the patient and the data is considered in light of the clinical record of the patient. ClinVar is a knowledge repository. As such it goes beyond what would be sent in a non-research report. At most a trio would be referred to in a lab test. Again, a clinvar submission assumes a grouping of individuals with a singular variant focus. For Healthcare transactions, the focus is on a singular individual with a grouping of variants found in a particular test. </p>
<p>ClinVar treats therapeutic implications with the same elements as it uses for diagnostic implications. The CinVar clinical significance description field maps into both the Diagnostic implication and Therapeutic implications.<br>
e.g. A 'drug response'  association in a ClinVar entry would be mapped with Therapeutic implication. A 'Pathogenic' association in a ClinVar entry would be mapped to Diagnostic implication<br>
ClinVar has 'protective' in clinical significance description value set and 'risk factor.'  Protective is a different meaning than benign. Similarly, 'risk factor' is a different implication than pathogenic. However, putting them in the same value set as our current terms will complicate the current meaning of the data element - i.e. make it harder to find all the 'pathogenic' variants for a disease as some instances of data might have 'risk factor' and not use 'pathogenic' or 'likely pathogenic.' Should be discussed. There is the Prognosis component in the CG IG which could be used.<br>
Also, ClinVar has 'confers sensitivity' in clinical significance. Might consider how to handle this in the Therapeutic profile as drug sensitivity. The CG IG prognosis component could be used.</p>
<p>The variantSet variant in ClinVar maps well into the CG IG variant profile. ClinVar associates chromosomal position information data elements in one group. The ClinVar element variant type maps to multiple variant elements (component:dna-chg-type, component:molecular-consequence, component:amino-acid-chg-type).<br>
The ClinVar variantSet variant has referenceCopyNumber, this is not in the CG IG.<br>
The CG IGs components for Variant component:molecular-consequence and Diagnostic Implication component:functional-effect gives a greater vocabulary for describing the consequences of the variant than the ClinVar API.</p>
<p>ClinVar API assertionCriteria element is not specifically present in the CG IG, but it is part of the attribution elements in FHIR which indicate the observer.</p>
<p>POPULATIONS? observedIn element in the ClinVar API is interesting, somewhat similar to CG IG FHIR use of patient but meant for describing the group of subjects the ClinVar submission record is about. Generally observedIn looks more like what would be expected to be seen in a research study. The items are interesting but not a report for an individual patient's genomic result. The clinicalFeatures field could be mapped to Condition sent with Patient. But with FHIR access to the Patient clinical features is typically a given in the recipient system. One could envision using Family history to report the types of data seen here for a Trio analysis. A data element like observedIn is well suited for a knowledge repository and something like it could be used to return Population results from an EMR.</p>



<a name="257068623"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257068623" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257068623">(Oct 11 2021 at 14:02)</a>:</h4>
<p><span class="user-mention" data-user-id="194730">@Liz Amos</span> In my spreadsheet, I've two tabs. I can do a line by line mapping from variantSet:variant to the CG variant if you need it. The mapping is pretty straightforward, except where noted above. I looked at all the data elements in the API submission model. Happy to have someone from ClinVar comment on my notes (<a href="https://docs.google.com/spreadsheets/d/1fLNIj5QXRbfBQqvUoolC7m6AvyYg9x-dCkvKqMsEkKM/edit?usp=sharing">https://docs.google.com/spreadsheets/d/1fLNIj5QXRbfBQqvUoolC7m6AvyYg9x-dCkvKqMsEkKM/edit?usp=sharing</a> ). The variantType field needed to be mapped to multiple fields. Also, good to think about the under used CG IG FHIR component: prognosis in Genomic implications.</p>



<a name="257070432"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257070432" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257070432">(Oct 11 2021 at 14:16)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> I think you might want to think about the prognosis component in implication. It's visible in the Structure-def view but hidden in the differential... <span class="user-mention" data-user-id="196553">@Jamie Jones</span> any thoughts how to make the elements in the differential for <a href="http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-implication.html">http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-implication.html</a> more visible in the Diagnostic and Therapeutic profiles? I wonder if it would be useful  make the derived from slices, the genetic reference and the prognosis component more visible in the derivative profiles.</p>



<a name="257072738"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257072738" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257072738">(Oct 11 2021 at 14:33)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> Thanks for getting this started. Do you mind if I make some formatting suggestions for your spreadsheet? I don't know about anyone else, but with all the overlapping text, I find it hard to read?</p>



<a name="257073856"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257073856" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257073856">(Oct 11 2021 at 14:41)</a>:</h4>
<p>I requested write access</p>



<a name="257077513"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257077513" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257077513">(Oct 11 2021 at 15:08)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> I looked over their entire submission model</p>



<a name="257077769"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257077769" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257077769">(Oct 11 2021 at 15:10)</a>:</h4>
<p>(deleted)</p>



<a name="257077796"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257077796" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257077796">(Oct 11 2021 at 15:10)</a>:</h4>
<p>(deleted)</p>



<a name="257077900"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257077900" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257077900">(Oct 11 2021 at 15:11)</a>:</h4>
<p>(deleted)</p>



<a name="257078325"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257078325" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257078325">(Oct 11 2021 at 15:14)</a>:</h4>
<p>The observedIn might be interesting for a Genetics for Populations profile.</p>



<a name="257078743"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257078743" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257078743">(Oct 11 2021 at 15:17)</a>:</h4>
<p>I just had a really hard time reading everything you put into the spreadsheet. I am making some formatting changes now to ensure we can read all the text.</p>



<a name="257078812"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257078812" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257078812">(Oct 11 2021 at 15:18)</a>:</h4>
<p>Yep</p>



<a name="257079233"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257079233" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257079233">(Oct 11 2021 at 15:21)</a>:</h4>
<p>It is nice how chromosomalcoordinates in the ClinVar Submission model groups things. But the native FHIR query get's uglier as we nest more and more</p>



<a name="257079320"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257079320" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257079320">(Oct 11 2021 at 15:21)</a>:</h4>
<p>e.g. if we we're to create a component to organize like the ClinVar API has done, and put all the chromosomal position information there.</p>



<a name="257102250"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Clinvar%20Submission%20API/near/257102250" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Clinvar.20Submission.20API.html#257102250">(Oct 11 2021 at 18:41)</a>:</h4>
<p><span class="user-mention" data-user-id="194730">@Liz Amos</span>  hello. Could you verify with your contact that 'clinvarSubmission variantSet variant chromosomeCoordinates accession' is meant to be something like a RefSeek accession? I assume it is but...</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
