---
layout: page
title: "FHIR Chat  · CNV size · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/CNV.20size.html">CNV size</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="167467077"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/CNV%20size/near/167467077" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/CNV.20size.html#167467077">(Jun 06 2019 at 09:52)</a>:</h4>
<p>our lab send the size of a CNV with this valueSet (Chromosome, p-Arm, Q-Arm, partial p-Arm, partial q-Arm, focal, non-focal, cluster) do we see a need to have a component for this in our IG?</p>



<a name="167526254"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/CNV%20size/near/167526254" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/CNV.20size.html#167526254">(Jun 06 2019 at 21:51)</a>:</h4>
<p>The use case I'm familiar with is the need to know which genes are affected by a CNV. Seems like even if the lab tells me that there are extra copies of, say, part of the p-arm, it's not telling me much more than I already know if the lab gives me some indication of (fuzzy) start and end coordinates.</p>



<a name="167614022"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/CNV%20size/near/167614022" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/CNV.20size.html#167614022">(Jun 07 2019 at 21:08)</a>:</h4>
<p>I haven’t see a ton of examples in this space, but i can’t say i have seen that level of detail in many reports. Not sure if they are broadly applicable or not?</p>



<a name="167840645"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/CNV%20size/near/167840645" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Dora Walter <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/CNV.20size.html#167840645">(Jun 11 2019 at 11:20)</a>:</h4>
<p>The size of CNV is important to know in order to draw conclusions about potentially therapy-relevant genes.<br>
In my opinion, a more detailed description of the size of CNV is helpful if such information as affected genes are not reported by the laboratory.<br>
However, I have to agree with Bob that the size report does not provide much added value. It only limits the amount of potentially affected genes.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
