---
layout: page
title: "FHIR Chat  · ComplexMutations · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html">ComplexMutations</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="153991636"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991636" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991636">(Aug 29 2018 at 16:07)</a>:</h4>
<p><span class="user-mention" data-user-id="196701">@Gideon Giacomelli</span> <br>
trying to extract extra topics from the panel discussion:<br>
You already can capture all the mutations in the DescribedVariant profile in Observation(Described Variant).component:genomic-dna-chg. (Genomica DNA change (gHGVS))</p>



<a name="153991637"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991637" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991637">(Aug 29 2018 at 16:08)</a>:</h4>
<p>HGVS lets you express deletions, indels, copy number changes etc.</p>



<a name="153991639"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991639" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991639">(Aug 29 2018 at 16:16)</a>:</h4>
<p>I don't see the need of a new resource like geneStudy. Yes we use Observation a lot, but with different profiles. Until now i haven't encountered a restriction which legitimates the need of a new resource.</p>



<a name="153991857"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991857" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Gideon Giacomelli <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991857">(Aug 30 2018 at 09:27)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span> <span class="user-mention" data-user-id="191913">@Kevin Power</span>  thank you for the detailed reply.  As far as I understand it, if I have one Observation per variant I want to represent this mutation in a sequence resource  and for a complex structural mutation I would connect this observation to several sequences.<br>
We should also take into account that Whole Genome Sequencing can result in several thousand mutations therefore I am not sure if this is feasible.<br>
Also if a somatic mutation was found I want to link this to two samples a tumor sample and a control sample (mostl likely from blood) which is not possible right now, or is it? <br>
I don't know if this is the best way to ask all these questions here of if there is a better way for me to proceed.</p>



<a name="153991877"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991877" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991877">(Aug 30 2018 at 11:43)</a>:</h4>
<p>I'm not sure if you need the sequence resource for that. Found Variants are captured as an Observation(Described Variant Profile)</p>



<a name="153991878"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991878" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991878">(Aug 30 2018 at 11:44)</a>:</h4>
<p>Quote from the other Thread:<br>
The lab returns all studied genes as Observations(Described Variant). I have one Observation per found variant or per gene. What was looked at is Observation(Described Variant).component:gene-studied, the overall result (was something found) would be Observation(Described Variant).valueCodeableConcept.<br>
If a Variant was found the details are captured in the components.</p>



<a name="153991879"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991879" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991879">(Aug 30 2018 at 11:52)</a>:</h4>
<p>To distinguish between somatic and germline mutations there is:</p>



<a name="153991880"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991880" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991880">(Aug 30 2018 at 11:53)</a>:</h4>
<p>Observation(DescribedVariant).component:genomic-source-class</p>



<a name="153991904"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/ComplexMutations/near/153991904" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/ComplexMutations.html#153991904">(Aug 30 2018 at 13:16)</a>:</h4>
<p>I will just add that there are a couple of ways of referencing info about the specimen:<br>
<a href="http://build.fhir.org/observation-definitions.html#Observation.specimen" target="_blank" title="http://build.fhir.org/observation-definitions.html#Observation.specimen">Observation.specimen</a><br>
<a href="http://build.fhir.org/observation-definitions.html#Observation.bodySite" target="_blank" title="http://build.fhir.org/observation-definitions.html#Observation.bodySite">Observation.bodySite</a></p>
<p>I will also mention we have yet to really define how to appropriately structure those large changes.  I don't disagree with what has been said so far, but just like to be clear that we haven't nailed it down yet.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
