---
layout: page
title: "FHIR Chat  · COVID19 viral sequences? · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html">COVID19 viral sequences?</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="193567951"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193567951" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193567951">(Apr 10 2020 at 13:20)</a>:</h4>
<p>Posted to Listserv but looking for input here</p>



<a name="193567963"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193567963" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193567963">(Apr 10 2020 at 13:20)</a>:</h4>
<p>To enable tracking of virus spread and potentially targeted therapies, sharing the genetic information of a positive COVID19 test is desirable. Right now, there is not great guidance on providing discrete genetic data for microorganisms. Given the urgency. Until there is more formal guidance, for the statement: "Virus observed in patient A using specimen X, the virus is SARS-CoV-2, and the genetic sequence as evidence of this is P." The following could be used with FHIR R4 and has the ability to be retrieved from a system with a FHIR query.</p>
<p>With a Lab observation of a positive COVID19 test, here are two options ObservationVariant or MolecularSequence. Use Observation.dervierdFrom<br>
    pointing to either ObservationVariant (<a href="http://hl7.org/fhir/uv/genomics-reporting/variant.html" title="http://hl7.org/fhir/uv/genomics-reporting/variant.html">http://hl7.org/fhir/uv/genomics-reporting/variant.html</a>)<br>
        where reference, position, and change  are used<br>
        CHANGE: component:ref-allele and component:alt-allele<br>
        REFERENCE: component:genomic-ref-seq (LONIC code 48013-7)<br>
        POSITION: component:coordinate-system, component:variant-length, and either component:exact-start-end or component:outer-start-end or component:inner-start-end<br>
OR<br>
    pointing to MolecularSequence (<a href="https://www.hl7.org/fhir/molecularsequence.html" title="https://www.hl7.org/fhir/molecularsequence.html">https://www.hl7.org/fhir/molecularsequence.html</a>)<br>
        where reference is used to be more expressive position, change can be used<br>
        SEQUENCING DATA: MolecularSequence.observedSeq allows for large chunks of sequence data (such as an entire viral genome) to be related<br>
        MINIMALLY for REFERENCE MolecularSequence.referenceSeq.referenceSeqId</p>
<div class="codehilite"><pre><span></span>    Additionally: MolecularSequence.repository can be used to provide a link to a sequencing data file or repository.
</pre></div>


<p>A genomic reference for SARS-CoV-2 is code "NC_045512" with codesystem <a href="http://www.ncbi.nlm.nih.gov/refseq" title="http://www.ncbi.nlm.nih.gov/refseq">http://www.ncbi.nlm.nih.gov/refseq</a> <br>
*note a key is use a proper reference for  SARS-CoV-2, the name of the coronavirus associated with COVID19.</p>
<p>I would recommend ObservationVariant for deviations from a reference virus genome. MolecularSequence when a large contiguous chunk of sequence data is being reported (as in the case with sequencing the entire viral genome).</p>
<p>To retrieve data, use a query on the parent Observation with an appropriate list of LOINC codes for COVID19 detection and followed the derviedFrom link to MolecularSequence or ObservationVaraint.<br>
OR <br>
use a query for Observations and MolecularSequence with a SARS-CoV-2 sequence reference value for the reference id )such as code "NC_045512" with codesystem "<a href="http://www.ncbi.nlm.nih.gov/refseq" title="http://www.ncbi.nlm.nih.gov/refseq">http://www.ncbi.nlm.nih.gov/refseq</a>"). The field for reference is MolecularSequnece.referenceSeq.referenceSeqId OR Observation.component:genomic-ref-seq.code = LOINC code 48013-7 and observation.component:genomic-ref-seq.value.</p>
<p>Example ObservationVariant query<br>
GET /Observation?subject:Patient=123&amp;component-code-value-concept=http://loinc.org|48013-7$http://www.ncbi.nlm.nih.gov/refseq|NC_045512</p>
<p>Example MolecularSequence query<br>
GET<br>
/MolecularSequence:Pateint=123&amp;MolecularSequence.referenceSeq.referenceSeqId=http://loinc.org|48013-7$http://www.ncbi.nlm.nih.gov/refseq|NC_045512</p>



<a name="193572671"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193572671" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193572671">(Apr 10 2020 at 14:10)</a>:</h4>
<p>I like MolecularSequence for describing a set of variants of one sequence. But EHRs don't support MS. Not sure if that's important. I can describe a set of variants in one resource instance. Otherwise I would have to have a separate Observation for each variant. So if my reference is the whole viral sequence, I can describe many variations in the the new sequence, e.g.,</p>
<div class="codehilite"><pre><span></span>{
    &quot;resourceType&quot;: &quot;MolecularSequence&quot;,
    &quot;type&quot; : &quot;dna&quot;,
    &quot;coordinateSystem&quot;: 1,
    &quot;referenceSeq&quot;: {
        &quot;referenceSeqId&quot;: {
            &quot;coding&quot;: [
                {
                    &quot;system&quot;: &quot;http://www.ncbi.nlm.nih.gov/refseq&quot;,
                    &quot;code&quot;: &quot;NC_045512.2&quot;
                }
            ]}},
    &quot;variant&quot;: [
        {
            &quot;start&quot;: 60,
            &quot;end&quot;: 60,
            &quot;observedAllele&quot;: &quot;A&quot;,
            &quot;referenceAllele&quot;: &quot;T&quot;
        },
        {
            &quot;start&quot;: 27421,
            &quot;end&quot;: 27423,
            &quot;observedAllele&quot;: &quot;TAC&quot;,
            &quot;referenceAllele&quot;: &quot;ATA&quot;
        },
       ...etc...
    ]
}
</pre></div>



<a name="193600342"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193600342" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193600342">(Apr 10 2020 at 18:15)</a>:</h4>
<p>hmmmm. To me a positive covid test is an Observation. I didn't get the need to point to variant. If the virus was detected i maybe want to send the exact RNA pattern as a proof or for research. The actual Sequence is either a MolecularSequence or a Binary containing the sequence.</p>



<a name="193600384"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193600384" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193600384">(Apr 10 2020 at 18:15)</a>:</h4>
<p>Just spotted the "from the virus genome", yes then Variant would be correct.</p>



<a name="193600556"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193600556" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193600556">(Apr 10 2020 at 18:17)</a>:</h4>
<p>This reminds me that we still have to discuss the scope of MolecularSequence. I still don't like the redundancy to Variant. Maybe MolecularSequence could be scoped down, to actually represent a Sequence and using Variants to describe Variations to this Sequence.</p>



<a name="193600834"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193600834" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193600834">(Apr 10 2020 at 18:20)</a>:</h4>
<p>re MolecularSequence, I <em>really</em> like being able to list all the variants in a single resource, akin to a VCF file, which is what I can do in MS. Using Variant observation requires me to send a separate resource for each variant. This is overkill.</p>



<a name="193600892"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193600892" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193600892">(Apr 10 2020 at 18:21)</a>:</h4>
<p>Sequence is currently potentially a 4th way for Obs-variant to communicate the definitional variant observed in a patient.</p>



<a name="193600986"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193600986" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193600986">(Apr 10 2020 at 18:22)</a>:</h4>
<p>I get that point, but still think that it doesn't fit in the rest of our IG. For example: you can't point to this variants from Implications and other profiles.</p>



<a name="193601198"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193601198" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193601198">(Apr 10 2020 at 18:24)</a>:</h4>
<p>I agree it has great potential use in being a more concise way to send multiple variants, especially if we give it some support. Also agree it doesn't match the rest of our approach very well</p>



<a name="193601206"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193601206" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193601206">(Apr 10 2020 at 18:24)</a>:</h4>
<p>Also the example stated above replicates Variant usage with obs and ref allele + position + refSeq. It is the same, but instead of a Bundle it is a list stuck inside of MolSeq, not usable as individual Information objects. Somehow similar to contained resources.</p>



<a name="193601332"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193601332" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193601332">(Apr 10 2020 at 18:26)</a>:</h4>
<p>it's closer to a VCF file. You could make multiple Observations based on it</p>



<a name="193616854"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193616854" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193616854">(Apr 10 2020 at 20:57)</a>:</h4>
<p><a href="https://www.ncbi.nlm.nih.gov/bioproject/614995" title="https://www.ncbi.nlm.nih.gov/bioproject/614995">https://www.ncbi.nlm.nih.gov/bioproject/614995</a> is an example repository of genetic sequences. Right now the Utah Health Department is doing all our state's sequencing. They upload new virus samples as they go. And this is kinda cool as an end goal: <a href="https://nextstrain.org/ncov/global" title="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</a>  This works for treating things as isolates, but loses the clinical data associated with the patient - you don't know who or what test - which is what would come from a top level observation to is using derivedFrom to point to MolecularSequence or ObservationVariant.</p>



<a name="193617084"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193617084" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193617084">(Apr 10 2020 at 20:59)</a>:</h4>
<p>(deleted)</p>



<a name="193617145"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193617145" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193617145">(Apr 10 2020 at 20:59)</a>:</h4>
<p>Any genomic elements in the covid IG others have been working on?</p>



<a name="193617315"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193617315" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193617315">(Apr 10 2020 at 21:01)</a>:</h4>
<p>I only know this one COVID19 IG from Logica <a href="https://covid-19-ig.logicahealth.org/?mc_cid=15d567de77&amp;mc_eid=4a139423cc" title="https://covid-19-ig.logicahealth.org/?mc_cid=15d567de77&amp;mc_eid=4a139423cc">https://covid-19-ig.logicahealth.org/?mc_cid=15d567de77&amp;mc_eid=4a139423cc</a></p>



<a name="193617323"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193617323" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193617323">(Apr 10 2020 at 21:02)</a>:</h4>
<p>no advice yet in it</p>



<a name="193617407"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193617407" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193617407">(Apr 10 2020 at 21:02)</a>:</h4>
<p>There's an audacious inquiry one somewhere too</p>



<a name="193620315"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193620315" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Vassil Peytchev <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193620315">(Apr 10 2020 at 21:34)</a>:</h4>
<p><a class="stream" data-stream-id="226195" href="/#narrow/stream/226195-Covid-19-Response">#Covid-19 Response</a></p>



<a name="193621540"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193621540" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193621540">(Apr 10 2020 at 21:47)</a>:</h4>
<p>Probably would find it, but here is that IG:<br>
<a href="http://build.fhir.org/ig/AudaciousInquiry/fhir-saner/branches/master/index.html" title="http://build.fhir.org/ig/AudaciousInquiry/fhir-saner/branches/master/index.html">http://build.fhir.org/ig/AudaciousInquiry/fhir-saner/branches/master/index.html</a></p>



<a name="193643845"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193643845" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193643845">(Apr 11 2020 at 05:36)</a>:</h4>
<p><span class="user-mention" data-user-id="191916">@ Bob Milius</span>  suggested a variation on where to put variant which while different from how the current IG handles human sequences would limit the advice to using on MolecularSequence. Right now there is not a lot of sharing that includes clinical, patient level data, isolates of the virus are identified by the sequencing lab and the sequences posted to repositories. This leads me more towards MolecularSequence.</p>



<a name="193659772"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193659772" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193659772">(Apr 11 2020 at 13:09)</a>:</h4>
<p>I'm not sure what you mean by "limit the advice to using MolecularSequence." You can create an Observation with derivedFrom MolecularSequence and put the advice in that Obs, correct? And put both in a bundle.</p>



<a name="193788541"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193788541" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193788541">(Apr 13 2020 at 16:14)</a>:</h4>
<p>Reporting the existence of a covid19 infection is handled by several other IGs atm (PROMs from Switzerland, Logica for Infection reporting and some more)</p>



<a name="193788694"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193788694" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193788694">(Apr 13 2020 at 16:15)</a>:</h4>
<p>If a use-case needs the exchange of actual sequences, they could go with BAM files or VCF + refSequence.</p>



<a name="193788802"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193788802" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193788802">(Apr 13 2020 at 16:16)</a>:</h4>
<p>If there is a need to express this in FHIR you could use MolecularSequence.</p>



<a name="193789063"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193789063" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193789063">(Apr 13 2020 at 16:18)</a>:</h4>
<p>If there are implications which have to be reported, you would create these and point them to MolSeq via derivedFrom (because it is sliced open)</p>



<a name="193789126"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193789126" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193789126">(Apr 13 2020 at 16:18)</a>:</h4>
<p>If you need to point to a specific variant, you create a Variant Obs, pointing to MolSeq, and point the Implication to the Variant Obs</p>



<a name="193789649"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/193789649" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#193789649">(Apr 13 2020 at 16:24)</a>:</h4>
<p>I think this would be an important guidance in general.</p>



<a name="195579676"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/195579676" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#195579676">(Apr 28 2020 at 15:20)</a>:</h4>
<p>great! so the span of options provided at the start of this thread (along with query guidance provided) sounds acceptable. Shall I post to the more general Zulip thread? Do we need a committee vote on a guidance blurb? Or some other blessing from the committee?</p>



<a name="195581445"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/195581445" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#195581445">(Apr 28 2020 at 15:32)</a>:</h4>
<p>I think we will need to tighten it up with some examples. Also, where would this guidance be going (in the case we get something meaningful and helpful to say)</p>



<a name="195582339"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/195582339" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#195582339">(Apr 28 2020 at 15:38)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> just posting in the more generalized Zuip chat...but you co-chairs have more outlets for publication of information such as our webpages on the HL7 site etc... : ^ ) .... I can cook-up an example reference COIVD19 genome MolSeq profile for reference by a 'basic' lab positive observation (i'd be borrowing from Logica's draft IG).</p>



<a name="196302235"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/196302235" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#196302235">(May 05 2020 at 12:47)</a>:</h4>
<p>LOGIC focused on probe detection. not sure if its too far from the case, but on Friday CDC issued statement on ramping up viral genome collection: <a href="https://www.cdc.gov/media/releases/2020/p0501-SARS-CoV-2-transmission-map.html" title="https://www.cdc.gov/media/releases/2020/p0501-SARS-CoV-2-transmission-map.html">https://www.cdc.gov/media/releases/2020/p0501-SARS-CoV-2-transmission-map.html</a></p>



<a name="196304226"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/196304226" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Swapna Abhyankar <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#196304226">(May 05 2020 at 13:04)</a>:</h4>
<p>We created a LOINC code (currently in prerelease) for SARS-CoV-2 whole genome sequence: SARS coronavirus 2 whole genome [Nucleotide sequence] in Isolate by Sequencing (<a href="https://loinc.org/94764-8/" title="https://loinc.org/94764-8/">https://loinc.org/94764-8/</a>)</p>



<a name="196305704"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/196305704" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#196305704">(May 05 2020 at 13:16)</a>:</h4>
<p>FYI: please post on  <a href="#narrow/stream/226195-Covid-19-Response/topic/Viral.20whole.20genome.20sequencing" title="#narrow/stream/226195-Covid-19-Response/topic/Viral.20whole.20genome.20sequencing">https://chat.fhir.org/#narrow/stream/226195-Covid-19-Response/topic/Viral.20whole.20genome.20sequencing</a>  from here on. Thanks.</p>



<a name="196306393"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/COVID19%20viral%20sequences%3F/near/196306393" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/COVID19.20viral.20sequences.3F.html#196306393">(May 05 2020 at 13:21)</a>:</h4>
<p>Not sure its necessary to move it, but since you did, sure <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
