---
layout: page
title: "FHIR Chat  · Cytogenetic Nomenclature · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html">Cytogenetic Nomenclature</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="160024151"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160024151" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160024151">(Mar 05 2019 at 17:54)</a>:</h4>
<p>sorry for bringing this up here again, but maybe we can progress in the discussion here.<br>
To me this is still a finding,  a cytogenetic finding using a cytogenetic nomenclature (e.g. ISCN) to express what was found. Am i missing a fact why finding might off the table?</p>



<a name="160024518"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160024518" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160024518">(Mar 05 2019 at 17:59)</a>:</h4>
<p>We tentatively decided to keep the <a href="http://build.fhir.org/ig/HL7/genomics-reporting/obs-cytogenetic-nomenclature.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/obs-cytogenetic-nomenclature.html">profile</a> (name and all), and link to it from the <a href="http://build.fhir.org/ig/HL7/genomics-reporting/cytogenomics.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/cytogenomics.html">Cytogenomic Reporting</a> page.  The concern with changing the name to "finding" is that might imply a deeper level  of cytogenetic data than we are proposing as of today.  For some examples of the deeper cytogenetic data that we might consider some day, see the <a href="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=364" target="_blank" title="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=364">V2 guide</a></p>



<a name="160024658"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160024658" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160024658">(Mar 05 2019 at 18:00)</a>:</h4>
<p>That V2 guide was last published in 2014, and before we model FHIR IG profiles after it, the thinking is that we need SMEs to evaluate it's relevance in today's world of testing and reporting.</p>



<a name="160024978"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160024978" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160024978">(Mar 05 2019 at 18:04)</a>:</h4>
<p>To be honest, the V2 guide might still apply completely, or nearly completely.  The general sense from the WG as of today is that we don't have the right SMEs to confirm or deny.  And until we do, we want to keep it simple.</p>



<a name="160026248"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160026248" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160026248">(Mar 05 2019 at 18:20)</a>:</h4>
<blockquote>
<p>The concern with changing the name to "finding" is that might imply a deeper level  of cytogenetic data than we are proposing as of today.  </p>
</blockquote>
<p>I can follow this argumentation, but even if we would imply a deeper level than we have modeled yet. An Implementer could start with our profile and add other needed properties as component into his downstream use-case specific cytogenetic profile. (And share his structure with us in the best case).<br>
My motivation for keeping this discussion up is that i don't think we are using this profile to transport a cytogenetic nomenclature, but a cytogenetic finding/observation/aspect using this nomenclature.<br>
But if i am the only one with this issue i can accept keeping the profile name.</p>



<a name="160031413"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160031413" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160031413">(Mar 05 2019 at 19:15)</a>:</h4>
<p>Not sure how many of them we will use/not use, but there are various classes/type of "cytogenetic findings" articulated in the V2 Cyto IG that don't have the ISCN string contained in them.  So, I am fairly certain that will be will end up with different profiles, and maybe find a base "Cytogenetic Finding" profile - it just won't include the ISCN string.  So, if someone wanted to send Cyto data, I wouldn't expect them to profile off the profile that contains the ISCN string, they would likely do their own profile of Observation.</p>



<a name="160032116"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160032116" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160032116">(Mar 05 2019 at 19:23)</a>:</h4>
<p>Ok, what about having a cytogenetic findings profile, with one component (for now) which contains the ISCN String. Then we can other components as we need them.</p>



<a name="160032838"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160032838" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160032838">(Mar 05 2019 at 19:32)</a>:</h4>
<p>I not sure the other components semantically belong in the same profile as the ISCN string value.  I would compare it to putting the HGVS string in the Genetic Finding profile - that wouldn't make sense because you don't need the HGVS string to represent a Genotype or Haplotype - only need it in Variant.</p>



<a name="160038592"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160038592" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160038592">(Mar 05 2019 at 20:47)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span> Good question.  I depends on how you are finding is defined.  For example, if it's part of the lab report, cytogenetics report, genomics report, it's not usually a finding, but rather results (aka observations).  Lab results are coded with LOINC.  However, when the results are seen in the EHR and assessed by the physician, they usually note that the lab result is a finding of X or diagnosis of X and coded with ICD-10/SNOMED CT.   It appears that the definition you are using is the second one.  Is that correct?  If so, I wouldn't expect it to be in the primary genetics report reported by the laboratory (unless in the diagnostic section issued by the pathologist) as much as notation by a downstream receiver such as a clinical geneticist or oncologist.  Does that help address your question?</p>



<a name="160041894"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160041894" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160041894">(Mar 05 2019 at 21:29)</a>:</h4>
<p>FYI-  Commented on Previous Ballots asking that Genomics Guides are aligned with CAP Cytogenetic Checklist requirements (for accreditation of labs in US and those international customers).  Clem indicated many of the requirements would be in the CG Cytogenetics Guide not CG Guide.  </p>
<p>For accreditation, reports for cytogenetics studies and genomic copy number analysis using microarrays, require ISCN is used correctly in the report.  Constitutional Cases must be banded at the 400 band level, according to ISCN requirements too.</p>
<p>What's not clear to me and mentioned today is why all the cytogenetics testing, report information isn't in the Cytogenetics Implementation Guide and why we are discussing with the CG guide?  (especially if previous CG ballot comments were addressed based upon that premise?)  I thin this gets to some of <span class="user-mention" data-user-id="191451">@Patrick Werner</span> 's questions too. (please correct me if I've misunderstood.)</p>



<a name="160048768"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160048768" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160048768">(Mar 05 2019 at 22:51)</a>:</h4>
<p>The current Cyto IG is a V2 guide.  It might make sense to create an entirely new FHIR IG for Cyto - but new FHIR profiles roughly mathcing the V2 IG were drafted as part of the FHIR Genomics Reporting IG.  Without the proper vetting, we decided to remove the Cyto specifics from the FHIR IG --- except the "ISCN string".  So, for the first release, we are not going to be anywhere near to support full Cytogenetic reporting requirements.</p>



<a name="160108001"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160108001" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160108001">(Mar 06 2019 at 15:59)</a>:</h4>
<p><span class="user-mention" data-user-id="192198">@Bob Freimuth</span>  I think the model you created in the CG modeling sub-group would be informative here. It got into the guts of ISCN a bit.</p>



<a name="160111919"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160111919" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160111919">(Mar 06 2019 at 16:41)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> Not too deeply.  We broke apart the cytoband notation but that is as far as we went.  We didn't want to re-invent ISCN or go too far without a stakeholder pushing for more detail.</p>



<a name="160290800"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Cytogenetic%20Nomenclature/near/160290800" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Cytogenetic.20Nomenclature.html#160290800">(Mar 08 2019 at 13:56)</a>:</h4>
<p>precisely - but still worth describing what it means to break apart the cytoband notation. I think folks would find that useful</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
