---
layout: page
title: "FHIR Chat  · Describing TMB &amp; Other Tests · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html">Describing TMB &amp; Other Tests</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="168156251"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168156251" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168156251">(Jun 14 2019 at 17:45)</a>:</h4>
<p>Hi all, with the recent discussion on TMB it seems apparent that research will always be trying new test types and want to represent the results (and describe the test itself) in FHIR. This means concepts outside of LOINC will need to be conveyed, and we need to provide guidance on standardizing them or we will be getting a bunch of workarounds with data that are difficult to interpret or reuse.</p>



<a name="168156370"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168156370" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168156370">(Jun 14 2019 at 17:47)</a>:</h4>
<p>I'd love to see us bring back/update the "descriptive genetic finding" profile to help cover and unify these fringe cases so we can ensure the results get to the right places and are able to be seen in context.</p>
<p>Thoughts?</p>



<a name="168156482"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168156482" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168156482">(Jun 14 2019 at 17:48)</a>:</h4>
<p>Beyond a profile, guidance on how to send new/specific concepts as additional components/Observations/whole Reports is needed.</p>



<a name="168156866"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168156866" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168156866">(Jun 14 2019 at 17:53)</a>:</h4>
<p>I know our first focus is on data types rather than workflow, but since we mention the 'Abstract' profiles are not meant to be implemented, making sure there's at least a reasonably obvious place to put results that don't exactly fit into other 'Findings' seems very important.</p>



<a name="168159689"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168159689" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168159689">(Jun 14 2019 at 18:27)</a>:</h4>
<p>One thing we discussed on Tuesday - Do we need any sort of profile for this or not?  The general thinking on the call was no as many of these other 'findings' would fit simply as Observations.  We did discuss the need for additional guidance in the IG however about how to send something that is a genetic finding, but doesn't align with one of our existing profiles - so it would really be just an Observation.</p>
<p>Having said that, perhaps there is value in defining a profile to 'unify these fringe cases'?   I am not quite sure what all we would profile from Observation, but perhaps starting with a simple profile is helpful?</p>



<a name="168161431"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168161431" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168161431">(Jun 14 2019 at 18:49)</a>:</h4>
<p>It sounds like these tests may be reported as part of a more comprehensive genomics report and/or reported on their own as part of a typical lab report? If so, it might make sense to see how O&amp;O and LOINC are approaching them. In the mean time, we could at least begin with 'additional guidance'.</p>



<a name="168161580"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168161580" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168161580">(Jun 14 2019 at 18:51)</a>:</h4>
<blockquote>
<p>This means concepts outside of LOINC will need to be conveyed, </p>
</blockquote>
<p>I did want to ask <span class="user-mention" data-user-id="196553">@James Jones</span> a bit more about this comment.  What outside of LOINC are you referring too?  The Observation.code, Observation.value, or something else?</p>



<a name="168166357"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168166357" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168166357">(Jun 14 2019 at 19:52)</a>:</h4>
<p>Specifically we run into trouble where someone wants to send an Observation that doesn't align with any of the codes we have required in our current set of Observation profiles.</p>
<p>If we opened up the base genetic observation profile for implementation (in our textual guidance), it may solve most of the issue.</p>



<a name="168167930"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168167930" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168167930">(Jun 14 2019 at 20:11)</a>:</h4>
<p>For instance, MSI results have LOINC code(s) but we give no guidance on how to send them. (see <a href="https://r.details.loinc.org/LOINC/81711-4.html?sections=Comprehensive" target="_blank" title="https://r.details.loinc.org/LOINC/81711-4.html?sections=Comprehensive">https://r.details.loinc.org/LOINC/81711-4.html?sections=Comprehensive</a>)</p>



<a name="168171393"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168171393" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168171393">(Jun 14 2019 at 21:01)</a>:</h4>
<p>I got ya.  So it seems we all agree we need some textual guidance, but the outstanding question of "simple profile or Observation?" still remains.</p>



<a name="168324185"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168324185" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168324185">(Jun 17 2019 at 16:13)</a>:</h4>
<p>Discussion today suggested inclusion on Figure 1 (the genomics report "results" figure) of "Other Observations". In the text under the figure we would specifically mention that other results (TMB/MSI/various biomarker tests) may be included along with specific variants and other computable genetic findings.</p>



<a name="168324411"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168324411" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168324411">(Jun 17 2019 at 16:15)</a>:</h4>
<p>The question remains if we would like these observations to use a gene-studied component where possible or if there is any other guidance we wanted to give.</p>



<a name="168324854"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168324854" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Arthur Hermann <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168324854">(Jun 17 2019 at 16:20)</a>:</h4>
<p>This PDF with test results is a good example of the problem which we need to solve.. <a href="/user_uploads/10155/K5ufmbKB5E6xjFm7z6osVhJE/FoundationOne_CDx_Sample_Report.pdf" target="_blank" title="FoundationOne_CDx_Sample_Report.pdf">FoundationOne_CDx_Sample_Report.pdf</a></p>



<a name="168350872"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168350872" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168350872">(Jun 17 2019 at 21:53)</a>:</h4>
<p>Regarding whether or not these observations need the gene-studied component, I am afraid I don't know.  It seems OK for some, but not all?  Are there other common components that would be used?</p>
<p>I am not an expert for sure.  So I am basically just responding to keep the conversation going ... <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="168356655"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168356655" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Arthur Hermann <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168356655">(Jun 17 2019 at 23:24)</a>:</h4>
<p>I know this is contrary to best practices.. but I am certain that no matter what we do, there will be other types of information in the Observation that we don't have a LOINC code for, or any other standard way of allowing this information to be sent as part of an observation. For this reason, I would think we would want to create a method to allow these results to be reported somehow. Do we currently have any way to do so? If so, could we amplify the guidance on using the implementation guide for this scenario?</p>



<a name="168356774"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/168356774" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Arthur Hermann <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#168356774">(Jun 17 2019 at 23:27)</a>:</h4>
<p>I see that Jamie essentially said what I stated: To summarize my thoughts, I think we need another "Finding" profile and guidance on how to send genetic Observations that aren't specifically variants/genotypes/haplotypes or we may see some weird workarounds that create data that is difficult to interpret or reuse. It could either be lightweight textual and descriptive, or based on Grouper specifically modeling a test/panel (may need both options in the future).</p>
<p>Sorry for the duplication Jamie - but I completely agree with your train of thought!</p>



<a name="171197902"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/171197902" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#171197902">(Jul 18 2019 at 18:44)</a>:</h4>
<p>For TMB, is anyone reporting allelic frequency? We're potentially running into issues with this not being reported/standardized, so a report would be stating a much higher TMB than another on the same sample at a different frequency. Just a thought towards later developing a profile for this test, if it isn't already happening with O&amp;O...</p>



<a name="171239989"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/171239989" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#171239989">(Jul 19 2019 at 08:39)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@James Jones</span> <br>
Yes we are getting NAF for TMB from our lab</p>



<a name="171239993"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/171239993" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#171239993">(Jul 19 2019 at 08:40)</a>:</h4>
<p>Mutationslast (Tumor Mutational Burden): intermediär (7,3 Varianten/Megabase)</p>



<a name="171240197"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/171240197" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#171240197">(Jul 19 2019 at 08:43)</a>:</h4>
<p>i think the development of a TMB profile should be done by us as we are the domain experts.</p>



<a name="171240296"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/171240296" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#171240296">(Jul 19 2019 at 08:45)</a>:</h4>
<p>value would be the NAF and interpretation the coded interpretation</p>



<a name="171252572"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Describing%20TMB%20%26%20Other%20Tests/near/171252572" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Describing.20TMB.20.26.20Other.20Tests.html#171252572">(Jul 19 2019 at 12:25)</a>:</h4>
<p>asked about the ucum code here: <a href="#narrow/stream/179202-terminology/topic/UCUM.20Code.20variants.2FMegaBase" title="#narrow/stream/179202-terminology/topic/UCUM.20Code.20variants.2FMegaBase">https://chat.fhir.org/#narrow/stream/179202-terminology/topic/UCUM.20Code.20variants.2FMegaBase</a></p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
