---
layout: page
title: "FHIR Chat  · fusion genes · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html">fusion genes</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="167490789"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/fusion%20genes/near/167490789" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html#167490789">(Jun 06 2019 at 15:14)</a>:</h4>
<p>is there a consensus on how to capture variants-obs on fusion genes? I know the notation: ETV6-NTRK3 vs. ETV6:NTRK3 (i think the first one is more common).<br>
Would we capture this in gene studied id? Gene studied id has an extensible ValueSet binding on HGNC. To my knowledge ETV6-NTRK3 wouldn't be a valid HGNC value. If we would use the ETV6-NTRK3 format, what would be the CodeSystem? Our own one?: <a href="http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/fusion-gene" target="_blank" title="http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/fusion-gene">http://hl7.org/fhir/uv/genomics-reporting/CodeSystem/fusion-gene</a>?</p>



<a name="167524879"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/fusion%20genes/near/167524879" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html#167524879">(Jun 06 2019 at 21:34)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span>  you remind me of this ongoing but only kinda related question I have - I'm just curious why oncology knowledgebases generally represent variants at the protein level rather than at the DNA level?</p>



<a name="167611934"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/fusion%20genes/near/167611934" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html#167611934">(Jun 07 2019 at 20:37)</a>:</h4>
<p>I don’t think we ever discussed this and I am not sure what the right answer is? Is there a list of possible fusion genes anywhere we could reference in a new “fusion-gene” code system?</p>



<a name="167841892"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/fusion%20genes/near/167841892" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Dora Walter <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html#167841892">(Jun 11 2019 at 11:43)</a>:</h4>
<p>Oncological knowledge databases can, for example, provide information on questions such as the possible relationship between AR amplification and TMPRSS2-ERG gene fusion in prostate cancer. It is known that gene fusions occur in more than 50% of prostate adenocarcinoma, with TMPRSS2-ERG being one of the most common. In my opinion, both levels are important to identify the variants at the DNA level and the effects on protein levels (if known). </p>
<p>Maybe this could help for Fusion Gene Lists? <br>
FusionGDB (Fusion Gene annotation DataBase) available at <a href="https://ccsm.uth.edu/FusionGDB" target="_blank" title="https://ccsm.uth.edu/FusionGDB">https://ccsm.uth.edu/FusionGDB</a>.</p>



<a name="168030463"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/fusion%20genes/near/168030463" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html#168030463">(Jun 13 2019 at 09:56)</a>:</h4>
<p>i contacted the staff from FusionGDB and they confirmed that a Fusion between geneA-geneB is unique. Therefore this seems a good CodeSystem to include. And transport the fusions as FusionGDB ID encoded values.</p>



<a name="175756411"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/fusion%20genes/near/175756411" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html#175756411">(Sep 15 2019 at 16:42)</a>:</h4>
<p>regards stating the effect on the protein level. For others, the logic is that if there is a DNA change this may or may not be relevant at the protein sequence level. For example, if the change does not effect splicing or codons then at the protein sequence level there is no change. There can still be a change in the amount of protein but that is not a protein sequence change. (maybe obvious to everyone) &gt; <span class="user-mention" data-user-id="195522">@Dora Finkeisen</span> sound right?</p>



<a name="175758154"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/fusion%20genes/near/175758154" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Dora Walter <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/fusion.20genes.html#175758154">(Sep 15 2019 at 17:29)</a>:</h4>
<p>right</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
