---
layout: page
title: "FHIR Chat  · genotype and haplotype · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html">genotype and haplotype</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="153948831"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948831" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948831">(Mar 28 2018 at 21:25)</a>:</h4>
<p>In the CG IG,  figure 4 - genetic findings, there's a Genotype 844413-4 and a Haplotype 844414-2 observation. Are these observation.codes or observation.component.codes? I'm not sure how to say an observation is a genotype or a haplotype.</p>



<a name="153948835"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948835" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948835">(Mar 28 2018 at 21:36)</a>:</h4>
<p>Those would be used as Observation.code.  You would use Observation.valueString to represent the value of the genotype/haplotype, and then use the components listed in the 'Computable Genetic Finding' profile (gene, chromosome, dna region) as Observation.components.code values on the same observation.</p>



<a name="153948838"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948838" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948838">(Mar 28 2018 at 21:39)</a>:</h4>
<p>In general, the LOINC mentioned in the diagrams in the title would be Observation.code.  A better option is to review the profiles themselves for guidance like this.</p>



<a name="153948842"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948842" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948842">(Mar 28 2018 at 21:41)</a>:</h4>
<p>And now that you made me look at this, I notice a difference.  For Haplotype and Genotype, the diagram says to use valueString, but the profiles say to use valueCodeableConcept.  <span class="user-mention" data-user-id="191320">@Lloyd McKenzie</span> ?</p>



<a name="153948843"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948843" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948843">(Mar 28 2018 at 21:45)</a>:</h4>
<p>Also <span class="user-mention" data-user-id="191320">@Lloyd McKenzie</span>  - I think we noticed this before, but I see that 'obs-base' and 'obs-comp-gen-finding' and 'obs-genotype' (and likely others) show up on the profile pages with the display name of "CG Observations"?  Not sure where that is pulled from in the spreadsheets, but might be good to change if we can?</p>



<a name="153948845"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948845" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948845">(Mar 28 2018 at 21:52)</a>:</h4>
<p>It gets pretty confusing because on the profiles (like obs-genotype), it shows as the structure being derived from "CG Observations", but then looking at the URL shows its (correctly) pointing at obs-comp-gen-finding .  </p>
<p>It also then shows "CG Observations" multiple times as valid types for the .derivedFrom reference to other resources as well.</p>



<a name="153948846"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948846" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948846">(Mar 28 2018 at 21:54)</a>:</h4>
<p>I click on the Differential tab. There, if I hover over the different CG Observations, it shows they are pointing to different structure definitions</p>



<a name="153948848"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948848" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948848">(Mar 28 2018 at 21:59)</a>:</h4>
<p>Yup, the actual links are correct.  Its the display that is confusing because they all say the same thing.  Am hoping that is easy to fix, but will likely to have let Lloyd weigh in.</p>



<a name="153948867"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948867" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948867">(Mar 28 2018 at 22:13)</a>:</h4>
<p>valueCodeableConcept.  Diagrams will be fixed on my next commit</p>



<a name="153948868"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948868" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948868">(Mar 28 2018 at 22:13)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> Can you add a todo on the spreadsheet on "CG Observations"?</p>



<a name="153948874"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948874" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948874">(Mar 28 2018 at 22:21)</a>:</h4>
<p>Done</p>



<a name="153948876"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/153948876" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#153948876">(Mar 28 2018 at 22:22)</a>:</h4>
<p>And, shout outs to <span class="user-mention" data-user-id="192123">@Bret H</span> and <span class="user-mention" data-user-id="191916">@ Bob Milius</span>  for loading up our issue list today <span class="emoji emoji-1f642" title="slight smile">:slight_smile:</span></p>



<a name="259743035"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/259743035" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#259743035">(Oct 30 2021 at 17:48)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> Do you know if the intent of <a href="http://jira.hl7.org/browse/FHIR-14315">FHIR-14315</a> and <a href="http://jira.hl7.org/browse/FHIR-14316">FHIR-14316</a> were to include location coordinates for haplotypes and genotypes? Reason I ask is that I was trying to represent an intergenic haplotype, and I'm not sure I can do it without a refSeq and locations (<a href="https://pubmed.ncbi.nlm.nih.gov/23959643/">https://pubmed.ncbi.nlm.nih.gov/23959643/</a>).</p>



<a name="259746901"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/259746901" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#259746901">(Oct 30 2021 at 19:21)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> - TBH, I am not sure. For the most part, we have been using Haplotype/Genotype more as a container to ultimately represent a list of Variants, which is where you would find the refSeq and locations. However, perhaps that is not always appropriate.</p>
<p>I can't get to the full article. Do you mind presenting more detail on how you think we should represent haplotypes and genotypes in this use case? Are you thinking about adding some combination of genomic-ref-seq, coordinate-system, chromosome-identifer, exact-start-end, inner-start-end, outer-start-end that are on Variant to Haplotype and Genotype?</p>
<p>And sorry if this is a dumb question, but could you represent the intergenic variant(s) with the Variant profile, then group them with  Haplotype and/or Genotype?</p>
<p>Once again, I think you are on the edges or even a bit outside of what we were intending to support <span aria-label="smile" class="emoji emoji-1f642" role="img" title="smile">:smile:</span></p>



<a name="259747446"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/259747446" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#259747446">(Oct 30 2021 at 19:36)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="191913">@Kevin Power</span> . To be honest, I'm still thinking it through as well. (a preprint of article is here: <a href="https://digitalcommons.wayne.edu/cgi/viewcontent.cgi?article=1024&amp;context=humbiol_preprints">https://digitalcommons.wayne.edu/cgi/viewcontent.cgi?article=1024&amp;context=humbiol_preprints</a>)</p>
<p>In this case, there are a number of haplotypes defined over an intergenic region between EDA2R and AR genes. The haplotype is defined based on ~100 SNPs. It might be impractical to list each SNP as a variant if, for instance, I only want to communicate that patient has the 'EDA2R/AR alpha haplotype'. But I'd also like to include something such as a location that will allow someone to search and find this haplotype if they need to. </p>
<p>I was thinking of perhaps moving the coordinate-system, genomic-ref-seq, and exact-start-end components from Variant up into Genomic Finding might be a good start.</p>



<a name="259748047"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/259748047" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#259748047">(Oct 30 2021 at 19:44)</a>:</h4>
<p>We would certainly have to look at everything else that is based on Finding.</p>



<a name="259748278"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/259748278" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#259748278">(Oct 30 2021 at 19:46)</a>:</h4>
<p>It gets worse - the current genes component is 0..*, and in the draft GA4GH model, a genotype can include multiple refSeqs. Sounds like possibly a topic for the next round...</p>



<a name="259750068"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/259750068" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#259750068">(Oct 30 2021 at 20:32)</a>:</h4>
<p>This is the fixing I hope comes out of integrating <span class="user-mention" data-user-id="192198">@Bob Freimuth</span>’s IM work into the IG</p>



<a name="259877599"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/genotype%20and%20haplotype/near/259877599" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/genotype.20and.20haplotype.html#259877599">(Nov 01 2021 at 15:26)</a>:</h4>
<p><span class="user-mention silent" data-user-id="193751">Bob Dolin</span> <a href="#narrow/stream/179197-genomics/topic/genotype.20and.20haplotype/near/259747446">said</a>:</p>
<blockquote>
<p>Thanks <span class="user-mention silent" data-user-id="191913">Kevin Power</span> . To be honest, I'm still thinking it through as well. (a preprint of article is here: <a href="https://digitalcommons.wayne.edu/cgi/viewcontent.cgi?article=1024&amp;context=humbiol_preprints">https://digitalcommons.wayne.edu/cgi/viewcontent.cgi?article=1024&amp;context=humbiol_preprints</a>)</p>
<p>In this case, there are a number of haplotypes defined over an intergenic region between EDA2R and AR genes. The haplotype is defined based on ~100 SNPs. It might be impractical to list each SNP as a variant if, for instance, I only want to communicate that patient has the 'EDA2R/AR alpha haplotype'. But I'd also like to include something such as a location that will allow someone to search and find this haplotype if they need to. </p>
<p>I was thinking of perhaps moving the coordinate-system, genomic-ref-seq, and exact-start-end components from Variant up into Genomic Finding might be a good start.</p>
</blockquote>
<p>Here you can send the genotype with references back to the location of the variant instances on the originating FHIR server. but the basic variant data is important to have as well as the summation genotype if one wants to have the discrete details.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
