---
layout: page
title: "FHIR Chat  · harmonize somatic and PGx profiles · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html">harmonize somatic and PGx profiles</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="177073233"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/177073233" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#177073233">(Oct 01 2019 at 16:38)</a>:</h4>
<p>We have a set of profiles to deliver PGx implications and a separate profile to deliver Tumor specific treatment implications. The major difference in these two concepts is that the PGx implications are about how the patient's non-tumor tissue will effect the pharmacodynamics and pharmacokinetics of medications. The somatic profile is about the tumor. Our current mechanism of separation is through pre-coordinating Somatic in the profile name. But one could consider the tumor as a Focal subject. <span class="user-mention" data-user-id="195180">@Alexander Mankovich</span></p>



<a name="177073339"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/177073339" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#177073339">(Oct 01 2019 at 16:39)</a>:</h4>
<p>from <span class="user-mention" data-user-id="191451">@Patrick Werner</span> <br>
this could be done through a component in the profile which states: tumor/patient origin of the specimen. Theoretically this could be also be described in the Specimen, but i think it would be easier to code this in the Observation profile.</p>



<a name="177073380"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/177073380" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#177073380">(Oct 01 2019 at 16:39)</a>:</h4>
<p><span class="user-mention" data-user-id="195180">@Alexander Mankovich</span></p>



<a name="177264534"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/177264534" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Alexander Mankovich <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#177264534">(Oct 03 2019 at 17:03)</a>:</h4>
<p>I think Patrick's approach would be fine. Just to be clear, this proposal would replace somatic prognostic and somatic predictive implication (but not diagnostic) profiles correct? Is there still room to specify the associated-cancer component using the pgx profiles? I'm also wondering how predictive implications would be represented if the therapy was something other than a drug.</p>



<a name="177266295"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/177266295" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#177266295">(Oct 03 2019 at 17:20)</a>:</h4>
<p>It seems the major pre-coordination is in the Observation.code, (eg, "tbd-som-predictive"), which would tell the system what components to expect in that "knowledge observation" and how to interpret it semantically. </p>
<p>The main computational difference in the profiles (medication-implication and obs-som-predictive) is that one requires the <code>medication-assessed</code> component to be present, and the other requires the <code>associated-cancer</code> component.</p>
<p>However, our component slicing is currently "open" so someone could easily send an <code>associated-cancer</code> component on one of our PGx implications currently and vice versa. Similar arguments could be made about <code>associated-therapy</code>, but it gets messy with our current required cardinalities.</p>
<p>In general, we are looking for a balance here between having "too many boxes" (an overwhelming number of profiles not fitting an obvious pattern) and "big leaky buckets" (where a "bucket-like" implication profile would require a system to look at which components were populated to understand how to process it)... I don't have a solution for this yet, except thinking that our recent decision to frame the IG as general guidance instead of targeting specific use cases may suggest we ought to have one "bucket-like" implication to match obs-variant and then smaller use-case specific IGs/sections could derive use-case implications from that profile.</p>



<a name="177266455"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/177266455" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#177266455">(Oct 03 2019 at 17:22)</a>:</h4>
<p>That is, rather than have inherited profiles <code>add</code> components, they could be <code>restricting</code> them from the base implication. Something to consider in the future, in my opinion...</p>



<a name="181034764"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181034764" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181034764">(Nov 18 2019 at 17:08)</a>:</h4>
<p>Wanted to bump this thread as came up on call today, in context of going over Kevin's branch to join the 3 somatic profiles (available at <a href="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/index.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/index.html">http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/index.html</a>).</p>



<a name="181035203"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035203" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035203">(Nov 18 2019 at 17:13)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  <span class="user-mention" data-user-id="192302">@May Terry</span>  please see Alex's comments above. His comments indicate how he was planning to use associated cancer.  And Patricks comments on adding an indicator for specimen origin (what has the genetic variation). <span class="user-mention" data-user-id="196553">@James Jones</span>  worth considering.  The merging that Kevin did fits well with this approach. The main benefit and drawback is that you explicitly remove the ability to mix-match components.</p>



<a name="181035535"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035535" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035535">(Nov 18 2019 at 17:16)</a>:</h4>
<blockquote>
<p>The main benefit and drawback is that you explicitly remove the ability to mix-match components.</p>
</blockquote>
<p>How so?</p>



<a name="181035544"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035544" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035544">(Nov 18 2019 at 17:16)</a>:</h4>
<p>I want to highlight the need to indicate weather the target of therapy is a tumor, and weather the problem is with the patient's liver metabolism (pharmacokinetics) versus the tumor's receptiveness to treatment (pharmacodynamics).</p>



<a name="181035574"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035574" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035574">(Nov 18 2019 at 17:16)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  can't have an instance with a non-allowed component. that's all.</p>



<a name="181035649"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035649" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035649">(Nov 18 2019 at 17:17)</a>:</h4>
<blockquote>
<p><span class="user-mention silent" data-user-id="191913">Kevin Power</span>  can't have an instance with a non-allowed component. that's all.</p>
</blockquote>
<p>I didn't change anything that would add that restriction?</p>



<a name="181035718"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035718" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035718">(Nov 18 2019 at 17:18)</a>:</h4>
<p>(at least I don't think I did?)</p>



<a name="181035725"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035725" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035725">(Nov 18 2019 at 17:18)</a>:</h4>
<p>referring to Jaime's suggestion. Not your branch,</p>



<a name="181035758"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035758" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035758">(Nov 18 2019 at 17:18)</a>:</h4>
<p>ahhhhh <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="181035986"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181035986" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181035986">(Nov 18 2019 at 17:21)</a>:</h4>
<p>Yes, my top-down inheritance would only work for the use cases we identify. That said, I went to add a component onto a profile that appeared to have open slicing and received an error, but it is likely one from the validator and not specific to our spec.</p>



<a name="181036167"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036167" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036167">(Nov 18 2019 at 17:22)</a>:</h4>
<p>And I would prefer the fewer boxes in Kevin's approach, I just have to do a bit of homework on the components to make sure we can provide the right guidance for Observation.value, and confirm where this approach may impact other issues we have with the 1.0 Implication profiles in general.</p>



<a name="181036303"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036303" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036303">(Nov 18 2019 at 17:24)</a>:</h4>
<blockquote>
<p>That said, I went to add a component onto a profile that appeared to have open slicing and received an error, but it is likely one from the validator and not specific to our spec.</p>
</blockquote>
<p>Hmm, an error?  I see our examples generating "informational" messages for that situation, but not errors.</p>



<a name="181036339"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036339" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036339">(Nov 18 2019 at 17:24)</a>:</h4>
<p>I was getting a 'missing expected required component' error on an example that didn't make it into the build</p>



<a name="181036415"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036415" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036415">(Nov 18 2019 at 17:25)</a>:</h4>
<p>but I think that was just an issue with the validator interpreting open slicing in that case. Seems to work fine on other profiles and I'd have to do some troubleshooting to pin it down further.</p>



<a name="181036416"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036416" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036416">(Nov 18 2019 at 17:25)</a>:</h4>
<p>If I get the time today or tomorrow, I might push up additional changes:</p>
<ul>
<li>Try to combine the PGx profiles in a similar way that I did on the Somatic side</li>
<li>Renaming "Inherited Disease Pathogenicity" to "Inherited Disease Implication" for better alignment.</li>
</ul>



<a name="181036570"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036570" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036570">(Nov 18 2019 at 17:27)</a>:</h4>
<p>I like both those bullets, I'll think about how to pursue Observation.value here, if we really can have it subsume the previous value sets in a good way or if it should just be present/absent/etc and the values need their own components.</p>



<a name="181036571"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036571" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036571">(Nov 18 2019 at 17:27)</a>:</h4>
<blockquote>
<p>I was getting a 'missing expected required component' error on an example that didn't make it into the build</p>
</blockquote>
<p>Oh, well that sounds more like something that I made "required" by make it 1..* where before it might have been 0..* - what component?</p>



<a name="181036726"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036726" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036726">(Nov 18 2019 at 17:28)</a>:</h4>
<blockquote>
<p>I'll think about how to pursue Observation.value here, if we really can have it subsume the previous value sets in a good way or if it should just be present/absent/etc and the values need their own components.</p>
</blockquote>
<p>Yea, I suppose that would work - then have three different components (for the three different value sets)?</p>



<a name="181036845"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036845" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036845">(Nov 18 2019 at 17:30)</a>:</h4>
<p>I forget the example, but the component that was being asked for was already present in the profile (and the only one defined). Adding a second component that wasn't defined through a non-sensical error, so I think we can move past it unless we run into it again.</p>
<p>And yeah, I think each concept needs a defined component, I'll have some time this week to fiddle with it</p>



<a name="181036907"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181036907" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181036907">(Nov 18 2019 at 17:30)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> take care when combining the pgx profiles, the value sets are unique&gt; I'm not a fan of all the diversity that CPIC has in the value sets. There is large overlap. But having the profiles separated allows one to specifically indicate which value set goes with which LOINC code.</p>



<a name="181037400"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181037400" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181037400">(Nov 18 2019 at 17:35)</a>:</h4>
<p>the upshot: either the sender does the work (i.e. use the right profile) and the user just puts in the right place. OR the instance comes in and the user has logic to verify that the value is from the correct value set. something like this</p>



<a name="181054175"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181054175" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181054175">(Nov 18 2019 at 20:48)</a>:</h4>
<p>I went back into our Domain Analysis Model doc and searched for "impact" (our old term for "implication") and found that we reference <a href="https://www.fda.gov/drugs/science-and-research-drugs/table-pharmacogenomic-biomarkers-drug-labeling" target="_blank" title="https://www.fda.gov/drugs/science-and-research-drugs/table-pharmacogenomic-biomarkers-drug-labeling">https://www.fda.gov/drugs/science-and-research-drugs/table-pharmacogenomic-biomarkers-drug-labeling</a> when introducing PGx. Of note on that table, "Oncology" is quite prominent, reinforcing the idea that germline and somatic PGx implications may not be separate use cases needing individual profiles.</p>



<a name="181056439"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181056439" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181056439">(Nov 18 2019 at 21:10)</a>:</h4>
<p>Yea, seems like the biggest things differentiating our 3 types of implications today (inherited, somatic, PGx) are:</p>
<ul>
<li>some differences in components</li>
<li>fairly important differences in answer lists</li>
</ul>



<a name="181062156"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181062156" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181062156">(Nov 18 2019 at 22:12)</a>:</h4>
<p>Re: condition vs cancer vs phenotype, any reason we shouldn't be considering HPO (used by phenopackets alongside snomed)</p>



<a name="181062204"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181062204" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181062204">(Nov 18 2019 at 22:13)</a>:</h4>
<p>Phenotypic feature: <a href="https://mseqdr.org/hpo_browser.php?118" target="_blank" title="https://mseqdr.org/hpo_browser.php?118">https://mseqdr.org/hpo_browser.php?118</a>;</p>



<a name="181062281"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181062281" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181062281">(Nov 18 2019 at 22:14)</a>:</h4>
<p>100% should be an option - not sure if it can be the only option though.</p>



<a name="181068169"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181068169" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181068169">(Nov 18 2019 at 23:39)</a>:</h4>
<p>Also, wanted to note - I am holding off on further changes for now, mostly because of <span class="user-mention" data-user-id="192123">@Bret H</span> great point, which is spot on:</p>
<blockquote>
<p>take care when combining the pgx profiles, the value sets are unique&gt; I'm not a fan of all the diversity that CPIC has in the value sets. There is large overlap. But having the profiles separated allows one to specifically indicate which value set goes with which LOINC code.</p>
</blockquote>



<a name="181069354"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181069354" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181069354">(Nov 18 2019 at 23:56)</a>:</h4>
<p>Pushing the values into components fulfills this as well, but may introduce need for additional invariants on the profile</p>



<a name="181074251"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181074251" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181074251">(Nov 19 2019 at 01:34)</a>:</h4>
<p>The fact that we need invariants in this case feels like we don’t have something quite right yet. But yes that helps address the desire to flex the answer lists</p>



<a name="181298281"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/harmonize%20somatic%20and%20PGx%20profiles/near/181298281" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/harmonize.20somatic.20and.20PGx.20profiles.html#181298281">(Nov 21 2019 at 12:13)</a>:</h4>
<p>how often should invariants be used? seems like an attractive new toy in some ways</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
