---
layout: page
title: "FHIR Chat  · HGNC gene symbol · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html">HGNC gene symbol</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="181851308"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181851308" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181851308">(Nov 25 2019 at 18:50)</a>:</h4>
<p>During the DevDays i was asked what implementers should do if they don't know the HGNC Gene ID, german labs are only reporting the approved names and are afraid of lookups. <br>
Do you think it would be a good idea to create a HGNC approved symbols VS (BRCA2, EGFR, ...) and a component to put this into?</p>



<a name="181852474"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181852474" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181852474">(Nov 25 2019 at 19:02)</a>:</h4>
<p>Tough to support both id and name in my opinion, voted specifically to support the IDs over the names. </p>
<p>Multiple lookups are needed in the domain, how did they get the HGNC names themselves if not from a lookup? It seems that's the service they should look at upgrading to ease compliance :)</p>



<a name="181860454"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181860454" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181860454">(Nov 25 2019 at 20:40)</a>:</h4>
<p>Thats why i'm thinking of  a new component, coded to represent HGNC approved Symbols. <br>
This use case was brought to me by labs, and lately by the "german medical IT initiative" a quite big funding for german university hospitals to exchange data. If they have structured variant data, the genes are represented as symbols. I personally would implement a lookup into my solution, but there are implementers which will stuck to the symbol.</p>



<a name="181860586"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181860586" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181860586">(Nov 25 2019 at 20:42)</a>:</h4>
<p>Instead of not supporting this use case and telling them to implement lookups, i'd rather show them a component.</p>



<a name="181860805"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181860805" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181860805">(Nov 25 2019 at 20:45)</a>:</h4>
<p>What about groups "locked" into using Entrez IDs? We have to make a cut-off at some point but I agree the symbols are common currency. </p>
<p>Open slicing can currently allow them to send it instead with a different code, and it may be a good idea to get it on the list. It would probably be a re-vote ruling though (since our current guidance specifically states the name should be coded as display).</p>



<a name="181865094"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181865094" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181865094">(Nov 25 2019 at 21:30)</a>:</h4>
<p>Are we sure there's no way to leverage the other discussion re: <a href="http://identifiers.org" target="_blank" title="http://identifiers.org">identifiers.org</a> here? Since HGNC exposes multiple code systems:<br>
hgnc<br>
hgnc.symbol<br>
hgnc.family<br>
hgnc.genefamily<br>
could have the slicing on the component.value.Coding require either hgnc [id] or hgnc.symbol per that system (or something else that gets the job done).</p>



<a name="181869896"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181869896" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181869896">(Nov 25 2019 at 22:25)</a>:</h4>
<p>Whether we use <a href="http://identifiers.org" target="_blank" title="http://identifiers.org">identifiers.org</a> or just do our own 'HGNC Symbol' code system doesn't matter (to me) - I think we need to decide if we are comfortable enough with having the option to send the symbol.  I was betting this would be some feedback we would get (and quickly) from the implementer community.</p>



<a name="181871191"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181871191" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181871191">(Nov 25 2019 at 22:41)</a>:</h4>
<p>And FWIW - I happen to be OK allowing the symbol only.  I am fairly certain I supported the ID only approach, and still do on principled grounds.  But getting closer to implementation makes me OK reconsidering.</p>



<a name="181935662"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181935662" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181935662">(Nov 26 2019 at 15:59)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> <span class="user-mention" data-user-id="191451">@Patrick Werner</span> <span class="user-mention" data-user-id="196553">@Jamie Jones</span>  It is dangerous to use only HGNC gene symbols, the synonyms and symbols change. The HGNC identifier does not. You have a risk of reporting on the wrong gene if you rely on symbols. USE THE DISPLAY part of the Codeable concept to send the symbol.</p>



<a name="181935690"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181935690" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181935690">(Nov 26 2019 at 16:00)</a>:</h4>
<p>the HGNC code goes into the code portion</p>



<a name="181936299"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181936299" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181936299">(Nov 26 2019 at 16:05)</a>:</h4>
<p>i agree, i'm just talking about real-life. Our Reports are containing GENE Symbols only.</p>



<a name="181937522"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937522" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937522">(Nov 26 2019 at 16:16)</a>:</h4>
<p>Ya. That means there is a risk to patient care</p>



<a name="181937526"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937526" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937526">(Nov 26 2019 at 16:16)</a>:</h4>
<p>in real life</p>



<a name="181937609"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937609" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937609">(Nov 26 2019 at 16:17)</a>:</h4>
<p>I'm pretty flexible with most of the standard, but bad practices. HGNC ids versus symbols is one area I am less flexible due to the risk in patient care</p>



<a name="181937761"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937761" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937761">(Nov 26 2019 at 16:18)</a>:</h4>
<p>If you look at reports like <a href="http://info.foundationmedicine.com/hubfs/TMB/Lung-adeno_TMB-high3_Sample_Report.pdf?__hssc=231517312.1.1492277613863&amp;__hstc=231517312.eb86c5dc485da07f63c176c775e02567.1492184535979.1492184535979.1492277613863.2&amp;__hsfp=240191977&amp;hsCtaTracking=64859" target="_blank" title="http://info.foundationmedicine.com/hubfs/TMB/Lung-adeno_TMB-high3_Sample_Report.pdf?__hssc=231517312.1.1492277613863&amp;__hstc=231517312.eb86c5dc485da07f63c176c775e02567.1492184535979.1492184535979.1492277613863.2&amp;__hsfp=240191977&amp;hsCtaTracking=64859">Foundation One</a> or the lab <a href="https://www.cegat.de/en/for-physicians/medical-report/" target="_blank" title="https://www.cegat.de/en/for-physicians/medical-report/">we are working with</a></p>



<a name="181937791"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937791" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937791">(Nov 26 2019 at 16:19)</a>:</h4>
<p>Does not mean it is a safe practice to use only gene symbol.</p>



<a name="181937814"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937814" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937814">(Nov 26 2019 at 16:19)</a>:</h4>
<p>If you want we could just go backward and have one large text field</p>



<a name="181937823"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937823" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937823">(Nov 26 2019 at 16:19)</a>:</h4>
<p>: ^ )</p>



<a name="181937878"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937878" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937878">(Nov 26 2019 at 16:20)</a>:</h4>
<p>There is a cost in adopting FHIR</p>



<a name="181937913"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937913" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937913">(Nov 26 2019 at 16:20)</a>:</h4>
<p>part of that is tooling to handle the terminology pieces better.</p>



<a name="181937921"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937921" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937921">(Nov 26 2019 at 16:20)</a>:</h4>
<p>Just because the reports display only the names shouldn't stop the systems from being strict with the codes</p>



<a name="181937973"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181937973" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181937973">(Nov 26 2019 at 16:21)</a>:</h4>
<p>No it doesn't. But instead of users only filling out CodeableConcept.text with the Gene Symbol (which then wouldn't be validated) i'd prefer a component specifically for the symbol. Then we can do a validation of this component with an approved symbol ValueSet</p>



<a name="181938008"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181938008" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181938008">(Nov 26 2019 at 16:21)</a>:</h4>
<p>That's the point. Foundation one is being asked to use HGNC Gene ID to adopt the profiles</p>



<a name="181938024"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181938024" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181938024">(Nov 26 2019 at 16:22)</a>:</h4>
<p>if we leave it as it currently is</p>



<a name="181938084"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181938084" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181938084">(Nov 26 2019 at 16:22)</a>:</h4>
<blockquote>
<p>Just because the reports display only the names shouldn't stop the systems from being strict with the codes</p>
</blockquote>
<p>i agree. <br>
I am only reporting push back i got from implementers.</p>



<a name="181938171"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181938171" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181938171">(Nov 26 2019 at 16:23)</a>:</h4>
<p>is there enough information for you to push back to them? or are you unconvinced? <span class="user-mention" data-user-id="191451">@Patrick Werner</span></p>



<a name="181938567"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181938567" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181938567">(Nov 26 2019 at 16:28)</a>:</h4>
<p>i'm not sure tbh. We will see if i can convince them to use a lookup.</p>



<a name="181938653"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181938653" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181938653">(Nov 26 2019 at 16:29)</a>:</h4>
<p>"the Hl7 CG WG voted to use HGNC ID due to problem with overlap in gene symbols and the potential for Patient Harm" hope that is useful</p>



<a name="181941910"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/181941910" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#181941910">(Nov 26 2019 at 17:03)</a>:</h4>
<p>Not to be too snarky here, but a question we will likely get in the real world - are any of these concerns (overlap in gene symbols, symbols that have changed) for genes that labs report on today?  I honestly have no idea, and maybe even the possibility of an issue is too much to bear.</p>



<a name="182187181"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/182187181" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#182187181">(Nov 29 2019 at 15:48)</a>:</h4>
<p>Yes there are. We'll have to look back through work group calls to find the details.  Implementing a mapping between the symbols a lab has and HGNC ids is a small task compared to adopting FHIR - and it decreases a possibility in patient harm.</p>



<a name="191175671"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191175671" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191175671">(Mar 19 2020 at 21:08)</a>:</h4>
<p>What's the current status of this? Should resource instances that use HGNC gene symbol from <a href="http://genenames.org" target="_blank" title="http://genenames.org">genenames.org</a> validate without error against the IG?</p>



<a name="191178977"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191178977" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191178977">(Mar 19 2020 at 21:41)</a>:</h4>
<p>I'm getting a warning when I do this, and that's okay, but when I try to validate the same instance to a derived IG and I bind to a valueset  that contains a subset of HLA genes, I get errors. Might be an issue with my IG, but not sure. Here's the valueset I bound it to.</p>
<div class="codehilite"><pre><span></span>{
  &quot;resourceType&quot;: &quot;ValueSet&quot;,
  &quot;id&quot;: &quot;hla-gene-name-value-set&quot;,
  &quot;url&quot;: &quot;http://example.org/fhir/hla-reporting/ValueSet/hla-gene-name-value-set&quot;,
  &quot;version&quot;: &quot;0.1.0&quot;,
  &quot;name&quot;: &quot;HLAGeneName&quot;,
  &quot;title&quot;: &quot;HLA Gene Name Value Set&quot;,
  &quot;status&quot;: &quot;active&quot;,
  &quot;date&quot;: &quot;2020-03-19T15:52:47-05:00&quot;,
  &quot;description&quot;: &quot;HLA Gene Names&quot;,
  &quot;compose&quot;: {
    &quot;include&quot;: [
      {
        &quot;system&quot;: &quot;http://www.genenames.org&quot;,
        &quot;concept&quot;: [
          {
            &quot;code&quot;: &quot;HGNC:4931&quot;,
            &quot;display&quot;: &quot;HLA-A&quot;
          },
          {
            &quot;code&quot;: &quot;HGNC:4932&quot;,
            &quot;display&quot;: &quot;HLA-B&quot;
          },
          {
            &quot;code&quot;: &quot;HGNC:4933&quot;,
            &quot;display&quot;: &quot;HLA-C&quot;
          }
        ]
      }
    ]
  }
}
</pre></div>



<a name="191207558"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191207558" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191207558">(Mar 20 2020 at 07:04)</a>:</h4>
<p>what errors are you getting?</p>



<a name="191249202"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191249202" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191249202">(Mar 20 2020 at 14:27)</a>:</h4>
<p>And can you share the instance you are trying to validate?</p>
<p>You have probably seen in the Genomics Reporting IG, but here is the HGNC value set:<br>
<a href="http://build.fhir.org/ig/HL7/genomics-reporting/valueset-hgnc.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/valueset-hgnc.html">http://build.fhir.org/ig/HL7/genomics-reporting/valueset-hgnc.html</a></p>
<p>During validation, we do see errors like this from the validator (you can see them in <a href="http://build.fhir.org/ig/HL7/genomics-reporting/qa.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/qa.html">http://build.fhir.org/ig/HL7/genomics-reporting/qa.html</a>):</p>
<div class="codehilite"><pre><span></span>Path    Severity    Message
Observation/SNVexample: Observation.component[0].value.ofType(CodeableConcept).coding[0]    information Code System URI &quot;http://www.genenames.org/geneId&quot; is unknown so the code cannot be validated
</pre></div>



<a name="191252935"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191252935" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191252935">(Mar 20 2020 at 14:54)</a>:</h4>
<p>that would be expected, and is a info/warning so no error. If you really get an error something went wrong</p>



<a name="191262878"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191262878" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191262878">(Mar 20 2020 at 16:00)</a>:</h4>
<p>The error I get is when I try to validate against my constrained profile where it I have a binding of the gene-studied component value to the valueset I have above, and I'm requiring gene-studied to be present. It might be the tooling I'm using (shorthand).</p>
<p>Here's the warning when I validate against the Genomics Reporting IG:</p>
<div class="codehilite"><pre><span></span>$ java -jar ~/.local/bin/org.hl7.fhir.validator.jar cg_genotype_1.json -version 4.0.1 -ig &quot;http://hl7.org/fhir/uv/genomics-reporting/STU1&quot;
FHIR Validation tool Version 4.2.9-SNAPSHOT (Git# 849b65417a9c). Built 2020-03-17T15:22:07.704+11:00 (3 days old)
Detected Java version: 1.8.0_172 from /Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre on x86_64 (64bit). 3641MB available
Arguments: cg_genotype_1.json -version 4.0.1 -ig http://hl7.org/fhir/uv/genomics-reporting/STU1
Directories: Current = /Users/bmilius/Documents/src/fsh/tests, Package Cache = /Users/bmilius/.fhir/packages
  .. FHIR Version 4.0, definitions from hl7.fhir.r4.core#4.0.1
  .. connect to tx server @ http://tx.fhir.org
    (v4.0.1)
+  .. load IG from http://hl7.org/fhir/uv/genomics-reporting/STU1
  .. validate [cg_genotype_1.json]
Terminology server: Check for supported code systems for http://loinc.org
Success...validating cg_genotype_1.json:  error:0 warn:1 info:3
  Information @ Observation.value.ofType(CodeableConcept) (line 33, col16) : Binding for path Observation.value.ofType(CodeableConcept) has no source, so can&#39;t be checked
  Information @ Observation.value.ofType(CodeableConcept).coding[0] (line 34, col8) : Code System URI &quot;https://glstring.org&quot; is unknown so the code cannot be validated
  Information @ Observation.component[0].value.ofType(CodeableConcept).coding[0] (line 52, col12) : Code System URI &quot;http://www.genenames.org&quot; is unknown so the code cannot be validated
  Warning @ Observation.component[0].value.ofType(CodeableConcept) (line 51, col20) : None of the codes provided are in the value set http://hl7.org/fhir/uv/genomics-reporting/ValueSet/hgnc (http://hl7.org/fhir/uv/genomics-reporting/ValueSet/hgnc, and a code should come from this value set unless it has no suitable code) (codes = http://www.genenames.org#HGNC:4931)
</pre></div>


<p>and here it is when I validate against my IG that is constraining the Genomics Reporting IG</p>
<div class="codehilite"><pre><span></span>$ fv hla_genotype_1.json -version 4.0.1 -ig &quot;http://localhost:8000/ig/hla-reporting&quot;
FHIR Validation tool Version 4.2.9-SNAPSHOT (Git# 849b65417a9c). Built 2020-03-17T15:22:07.704+11:00 (3 days old)
Detected Java version: 1.8.0_172 from /Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre on x86_64 (64bit). 3641MB available
Arguments: hla_genotype_1.json -version 4.0.1 -ig http://localhost:8000/ig/hla-reporting
Directories: Current = /Users/bmilius/Documents/src/fsh/tests, Package Cache = /Users/bmilius/.fhir/packages
  .. FHIR Version 4.0, definitions from hl7.fhir.r4.core#4.0.1
  .. connect to tx server @ http://tx.fhir.org
    (v4.0.1)
+  .. load IG from http://localhost:8000/ig/hla-reporting
+  .. load IG from hl7.fhir.uv.genomics-reporting#1.0.0
  .. validate [hla_genotype_1.json]
Terminology server: Check for supported code systems for http://loinc.org
*FAILURE* validating hla_genotype_1.json:  error:1 warn:1 info:3
  Error @ Observation.component[0].value.ofType(CodeableConcept) (line 51, col20) : None of the codes provided are in the value set http://example.org/fhir/hla-reporting/ValueSet/hla-gene-name-value-set (http://example.org/fhir/hla-reporting/ValueSet/hla-gene-name-value-set, and a code from this value set is required) (codes = http://www.genenames.org#HGNC:4931)
  Information @ Observation.value.ofType(CodeableConcept) (line 33, col16) : Binding for path Observation.value.ofType(CodeableConcept) has no source, so can&#39;t be checked
  Information @ Observation.value.ofType(CodeableConcept).coding[0] (line 34, col8) : Code System URI &quot;https://glstring.org&quot; is unknown so the code cannot be validated
  Information @ Observation.component[0].value.ofType(CodeableConcept).coding[0] (line 52, col12) : Code System URI &quot;http://www.genenames.org&quot; is unknown so the code cannot be validated
  Warning @ Observation.value.ofType(CodeableConcept).coding[0].code (line 37, col37) : GLSC: Use GL String Code to report HLA genotype [matches(&#39;^hla#&#39;)]
</pre></div>


<p>and here's my instance (note that the meta profile changes depending on which IG I'm validating against)</p>
<div class="codehilite"><pre><span></span>{
  &quot;resourceType&quot;: &quot;Observation&quot;,
  &quot;id&quot;: &quot;HLA-A-GenotypeExample1&quot;,
  &quot;meta&quot;: {
    &quot;profile&quot;: [
      &quot;http://example.org/fhir/hla-reporting/StructureDefinition/hla-genotype&quot;
    ]
  },
  &quot;text&quot;: {
    &quot;status&quot;: &quot;generated&quot;,
    &quot;div&quot;: &quot;&lt;div xmlns=\&quot;http://www.w3.org/1999/xhtml\&quot;&gt;&lt;p&gt;&lt;b&gt;Generated Narrative with Details&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;id&lt;/b&gt;: HLA-A-GenotypeExample1&lt;/p&gt;&lt;p&gt;&lt;b&gt;meta&lt;/b&gt;: &lt;/p&gt;&lt;p&gt;&lt;b&gt;status&lt;/b&gt;: final&lt;/p&gt;&lt;p&gt;&lt;b&gt;category&lt;/b&gt;: Laboratory &lt;span style=\&quot;background: LightGoldenRodYellow\&quot;&gt;(Details : {http://terminology.hl7.org/CodeSystem/observation-category code &#39;laboratory&#39; = &#39;Laboratory)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;code&lt;/b&gt;: Genotype display name &lt;span style=\&quot;background: LightGoldenRodYellow\&quot;&gt;(Details : {LOINC code &#39;84413-4&#39; = &#39;Genotype display name)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;value&lt;/b&gt;: hla#3.25.0#HLA-A*01:01:01:01/HLA-A*01:02+HLA-A*24:02:01:01 &lt;span style=\&quot;background: LightGoldenRodYellow\&quot;&gt;(Details : {https://glstring.org code &#39;hla#3.25.0#HLA-A*01:01:01:01/HLA-A*01:02+HLA-A*24:02:01:01&#39; = &#39;hla#3.25.0#HLA-A*01:01:01:01/HLA-A*01:02+HLA-A*24:02:01:01)&lt;/span&gt;&lt;/p&gt;&lt;h3&gt;Components&lt;/h3&gt;&lt;table class=\&quot;grid\&quot;&gt;&lt;tr&gt;&lt;td&gt;-&lt;/td&gt;&lt;td&gt;&lt;b&gt;Code&lt;/b&gt;&lt;/td&gt;&lt;td&gt;&lt;b&gt;Value[x]&lt;/b&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;*&lt;/td&gt;&lt;td&gt;Gene studied [ID] &lt;span style=\&quot;background: LightGoldenRodYellow\&quot;&gt;(Details : {LOINC code &#39;48018-6&#39; = &#39;Gene studied [ID])&lt;/span&gt;&lt;/td&gt;&lt;td&gt;HGNC:4931 &lt;span style=\&quot;background: LightGoldenRodYellow\&quot;&gt;(Details : {http://www.genenames.org/geneId code &#39;HGNC:4931&#39; = &#39;HGNC:4931)&lt;/span&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;&quot;
  },
  &quot;status&quot;: &quot;final&quot;,
  &quot;category&quot;: [
    {
      &quot;coding&quot;: [
        {
          &quot;system&quot;: &quot;http://terminology.hl7.org/CodeSystem/observation-category&quot;,
          &quot;code&quot;: &quot;laboratory&quot;
        }
      ]
    }
  ],
  &quot;code&quot;: {
    &quot;coding&quot;: [
      {
        &quot;system&quot;: &quot;http://loinc.org&quot;,
        &quot;code&quot;: &quot;84413-4&quot;
      }
    ]
  },
  &quot;valueCodeableConcept&quot;: {
    &quot;coding&quot;: [
      {
        &quot;system&quot;: &quot;https://glstring.org&quot;,
        &quot;code&quot;: &quot;hla#3.25.0#HLA-A*01:01:01:01/HLA-A*01:02+HLA-A*24:02:01:01&quot;
      }
    ]
  },
  &quot;component&quot;: [
    {
      &quot;code&quot;: {
        &quot;coding&quot;: [
          {
            &quot;system&quot;: &quot;http://loinc.org&quot;,
            &quot;code&quot;: &quot;48018-6&quot;
          }
        ]
      },
      &quot;valueCodeableConcept&quot;: {
        &quot;coding&quot;: [
          {
            &quot;system&quot;: &quot;http://www.genenames.org&quot;,
            &quot;code&quot;: &quot;HGNC:4931&quot;
          }
        ]
      }
    }
  ]
}
</pre></div>


<p>Here's the snippet in the structure definition that I'm using to constrain genotype (created using shorthand/sushi)</p>
<div class="codehilite"><pre><span></span>      {
        &quot;id&quot;: &quot;Observation.component:gene-studied.value[x]&quot;,
        &quot;path&quot;: &quot;Observation.component.value[x]&quot;,
        &quot;short&quot;: &quot;If no HGNC code issued for this gene yet, NCBI gene IDs SHALL  be used.&quot;,
        &quot;definition&quot;: &quot;The information determined as a result of making the observation, if the information has a simple value.&quot;,
        &quot;comment&quot;: &quot;Used when observation has a set of component observations. An observation may have both a value (e.g. an  Apgar score)  and component observations (the observations from which the Apgar score was derived). If a value is present, the datatype for this element
should be determined by Observation.code. A CodeableConcept with just a text would be used instead of a string if the field was usually coded, or if the type associated with the Observation.code defines a coded value.  For additional guidance, see the [Notes section](https://hl7
.org/fhir/observation.html#notes) below.&quot;,
        &quot;requirements&quot;: &quot;An observation exists to have a value, though it might not if it is in error, or if it represents a group of observations.&quot;,
        &quot;min&quot;: 0,
        &quot;max&quot;: &quot;1&quot;,
        &quot;base&quot;: {
          &quot;path&quot;: &quot;Observation.component.value[x]&quot;,
          &quot;min&quot;: 0,
          &quot;max&quot;: &quot;1&quot;
        },
        &quot;type&quot;: [
          {
            &quot;code&quot;: &quot;CodeableConcept&quot;
          }
        ],
        &quot;constraint&quot;: [
          {
            &quot;key&quot;: &quot;ele-1&quot;,
            &quot;severity&quot;: &quot;error&quot;,
            &quot;human&quot;: &quot;All FHIR elements must have a @value or children&quot;,
            &quot;expression&quot;: &quot;hasValue() or (children().count() &gt; id.count())&quot;,
            &quot;xpath&quot;: &quot;@value|f:*|h:div&quot;,
            &quot;source&quot;: &quot;http://hl7.org/fhir/StructureDefinition/Element&quot;
          }
        ],
        &quot;mustSupport&quot;: false,
        &quot;isModifier&quot;: false,
        &quot;isSummary&quot;: true,
        &quot;binding&quot;: {
          &quot;strength&quot;: &quot;required&quot;,
          &quot;valueSet&quot;: &quot;http://example.org/fhir/hla-reporting/ValueSet/hla-gene-name-value-set&quot;
        },
</pre></div>



<a name="191353469"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191353469" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191353469">(Mar 21 2020 at 16:21)</a>:</h4>
<p>I am not seeing anything that  might cause those, but very possible I am missing it.  I am assuming you don't get any build errors/warnings on your IG?  Any chance you have your IG published somewhere?</p>



<a name="191393867"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191393867" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191393867">(Mar 22 2020 at 12:11)</a>:</h4>
<p>hi <span class="user-mention" data-user-id="191916">@ Bob Milius</span> the correct uri of a gene id instance would be: </p>
<div class="codehilite"><pre><span></span>  <span class="s2">&quot;valueCodeableConcept&quot;</span><span class="err">:</span> <span class="p">{</span>
        <span class="nt">&quot;coding&quot;</span><span class="p">:</span> <span class="p">[</span>
          <span class="p">{</span>
            <span class="nt">&quot;system&quot;</span><span class="p">:</span> <span class="s2">&quot;http://www.genenames.org/geneId&quot;</span><span class="p">,</span>
            <span class="nt">&quot;code&quot;</span><span class="p">:</span> <span class="s2">&quot;HGNC:4931&quot;</span>
          <span class="p">}</span>
</pre></div>



<a name="191416242"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191416242" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191416242">(Mar 22 2020 at 21:06)</a>:</h4>
<p>Would that have caused the error <span class="user-mention" data-user-id="191451">@Patrick Werner</span> ?</p>



<a name="191418547"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191418547" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191418547">(Mar 22 2020 at 22:03)</a>:</h4>
<p>possibly. <span class="user-mention" data-user-id="191916">@ Bob Milius</span>  your ValueSet also has to include via <a href="http://www.genenames.org/geneId" target="_blank" title="http://www.genenames.org/geneId">http://www.genenames.org/geneId</a> (for geneIds)<br>
I provided a full HGNC CodeSystem and ValueSet to test in the other thread about the HGNC ValueSet</p>



<a name="191481616"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191481616" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191481616">(Mar 23 2020 at 14:36)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="191451">@Patrick Werner</span> ! I just looked at your files. Does your codesystem for HGNC include all of the geneIds? Is it part of your IG? I'm wondering if I need to include the entire HGNC codesystem in my IG, and then include only the HLA genes in my valueset?</p>



<a name="191482182"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HGNC%20gene%20symbol/near/191482182" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HGNC.20gene.20symbol.html#191482182">(Mar 23 2020 at 14:40)</a>:</h4>
<p>Yes our CS contains all geneIds, and yes you could include the whole CS into your IG, and only define the ones you are using in a VS. <br>
You also could create a sub-set of the geneIds into a CS, same url "http://www.genenames.org/geneId"and set content to fragment, and create a VS to include the codes you are using.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
