---
layout: page
title: "FHIR Chat  · hgvs component(s) · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html">hgvs component(s)</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="178245930"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178245930" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178245930">(Oct 15 2019 at 23:50)</a>:</h4>
<p>HGVS is undoubtedly important to share in these messages in regards to variant observations. However, it is not crystal clear how the CG IG is scoping what is considered a valid or proper hgvs expression.  It is my reading of the HGVS specification that all HGVS expressions should (or actually MUST?) include the reference sequence in the form &lt;ref seq&gt;:&lt;position+change expression&gt;. Certainly, the more interesting p., c., g., n., m. portion of an HGVS expression is fairly useless without the reference sequence on which it is based, at least in terms of sharing structured and computationally useful variants. </p>
<p>It seems like the CG IG is essentially parsing out the hgvs components like ref seq/accessions and in the case of genomic data the start, end, ref and alt.  Doing so makes it easy to ignore situations whereby someone does not pass a complete and proper HGVS expression in that dedicated component, but this is an interesting and confusing aspect of the guide IMO. If the guide is trying to be flexible in allowing users to share whatever form of HGVS producers have then it should be clarified that it is purely informational and not a standard for representing the variation that it presumably represents. By not clearly stating this it can lead implementers to presume that HGVS in "any" form is potentially a legitimate means for sharing variation using the CG IG. Folks believe this now and without stating this explicitly will continue to foster this dangerous practice.</p>
<p>Additionally, if the CG IG spec does want to parse out and pass the structured bits of the HGVS expression like is done for genomic dna, and mostly for transcript dna, then it should also do so for amino acid (I assume).  Currently, there is not amino-acid dna ref seq, even though there is one for transcript and genomic ref seqs.  Also, if one was to use the amino acid component there should be a separate indicator to help the consuming system know whether or not the passed AA change is a one-letter or three-letter expression. While this seems obvious it is not necessarily something that can be easily determined by simply inspecting the data (even though it seems so).</p>
<p>Also, since the CG IG does need to pass hgvs representations, then you may want to consider providing a method for adding multiple transcript and amino acid representations (and their corresponding parsed elements, if that is the approach that is chosen in general).  Many genomic variants will have multiple transcript forms, many of which may derive different amino acid changes. Being able to only express one line of genomic-dna-&gt;trx-dna-&gt;aa changes is flexible in the sense that it allows you to express several forms of a variant, but it is also constrained in that it does not allow you to express several forms of the transcript that the genomic dna may map to (and the corresponding aa changes). It seems like an arbitrary constraint or flexibility depending on your perspective.</p>
<p>It would be helpful to understand if there's a method to how the CG IG is ultimately planning to handle the sharing of hgvs standard forms and whether it will allow multiple and complete representation if desired.  Or will it (should it) only provide one of the forms in a message at a time so as to avoid all the complexity and thus requiring implementers to pick a form that best represents what was actually measured.  That would be my vote.</p>



<a name="178247299"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178247299" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178247299">(Oct 16 2019 at 00:16)</a>:</h4>
<p>a couple more questions... <br>
- Is it permissible to send the hgvs values with the accession/ref-seq portion?<br>
- since there is only a gHGVS, cHGVS and pHGVS component, then how would someone send m., n. r. hgvs expressions?</p>



<a name="178250959"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178250959" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178250959">(Oct 16 2019 at 01:31)</a>:</h4>
<p>also, are the following components linked to genomic variant representation or can they be applied to cDNA, aa representations?<br>
- coordinate system<br>
- exact-start-end<br>
- inner/outer-start-end<br>
- ref-allele<br>
- alt-allele</p>
<p>If any or all of these can be used for any type of reference sequence basis then how does one convey that in the data being passed to the consumer?  If these are all fixed to genomic data then is it presumed that the genomic ref seq and/or the genomic-build + chromosome are required in order to understand which reference sequence these components are dependent on?</p>



<a name="178281053"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178281053" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178281053">(Oct 16 2019 at 11:54)</a>:</h4>
<p>also, related...<br>
- the component "dna-change-type" seems to align - namewise - with the "dna-change" component. The dna-change component is constrained to the cHGVS expression.  </p>
<p>So, if you don't have a cHGVS expression can you still have a "dna-change-type" component and would it work equally to describe the dna-change type regardless if it was tied to a genomic-dna-change or a coding dna-change?  </p>
<p>I'm guessing that the dna-change-type can always work for either genomic-dna-change (gHGVS) or dna-change (cHGVS) because these two should always have the same type regardless of whether it is a cDNA or genomic representation of the same change.  Please confirm that this assumption is correct.  (It may be worth noting in the IG under the dna-change_type component so that folks clearly understand that it is about the DNA change regardless if it is on a genomic, mito or transcript (NM) ref seq, since all of these actually represent DNA sequences.)</p>



<a name="178286469"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178286469" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178286469">(Oct 16 2019 at 13:09)</a>:</h4>
<p>My understanding is that we are hoping to align variant more with the GA4GH variant representation and our own IM modelling work in the near future. I agree that our lack of details around the intended usage of HGVS in its various forms is currently misleading and confusing... Is it treated mainly as a way for labs currently only sending hgvs to use our spec?</p>



<a name="178287101"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178287101" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178287101">(Oct 16 2019 at 13:16)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="196553">@James Jones</span> that is helpful to know. If it could be added to the current version of the spec so early adopters and reviewers understand the intention and volatility of this area of the spec. I assume this is a general understanding of the group, if so, stating it clearly in the spec on the various profiles would be helpful.</p>



<a name="178288104"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178288104" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178288104">(Oct 16 2019 at 13:27)</a>:</h4>
<p>I am pretty swamped today, and will have to admit I only did a quick review - but a few thoughts on HGVS:</p>
<ul>
<li>
<p>While we have HGVS CodeSystem and ValueSet, there is nothing in place today to validate what is being sent as the string.  Perhaps at some point, we can work with Grahame to do that.  And even then, most if not all systems will likely perform their own validation in whatever way they chose, or chose not to perform validation because labs today send a variety of things as HGVS.  I wish we could get everyone to follow every standard perfectly, but it simply won't happen.<br>
** And yes, we can and should do what we can to move labs in that direction.  I think having a common spec for them to use with some structure to it is a great start.  Then we can evolve it.</p>
</li>
<li>
<p>Maybe I am reading to much into the comment about "the IG does need to pass hgvs representations" - none of the HGVS components are  required, so not sure what that means.</p>
</li>
<li>I think the whole spec is a bit 'volatile' right now, and I am not sure this area is particularly more so than others.  However, if you have any recommendations for additional guidance we can provide, we will start considering trackers for the next release of the IG, so we would welcome some in this area.</li>
</ul>



<a name="178290960"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178290960" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178290960">(Oct 16 2019 at 13:55)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> +1 <br>
 I think it would be difficult to enforce without providing an HGVS parser to really ensure proper use of HGVS. Today, labs often send incomplete HGVS strings in their PDF version of reports. HGVS is a convention like ISCN and the GL strings that are used in HLA typing. I do not think the CG WG can easily replace due diligence on the part of the implementation community and recipients of  reports.  Locally, I suppose our EMR vendor could be asked to build an HGVS validation parser...but  labs would want to ensure data integrity so that data will be used appropriately by the downstream (recipient) system. </p>
<p>aside: some group might suggest a single field for HGVS. The multiple fields in our current IG for HGVS make it easier for a recipient system to know that the HGVS string is c, g, etc....without having to parse the HGVS string. It is a convention to aid recipient systems. The roots of this convention are historical and related to alignment with the HL7v2 LRI.</p>



<a name="178335509"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178335509" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178335509">(Oct 16 2019 at 23:07)</a>:</h4>
<p>I might also suggest that we consider a new component, for the SPDI representation of the variant. For variants with known breakpoints, I am finding SPDI to be incredibly useful - easy to validate, can be translated into a canonical representation, will soon be supported by ClinVar, etc.</p>



<a name="178389572"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178389572" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178389572">(Oct 17 2019 at 15:01)</a>:</h4>
<p>my recommendation is to NOT treat the hgvs as a codeableconcept at all, but instead as a string (as Kevin stated). I agree with all the statements above about the inability to control how folks will use hgvs and apply it and thus a "string" is the best we can (and should) do for now.  </p>
<p>Following <span class="user-mention" data-user-id="192123">@Bret H</span> 's aside above on a single field. I couldn't agree more. One component for HGVS that allows multiple instances (0..*) .  Then compose the component to have a qualifier that would allow the sender and reciever to understand the "kind or type" of hgvs (g, m, n, c, p, r, etc..).</p>
<p>It feels now that by making the hgvs a "coding" with a system, code, display structure that it is a kind of real "code" or "identifier" which is where we seem to stratal the line between a controlled standard representation (format) and a string that we can't dictate the structure of.</p>
<p>I'm not sure if the components can provide additional attributes to help qualify the contents of the component but if possible then it may be a way to bundle this whole concern into a single component which we can later evolve into something more formal.  I feel like this issue around the use of HGVS will continue to bog us down otherwise.</p>
<p>Here's the concept...</p>
<p>Under "Variant Profile"<br>
replace components dna-change, genomic-dna-change and amino-acid-dna-change with...<br>
component:hgvs-representation .  (cardinality 0..*)<br>
code .  CodeableConcept 1..1  <br>
value[x]  Text 1..1<br>
extension or modifierExtension . (not sure if this is really acceptable - but we could add one simple extension or modifierExtension to provide additional qualifiers about the hgvs expression - like whether it is genomic, rna, protein, non-coding, etc..  We could also add things like whether it is a single-letter or three-letter amino acid version, or even the version of hgvs that the producer used, etc...)</p>
<p>I think this would be a good way to consolidate the issue into a single component while keeping it flexible enough to allow folks to use it in any of the forms they currently do now.  It will also avoid the perception that these current HGVS components are more formal that they are really designed to be.</p>



<a name="178389947"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178389947" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178389947">(Oct 17 2019 at 15:05)</a>:</h4>
<p>I too support the notion of including SPDI representation somehow like <span class="user-mention" data-user-id="193751">@Bob Dolin</span> .  But I am concerned about component-creep in these profiles. It may be time to step back and consider the various classes of "representation" versus "identifiers" . (dbsnp, spdi, hgvs, cosmic, clinvar, etc...) and see if we can bucket them into a small number of component classes without creating specialized components for every case now or in the future. Again, not sure if the component structure is amenable to supporting "modifiers" or "qualifiers" to bundle kinds of components together under a single specification type.</p>



<a name="178390280"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178390280" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178390280">(Oct 17 2019 at 15:08)</a>:</h4>
<p>Also, when/if there is some re-org around the hgvs components, we should consider the fact that haplotypes and genotypes can be expressed with hgvs representation as well. I do not see how this can be done under the haplotype and genotype profiles currently. Again, maybe this is an area of the hgvs spec we feel is not really used by the community and thus does not need to be included in the standard.  If it is an oversite, then we should consider how an hgvs component can be used for those profiles as well.</p>



<a name="178390944"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178390944" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178390944">(Oct 17 2019 at 15:16)</a>:</h4>
<p>If an HGVS is received, how do we expect it to be used?  Would anyone ever try to compute against it or is it really for human rendering and possible look-up?</p>



<a name="178391458"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178391458" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178391458">(Oct 17 2019 at 15:22)</a>:</h4>
<p>I don't know of any clinical application that would compute against it. If so, I think it would raise serious concerns.  It is primarily for human rendering. Unfortunately, this is a very challenging area of this domain, as it is a representation that looks and feels like it can be used for computational purposes.  So people do use it that way (hopefully in research settings only).  But that said, there's a reckoning that will be coming soon to settle this matter once and for all, hopefully. I think HL7 CG group is the place to put a stake in the ground on this matter. Erring on the side of caution of course.</p>
<p>If there are others out there that consider HGVS a viable clinical computationally useful form then they should chime in.</p>
<p>I should add, that hgvs, when adhered to and validated can be used computationally to the extent any other nomenclature can. Do we have other examples of nomenclatures that are used for clinical computational practices?  That may help shed light on how, what and where it should be used.</p>



<a name="178394963"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178394963" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178394963">(Oct 17 2019 at 15:58)</a>:</h4>
<p>We did have some talks about whether this should be a 'string' or CodeableConcept.  We landed on CC with the thinking that it would be treated somewhat like UCUM, and (hopefully) move everyone towards being more consistent in the sending of the data, even with the recognition that many may still treat it like 'text' at first.  We did the same thing with the ISCN Karyotype string.  Will be curious if we get feedback on that approach from the community.</p>



<a name="178399126"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178399126" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178399126">(Oct 17 2019 at 16:45)</a>:</h4>
<p>UCUM is post coordinating, but there's strict validation rules.  You can always validate that a code is valid and determine what it means.  Is that the case here?</p>



<a name="178400510"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178400510" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178400510">(Oct 17 2019 at 17:00)</a>:</h4>
<p>There are some fairly strict rules (dig around <a href="http://varnomen.hgvs.org/" target="_blank" title="http://varnomen.hgvs.org/">http://varnomen.hgvs.org/</a> if you want), and there are some HGVS validators out there.  However, not all producers of the data (actually, probably few of them) will consistently produce values that will meet all the rules.</p>



<a name="178400612"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178400612" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178400612">(Oct 17 2019 at 17:01)</a>:</h4>
<p>The hope is that using CC nudges producers of data in the right direction.</p>



<a name="178404130"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178404130" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178404130">(Oct 17 2019 at 17:45)</a>:</h4>
<p>Invalid codes are possible in most code systems.  That just means systems are non-conformant.  We can highlight that it's a known problem consumers should watch for, but that doesn't mean it's not a 'code'.</p>



<a name="178412935"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178412935" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178412935">(Oct 17 2019 at 19:20)</a>:</h4>
<p>thank you for the quick turnaround on this. <br>
As a community implementer, here's my takeaway from this conversation. Please correct me if I'm mistaken.</p>
<p>When sharing HGVS representations of variants (observed or derived) I can simply put what we normally would use on reports in the code value. Even if it isn't always consistent. </p>
<p>And, if in the future I need to rely on the hgvs information for computational purposes then I should coordinate that with the various systems I am sharing data with to make sure we are all coding things to some commonly accepted practices for a given version of the HGVS recommendations.</p>



<a name="178703944"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178703944" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178703944">(Oct 21 2019 at 22:11)</a>:</h4>
<p><span class="user-mention" data-user-id="209510">@Larry Babb</span> and <span class="user-mention" data-user-id="193751">@Bob Dolin</span>   sounds like some good geforge tracker statements.  regards SPDI and regards single component. But both would require implementors to follow governance practices outside HL7. Which is okay but I  did want to mention it. ..at any rate, if you do make gforge items, please add links here.</p>



<a name="178704406"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178704406" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178704406">(Oct 21 2019 at 22:18)</a>:</h4>
<p><span class="user-mention" data-user-id="209510">@Larry Babb</span> <span class="user-mention" data-user-id="191913">@Kevin Power</span> <span class="user-mention" data-user-id="196553">@James Jones</span>  but I would add that you should indicate the reference etc... in other words, try to send good HGVS, not incomplete HGVS. The implementor is responsible for the quality of their data.</p>



<a name="178757878"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/178757878" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#178757878">(Oct 22 2019 at 13:53)</a>:</h4>
<p>As we are wrapping up the publishing of the first release, we are working on what is next.  If you would like these considered, please log trackers.</p>



<a name="179438390"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/179438390" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#179438390">(Oct 30 2019 at 14:39)</a>:</h4>
<p>I'm not sure that SPDI is more than another attribute/id (useful possibly, but there will be others going forward too) and it I'm not sure I want to enable the continued illusion that HGVS is a sound computable identifier by bolstering the constraints on how it is validated. I'm going to hold out hope for the effort to planned around combining the IM work with the CG IG spec to help bolster the overall approach.</p>



<a name="179441834"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/179441834" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#179441834">(Oct 30 2019 at 15:11)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="209510">@Larry Babb</span> , I've struggled with HGVS too. One of the things I really like about SPDI is that it has the notion of a canonical form and a nice set of services that let you convert variants in other formats into SPDI canonical form. (<a href="https://api.ncbi.nlm.nih.gov/variation/v0/" target="_blank" title="https://api.ncbi.nlm.nih.gov/variation/v0/">https://api.ncbi.nlm.nih.gov/variation/v0/</a>)</p>



<a name="179458787"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/hgvs%20component%28s%29/near/179458787" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/hgvs.20component(s).html#179458787">(Oct 30 2019 at 17:55)</a>:</h4>
<p>I agree <span class="user-mention" data-user-id="193751">@Bob Dolin</span> I'm thinking that the GA4GH VR 1.0 digest serves the same purpose since it incorporates the SPDI algorithm for normalizing. But we could add both I suppose, seems like there's no limit to the number of attributes/identifiers/etc... that we can add to a variant. ;)</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
