---
layout: page
title: "FHIR Chat  · HLA Haplotype from NGS data · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html">HLA Haplotype from NGS data</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="212359090"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212359090" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212359090">(Oct 05 2020 at 21:27)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="191916">@ Bob Milius</span> . Is there open source software that computes HLA haplotypes from short-read NGS data?</p>



<a name="212366887"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212366887" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Joel Schneider <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212366887">(Oct 05 2020 at 23:02)</a>:</h4>
<p>I haven't followed it closely, but some work has been done on deriving HLA genotyping from typical WGS data, e.g.:<br>
<a href="https://doi.org/10.1016/j.humimm.2015.07.183">https://doi.org/10.1016/j.humimm.2015.07.183</a></p>
<p>However, the HLA genotyping computable from typical (short read) WGS data appears to have limited accuracy.</p>
<p>My understanding is, to get really accurate HLA results via NGS methods, custom amplicons/probes are needed.</p>
<p>(I'm not aware of open source software for this, but the short-read NGS data may have intrinsic limitations regarding HLA.)</p>



<a name="212367479"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212367479" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Joel Schneider <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212367479">(Oct 05 2020 at 23:08)</a>:</h4>
<p>Another related article:<br>
<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3575-z">https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3575-z</a></p>



<a name="212369175"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212369175" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Joel Schneider <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212369175">(Oct 05 2020 at 23:29)</a>:</h4>
<p>If you're interested in (probabilistic) HLA haplotype predictions based on population-level haplotype frequencies, HaploStats may be of some use:<br>
<a href="https://haplostats.org">https://haplostats.org</a></p>



<a name="212369427"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212369427" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212369427">(Oct 05 2020 at 23:32)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="191926">@Joel Schneider</span> , I will take a look at these. I was kinda suspecting limited accuracy, but thought it might still be an interesting exercise.</p>



<a name="212480482"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212480482" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212480482">(Oct 06 2020 at 20:21)</a>:</h4>
<p>Most (if not all) clinical labs reporting HLA genotyping use commercial software, e.g., <a href="https://www.gendx.com/product_line/ngsengine/">GenDx NGSengine</a>, <a href="https://labproducts.caredx.com/software/assign/assign-trusight">CareDx Assign TruSight HLA</a>, <a href="https://www.omixon.com/products/hla-explore">Omixon HLA Explore</a>, etc. NMDP spent some time working on an open-source pipeline for HLA allele assignment, but it's still pretty incomplete. As part of our Sync4Genes project, we're developing a GenDx to FHIR tool to create HLA reports compliant with our HLA Reporting IG. We hope to work with other vendors to develop similar tools.</p>



<a name="212480879"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212480879" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212480879">(Oct 06 2020 at 20:25)</a>:</h4>
<p>btw, I assume when <span class="user-mention" data-user-id="193751">@Bob Dolin</span> mentioned HLA haplotypes, you meant HLA alleles (gene level variation). In the HLA community, HLA haplotypes refer to different gene-level alleles from different HLA genes on the same chromosome (e.g., HLA-A, HLA-B, HLA-C)</p>



<a name="212489975"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/212489975" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#212489975">(Oct 06 2020 at 21:44)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="191916">@ Bob Milius</span> , yes, that's what I meant. And thanks for the info.</p>



<a name="217944369"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/217944369" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#217944369">(Nov 25 2020 at 22:00)</a>:</h4>
<p><span class="user-mention" data-user-id="191916">@ Bob Milius</span> <span class="user-mention" data-user-id="191926">@Joel Schneider</span> Just a quick fyi, I did find some open source software, xHLA [https://github.com/humanlongevity/HLA], that allowed me to compute HLA alleles from WGS sequencing data.</p>



<a name="218310501"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/218310501" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#218310501">(Nov 30 2020 at 16:32)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="193751">@Bob Dolin</span> . I'm not familiar with that code. I'll ask our bioinformatics group to see if they have any opinions about it.</p>



<a name="218375407"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/HLA%20Haplotype%20from%20NGS%20data/near/218375407" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Joel Schneider <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/HLA.20Haplotype.20from.20NGS.20data.html#218375407">(Dec 01 2020 at 02:58)</a>:</h4>
<p>HLA- A, B, C, DPB1, DQB1, and DRB1 are indeed the main HLA genes we currently use for tissue type matching and transplantation.</p>
<p>However, it looks like the license from Human Longevity, Inc. ("HLI") only allows for non-commercial academic research use.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
