---
layout: page
title: "FHIR Chat  · How to represent Exon &amp; MSI marker data · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html">How to represent Exon &amp; MSI marker data</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="271516448"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271516448" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271516448">(Feb 11 2022 at 00:46)</a>:</h4>
<p>Hi,<br>
The lab I am working with would like to store Exon # in FHIR (Variant Resource) but I can't seem to find any components relating to Exon data?<br>
Regardless of whether this data is included/not included in current IG, is there any alternate ways to represent and store this specific Exon# data?<br>
Below is an image of the report (NF1).<br>
<a href="/user_uploads/10155/89CE0oyIFbKIuF9LhOr1u_oh/image.png">image.png</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/89CE0oyIFbKIuF9LhOr1u_oh/image.png" title="image.png"><img src="/user_uploads/10155/89CE0oyIFbKIuF9LhOr1u_oh/image.png"></a></div><p>Also, the lab would also want to store marker results to represent MSI results.<br>
Current MSI profile is only able to represent Stable, HIGH, LOW and I don't think you can store Marker Results within the same Observation Profile. Do you need to have them stored in a separate Variant Profile? If so how would you store Marker Data like below in a Variant Profile?<br>
For example, the lab would result in MSI-HIGH if certain number of tested markers results return true.</p>
<p>Thank you!</p>



<a name="271548763"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271548763" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271548763">(Feb 11 2022 at 09:20)</a>:</h4>
<p>Hi,<br>
for MSI: you can derviceFrom the MSI Observation from the individual marker results. Each Marker as its own Observation.</p>



<a name="271548859"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271548859" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271548859">(Feb 11 2022 at 09:21)</a>:</h4>
<p>Exon Number can be put into: <a href="http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-variant-definitions.html#Observation.component:cytogenetic-location">http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-variant-definitions.html#Observation.component:cytogenetic-location</a></p>



<a name="271635228"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271635228" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271635228">(Feb 11 2022 at 20:50)</a>:</h4>
<p>Don't forget to use a reference accession id with the Exon number. Otherwise it's pretty useless and requires inference for a computer to use. If the lab expects this to be used for display only there is a .text section of the observation that could be used. OR they can send a rendered report with the Diagnostic Report....depends on what is trying to be accomplished. Patrick's pointed you to the element for providing cytogenic coordinates.</p>



<a name="271636838"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271636838" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271636838">(Feb 11 2022 at 21:06)</a>:</h4>
<p>Look to the other components for handing 'deletion' or the HGVS. Your HGVS should have a reference in it to be complete.</p>



<a name="271647110"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271647110" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271647110">(Feb 11 2022 at 22:28)</a>:</h4>
<p>If you use the component Patrick mentioned, you should be aware that other implementers might have chromosome/arm/band data in that component (like 1q21.1). Another possible option to capture the Exon # is to take advantage of the open slicing of Observation.component[], and use this LOINC code:</p>
<p><a href="https://loinc.org/47999-8">https://loinc.org/47999-8</a> (DNA region name [Identifier])</p>
<blockquote>
<p>A human readable name for the region of interest. Typically Exon #, Intron # or other. NOTE: This is not standardized and is mainly for convenience and display purposes.</p>
</blockquote>
<p>You could use that code, and then use 'valueString' to deliver the the result of "Exon 49" or something.</p>



<a name="271671467"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271671467" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271671467">(Feb 12 2022 at 05:43)</a>:</h4>
<p>using an extension:exon number is also an option but extensions can be ignored</p>



<a name="271744262"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271744262" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271744262">(Feb 13 2022 at 12:40)</a>:</h4>
<p><span class="user-mention silent" data-user-id="192123">Bret H</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/271671467">said</a>:</p>
<blockquote>
<p>using an extension:exon number is also an option but extensions can be ignored</p>
</blockquote>
<p>everything without a MS flag can be ignored ;-)</p>



<a name="271744274"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271744274" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271744274">(Feb 13 2022 at 12:41)</a>:</h4>
<p><span class="user-mention silent" data-user-id="191913">Kevin Power</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/271647110">said</a>:</p>
<blockquote>
<p>If you use the component Patrick mentioned, you should be aware that other implementers might have chromosome/arm/band data in that component (like 1q21.1). Another possible option to capture the Exon # is to take advantage of the open slicing of Observation.component[], and use this LOINC code:</p>
<p><a href="https://loinc.org/47999-8">https://loinc.org/47999-8</a> (DNA region name [Identifier])</p>
<blockquote>
<p>A human readable name for the region of interest. Typically Exon #, Intron # or other. NOTE: This is not standardized and is mainly for convenience and display purposes.</p>
</blockquote>
<p>You could use that code, and then use 'valueString' to deliver the the result of "Exon 49" or something.</p>
</blockquote>
<p>this answer is better than mine. Thanks Kevin</p>



<a name="271751996"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271751996" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271751996">(Feb 13 2022 at 15:51)</a>:</h4>
<p>And extensions <em>with</em> a MS flag <em>can't</em> be ignored - at least not if you claim conformance to the profile that declares the MS...</p>



<a name="271774038"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271774038" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271774038">(Feb 14 2022 at 00:21)</a>:</h4>
<p><span class="user-mention silent" data-user-id="192123">Bret H</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/271635228">said</a>:</p>
<blockquote>
<p>Don't forget to use a reference accession id with the Exon number. Otherwise it's pretty useless and requires inference for a computer to use. If the lab expects this to be used for display only there is a .text section of the observation that could be used. OR they can send a rendered report with the Diagnostic Report....depends on what is trying to be accomplished. Patrick's pointed you to the element for providing cytogenic coordinates.</p>
</blockquote>
<p>Thanks for all the response eveyone.</p>
<p>To clarify my understanding of the answers I have received...<br>
<span class="user-mention" data-user-id="192123">@Bret H</span>  you are saying "depending on the use of the fhir resource, whether it be for system to system communication (like messaging) or for display purpose only (like SMART on FHIR) or big data use, you can utilize the value.text or observation.text element?"<br>
So it's perfectly fine to store data as text? but if you store just as text it's hard to query the values right? (from what I know, HAPI FHIR servers or Azure FHIR API or FHIR itself has query-able elements and I believe just text can't be queried)</p>
<p>There isn't a strict rule for implementers to use CodeableConcept over regular text as values?<br>
Thanks</p>



<a name="271774242"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271774242" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271774242">(Feb 14 2022 at 00:25)</a>:</h4>
<p><span class="user-mention silent" data-user-id="191451">Patrick Werner</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/271744274">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="191913">Kevin Power</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/271647110">said</a>:</p>
<blockquote>
<p>If you use the component Patrick mentioned, you should be aware that other implementers might have chromosome/arm/band data in that component (like 1q21.1). Another possible option to capture the Exon # is to take advantage of the open slicing of Observation.component[], and use this LOINC code:</p>
<p><a href="https://loinc.org/47999-8">https://loinc.org/47999-8</a> (DNA region name [Identifier])</p>
<blockquote>
<p>A human readable name for the region of interest. Typically Exon #, Intron # or other. NOTE: This is not standardized and is mainly for convenience and display purposes.</p>
</blockquote>
<p>You could use that code, and then use 'valueString' to deliver the the result of "Exon 49" or something.</p>
</blockquote>
<p>this answer is better than mine. Thanks Kevin</p>
</blockquote>
<p>This really helps me clear out and understand better in how to implement!<br>
I personally concur with <span class="user-mention" data-user-id="191451">@Patrick Werner</span>  and <span class="user-mention" data-user-id="191913">@Kevin Power</span> .<br>
LOINC 47999-8 as a component of the observation (where cytogenomics guide isn't available) is more direct for me (as a non-expert). <br>
So from my understanding, using the above code, I can just store Exon 49 (the number 49 as shown in above image)? </p>
<p>Is there a difference between Component and Extensions? </p>
<p>Thanks!</p>



<a name="271774406"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271774406" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271774406">(Feb 14 2022 at 00:28)</a>:</h4>
<p><span class="user-mention silent" data-user-id="191451">Patrick Werner</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/271548763">said</a>:</p>
<blockquote>
<p>Hi,<br>
for MSI: you can derviceFrom the MSI Observation from the individual marker results. Each Marker as its own Observation.</p>
</blockquote>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span>  thanks for the response!</p>
<p>Questions on the MSI.<br>
Would the individual Marker be observation variant profile?<br>
If so where would you place the marker? would it be the gene studied component? (sorry for the lack of knowledge :( and thanks for all the help!)</p>



<a name="271839717"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/271839717" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#271839717">(Feb 14 2022 at 15:14)</a>:</h4>
<p><span class="user-mention silent" data-user-id="364604">SangHo Kim</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/271774242">said</a>:</p>
<blockquote>
<p>So from my understanding, using the above code, I can just store Exon 49 (the number 49 as shown in above image)? <br>
Is there a difference between Component and Extensions? <br>
Thanks!</p>
</blockquote>
<p>To your first question, yes. The Observation.component[].value[x] is a polymorphic element, so there is a choice of data types. You can see them here:<br>
<a href="http://hl7.org/fhir/R4/observation-definitions.html#Observation.component.value_x_">http://hl7.org/fhir/R4/observation-definitions.html#Observation.component.value_x_</a></p>
<p>To your second question, 'component' is a generic array of name/value pairs that can be delivered as part of a single observation, and should be used for <code>any supporting result that cannot reasonably be interpreted and used outside the scope of the Observation it is a component of.</code> In the case of our genomics reporting implementation guide, we use components to deliver 'annotations' about the concepts represented in our IG (review the <a href="http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-variant.html">Variant</a> profile to see more). For additional background, I would recommend reading up on the entire Observation resource, then closely review 'component' and the section about grouping of observations:<br>
<a href="http://hl7.org/fhir/R4/observation.html">http://hl7.org/fhir/R4/observation.html</a><br>
<a href="http://hl7.org/fhir/R4/observation-definitions.html#Observation.component">http://hl7.org/fhir/R4/observation-definitions.html#Observation.component</a><br>
<a href="http://hl7.org/fhir/R4/observation.html#obsgrouping">http://hl7.org/fhir/R4/observation.html#obsgrouping</a></p>
<p>Regarding extensions, it is a core FHIR topic that you will certainly want to read more about - this is the place to go:<br>
<a href="http://hl7.org/fhir/R4/extensibility.html">http://hl7.org/fhir/R4/extensibility.html</a></p>
<p>To compare the two, component is a 'built in' way to deliver supporting results that are specific to the Observation resource. By contrast, extensions can be made to almost anything in FHIR as needed by certain use cases. In either case, it is vital to understand the underlying principles first, then closely review how implementation guides (like Genomics Reporting) use those principles to make more standardized implementations possible.</p>



<a name="272776804"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/272776804" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#272776804">(Feb 22 2022 at 09:13)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> <br>
Thanks for your response!<br>
As a non-expert it is very interesting learning FHIR and how to express Genomic data in FHIR.<br>
But it is extremely hard too :p</p>



<a name="272807509"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/272807509" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#272807509">(Feb 22 2022 at 14:11)</a>:</h4>
<p><span class="user-mention silent" data-user-id="364604">SangHo Kim</span> <a href="#narrow/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data/near/272776804">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="191913">Kevin Power</span> <br>
Thanks for your response!<br>
As a non-expert it is very interesting learning FHIR and how to express Genomic data in FHIR.<br>
But it is extremely hard too <span aria-label="stuck out tongue" class="emoji emoji-1f61b" role="img" title="stuck out tongue">:stuck_out_tongue:</span></p>
</blockquote>
<p>One of my favorite old sayings applies here - "If it were easy, everyone would do it" <span aria-label="smile" class="emoji emoji-1f642" role="img" title="smile">:smile:</span> </p>
<p>Joking aside, you are asking a lot of great questions, and we really do appreciate you bringing your use cases and your questions. They will make our IG better in the future.</p>



<a name="272824381"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/How%20to%20represent%20Exon%20%26%20MSI%20marker%20data/near/272824381" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/How.20to.20represent.20Exon.20.26.20MSI.20marker.20data.html#272824381">(Feb 22 2022 at 16:05)</a>:</h4>
<p>I am receiving a lot of great and warm help here. I should be the one thanking!<br>
Thought, it's great that the discussions and use case is of help! <span aria-label="blush" class="emoji emoji-1f60a" role="img" title="blush">:blush:</span></p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
