---
layout: page
title: "FHIR Chat  · how to express asco scale results in observation profiles · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html">how to express asco scale results in observation profiles</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="272080745"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272080745" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272080745">(Feb 16 2022 at 07:18)</a>:</h4>
<p>Hi,<br>
how would you express ASCO Scale results in an obervation profile?</p>
<p>Would you just have Observation.interpretation profile as positive if scale = 1, negative if scale = 5, and inconclusive if scale between 2~4?</p>
<p>thanks!</p>



<a name="272110669"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272110669" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272110669">(Feb 16 2022 at 12:45)</a>:</h4>
<p>do you want to have the ASCO Scale by name?</p>



<a name="272113451"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272113451" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272113451">(Feb 16 2022 at 13:13)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> <br>
I am not quiet sure but from what I have communicated with the lab personnel they determine positive/negative/greyscale result of the HER2 test depending on the ASCO scale values.<br>
So I was wondering if there were specific component that can be attached to observation interpretation to show that the observation interpretation value of Positive/Negative/Inconclusive was made based on the ASCO scale.</p>



<a name="272113717"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272113717" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272113717">(Feb 16 2022 at 13:16)</a>:</h4>
<p>I would suggest using Diagnostic Interpretation  or a specific Observation. In either case, the variant could be referenced from the Observation using the derived from. The Variant profile is about the variant not the higher level implication, like an ASCO score. does that help?</p>



<a name="272114056"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272114056" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272114056">(Feb 16 2022 at 13:19)</a>:</h4>
<p>(for those wandering into this thread, there is also a stroke assessment out there named ASCO. And, some others. But here the context is Oncology)</p>



<a name="272114430"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272114430" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272114430">(Feb 16 2022 at 13:23)</a>:</h4>
<p><span class="user-mention" data-user-id="364604">@SangHo Kim</span>   do you have a more recent reference than <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6986968/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6986968/</a> ?</p>



<a name="272115431"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272115431" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272115431">(Feb 16 2022 at 13:31)</a>:</h4>
<p><span class="user-mention" data-user-id="192302">@May Terry</span> in mCode would ASCO score be treated as simple Observation with a specific .code and .interpretation and .value? I did not see a specific example to post here, but I presume it would be.</p>



<a name="272116917"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272116917" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272116917">(Feb 16 2022 at 13:44)</a>:</h4>
<p>As a classic biomarker,  LOINC codes will be the most straight forward bet. For the gene amplification result, you could put HER2 into gene-studied of a Variant profile instance with the component (to be precise as you can, add a reference for HER2 or use the other components to indicate amplification). But you only really have the gene name that would be able to be used in variant.</p>
<p>That is, use a standard Observation for the amplification result with a link to a Variant profile instance with HER2 (i.e. a variant profile instance that identifies HER2 gene, be good to add the positional information but Gene ID at least). The test does not have more information than that to communicate. </p>
<p>Probes could be added - the WG is working on a study meta data use case to illustrate ways to communicate the probe/probe-set. But Plan Definition could be useful along with MolecularSequence Resource.</p>



<a name="272122673"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272122673" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272122673">(Feb 16 2022 at 14:28)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="192123">@Bret H</span> for the detailed response.<br>
It's a lot for me to process personally.<br>
just to clarify,</p>
<ol>
<li>you're saying MolecularSequence Resource can be used to convey information of a cytogenomics result?</li>
<li>
<p>I was going to ask how to express the Amplification part but you mentioned about component. Looking at Observation genomics profile link (<a href="http://hl7.org/fhir/observation-genetics-definitions.html#Observation.extension:CopyNumberEvent">http://hl7.org/fhir/observation-genetics-definitions.html#Observation.extension:CopyNumberEvent</a>) would this copy number change event be acceptable? It is under extension though but the loinc code and valueset has amplification/deletion/LOH.<br>
<a href="/user_uploads/10155/6MwlrKe30vvIexbAYHQYa6fE/스크린샷-2022-02-16-오후-11.21.47.png">스크린샷-2022-02-16-오후-11.21.47.png</a> </p>
<div class="message_inline_image"><a href="/user_uploads/10155/6MwlrKe30vvIexbAYHQYa6fE/스크린샷-2022-02-16-오후-11.21.47.png" title="스크린샷-2022-02-16-오후-11.21.47.png"><img src="/user_uploads/10155/6MwlrKe30vvIexbAYHQYa6fE/스크린샷-2022-02-16-오후-11.21.47.png"></a></div></li>
<li>
<p>Yes, the lab would like to store those "lab test information" data in FHIR as well. Things like Probe set, probe, karyotyped cell count, certain test methods within the banding method , etc but atm I don't see any element to express unless it is included by adding customized components?</p>
</li>
</ol>



<a name="272123166"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272123166" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> SangHo Kim <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272123166">(Feb 16 2022 at 14:32)</a>:</h4>
<p>Also, as an implementer it is also very hard to decide (I think this genomics field is so complex) when or how to build a diagnostic report when interpretations or how observations are broken down is very depended on the Karyotype results. (As in for developers to automate the building of the Diagnostic report for cytogenomics  results is hard due to such broad outcomes of Karyotypes) :( I may be wrong due to lack of expertise in the genomics field but as far as what I am going through atm. just sharing</p>



<a name="272127349"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272127349" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272127349">(Feb 16 2022 at 15:03)</a>:</h4>
<p>Based on that shared URL, yes, it would be a specific code and interpretation. <br>
<span class="user-mention silent" data-user-id="192123">Bret H</span> <a href="#narrow/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles/near/272115431">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="192302">May Terry</span> in mCode would ASCO score be treated as simple Observation with a specific .code and .interpretation and .value? I did not see a specific example to post here, but I presume it would be.</p>
</blockquote>



<a name="272134622"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272134622" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272134622">(Feb 16 2022 at 15:56)</a>:</h4>
<p>since there's been some discussion here on higher level types of molecular changes. I'd like to mention Sequence Ontology <a href="http://www.sequenceontology.org">http://www.sequenceontology.org</a> It's a first-class, genomics-focused, ontology for describing genetic events. The terms can be really useful and, being an ontology, it has subsumption. Karen Eilbeck is the leader, she was one of the team of Craig Venter that annotated one of the first human genome drafts. The terms are extremely comprehensive and have various levels of granularity.</p>



<a name="272138191"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272138191" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272138191">(Feb 16 2022 at 16:19)</a>:</h4>
<p><span class="user-mention" data-user-id="364604">@SangHo Kim</span>  Yep. The copy number for HER2 gene can be indicated there, I think it is a good choice (and it's nice to define the HER2 gene region with position and a genome build reference sequence accession -  but the use case is more for calculation than presentation of the result)</p>
<p><span class="user-mention silent" data-user-id="364604">SangHo Kim</span> <a href="#narrow/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles/near/272122673">said</a>:</p>
<blockquote>
<ol start="2">
<li>I was going to ask how to express the Amplification part but you mentioned about component. Looking at Observation genomics profile link (<a href="http://hl7.org/fhir/observation-genetics-definitions.html#Observation.extension:CopyNumberEvent">http://hl7.org/fhir/observation-genetics-definitions.html#Observation.extension:CopyNumberEvent</a>) would this copy number change event be acceptable? It is under extension though but the loinc code and valueset has amplification/deletion/LOH.<br>
<a href="/user_uploads/10155/6MwlrKe30vvIexbAYHQYa6fE/스크린샷-2022-02-16-오후-11.21.47.png">스크린샷-2022-02-16-오후-11.21.47.png</a> </li>
</ol>
</blockquote>



<a name="272138626"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272138626" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272138626">(Feb 16 2022 at 16:22)</a>:</h4>
<p><span class="user-mention silent" data-user-id="364604">SangHo Kim</span> <a href="#narrow/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles/near/272122673">said</a>:</p>
<blockquote>
<ol start="3">
<li>Yes, the lab would like to store those "lab test information" data in FHIR as well. Things like Probe set, probe, karyotyped cell count, certain test methods within the banding method , etc but atm I don't see any element to express unless it is included by adding customized components?</li>
</ol>
</blockquote>
<p>PlanDefinition can be used. I would suggest looking at the Emerge example for study meta data at the moment. On a call the WG was looking at a proposal for standardizing the handling. The discussion involved the Orders/Observations WG as well. Right now, I would consider PlanDefinition for 'lab test information' for anything that defines the test.</p>



<a name="272138884"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/how%20to%20express%20asco%20scale%20results%20in%20observation%20profiles/near/272138884" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles.html#272138884">(Feb 16 2022 at 16:24)</a>:</h4>
<p><span class="user-mention silent" data-user-id="364604">SangHo Kim</span> <a href="#narrow/stream/179197-genomics/topic/how.20to.20express.20asco.20scale.20results.20in.20observation.20profiles/near/272122673">said</a>:</p>
<blockquote>
<ol>
<li>you're saying MolecularSequence Resource can be used to convey information of a cytogenomics result?</li>
</ol>
</blockquote>
<p>I am saying that MolecularSequence could be used to describe, define, the probe used. In relation to a Plan Definition for the 'lab test information'</p>
<p>Generally not captured discretely. Which brings up a good point. Not everything is going to need to be discrete to accomplish the job of transmitting the information. It's tricky to determine how much to make discrete versus simply send as a string. Questions to ask are: how will the data element be used? what's the purpose in sending it? will it be computed upon? does it need to be accessible as a thing that a computer will do a calculation with?</p>
<p>Not everything a lab sends is valuable downstream - maybe a bit shocking but some of the information is regulatory (like a signing a EULA before being able to install software you've paid for - you're going to sign weather you like it or not as you already decided to buy it or use it) - best to focus on the data elements that are going to need to be calculated upon. Display elements are a little trickier, each system may have a different idea of how the User interface should look (e.g. Maybe they don't display the HGVS. But you would still want to send it. Or in the UI they are expecting one field that conveys 'GeneX Exon#' as a single element). </p>
<p>We've tried to use the Implications to provide a more reliable way for some of the clinically valuable, higher level concepts that are needed. This allows one to focus the Variant, Haplotype and Genotype on the discrete, genetic <em>variation</em> observation. You can use it to get  a gene name added Or you could take the geneID component and slice Observation.component, but that's just my opinion.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
