---
layout: page
title: "FHIR Chat  · Joint openEHR meeting, November · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html">Joint openEHR meeting, November</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="167762936"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/167762936" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Grahame Grieve <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#167762936">(Jun 10 2019 at 14:19)</a>:</h4>
<p>The openEHR community has asked whether we (FHIR community) are interested in a doing a joint FHIR / openEHR meeting around alignment in the genomics space somewhere in Europe as a satellite meeting to DevDays in Amsterdam</p>



<a name="167763076"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/167763076" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Grahame Grieve <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#167763076">(Jun 10 2019 at 14:21)</a>:</h4>
<p>with regard to their interest in genomics, there is ongoing work about the modelling of genetic variant archetypes with openEHR, and  <span class="user-mention" data-user-id="191650">@Ian McNicoll</span> says: </p>
<blockquote>
<p>The rationale is that this can lead to broader international input on the genomics models (and from a wider perspective than just reporting) and that there is a bunch of prior art in terms of clinical phenotypics- both from HighMED and Genomics England use and while this may not be immediately applicable but should at least act as a kick-off point for discussion and development.  The prime goal here is not to be offering an alternative set of models but examining if/how using openEHR methodology can get us to the same GA4GH endpoint more efficiently. We have to cover the same sort of ground internally, in any case.</p>
</blockquote>



<a name="167780084"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/167780084" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jim Steel <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#167780084">(Jun 10 2019 at 17:52)</a>:</h4>
<p>We (CSIRO, not me specifically) are certainly interested and active in that space. <span class="user-mention" data-user-id="191345">@Alejandro Metke</span> would be the most obvious person to participate in that, but he won't be able to be at DevDays as he'll be at AMIA. We might try to send someone else though</p>



<a name="167856883"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/167856883" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Liz Amos <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#167856883">(Jun 11 2019 at 14:45)</a>:</h4>
<p>From Clem McDonald: Need to distinguish between reporting of genetic reports to clinicians from the reporting of raw results from sequencing (different approaches). Some standards are established for sequencing results. Also be aware of active work on phenotyping from the Monarch Initiative (<a href="https://monarchinitiative.org/" target="_blank" title="https://monarchinitiative.org/">https://monarchinitiative.org/</a>) and the eMERGE study, both funded by NHGRI/NIH.</p>



<a name="167941983"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/167941983" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Ian McNicoll <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#167941983">(Jun 12 2019 at 12:39)</a>:</h4>
<p>The primary focus here is on genetic reports to clinicians. <a href="https://openehr.org/ckm/templates/1013.26.223" target="_blank" title="https://openehr.org/ckm/templates/1013.26.223">https://openehr.org/ckm/templates/1013.26.223</a> but also within the context of a rich phenotypics dataset e.g. <a href="https://ckm.apperta.org/ckm/#showProject=1051.61.28" target="_blank" title="https://ckm.apperta.org/ckm/#showProject=1051.61.28">https://ckm.apperta.org/ckm/#showProject=1051.61.28</a> - this is currently being used in a number of UK hospitals to gather front-line data and curate it before reporting. Just to be clear our goal in this new project is to sit inside the GA4GH output, in terms of exchanging data but see if/how using openEHR can help in the definition of genomics variant models for persistence in an EHR, as well as being a basis for the very extensive FHIR profiling work that will be required for the clinical phenotypics.</p>



<a name="168027950"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/168027950" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Birger <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#168027950">(Jun 13 2019 at 09:18)</a>:</h4>
<p>I think this would also be of interest for the German HiGHmed consortium! So also a +1 from my side</p>



<a name="168029799"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/168029799" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#168029799">(Jun 13 2019 at 09:46)</a>:</h4>
<p>depending on actual date and location this could be interesting to us</p>



<a name="168764022"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/168764022" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> GIanluigi Zanetti <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#168764022">(Jun 22 2019 at 20:23)</a>:</h4>
<p>Hi! we (CRS4) will be definitely interested in participating.</p>



<a name="169175208"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/169175208" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a>  Bob Milius <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#169175208">(Jun 27 2019 at 21:03)</a>:</h4>
<p><span class="user-mention" data-user-id="192198">@Bob Freimuth</span></p>



<a name="176209276"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/176209276" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#176209276">(Sep 20 2019 at 16:43)</a>:</h4>
<p>Echo Clem's comment on identifying the specific use cases. Clinical reporting is a different endpoint than enabling reanalysis or research. <span class="user-mention" data-user-id="192198">@Bob Freimuth</span> given the modeling aspects, I would recommend Bob Freimuth be requested to participate. He leads or Information Modeling subgroup.</p>



<a name="176209628"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/176209628" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Mark Kramer <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#176209628">(Sep 20 2019 at 16:47)</a>:</h4>
<p><span class="user-mention" data-user-id="191650">@Ian McNicoll</span> I would be interested in attending such a meeting. I work on the  mCODE (minimal Cancer Oncology Data Elements) project, which up to now, has been US-focused. But we realize there is a demand for an international standard and want to work towards that.</p>



<a name="176505438"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/176505438" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#176505438">(Sep 24 2019 at 20:57)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span>  did the meeting already take place? I realize Grahame was talking about DevDays. Was there documentation of the outcome from the meeting?</p>



<a name="177765708"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/177765708" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Mark Kramer <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#177765708">(Oct 09 2019 at 23:35)</a>:</h4>
<p>I thought the proposal was to meet in November, prior to DevDays in A’dam</p>



<a name="177772087"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/177772087" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#177772087">(Oct 10 2019 at 01:44)</a>:</h4>
<p>ahh, that makes sense. Thanks <span class="user-mention" data-user-id="191447">@Mark Kramer</span></p>



<a name="179478129"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/179478129" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#179478129">(Oct 30 2019 at 21:14)</a>:</h4>
<blockquote>
<p><span class="user-mention silent" data-user-id="192123">Bret H</span>  I would recommend Bob Freimuth be requested to participate. He leads or Information Modeling subgroup.</p>
</blockquote>
<p>I'd be happy to contribute to the discussion, but I will be at the AMIA Annual Symposium that week.</p>



<a name="180594635"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/180594635" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jim Steel <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#180594635">(Nov 13 2019 at 05:56)</a>:</h4>
<p>Was there any resolution on a time/place/agenda for this meeting?</p>



<a name="180597327"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Joint%20openEHR%20meeting%2C%20November/near/180597327" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Grahame Grieve <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Joint.20openEHR.20meeting.2C.20November.html#180597327">(Nov 13 2019 at 06:55)</a>:</h4>
<p>I think it fell through</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
