---
layout: page
title: "FHIR Chat  · Level of Evidence versus Clinical Significance · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html">Level of Evidence versus Clinical Significance</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="180608129"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180608129" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180608129">(Nov 13 2019 at 09:52)</a>:</h4>
<p>During yesterdays call this topic was discussed again without a clear motion for a concrete solution: <a href="https://confluence.hl7.org/pages/viewpage.action?pageId=66931175#CG-2019-11-12-Topic3:LOINCupdates" target="_blank" title="https://confluence.hl7.org/pages/viewpage.action?pageId=66931175#CG-2019-11-12-Topic3:LOINCupdates">https://confluence.hl7.org/pages/viewpage.action?pageId=66931175#CG-2019-11-12-Topic3:LOINCupdates</a></p>



<a name="180608152"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180608152" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180608152">(Nov 13 2019 at 09:52)</a>:</h4>
<p>This thread is to discuss the issue and create a proposed resolution to vote on during a weekly call.</p>



<a name="180683645"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180683645" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180683645">(Nov 13 2019 at 23:11)</a>:</h4>
<p>So, some background as a starting point:</p>
<ul>
<li>
<p>A good starting point for everyone to learn (or remember) what we did is to review the diagram here: <a href="http://build.fhir.org/ig/HL7/genomics-reporting/general.html#genomic-observations" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/general.html#genomic-observations">http://build.fhir.org/ig/HL7/genomics-reporting/general.html#genomic-observations</a> </p>
</li>
<li>
<p>Our base  "Implication" profile (abstract, never directly used) contains a component slice for 'evidence-level' (code = 93044-6, answer list = LL5356-2 is preferred).  Since this is in our base implication profile, it becomes a component on all of our implication profiles.<br>
** So therefore, all implications will have a component to deliver the evidence level</p>
</li>
<li>
<p>We have 3 general categories of implication profiles:<br>
** Inherited Disease Pathogenicity (IDP)<br>
** Medication - Supports PGx use cases<br>
** Somatic</p>
</li>
<li>
<p>IDP profile uses code 53037-8, values of Path/Likely Path/Uncertain/Likely Benign/Benign</p>
</li>
<li>
<p>Medication Implication is another abstract (never directly used) profile.  It defines a single component for 'medication-assessed' since it applies to all the PGx profiles.</p>
</li>
<li>
<p>Medication (PGx) has 4 concrete (directly used) profiles for the different types of PGx implications - each has its own code + answer list that is specific to the type of implication:<br>
** Efficacy<br>
** Metabolism<br>
** Transporter<br>
** High Risk</p>
</li>
<li>
<p>Somatic Implication is another abstract (never directly used) profile.  It defines a single component for 'associated-cancer' since it applies to all of the Somatic profiles.</p>
</li>
<li>
<p>Somatic has 3 concrete (directly used) profiles for the different types of Somatic implications - each has its own code + answer list that is specific to the type of implication:</p>
</li>
<li>
<ul>
<li>Somatic Diagnostic<br>
** Somatic Prognostic<br>
** Somatic Predictive</li>
</ul>
</li>
</ul>



<a name="180685969"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180685969" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180685969">(Nov 13 2019 at 23:47)</a>:</h4>
<p>After considering further, I do not agree with is the idea to use 53037-8 as the Observation.code in our somatic implication profiles.  I just don't see somatic related variants tagged with a 'clinical significance' of things like "pathogenic" or "benign" - so it does feel like we need a new code to describe the 'level of significance' of the somatic variants - and this is where the AMP tiering (or other values) could be delivered.  </p>
<p>I don't have it all quite worked out in my head yet, but maybe this will trigger some good discussion since this went all day being silent after Patrick got it started <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="180686558"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180686558" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180686558">(Nov 13 2019 at 23:57)</a>:</h4>
<p>Since most of this is going to revolve around our somatic profiles, you can see more detail about them all here: <a href="http://build.fhir.org/ig/HL7/genomics-reporting/somatics.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/somatics.html">http://build.fhir.org/ig/HL7/genomics-reporting/somatics.html</a></p>



<a name="180691137"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180691137" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180691137">(Nov 14 2019 at 01:18)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="191913">@Kevin Power</span> ,</p>
<p>I wonder if we can revisit if we really need eleven implication profiles. I'm kinda wondering, if in trying to create a specific profile for each type of implication, are we creating confusion knowing which profile to use (e.g. when would you use the Medication Transporter implication; how does somatic predictive differ from somatic prognostic) and are we going to need even more implication profiles as we want to capture additional types of implications (e.g. variant is a risk factor, presence of a variant suggests a mis-diagnosis). I'm not sure I see why a single generic implication profile, that has a component for the type of interaction, that allows for references to medications, etc, wouldn't work.</p>



<a name="180694176"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180694176" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180694176">(Nov 14 2019 at 02:24)</a>:</h4>
<p>I think we could make a single one work, but I do think we should consider that separate from this thread and keep it focused?  Unless we can’t come to a reasonable conclusion to this question</p>



<a name="180737053"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180737053" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180737053">(Nov 14 2019 at 14:49)</a>:</h4>
<p>i think Bob's question is relevant in terms of it implies that both level of evidence and clinical significance may be better suited as components and not codes that determine individual profiles</p>



<a name="180737642"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180737642" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180737642">(Nov 14 2019 at 14:54)</a>:</h4>
<p>Fair points.  Level of evidence already is a component, so +1 <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>
<p>It seems to me like the clinical significance does align with the code/value pair of the Observation.  It works really nicely for the IDP side.  But doesn't seems as neat and tidy on the PGx or Somatic side.</p>



<a name="180738029"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180738029" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180738029">(Nov 14 2019 at 14:58)</a>:</h4>
<p>I think we should consider high-risk-allele, as in our current example (<a href="http://build.fhir.org/ig/HL7/genomics-reporting/HLAB1502-pgx-example.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/HLAB1502-pgx-example.html">http://build.fhir.org/ig/HL7/genomics-reporting/HLAB1502-pgx-example.html</a>). Is the idea of communicating a high-risk allele that different from communicating an allele with high clinical significance?</p>



<a name="180738132"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180738132" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180738132">(Nov 14 2019 at 14:59)</a>:</h4>
<p>I think our overlapping/disjointed profiles here stem from not having a good definition of what we mean by clinical significance itself, if we can break that down into use-case-specific meanings it may be helpful.</p>



<a name="180738277"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180738277" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180738277">(Nov 14 2019 at 15:01)</a>:</h4>
<p>what about predicted adverse drug responses? I know <span class="user-mention" data-user-id="192123">@Bret H</span> was concerned with our current options/guidance to display those. That naively seems like a form of clinical significance that should be in scope to me</p>



<a name="180738630"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180738630" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180738630">(Nov 14 2019 at 15:04)</a>:</h4>
<p>Most of the PGx profile name/thoughts came from this: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5253119/" target="_blank" title="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5253119/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5253119/</a></p>



<a name="180738784"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180738784" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180738784">(Nov 14 2019 at 15:06)</a>:</h4>
<p>So, that fact that 'HLA-B*15:02' was detected means you are positive for a high risk phenotype.  So, I think my point is that there is not really a single list of 'clinical significance' answers that we can use in the PGx and Somatic spaces.</p>



<a name="180742446"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180742446" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180742446">(Nov 14 2019 at 15:43)</a>:</h4>
<p>Thanks for sharing that article, I had misplaced it haha. I'm not sure we have to respond by making individual profiles for each row in the mentioned table though, it was our first approach and after trying to build out some examples with it, a gap analysis and feedback from others seems definitely in order</p>



<a name="180742663"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180742663" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180742663">(Nov 14 2019 at 15:46)</a>:</h4>
<p>I think the fact that little of the approach extends to the cancer space is a strike against it, but it may just be we haven't talked through it sufficiently yet. Has the IM group done anything on level of evidence or clinical significance in its various forms?</p>



<a name="180743652"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180743652" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180743652">(Nov 14 2019 at 15:55)</a>:</h4>
<blockquote>
<p>So, that fact that 'HLA-B*15:02' was detected means you are positive for a high risk phenotype.</p>
</blockquote>
<p>Is our use of "Allele" here inconsistent (e g. with components in Variant)?</p>



<a name="180744066"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180744066" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180744066">(Nov 14 2019 at 15:59)</a>:</h4>
<p>The idea with the most discrete profiles is that enforcing the rules becomes easier.  If you have a generic profile that <em>could</em> be 1 of X things, you have to write things up in text to say how to use the generic thing.  However, have the discrete profiles implies you know all the use cases, which is often problematic.</p>
<p>I would argue it does extend to the somatic space.  How you express the 'signficance' of something varies depending on what was found, and sometimes it can have different categories of things to say:  In this <a href="https://jmd.amjpathol.org/article/S1525-1578(16)30223-9/fulltext#sec4" target="_blank" title="https://jmd.amjpathol.org/article/S1525-1578(16)30223-9/fulltext#sec4">article</a>, which  we at least partially used to help define our somatic approach, I like this section:</p>
<blockquote>
<p>Somatic variants include SNVs, indels, fusion genes resulting from genomic rearrangements, and CNVs. Unlike interpretation of germline sequence variations, which focuses on pathogenicity of a variant for a specific disease or disease causality, interpretation of somatic variants should be focused on their impact on clinical care. A variant can be considered a biomarker that affects clinical care if it predicts sensitivity, resistance, or toxicity to a specific therapy, alters the function of the gene, which can be targeted by approved or investigational drugs, serves as an inclusion criterion for clinical trials, influences disease prognosis, assists in establishing a diagnosis of a cancer, or warrants implementing surveillance measures for early cancer detection. Clinical impacts should, therefore, include therapeutic, prognostic, diagnostic, and preventive actions. The clinical impact of a given variant should be determined according to currently available evidence. Evidence used for variant categorization can be weighed differently based on its significance in clinical decision making.</p>
</blockquote>



<a name="180744286"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180744286" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180744286">(Nov 14 2019 at 16:01)</a>:</h4>
<p>And yea, we can consider removing Allele in case it is confusing.  We just tried to align with the paper (though not sure we do it well through out).</p>



<a name="180745085"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180745085" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180745085">(Nov 14 2019 at 16:08)</a>:</h4>
<p>Great reference again. Wasn't the AMP list the one that LOINC wanted removed from the level of evidence concept though?</p>



<a name="180745258"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180745258" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180745258">(Nov 14 2019 at 16:10)</a>:</h4>
<p>My thought on another thread was that we might not have to reduce the number of profiles but could consider making the abstract profile large and the implementable ones restrict from it</p>



<a name="180745290"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180745290" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180745290">(Nov 14 2019 at 16:10)</a>:</h4>
<p>Rather than building complexity in diverse directions</p>



<a name="180745634"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180745634" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180745634">(Nov 14 2019 at 16:13)</a>:</h4>
<p>This would also allow leaving the slicing open (think having the list of components extensible) in a way that hopefully prevents muddling use cases</p>



<a name="180745772"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180745772" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180745772">(Nov 14 2019 at 16:14)</a>:</h4>
<p>Yea, the AMP guidelines (and MVLD before it, which we also leveraged <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728662/" target="_blank" title="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728662/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728662/</a> ), really seem to blur/mix the concepts of significance and evidence - because often the amount/type of evidence helps determine the significance.</p>



<a name="180745968"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180745968" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180745968">(Nov 14 2019 at 16:16)</a>:</h4>
<p>I think that led us to the proposal to have, for example, a "diagnostic level of evidence," a "prognostic level of evidence," and a "therapeutic level of evidence"</p>



<a name="180746065"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180746065" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180746065">(Nov 14 2019 at 16:17)</a>:</h4>
<p>Since the field wants to have different criteria and interpretations for evidence going towards different use cases</p>



<a name="180746210"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180746210" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180746210">(Nov 14 2019 at 16:18)</a>:</h4>
<p>I've played around a bit with the VICC meta-knowledgebase, and what's out there currently is tough to wrangle into our profiles (but possible)</p>



<a name="180746646"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180746646" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180746646">(Nov 14 2019 at 16:23)</a>:</h4>
<p>One intermediate step that seems mostly reasonable would be to collapse the implementable somatic implications into Somatic Implication, add a 'type' to indicate diagnostic, prognostic, predictive, and maybe some additional guidance as to how to represent these types using the other components.  Could do the same on the PGx side.</p>



<a name="180746766"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180746766" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180746766">(Nov 14 2019 at 16:24)</a>:</h4>
<p>do you have a online reference for VICC KB?</p>



<a name="180747278"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180747278" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180747278">(Nov 14 2019 at 16:28)</a>:</h4>
<p><a href="https://search.cancervariants.org/" target="_blank" title="https://search.cancervariants.org/">https://search.cancervariants.org/</a></p>



<a name="180747436"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180747436" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180747436">(Nov 14 2019 at 16:30)</a>:</h4>
<p>Nice, thanks.</p>



<a name="180747695"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180747695" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180747695">(Nov 14 2019 at 16:33)</a>:</h4>
<p>Does VICC have a 'meta data dictionary' sort of thing that defines their value sets and corresponding definitions for the coded values?</p>



<a name="180751125"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180751125" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180751125">(Nov 14 2019 at 17:08)</a>:</h4>
<p>Looking at the somatic profiles, I'm not 100% sure on the difference between prognostic and predictive, so we may want to revisit definitions as well. As for the medication profiles, I wonder if we need the medication transporter profile.</p>



<a name="180751585"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180751585" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180751585">(Nov 14 2019 at 17:13)</a>:</h4>
<p>I'm not sure there's any difference from a workflow perspective between interpreting a transport implication compared to an overall efficacy implication</p>



<a name="180751798"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180751798" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180751798">(Nov 14 2019 at 17:15)</a>:</h4>
<p>Has anyone seen if phenopackets overlaps this space and made any design decisions we may be interested in?</p>



<a name="180752999"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180752999" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180752999">(Nov 14 2019 at 17:28)</a>:</h4>
<blockquote>
<p>Looking at the somatic profiles, I'm not 100% sure on the difference between prognostic and predictive, so we may want to revisit definitions as well. As for the medication profiles, I wonder if we need the medication transporter profile.</p>
</blockquote>
<p>Yea, we probably need to try to clarify a little more.  It seems delineating between those two is a common problem considering I found a paper titled "Prognostic or Predictive? It's Time to Get Back to Definitions!" <a href="https://ascopubs.org/doi/10.1200/JCO.2011.38.3729" target="_blank" title="https://ascopubs.org/doi/10.1200/JCO.2011.38.3729">https://ascopubs.org/doi/10.1200/JCO.2011.38.3729</a> <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="180753056"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180753056" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180753056">(Nov 14 2019 at 17:29)</a>:</h4>
<p>sounds like we ought to propose a simpler model :)</p>



<a name="180753225"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180753225" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180753225">(Nov 14 2019 at 17:30)</a>:</h4>
<p>Touche' <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="180754917"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180754917" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180754917">(Nov 14 2019 at 17:47)</a>:</h4>
<p>Good thing this implication stuff was identified as a key theme for STU2  <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="180775389"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180775389" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180775389">(Nov 14 2019 at 21:12)</a>:</h4>
<p>BTW <span class="user-mention" data-user-id="193751">@Bob Dolin</span>  - Regarding  your question about predictive/prognostic.  I have absolutely no idea how I missed this, but the following is currently in our IG on the Somatics page (and I just noticed some typos in there as well):</p>
<blockquote>
<p>NOTE: While it would seem like prognostic and predictive implications are expressing the same information, this is not always the case. A medication may improve outcomes without necessarily significantly implicationing a tumor. Similarly a medication may implication a tumor but not significantly improve outcomes. Both types of implications may be relevant and reporting organizations should share what is currently known. The work group acknowledges the profiles in this section are drafts and alternative models will be considered in the next round of balloting. </p>
</blockquote>



<a name="180775546"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180775546" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180775546">(Nov 14 2019 at 21:14)</a>:</h4>
<p>yep that note has been there all year and our "impact-to-implication" swap left a couple of these I had trouble tracking down <span aria-label="upside down" class="emoji emoji-1f643" role="img" title="upside down">:upside_down:</span></p>



<a name="180775626"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180775626" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180775626">(Nov 14 2019 at 21:15)</a>:</h4>
<p>Indeed <span aria-label="frown" class="emoji emoji-1f641" role="img" title="frown">:frown:</span> Call this a 'search and replace' failure</p>



<a name="180775627"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180775627" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180775627">(Nov 14 2019 at 21:15)</a>:</h4>
<p>(deleted)</p>



<a name="180777287"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180777287" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180777287">(Nov 14 2019 at 21:32)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  I don't understand what the note is trying to say. Tumor type and/or specific mutations can be indications for specific medications.  The note almost reads as though the implication of using a medication is a change in prognosis, which I suppose is true, but seems kinda a weird way to say it.</p>



<a name="180778138"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778138" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778138">(Nov 14 2019 at 21:40)</a>:</h4>
<p>I should have also included these definitions from the IG (with <strong>bolded</strong> part by me to show that I think that  our said is right):</p>
<blockquote>
<p>The Somatic Prognostic Implication profile indicates that the associated genetic findings have implications for the overall outcome for the cancer patient (either positive or negative). <strong>Those outcomes might be asserted on their own or asserted presuming the patient is receiving the indicated combination of medications and other therapies.</strong> This can be used to recommend or discourage particular therapeutic approaches based on current evidence.</p>
<p>The Somatic Predictive Implication profile indicates the implication the associated genetic findings on the susceptability/responsiveness of a particular medication or medication combination on the specified cancer type. </p>
</blockquote>



<a name="180778359"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778359" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778359">(Nov 14 2019 at 21:43)</a>:</h4>
<p>okay so our TBD guidance for tbd-prognostic "The medication or medication class whose implication on the cancer is being predicted. (Same as somatic-predictive-medication)," is a bit of an oversimplification</p>



<a name="180778469"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778469" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778469">(Nov 14 2019 at 21:44)</a>:</h4>
<p>is it implied in the prognostic profile that the medication/therapy are already in place (as in, context for the observation)?</p>



<a name="180778672"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778672" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778672">(Nov 14 2019 at 21:46)</a>:</h4>
<p>That guidance seems to be saying that the Finding is the subject of the observation-statement, and that the components are a context. Thus the profile shouldn't be saying a particular medication or therapy is going to increase/decrease prognosis</p>



<a name="180778713"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778713" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778713">(Nov 14 2019 at 21:47)</a>:</h4>
<p>I doubt it <span aria-label="frown" class="emoji emoji-1f641" role="img" title="frown">:frown:</span></p>
<p>At one point, I think we did propose a single 'somatic implication' profile, and it would have value/components like the items highlighted as "Somatic Interpretive" in this diagram:<br>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728662/figure/F1/" target="_blank" title="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728662/figure/F1/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728662/figure/F1/</a></p>



<a name="180778734"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778734" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778734">(Nov 14 2019 at 21:47)</a>:</h4>
<p>rather that the presence of the finding given the associated therapy indicates a certain prognosis</p>



<a name="180778747"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778747" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778747">(Nov 14 2019 at 21:47)</a>:</h4>
<p>We certainly needed this level of review during a ballot cycle <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="180778766"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180778766" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180778766">(Nov 14 2019 at 21:47)</a>:</h4>
<p>I'd be interested in <span class="user-mention" data-user-id="191451">@Patrick Werner</span> 's opinion. I've not seen a report that says "prognosis is X, but prognosis for those taking these medications is Y". Reports seem to focus more on trying to suggest what some viable treatments might be, given the tumor type and/or specific variants / biomarkers.</p>



<a name="180779906"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180779906" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180779906">(Nov 14 2019 at 21:58)</a>:</h4>
<p>The 'medication' component on Prognostic is 0..*, so you could deliver the prognosis without any medications?  I suppose if you know what the meds are now, and you can tailor the prognosis, this gives you that ability.</p>
<p>And I think if you were going to deliver <em>"what some viable treatments might be, given the tumor type and/or specific variants / biomarkers"</em>, I think you would use Predictive right?</p>
<p>I did find this <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5637861/" target="_blank" title="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5637861/">paper</a> which used the following definitions (if it helps/hurts, I don't know <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span>):</p>
<blockquote>
<p>Biomarkers can be classified into four types: diagnostic, prognostic, predictive, and therapeutic. </p>
<p>A diagnostic biomarker allows the early detection of the cancer in a noninvasive way and thus the secondary prevention of the cancer. </p>
<p>A predictive biomarker allows predicting the response of the patient to a targeted therapy and so defining subpopulations of patients that are likely going to benefit from a specific therapy. </p>
<p>A prognostic biomarker is a clinical or biological characteristic that provides information on the likely course of the disease; it gives information about the outcome of the patient. </p>
<p>A therapeutic biomarker is generally a protein that could be used as target for a therapy.</p>
</blockquote>



<a name="180780097"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180780097" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180780097">(Nov 14 2019 at 22:01)</a>:</h4>
<p>ok, thanks <span class="user-mention" data-user-id="191913">@Kevin Power</span> . Now I see the difference between our somatic prognostic and somatic predictive profiles.</p>



<a name="180780618"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180780618" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180780618">(Nov 14 2019 at 22:07)</a>:</h4>
<p>i think medication is 0..* because you could instead populate the <code>therapy</code> component, or consider the finding as a biomarker in its own context</p>



<a name="180818026"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180818026" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180818026">(Nov 15 2019 at 10:12)</a>:</h4>
<blockquote>
<p>I'd be interested in <span class="user-mention silent" data-user-id="191451">Patrick Werner</span> 's opinion. I've not seen a report that says "prognosis is X, but prognosis for those taking these medications is Y". Reports seem to focus more on trying to suggest what some viable treatments might be, given the tumor type and/or specific variants / biomarkers.</p>
</blockquote>
<p>This is also true for our lab, we only get predictive Implications: Try medication X based on the found variants YZ.</p>



<a name="180818224"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180818224" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180818224">(Nov 15 2019 at 10:15)</a>:</h4>
<p>great articles <span class="user-mention" data-user-id="191913">@Kevin Power</span> ! They really helped.</p>



<a name="180818424"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180818424" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180818424">(Nov 15 2019 at 10:18)</a>:</h4>
<p>I think we should reduce the number of Implication profiles. diagnostic, prognostic, predictive could be a code of a generic profile. Validation shouldn't be too hard as we can define invariants based on the implication type code to enforce cardinalities.</p>



<a name="180818779"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180818779" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180818779">(Nov 15 2019 at 10:25)</a>:</h4>
<p>So comming back to the actual topic:<br>
I'm in favour of Motion B: Consider creating new codes for diagnostic, therapeutic, and prognostic significance</p>



<a name="180818968"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180818968" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180818968">(Nov 15 2019 at 10:28)</a>:</h4>
<p>Otherwise we only could distinguish the different significances by the used profile (which is bad practice as a profile shouldn't add semantical meaning), using different codes would be better for queries.</p>



<a name="180819182"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180819182" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180819182">(Nov 15 2019 at 10:30)</a>:</h4>
<p>These could be achieved through having a single Implication profile with it's code bound (extensible?) to a VS with the new codes for: diagnostic, therapeutic, and prognostic significance</p>



<a name="180819400"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180819400" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180819400">(Nov 15 2019 at 10:33)</a>:</h4>
<p>Level of evidence would be a component with a preferred? binding to PharmGKB level of evidence?</p>



<a name="180840528"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180840528" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180840528">(Nov 15 2019 at 15:09)</a>:</h4>
<p>OK - seems to be enough input saying that is the right approach.  I should have some time today, so maybe I can take a stab setup a branch for our IG and mock up what a single somatic implication profile would look like</p>



<a name="180841284"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180841284" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180841284">(Nov 15 2019 at 15:16)</a>:</h4>
<p>If it seems to work, we can probably replicate it for PGx</p>



<a name="180882747"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180882747" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180882747">(Nov 15 2019 at 23:35)</a>:</h4>
<p>OK - I have pushed up a proposal.  Once it builds, you can review it here:<br>
<a href="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/index.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/index.html">http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/index.html</a></p>
<p>I won't give too many details as to what I did or what I think there is likely still to do - One because it is late on Friday and I am ready to go home.  <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span> Two I want everyone to review it without me giving too many directions and you can act more like a fresh set of eyes.</p>



<a name="180892921"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180892921" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180892921">(Nov 16 2019 at 03:18)</a>:</h4>
<p>I need to think more about suggesting use of Observation.value with the list on <a href="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/valueset-somatic-classes.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/valueset-somatic-classes.html">http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/valueset-somatic-classes.html</a>... Our codes are usually questions with values to answer them, so could phrase it as "what class of somatic implication/biomarker was found?" (may need to be careful with Variant's "Absent" value...)</p>



<a name="180912899"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180912899" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180912899">(Nov 16 2019 at 13:49)</a>:</h4>
<p>You nailed one of my biggest questions. I did it that way for two reasons:</p>
<ul>
<li>The papers I referenced and the example reports Swapna included in her analysis seemed to show that the class was a key element and would be included on every implication. </li>
<li>I considered having the values that were used as the values in the three profiles - but I wasn’t sure there will be much consistency in the possible value sets for the three classes?</li>
</ul>



<a name="180917540"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180917540" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180917540">(Nov 16 2019 at 16:02)</a>:</h4>
<p>Great work! Here are my two Euro cents <span aria-label="wink" class="emoji emoji-1f609" role="img" title="wink">:wink:</span> <br>
Having the class as value seems strange and unusual. To me the class should be Observation code, as it qualifies the kind of Observation. <br>
So Observation.code could be bound to the classes ValueSet. Value should be the result like: found/not found etc.</p>



<a name="180917572"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180917572" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180917572">(Nov 16 2019 at 16:03)</a>:</h4>
<p>Ahh duh!  Sometimes the obvious answer just skips right by <span aria-label="grinning" class="emoji emoji-1f600" role="img" title="grinning">:grinning:</span></p>



<a name="180917708"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180917708" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180917708">(Nov 16 2019 at 16:07)</a>:</h4>
<p>I just find myself always thinking our Observations need a fixed, single code since we do that in so many places.</p>



<a name="180924870"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180924870" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180924870">(Nov 16 2019 at 19:23)</a>:</h4>
<p>Yeah in general our approach has been fixed code per profile but fixed code per use case aligning to a single profile seems good for this</p>



<a name="180962737"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180962737" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180962737">(Nov 17 2019 at 16:12)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>   thanks for this. I like where you're going. There are scenarios where cancer treatment is also predicted based on germline mutations (e.g. PARP therapy in breast cancer with germline BRCA mutation), on biomarkers (e.g. immunotherapy if MSI-high), etc. Since the report may also have this information, perhaps we shouldn't restrict somatic implications to just somatic variants?</p>



<a name="180962739"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/180962739" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#180962739">(Nov 17 2019 at 16:12)</a>:</h4>
<p>and a couple of in the weeds thoughts: [1] perhaps associated-cancer should be 0..* (e.g. a variant is predictive that a treatment will work in one of these kinds of cancer); [2] somatic-predictive: at some point I think it would be good to revisit the value set. For instance, what is the difference between "responsive" and "sensitive"?</p>



<a name="181018533"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181018533" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181018533">(Nov 18 2019 at 14:14)</a>:</h4>
<blockquote>
<p>There are scenarios where cancer treatment is also predicted based on germline mutations (e.g. PARP therapy in breast cancer with germline BRCA mutation), on biomarkers (e.g. immunotherapy if MSI-high), etc. Since the report may also have this information, perhaps we shouldn't restrict somatic implications to just somatic variants?</p>
</blockquote>
<p>I don't know that we do restrict that anywhere in the profile, but I assume you are probably talking about the wording on the Somatic page?</p>
<blockquote>
<p>perhaps associated-cancer should be 0..* (e.g. a variant is predictive that a treatment will work in one of these kinds of cancer)</p>
</blockquote>
<p>That is fair (I think).  Anyone else have a problem with that?</p>
<blockquote>
<p>somatic-predictive: at some point I think it would be good to revisit the value set. For instance, what is the difference between "responsive" and "sensitive"?</p>
</blockquote>
<p>I am afraid I don't know, we would need someone with more expertise to weigh in.  The list of answers came directly from the MVLD link, but it doesn't provide any further definition.</p>



<a name="181019777"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181019777" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181019777">(Nov 18 2019 at 14:28)</a>:</h4>
<blockquote>
<blockquote>
<p>perhaps associated-cancer should be 0..* (e.g. a variant is predictive that a treatment will work in one of these kinds of cancer)</p>
</blockquote>
<p>That is fair (I think).  Anyone else have a problem with that?</p>
</blockquote>
<p><span aria-label="+1" class="emoji emoji-1f44d" role="img" title="+1">:+1:</span></p>



<a name="181019966"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181019966" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181019966">(Nov 18 2019 at 14:30)</a>:</h4>
<blockquote>
<blockquote>
<p>There are scenarios where cancer treatment is also predicted based on germline mutations (e.g. PARP therapy in breast cancer with germline BRCA mutation), on biomarkers (e.g. immunotherapy if MSI-high), etc. Since the report may also have this information, perhaps we shouldn't restrict somatic implications to just somatic variants?</p>
</blockquote>
</blockquote>
<p>Yes, there is no restriction in the profile. But the name somatic-implication. Should we just rename it to implication?</p>



<a name="181020121"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020121" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020121">(Nov 18 2019 at 14:32)</a>:</h4>
<p>If we're naming it implication we should consider alignment with other PGx cases and inherited diseases ;)</p>



<a name="181020138"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020138" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020138">(Nov 18 2019 at 14:32)</a>:</h4>
<blockquote>
<p>Yes, there is no restriction in the profile. But the name somatic-implication. Should we just rename it to implication?</p>
</blockquote>
<p>Well, can only rename to implication if we do a broader combining of our other areas (Jamie beat me to it <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span>)</p>



<a name="181020196"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020196" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020196">(Nov 18 2019 at 14:33)</a>:</h4>
<p>re: the predictive value set, any strong reasons we can't use the one for medication efficacy?</p>



<a name="181020274"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020274" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020274">(Nov 18 2019 at 14:34)</a>:</h4>
<p>In this case, I think Somatic Implication still applies?  The implication itself is still a cancer right even if the variant is germline?</p>



<a name="181020290"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020290" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020290">(Nov 18 2019 at 14:34)</a>:</h4>
<p><a href="https://r.details.loinc.org/AnswerList/LL539-8.html" target="_blank" title="https://r.details.loinc.org/AnswerList/LL539-8.html">https://r.details.loinc.org/AnswerList/LL539-8.html</a></p>



<a name="181020345"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020345" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020345">(Nov 18 2019 at 14:35)</a>:</h4>
<p>I know there are some objections to conflating 'cancer' and 'somatic' since you can have somatic mutations that are benign, etc</p>



<a name="181020363"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020363" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020363">(Nov 18 2019 at 14:35)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@James Jones</span> - I think that makes sense, but still no definitions on the terms in the medication efficacy answer list.</p>



<a name="181020480"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020480" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020480">(Nov 18 2019 at 14:37)</a>:</h4>
<p>yes, we would need to nail some better ones down, and also consider removing 'benign' and 'presumed benign'</p>



<a name="181020633"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020633" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020633">(Nov 18 2019 at 14:38)</a>:</h4>
<p>I guess 'benign' means 'no effect on drug efficacy'?</p>



<a name="181020681"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020681" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020681">(Nov 18 2019 at 14:39)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span>  had a proposal to get something simple and ordinal here that I liked, not sure why we didn't pick it up back then.</p>



<a name="181020873"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181020873" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181020873">(Nov 18 2019 at 14:41)</a>:</h4>
<p>Continued work on AnswerLists / definitions need to be part of this theme for sure.</p>



<a name="181021636"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181021636" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181021636">(Nov 18 2019 at 14:50)</a>:</h4>
<p>There's a couple places it looks like we might have been intending to restrict the profile to just somatic variants: [1] derivedFrom points to genotype/haplotype/variant but not to a biomarker; [2] Definitions of somatic classes (e.g. "finding of whether a particular somatic genotype/haplotype/variation..."</p>



<a name="181021810"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181021810" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181021810">(Nov 18 2019 at 14:52)</a>:</h4>
<p>[1] how do you define a 'biomarker'?<br>
[2] I can tweak some wording when I push up a new change here shortly.</p>



<a name="181022189"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181022189" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181022189">(Nov 18 2019 at 14:56)</a>:</h4>
<p>biomarker - I think we wound up treating these as just "other observations"</p>



<a name="181024382"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181024382" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181024382">(Nov 18 2019 at 15:19)</a>:</h4>
<p>biomarker: i agree that we need something like a biomarker profile.</p>



<a name="181024483"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181024483" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181024483">(Nov 18 2019 at 15:20)</a>:</h4>
<p>Just implementing one for a german project. similar to the mcode TumormarkerTest profile.</p>



<a name="181024536"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181024536" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181024536">(Nov 18 2019 at 15:20)</a>:</h4>
<p>And i would see this profile to be part of our IG and mcode deriving from it</p>



<a name="181024580"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181024580" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181024580">(Nov 18 2019 at 15:21)</a>:</h4>
<p>or at least mCODE ensuring alignment with it.  :-)</p>



<a name="181024595"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181024595" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181024595">(Nov 18 2019 at 15:21)</a>:</h4>
<p>:-)</p>



<a name="181024915"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181024915" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181024915">(Nov 18 2019 at 15:25)</a>:</h4>
<p>My wording was sub-optimal. Sorry. I meant that CG also needs a profile for Tumormarker Tests.</p>



<a name="181024976"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181024976" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181024976">(Nov 18 2019 at 15:26)</a>:</h4>
<p>And this should be aligned with mcode(mcode being the more specific IG)</p>



<a name="181297521"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181297521" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181297521">(Nov 21 2019 at 12:01)</a>:</h4>
<p>Start with the mCode approach.</p>



<a name="181297642"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181297642" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181297642">(Nov 21 2019 at 12:03)</a>:</h4>
<p>I am not sure if this thread is on point anymore. Start a different thread for Biomarker please.</p>



<a name="181297959"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181297959" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181297959">(Nov 21 2019 at 12:08)</a>:</h4>
<blockquote>
<p>So comming back to the actual topic:<br>
I'm in favour of Motion B: Consider creating new codes for diagnostic, therapeutic, and prognostic significance</p>
</blockquote>
<p>This seems to the point of the thread and a good place to end. As I recall, this was the determination at the work group meeting as well.  But the word evidence is important. Level of evidence is an important, expected, concept.</p>



<a name="181298128"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Level%20of%20Evidence%20versus%20Clinical%20Significance/near/181298128" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Level.20of.20Evidence.20versus.20Clinical.20Significance.html#181298128">(Nov 21 2019 at 12:11)</a>:</h4>
<p>as an approach <a href="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/somatic-implication.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/somatic-implication.html">http://build.fhir.org/ig/HL7/genomics-reporting/branches/kpower_SomaticImplication/somatic-implication.html</a> mixes all kinds of concepts. Treatment efficacy prediction is not the same as prognosis for cancer survival.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
