---
layout: page
title: "FHIR Chat  · MolecularVariant and Observation · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html">MolecularVariant and Observation</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="254600604"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254600604" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254600604">(Sep 23 2021 at 20:34)</a>:</h4>
<p><span class="user-mention" data-user-id="195180">@Alexander Mankovich</span>  <span class="user-mention" data-user-id="196553">@Jamie Jones</span>  <span class="user-mention" data-user-id="191451">@Patrick Werner</span> </p>
<p>Today during Q3 of our WGM, Alex asked a question about representing the equivalent of "variant x not found".  I didn't see the question in the chat until after the meeting so I am replying here.</p>
<p>Creating a toy example for illustrative purposes:  Var123 is a T at position 500 on sequence SEQ999 instead of a G</p>
<p>Therefore, I interpret the statement "Var123 not found" to mean "the T allele was not found in this subject's genome at the position homologous to position 500 on sequence SEQ999".</p>
<p>If it is known that the T allele was not found at that position, then something else must have been observed there instead, so let's flip the statement to the positive and assume G/A was found there instead:<br>
"G/A found in this subject's genome at the position homologous to position 500 on sequence SEQ999"<br>
Notice that I chose G/A rather than G/G to make the point that that absence of one allele does not imply the presence of another. :-)</p>
<p>Note that MolecularVariant does not yet formally support genotypes, but we can put that detail in a black box and express the idea in the positive:<br>
Observation for patient A points to an instance of MolVar that defines the G/A genotype at position 500 on sequence SEQ999</p>
<p>I would then create a second Observation <em>derivedFrom</em> the first, which expresses the conclusion that "the T allele was not found":<br>
Observation for patient A derivedFrom &lt;previous Obs&gt;, ref&lt;MolVar/T allele&gt;, value = "absent" or "negative"</p>
<p>Decoupling what was actually observed from the interpretations or conclusions that are based on it (whether stated in the positive or negative) allows for re-interpretation of those conclusions without altering the primary Observation.</p>
<p>Does that help?  I'm interested in your thoughts, especially if they differ from the approach I outlined above.</p>



<a name="254601431"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254601431" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254601431">(Sep 23 2021 at 20:40)</a>:</h4>
<p><span class="user-mention" data-user-id="192198">@Bob Freimuth</span> I'm sorry I missed the discussion. Another two options: [1] explicitly marking an observation as 'absent'; [2] use the region-studied profile to infer variants not found.</p>



<a name="254602330"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254602330" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254602330">(Sep 23 2021 at 20:45)</a>:</h4>
<p>I agree we should consider the upstream use cases for why we would want to say 'variant X not found'.</p>



<a name="254607387"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254607387" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254607387">(Sep 23 2021 at 21:20)</a>:</h4>
<p>Even in our IG (Variant profile) today, we do not have guidance on how to consistently indicate 'Var123 was not found" - most likely answer is the  <code>.value = absent, component[*] = variant of interest</code> approach. If/When we introduce MolecularVariant, I would like to see us document our guidance. What Bob F said makes sense to me, though it is a bit indirect. It is possible that those use cases have specific <code>code</code>s available, and perhaps we should consider how we enable that?</p>



<a name="254607875"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254607875" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254607875">(Sep 23 2021 at 21:23)</a>:</h4>
<p>One problem I see with creating explicit observations for variant=absent is that there are potentially a LOT of absent variants in a studied region.</p>



<a name="254608075"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254608075" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254608075">(Sep 23 2021 at 21:25)</a>:</h4>
<p>Agreed, which is what makes the comment from <span class="user-mention" data-user-id="196553">@Jamie Jones</span> very important - how many use cases do we have that require this, and then we come up with guidance to match it.</p>



<a name="254610937"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254610937" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Alexander Mankovich <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254610937">(Sep 23 2021 at 21:49)</a>:</h4>
<p>Thanks for clarifying Bob. I think this approach makes sense, and like Bob D.'s addition with the region-studied profile. What I'm trying to satisfy is the ability to express that, e.g. there was no variant found in the KRAS gene, or we found a variant encoding a BRAF V600E but not elsewhere on the BRAF gene. </p>
<p>I understand there are too many instances to report all the not-found variants in a gene, and my thinking was primarily surrounding use cases driven by clinical trial matching and therapies where resistance to targeted therapies can be inferred from specific mutations like MEK1 P124Q/S and BRAF inhibitors. This may also just be something that sits on the interpretation side of things and may even be outside the scope of molecularvariant - I'll do a bit more digging and see if I can flesh out the more routine use cases and go from there.</p>



<a name="254611346"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254611346" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254611346">(Sep 23 2021 at 21:53)</a>:</h4>
<p><span class="user-mention silent" data-user-id="195180">Alexander Mankovich</span> <a href="#narrow/stream/179197-genomics/topic/MolecularVariant.20and.20Observation/near/254610937">said</a>:</p>
<blockquote>
<p>This may also just be something that sits on the interpretation side of things and may even be outside the scope of molecularvariant - I'll do a bit more digging and see if I can flesh out the more routine use cases and go from there.</p>
</blockquote>
<p>That feels right to me.</p>



<a name="254611657"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254611657" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254611657">(Sep 23 2021 at 21:56)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/MolecularVariant.20and.20Observation/near/254602330">said</a>:</p>
<blockquote>
<p>I agree we should consider the upstream use cases for why we would want to say 'variant X not found'.</p>
</blockquote>
<p>Biomarker testing is a use-case. You are testing for presence/absence of defined variants</p>



<a name="254611750"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254611750" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254611750">(Sep 23 2021 at 21:56)</a>:</h4>
<p>+1 to <span class="user-mention" data-user-id="195180">@Alexander Mankovich</span> 'spost</p>



<a name="254612092"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254612092" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254612092">(Sep 23 2021 at 21:59)</a>:</h4>
<p>I'd like to evaluate approaching genetic biomarker testing with the same pattern we want to propose for karyotype/protein/other biomarker testing.</p>



<a name="254612564"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254612564" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254612564">(Sep 23 2021 at 22:02)</a>:</h4>
<p>Is a statement of not having a certain variant independently useful as an observation without knowing that it is being used as a biomarker test?</p>



<a name="254612676"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254612676" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254612676">(Sep 23 2021 at 22:03)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> - Sorry, I guess I missed that. What pattern is being proposed?</p>



<a name="254612999"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254612999" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254612999">(Sep 23 2021 at 22:06)</a>:</h4>
<p>We're still working on a proposal, but a request has been made for our WG to create guidance for other types of biomarkers that may be included in reports</p>



<a name="254613153"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254613153" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254613153">(Sep 23 2021 at 22:07)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/MolecularVariant.20and.20Observation/near/254612999">said</a>:</p>
<blockquote>
<p>We're still working on a proposal, but a request has been made for our WG to create guidance for other types of biomarkers that may be included in reports</p>
</blockquote>
<p>What request is that? (feel like I must have been asleep through 1/2 of the WGM <span aria-label="smile" class="emoji emoji-1f642" role="img" title="smile">:smile:</span>)</p>



<a name="254613224"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254613224" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254613224">(Sep 23 2021 at 22:08)</a>:</h4>
<p>The default approach for these tests is to put most of the complexity of the test in Observation.code, and have a quantifiable value with an interpretation, all as one Observation</p>



<a name="254613297"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254613297" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254613297">(Sep 23 2021 at 22:08)</a>:</h4>
<p>So, "Biomarker" becomes "Something that has it's own <code>Observation.code</code> ?"</p>



<a name="254613877"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254613877" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254613877">(Sep 23 2021 at 22:13)</a>:</h4>
<p>I would say Bob's "second Observation" derivedFrom the first with an interpretation could be called a biomarker</p>



<a name="254613996"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254613996" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254613996">(Sep 23 2021 at 22:14)</a>:</h4>
<p>There's an option to push the complexity of the specific test from Observation.code to a value</p>



<a name="254614633"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254614633" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254614633">(Sep 23 2021 at 22:20)</a>:</h4>
<p>I could see that becoming a third type of implication, and would want us to review other work in the biomarker/interpretation space.</p>



<a name="254614682"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254614682" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254614682">(Sep 23 2021 at 22:20)</a>:</h4>
<p>There are literally infinite things that are not found at any given location, because there are an infinite number of possible variants that could occur.  I think it is much more valuable (from a data re-use perspective), though it requires two separate statements, to first record what <em>was</em> seen and then make a second observation that interprets 1..* primary observations in the context of a specific use case.</p>



<a name="254617665"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254617665" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254617665">(Sep 23 2021 at 22:49)</a>:</h4>
<p><span class="user-mention" data-user-id="195180">@Alexander Mankovich</span> To your use case re:  BRAF V600E </p>
<p>This is an example of how things get a little tricky.  Please bear with me as I break this apart to the point where most will stop reading. <span aria-label="wink" class="emoji emoji-1f609" role="img" title="wink">:wink:</span> </p>
<p>V600E is (obviously) a protein variant, which would not be observed directly by a genomic test.  It is a prediction of the protein sequence that would result if the genomic sequence that was observed was transcribed and translated, assuming there isn't any regulatory or processing weirdness due to things we didn't see or don't yet understand.  Furthermore, the V to E change is degenerate, since there are 4 possible codons for V (GTn) and 2 for E (GAr).  This means that there are several possible changes that could have occurred: GTG =&gt; GAG, GTA =&gt; GAA, GTC =&gt; GAG, etc.  Recording only the protein change does not allow anyone to dig deeper to ask questions about whether a specific genomic change was present.  Finally, I personally do not think it would be accurate to record "V600E" as a primary observation <em>unless</em> it was detected through protein sequencing (or similar).  If it was predicted based on a genomic test, then I'd chain a few things together.</p>
<p>MolVar1: T&gt;A at pos xyz in BRAF gene (corresponding to the GTG=&gt;GAG change in codon 600)<br>
MolVar2: V&gt;E at pos 600 in BRAF protein<br>
MolVar1 has a VariantRelationship of "predicted translation" with MolVar2</p>
<p>Obs1 (primary): MolVar1<br>
Obs2 (derivedFrom Obs1): MolVar2</p>
<p>You are correct that interpretation is outside the scope of MolVar, but you may not be surprised that I have ideas about that as well. <span aria-label="wink" class="emoji emoji-1f609" role="img" title="wink">:wink:</span></p>



<a name="254618246"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254618246" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254618246">(Sep 23 2021 at 22:54)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> As you know, I think the concept of "region studied" is very important and we need to support it.  Currently, it contains information that I think could belong in different structures.  For example, I think test metadata should be in the method rather than in an Observation.  I'd like to try to tease those things apart a bit as we get into the modeling of the report and procedure/method.</p>



<a name="254618551"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/254618551" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#254618551">(Sep 23 2021 at 22:57)</a>:</h4>
<p>thanks <span class="user-mention" data-user-id="192198">@Bob Freimuth</span> . I'm all for that - I'm finding more and more that there are key pieces of the test itself that provide valuable context to results interpretation (such as coverage region, types of variants the test can detect, etc).</p>



<a name="257071172"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularVariant%20and%20Observation/near/257071172" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularVariant.20and.20Observation.html#257071172">(Oct 11 2021 at 14:22)</a>:</h4>
<p>If it is a well defined variant that is NOT observed (i.e. absent). Seems like one could describe the specific variant with Variant and use the 'Absent' value in the  variant value. <span class="user-mention" data-user-id="192198">@Bob Freimuth</span> <span class="user-mention" data-user-id="196553">@Jamie Jones</span>  what is missing with this approach? <span class="user-mention" data-user-id="193751">@Bob Dolin</span> obviously not useful to do this for every variant and it does not capture sufficiently the actual sequence of the patient, but I wonder if it would meet Alexander's use case well enough.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
