---
layout: page
title: "FHIR Chat  · MolecularSequence · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html">MolecularSequence</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="193792567"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/193792567" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#193792567">(Apr 13 2020 at 16:51)</a>:</h4>
<p>I thought a lot about MolSeq. To me MolSeq was the FHIR version of VCF and/or BAM. Now i think it is more like a/some BAM file(s) (or other sequence formats). MolecularSequence could be similar to DocumentReference. Being either a reference to an existing Sequence(file) while adding some meta information to it or providing the actual content (sequenceStrings).<br>
MolSeq.variant and MolSeq.structureVariant are duplicating our IG Obs. MolSeq quality is too bio-informatics for me to understand fully, is this reported and tested for every sequencing? Or is this some quality information which has to be tested i certain time patterns?</p>



<a name="193793103"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/193793103" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#193793103">(Apr 13 2020 at 16:57)</a>:</h4>
<p>Questions/potential Todos:</p>
<ul>
<li>String is limited to 1MByte in FHIR, is this enough for sequence data? We have to warn the implementer about the limitation or choose another data type</li>
<li>Observations and DiagReport from our IG have to be extended/checked for a slice: Obs.derivedFrom pointing to MolSeq</li>
<li>finding a solution to the duplicated variant issue in MolSeq.</li>
</ul>



<a name="193793313"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/193793313" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#193793313">(Apr 13 2020 at 16:59)</a>:</h4>
<p>String shouldn't be a limitation if one uses <br>
pointer Σ   0..*    Reference(MolecularSequence)    Pointer to next atomic sequence</p>



<a name="193793349"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/193793349" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#193793349">(Apr 13 2020 at 16:59)</a>:</h4>
<p>Our guidance on how to break long sequences up is very thin but the functionality is there</p>



<a name="193794170"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/193794170" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#193794170">(Apr 13 2020 at 17:07)</a>:</h4>
<p>"Complete genetic sequence information, of which specific genetic variations are a part, is reported by reference to the GA4GH repository." This shouldn't be only tied to GA4GH, right?</p>



<a name="193794272"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/193794272" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#193794272">(Apr 13 2020 at 17:08)</a>:</h4>
<p>used to be there as an example, any reference is fine</p>



<a name="195580812"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/195580812" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#195580812">(Apr 28 2020 at 15:28)</a>:</h4>
<p>" MolSeq quality is too bio-informatics for me to understand fully, is this reported and tested for every sequencing? " MolSeq is only used when useful (much like the other functions we have). You're thinking too deeply, I think : ^ ) and we've had lots of discussions on the purpose of MolSeq without standing on a final definition. But, in my opinion its a place to send contiguous chunks of sequence data, can be used as a reference sequence itself, useful for things like entire viral genomes. <span class="user-mention" data-user-id="192198">@Bob Freimuth</span> <span class="user-mention" data-user-id="193751">@Bob Dolin</span> <span class="user-mention" data-user-id="209510">@Larry Babb</span>   might consider MolSeq as the Allele representation in our FHIR IG of the allele in the Information Model Bob is guiding. Where we're reporting the sequence that is in a specimen/sample/patient - not so much the variant. That MolSeq can also hold variant information is indeed a point of confusion potentially. But our group needs to provide a final determination. Something to consider.</p>



<a name="195582680"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/195582680" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#195582680">(Apr 28 2020 at 15:40)</a>:</h4>
<p>For MolecularSequence, I think I have a kinda basic question - is a MolecularSequence a sequence from a single molecule?</p>



<a name="195582724"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/195582724" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#195582724">(Apr 28 2020 at 15:41)</a>:</h4>
<p>I would say yes. that is the intent</p>



<a name="195582806"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/195582806" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#195582806">(Apr 28 2020 at 15:41)</a>:</h4>
<p>Reason I ask is that I think that will clarify whether we use it for, say, VCF, BAM, PrecisionFDA quality metrics, etc.</p>



<a name="195582917"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/195582917" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#195582917">(Apr 28 2020 at 15:42)</a>:</h4>
<p>in the past, Bob M did some good work showing how MolSeq hierarchically referred to profiles could be used to represent HLA alleles</p>



<a name="195583697"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/195583697" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#195583697">(Apr 28 2020 at 15:48)</a>:</h4>
<p>We may need to profile seq or extend it to make this more clear. I think initially it was hoped it be about to cover all of those cases though I haven't seen meaningful examples for each</p>



<a name="200465395"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/200465395" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#200465395">(Jun 10 2020 at 19:32)</a>:</h4>
<p>I've reviewed the IM model report and see opportunities to support in MolecularSequence the concepts of "simple sequence", "resolvable sequence", and "relative sequence." "Formatted sequence" may need some help. I struggle to see how any of the current quality fields are useful for these sequence representations, as they are aimed at comparing VCF results. Is there meaningful quality information we can capture at the single molecule level? The structure of MolecularSequence could allow us much more freedom to convey poor quality/no-call regions than what we could do in Observation.</p>
<p>Separately, we need to take a hard look at support for VCF (or gVCF/clinical+VCF and possibly other NGS data) and determine if it should be delegated to another resource.</p>



<a name="202042912"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/202042912" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Arthur Hermann <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#202042912">(Jun 26 2020 at 00:58)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/MolecularSequence/near/200465395">said</a>:</p>
<blockquote>
<p>I've reviewed the IM model report and see opportunities to support in MolecularSequence the concepts of "simple sequence", "resolvable sequence", and "relative sequence." "Formatted sequence" may need some help. I struggle to see how any of the current quality fields are useful for these sequence representations, as they are aimed at comparing VCF results. Is there meaningful quality information we can capture at the single molecule level? The structure of MolecularSequence could allow us much more freedom to convey poor quality/no-call regions than what we could do in Observation.</p>
<p>Separately, we need to take a hard look at support for VCF (or gVCF/clinical+VCF and possibly other NGS data) and determine if it should be delegated to another resource.</p>
</blockquote>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> <span class="user-mention" data-user-id="192123">@Bret H</span> <span class="user-mention" data-user-id="191913">@Kevin Power</span> <br>
I just found out that our Oncologists do want the actual VCF file sometimes (not sure who uses/interrogates it but he was quite clear). Therefore I agree that being able to add a VCF to the message should be have a clearly established method in the IG</p>



<a name="202457626"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/202457626" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Arthur Hermann <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#202457626">(Jun 30 2020 at 16:05)</a>:</h4>
<p>(deleted)</p>



<a name="205114168"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/205114168" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#205114168">(Jul 27 2020 at 13:07)</a>:</h4>
<p>a VCF file could be treated like MEDIA and use the Reference data type.  However, if one is sending VCF , one can still use the variant profile for variants  too.</p>



<a name="205114325"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/MolecularSequence/near/205114325" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/MolecularSequence.html#205114325">(Jul 27 2020 at 13:09)</a>:</h4>
<p>not sure how the discussion on VCF relates to Molecular Sequence. I always thought of MolecularSequence as particularly useful for LONG -contiguous nucleotide information  - like you see in a RefSeek record.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
