---
layout: page
title: "FHIR Chat  · query variants in a specific region · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html">query variants in a specific region</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="167304912"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167304912" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167304912">(Jun 04 2019 at 15:23)</a>:</h4>
<p>discussion started in the call:</p>



<a name="167305604"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167305604" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167305604">(Jun 04 2019 at 15:31)</a>:</h4>
<p>There was some discussion in Jan's connectathon (i'll try to track down the chats), but essentially this question leads to a discussion of operations and handling of data in implementation of a server. E.g. my server might ingest observations for height in cm but always provide height in mm. Or to put another way, my server might ingest distinct HGVS but be able to provide aggregated variant results. This does require a bioinformatic calculation. I would argue that the calculation is inevitable in genomics and we can use operations to let implementers of FHIR know that they need to consider the bioinformatic calculations when implementing. Thoughts?</p>



<a name="167306098"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167306098" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167306098">(Jun 04 2019 at 15:36)</a>:</h4>
<p>We have a PGx use case - upon order entry of a medication that has a drug-gene interaction, we query a genomic data server for variants in the related gene. In the response, if no variants are found, we need to differentiate between [1] region was studied and no variants were found; [2] region wasn't studied. We had been using the region-studied profile, with an observation value of 'negative' for case [1], and 'unknown' for case [2].</p>



<a name="167307109"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167307109" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167307109">(Jun 04 2019 at 15:46)</a>:</h4>
<p>We were thinking you could attach an Overall Interpretation with "Inconclusive" on the DiagnosticReport</p>



<a name="167307431"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167307431" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167307431">(Jun 04 2019 at 15:48)</a>:</h4>
<p>I suspect we need a more granular approach, for when the queried region covers subregions that were / weren't studied. I was thinking of experimenting with a component of the region-studied profile</p>



<a name="167326421"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167326421" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167326421">(Jun 04 2019 at 19:05)</a>:</h4>
<p>Okay, so the idea then would be to return multiple regionstudied profiles classifying the callability of the subregions of the region queried? Would they each 'hasMember' the relevant findings?</p>



<a name="167326657"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167326657" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167326657">(Jun 04 2019 at 19:08)</a>:</h4>
<p>I was thinking that inclusion of hasMember links would be too much overhead (and not sure I'd use it). Also, trying to come up with a design that avoids the need for hundreds of regionStudied profiles. One idea might be to have a single regionStudied profile that identifies itself as 'no call regions', and then just include all the no call subregions in some type of compact list structure.</p>



<a name="167326811"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167326811" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167326811">(Jun 04 2019 at 19:10)</a>:</h4>
<p>Yeah I worry about hasMember too, but I know Bret was a fan of it in January for the simpler use case...</p>



<a name="167327483"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167327483" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167327483">(Jun 04 2019 at 19:18)</a>:</h4>
<p>This thread overlaps with another one: <a href="#narrow/stream/189875-genomics-.2F.20eMerge.20Pilot/topic/Guidance.20re.20region.20studied" title="#narrow/stream/189875-genomics-.2F.20eMerge.20Pilot/topic/Guidance.20re.20region.20studied">https://chat.fhir.org/#narrow/stream/189875-genomics-.2F.20eMerge.20Pilot/topic/Guidance.20re.20region.20studied</a>. So <span class="user-mention" data-user-id="209510">@Larry Babb</span>  and <span class="user-mention" data-user-id="196626">@Mullai Murugan</span>  may also be interested.  And if I could digress a bit more Jamie, I tend to think of the RegionStudied profile as a swiss army knife that can be used to annotate arbitrary regions - not just to say what was studied, but also to convey, for instance, analytic accuracy annotations. If we were to go with something like a single 'no call' or 'low quality' profile, where we want to include a potentially long list of subregions, I'm not sure whether or not FHIR currently supports any kind of compact list structure to do this?</p>



<a name="167329690"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167329690" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167329690">(Jun 04 2019 at 19:44)</a>:</h4>
<p>I don't see many good options for that sort of list structure... besides DocumentRefence to a CSV/etc, which I believe was mentioned on the other thread</p>



<a name="167330213"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167330213" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167330213">(Jun 04 2019 at 19:51)</a>:</h4>
<p>The LOINC definition for the code we are using for this profile (53041-0) is "A set of observations which comprise the definition of a contiguous sequence of nucleotides in the genome. This is intended for use in genetic test coverage panels in genetic test reporting."</p>
<p>If we are returning non-contiguous regions some may want us to update that description...</p>



<a name="167330383"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167330383" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167330383">(Jun 04 2019 at 19:53)</a>:</h4>
<p>I would love for this profile to be able to annotate arbitrary regions, so we ought to iron it out if we can.</p>



<a name="167330649"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167330649" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167330649">(Jun 04 2019 at 19:57)</a>:</h4>
<p>If were were able to that, I think we'd also be able to address Mullai's use case as well.  Maybe it's a question for Grahame or Lloyd, whether or not there is something like the SampledData data type (<a href="https://www.hl7.org/fhir/datatypes.html#SampledData" target="_blank" title="https://www.hl7.org/fhir/datatypes.html#SampledData">https://www.hl7.org/fhir/datatypes.html#SampledData</a>) that might work here? (I kinda shy away from CSV because you can't really standardize what the columns are).</p>



<a name="167331385"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167331385" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167331385">(Jun 04 2019 at 20:07)</a>:</h4>
<p>I think the main issue with using observation.value is that it's not a simple answer to the (implied) question, "what region was studied"</p>



<a name="167331522"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167331522" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167331522">(Jun 04 2019 at 20:09)</a>:</h4>
<p>The code is called "DNA region of interest" however, so we may still be able to use it but have to be clear about how and what we are annotating</p>



<a name="167332571"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167332571" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167332571">(Jun 04 2019 at 20:22)</a>:</h4>
<p>In particular, I don't see much info for the region coverage component "TBD-RegionCoverage". Could this component do everything you need or do we need an additional component?</p>



<a name="167332968"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167332968" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167332968">(Jun 04 2019 at 20:28)</a>:</h4>
<p>Or is RegionCoverage intended to be literal read coverage? I wasn't sure and can't find any other notes on it</p>



<a name="167335799"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167335799" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167335799">(Jun 04 2019 at 21:05)</a>:</h4>
<p>Right. That's why I agree with removing observation.value, and was thinking maybe we want to come up with some additional components, that provide different types of annotations on the region in question. I believe RegionCoverage was suggested by Mullai - they have examples of reports that show the percent coverage of different regions. But there are potentially so many non-contiguous subregions, for quality annotations, for no-call regions, etc, that the main blocker in my mind gets back to how to send all these subregions in a compact way.</p>



<a name="167381241"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167381241" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167381241">(Jun 05 2019 at 12:05)</a>:</h4>
<p>While I agree the aim of being able to "query variants in a specific region" from the data that is shared by a lab to a clinical system is probably one of the top 10 primary use cases that the clinical and research communities hope to achieve someday. I believe this is a stretch at best based on the resources, classes and data types that we are providing to date.  <br>
First, using HGVS expressions as a basis for unpacking and computationally identifying precisely what was observed "while tempting and used quite predominantly today" is dangerous and likely a non-starter in a clinical setting (that is my strong opinion).  HGVS has a dominant place in helping humans display and communicate with reasonable accuracy one of the many representations of a given variant. So I would caution folks for depending on this approach too broadly. It may work for some use cases but it should be applied with caution and based on lots of available information on the pitfalls that can lead to errors or misrepresentation of the originally observed values.<br>
In terms of the regions studied and variant representations, both of which are critical to get right in order to be able to achieve the aim of this thread's concern, FHIR will need to eventually provide much more precise datatypes to represent Sequences, Locations, Alleles, Haplotypes, Genotypes, etc... in a definitional (non observational) form. The current flexibilities and choices for the variety of ways to represent variants within either within the broadly scoped Observation profiles and MolecularSequence Resource provides an environment to do the same thing in many, many different ways.  So we will need implementation guidance merely focused on all the ways to properly and consistently apply these Variant-related Observation Profiles and MolecularSequence resource to help adopters consistently share SNPs, inDels, CNVs, Hapl, Geno, Comp Hets, Structural variants, etc...  <br>
The most fundamental issue that should be addressed well before we start solving how to use these super flexible forms of variation is to separate the Observational from the Definitional forms of the data.  I would strongly recommend considering how to align with the GA4GH genomic knowledge standards variation representation efforts that are just now piloting a specification and roadmap for the datatypes that could greatly reduce the complexity in FHIR genomics data sharing as well as provide consistency such that computational and precise sharing and use can be achieved.</p>
<p>Sorry I don't have anything to directly contribute to helping solve this region studied and observation.component extension potential plan for providing a fairly fragile architecture for determining how to query variants in specific regions using the existing profiles, extensions and resources in FHIR, but this is precisely the kind of design that begs for better datatypes to represent the definitional components needed to support sound bioinformatics use of the data.</p>



<a name="167386335"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167386335" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167386335">(Jun 05 2019 at 13:09)</a>:</h4>
<p>One tool you do have available to you is Operation.  If the answer to a question can't be determined from a simple element match but can be determined by the complex interplay of several elements in the target resource, Operation can be used to do that.  (The tricky question is how to appropriately index to support implementing the operation...)</p>



<a name="167399132"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167399132" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167399132">(Jun 05 2019 at 15:15)</a>:</h4>
<p><span class="user-mention" data-user-id="191320">@Lloyd McKenzie</span> Just out of curiosity, not saying this is the way to go, but rather thinking through options, is there a way to communicate a list of integer ranges as an observation (or component) value? Some like value="1-10,20-73,114-275"?</p>



<a name="167399927"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167399927" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167399927">(Jun 05 2019 at 15:24)</a>:</h4>
<p>As separate components, yes.  As a single value, only if you use the sampledData data type.</p>



<a name="167797952"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167797952" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167797952">(Jun 10 2019 at 21:25)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@James Jones</span> If we were to go forward with a proposal for a set of GACS Operations, it could simplify the representation of 'region wasn't studied' - rather than trying to represent this in the FHIR Genomics report, we could just make it an error response (e.g. <strong>422</strong>: Error. No data present for requested region for patient). What do you think?</p>



<a name="167798121"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/167798121" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#167798121">(Jun 10 2019 at 21:27)</a>:</h4>
<p>Yes, I was suggesting an error response at the app level at the WGM. Could also generate an Observation with status "cancelled" (aka aborted) with a dataAbsentReason for context</p>



<a name="168415685"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/168415685" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#168415685">(Jun 18 2019 at 16:04)</a>:</h4>
<p>Anything concrete we think we can say here before publication?</p>



<a name="168415879"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/168415879" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#168415879">(Jun 18 2019 at 16:06)</a>:</h4>
<p>We just passed a tracker to revamp <a href="http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html">http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html</a> with our latest guidance on querying, so any updates are very appreciated (:</p>



<a name="168423532"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/168423532" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#168423532">(Jun 18 2019 at 17:15)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@James Jones</span> I'd like to bring forward a proposal for a set of GACS Operations, after we publish the IG.</p>



<a name="168446814"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/168446814" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#168446814">(Jun 18 2019 at 19:22)</a>:</h4>
<p>We'll be updating the the standard search parameters to R4 on that page shortly, and can mention that the need for advanced operations to meet complex querying use cases.</p>



<a name="182804312"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182804312" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182804312">(Dec 06 2019 at 21:02)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span>, <span class="user-mention" data-user-id="191913">@Kevin Power</span>  I updated tracker <a href="https://jira.hl7.org/browse/FHIR-25250" target="_blank" title="https://jira.hl7.org/browse/FHIR-25250">https://jira.hl7.org/browse/FHIR-25250</a> to link to a more detailed proposal for an $extract-simple-variants operation.</p>



<a name="182804462"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182804462" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182804462">(Dec 06 2019 at 21:04)</a>:</h4>
<p>That's great! I will get it on the agenda for Monday (I do have a conflict halfway through so may miss covering it)</p>



<a name="182805984"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182805984" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182805984">(Dec 06 2019 at 21:18)</a>:</h4>
<p>Thanks Jamie</p>



<a name="182807610"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182807610" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182807610">(Dec 06 2019 at 21:38)</a>:</h4>
<p>Good stuff <span class="user-mention" data-user-id="193751">@Bob Dolin</span> - I did a quick review and added a few comments on your document.</p>



<a name="182808234"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182808234" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182808234">(Dec 06 2019 at 21:47)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="191913">@Kevin Power</span> . I think your questions get at some  fundamental design decisions worth considering - I can, for instance, define an operation that has a fully locked in response (e.g. just variants that have an exact-start-end, which is therefore a required component), or, I can have a broader operation, which meets more use cases, but which might then need some optionality in the response (e.g. optionally respond back with exact-start-end or inner/outer ranges or copy count, etc). I lean towards a more locked in response, where we'd just have to create other operations to accommodate other use cases.</p>



<a name="182810004"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182810004" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182810004">(Dec 06 2019 at 22:10)</a>:</h4>
<p>Great to see it spelled out like this, but I do have similar questions to Kevin's comments in the doc. I'll be looking at other operation definitions to get some more context for my feedback</p>



<a name="182810136"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182810136" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182810136">(Dec 06 2019 at 22:12)</a>:</h4>
<p>Makes sense, and in that case it feels like we need to be more precise in the definition.  Such as indicating to only return Variants when the exact coordinates are within the given range.  Maybe describe the desired behavior in edge cases like - what if given range is 10:20, and the variant is a deletion from 19-23?  Or an insertion of 3 nucleotides at position 9?</p>



<a name="182810295"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182810295" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182810295">(Dec 06 2019 at 22:14)</a>:</h4>
<p>I'd like to keep the ideology of being as broad as possible in what it returns, and based on the reference sequence provided. In your example I would say return the first one as the variant's start position in the reference sequence is within the range, but exclude the second one.</p>



<a name="182810980"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182810980" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182810980">(Dec 06 2019 at 22:24)</a>:</h4>
<p>I am also thinking it should have to accommodate the fuzzy ends. We provided 2 ways to give position and this should cover them.</p>
<p>I think this could be easily done by returning variants that don't have exact start-end if their "inner-start" or "inner-end" are in the queried range...</p>



<a name="182812054"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182812054" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182812054">(Dec 06 2019 at 22:41)</a>:</h4>
<p>The fuzzy ends option could be parameterized allowing the client to decide if they want those or not I suppose</p>



<a name="182813146"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182813146" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182813146">(Dec 06 2019 at 22:56)</a>:</h4>
<p>I kinda feel like variants with precise end points should be handled differently from those with imprecise end points (e.g. separate operations). Reason is that there may be other information needed for the latter case.</p>
<p>As for what to return, I'd suggest we adopt conventions common to many of the Broad and GA4GH specifications, where you return variants that overlap the requested region. In other words, in the following picture, any of variants 1-5 would be returned in response to a query for variants in range x..y.</p>



<a name="182813167"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182813167" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182813167">(Dec 06 2019 at 22:56)</a>:</h4>
<p><a href="/user_uploads/10155/YVMqWqxGgy2Qf9Bv-VF9ijZQ/pasted_image.png" target="_blank" title="pasted_image.png">pasted image</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/YVMqWqxGgy2Qf9Bv-VF9ijZQ/pasted_image.png" target="_blank" title="pasted image"><img src="/user_uploads/10155/YVMqWqxGgy2Qf9Bv-VF9ijZQ/pasted_image.png"></a></div>



<a name="182813334"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182813334" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182813334">(Dec 06 2019 at 22:59)</a>:</h4>
<p>SPDI differentiates variants with / without precise breakpoints. Maybe a better name for this operation is something like $find-precise-variants.</p>



<a name="182815007"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182815007" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182815007">(Dec 06 2019 at 23:25)</a>:</h4>
<p>That picture looks perfect to me, do you have a reference link too?</p>



<a name="182815473"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182815473" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182815473">(Dec 06 2019 at 23:33)</a>:</h4>
<p>I drew the picture, to confirm my understanding of how tabix indexing works. I'll see if I can find some good references.</p>



<a name="182815499"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182815499" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182815499">(Dec 06 2019 at 23:33)</a>:</h4>
<p>I was going to do the same, so that's great! </p>
<p>Are the end points in terms of the refseq??</p>



<a name="182815619"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182815619" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182815619">(Dec 06 2019 at 23:35)</a>:</h4>
<p>Yes. start and end positions would both be on the same refSeq</p>



<a name="182816433"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/182816433" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#182816433">(Dec 06 2019 at 23:48)</a>:</h4>
<p>An insertion just before the window may be a good counter example</p>



<a name="183019936"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183019936" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183019936">(Dec 10 2019 at 01:38)</a>:</h4>
<p>I did a little analysis, to see the extent that return parameters would need to differ between a $find-precise-variants and $find-imprecise-variants. </p>
<p>First finding is that in some variant representations (e.g. VCF), imprecise variants have confidence intervals around start and end positions, and these are not currently considered in Tabix indexing. For instance, in this picture, variant 6 would not be returned in a call for variants in x..y.</p>



<a name="183019951"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183019951" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183019951">(Dec 10 2019 at 01:38)</a>:</h4>
<p><a href="/user_uploads/10155/4guH2kQl5ME7AT86y3kAze9i/pasted_image.png" target="_blank" title="pasted_image.png">pasted image</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/4guH2kQl5ME7AT86y3kAze9i/pasted_image.png" target="_blank" title="pasted image"><img src="/user_uploads/10155/4guH2kQl5ME7AT86y3kAze9i/pasted_image.png"></a></div>



<a name="183019975"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183019975" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183019975">(Dec 10 2019 at 01:39)</a>:</h4>
<p>Next, I tried to consider what other Variant components might be needed, and I came up with this list: </p>
<p>1..1 component:dna-chg-type (e.g. inversion)<br>
1..1 component:copy-number<br>
1..1 component:outer-start-end<br>
1..1 component:inner-start-end<br>
1..1 cytogenomic-nomenclature</p>



<a name="183019988"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183019988" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183019988">(Dec 10 2019 at 01:39)</a>:</h4>
<p>Finally, I tried to consider some of the more complex structural variants, such as rearrangements with multiple imprecise breakends. Here's an example from VCF 4.3 spec:</p>



<a name="183019995"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183019995" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183019995">(Dec 10 2019 at 01:39)</a>:</h4>
<p><a href="/user_uploads/10155/R7q5VPDzfQfThM1eN2NFuyUS/pasted_image.png" target="_blank" title="pasted_image.png">pasted image</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/R7q5VPDzfQfThM1eN2NFuyUS/pasted_image.png" target="_blank" title="pasted image"><img src="/user_uploads/10155/R7q5VPDzfQfThM1eN2NFuyUS/pasted_image.png"></a></div>



<a name="183020004"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183020004" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183020004">(Dec 10 2019 at 01:39)</a>:</h4>
<p><a href="/user_uploads/10155/8vSMH21sPq0k0HdK5c5mAZ-W/pasted_image.png" target="_blank" title="pasted_image.png">pasted image</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/8vSMH21sPq0k0HdK5c5mAZ-W/pasted_image.png" target="_blank" title="pasted image"><img src="/user_uploads/10155/8vSMH21sPq0k0HdK5c5mAZ-W/pasted_image.png"></a></div>



<a name="183020017"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183020017" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183020017">(Dec 10 2019 at 01:39)</a>:</h4>
<p>Overall, I kinda get the sense that imprecise variants are sufficiently different as to warrant their own service.</p>



<a name="183020641"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183020641" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183020641">(Dec 10 2019 at 01:52)</a>:</h4>
<p>Thanks Bob - good stuff. I agree with all the principles at this point. Still not sure about the name “precise” though?  Maybe it’s ok?  What do others think?</p>



<a name="183020831"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183020831" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183020831">(Dec 10 2019 at 01:56)</a>:</h4>
<p>If it helps, SPDI also refer to these as variants with known breakpoints.  VCF uses Precise and Imprecise.</p>



<a name="183022318"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183022318" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183022318">(Dec 10 2019 at 02:30)</a>:</h4>
<p>What fields would be different for $extract-imprecise-variants?</p>



<a name="183068455"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183068455" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183068455">(Dec 10 2019 at 15:12)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="196553">@Jamie Jones</span> . In this picture, the components added for imprecise are in red, and I've also tried to highlite changes  to components that are there for precise:</p>



<a name="183068456"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183068456" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183068456">(Dec 10 2019 at 15:13)</a>:</h4>
<p><a href="/user_uploads/10155/zDGD6quW2_7LBdZNy0j9S1sd/pasted_image.png" target="_blank" title="pasted_image.png">pasted image</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/zDGD6quW2_7LBdZNy0j9S1sd/pasted_image.png" target="_blank" title="pasted image"><img src="/user_uploads/10155/zDGD6quW2_7LBdZNy0j9S1sd/pasted_image.png"></a></div>



<a name="183070030"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183070030" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183070030">(Dec 10 2019 at 15:28)</a>:</h4>
<p>Thanks for your thorough explanation. My main concern with splitting the operation into two is that we already lumped all variants together into one profile, so our approach doesn't feel very consistent...</p>
<p>$extract-known-variants sounds good to me, but would be misleading if we are lacking these outliers.</p>



<a name="183071422"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/183071422" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#183071422">(Dec 10 2019 at 15:43)</a>:</h4>
<p>Considering our state, I would still suggest we just go with the name 'find-variants', keep the scope defined for now as "Returning variants with precise and well understood start and end."  If we do decide later we need a different operation for 'imprecise' variants, we can reconsider names at that point.</p>
<p>Just my recommendation ...</p>



<a name="185637399"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/185637399" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#185637399">(Jan 14 2020 at 20:35)</a>:</h4>
<p>The first step towards implementing this operation is enumerating the different ways these Observations may be stored in a server to be returned. </p>
<p>(see <a href="http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/find-variants.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/find-variants.html">http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/find-variants.html</a> for latest updates of the proposal)</p>
<p>Before considering deriving new Observations from raw sequencing data, the suggested <code>IN</code> parameters for the operation naively seem to interact with the following fields in FHIR variant Observations that may be present on the server, to see if they match the search criteria:</p>
<p>*subject<br>
*component:genomic-ref-seq<br>
*component:coordinate-system = ‘0-based interval counting’<br>
*component:exact-start-end</p>
<p>My understanding is that these fields could be normalized, so that a variant in the queried range but represented in a 1-based coordinate system or on a different versioned/scoped reference sequence would also be flagged and returned in the bundle. </p>
<p>One may additionally have to look outside of those fields on Variant, since they are all 0..1. This information could be represented via g.HGVS, c.HGVS, variation-code (e.g., clinvar), dbSNP-id. It may even possibly be opened up to variants that are only represented in terms of p.HGVS and/or transcript reference sequences (although we may have trouble determining the required genomic ref_allele and alt_allele in that case, if not populated). There is also the concern that some may use the "Build+chromosome number" approach to identifying a reference sequence.</p>
<p>Best practices seem to include identifying these fields and providing plain language guidance on how to approach normalizing them for the required computation or stating incompatibilities. Any volunteers??</p>



<a name="185639152"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/185639152" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#185639152">(Jan 14 2020 at 20:55)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> The most effective normalization approach I've found so far is to convert everything to canonical SPDI.</p>



<a name="185639483"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/185639483" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#185639483">(Jan 14 2020 at 20:58)</a>:</h4>
<p>Were you thinking that this discussion would go on the page that describes the operation? If so, I can add a section</p>



<a name="185639736"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/185639736" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#185639736">(Jan 14 2020 at 21:01)</a>:</h4>
<p>We don't mention SPDI anywhere in the IG yet so a section here would be very valuable, in my opinion.</p>



<a name="185654072"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/185654072" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#185654072">(Jan 14 2020 at 23:48)</a>:</h4>
<p>Two things:<br>
1 - Do we think this is most (only?) relevant in the 'GACS' type scenario? We will have to be careful about how many expectations we put on potential receivers of this data - many won't have the ability to perform normalization type activities. <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span><br>
2 - I can see adding a new component for SPDI so it has a first class home in our IG.</p>



<a name="186444150"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186444150" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186444150">(Jan 23 2020 at 22:39)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  Kevin, regarding the find-variants operation [http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/find-variants.html], you made a comment last week in the genomics/committers channel, that by having the RegionStudied observation reflect back the query parameters, we are actually changing the meaning of the range-examined component.</p>
<p>It's a good point, and I wanted to follow up on it. We need to consider targeted testing, WES, and WGS; and we need to consider not only 'here's what I studied', but potentially also 'within the region I studied, these are the uncallable subregions'. </p>
<p>So, I think we should change the OUT parameter from this:</p>
<div class="codehilite"><pre><span></span>1..1 Observation (RegionStudied)
    1..1 component:ranges-examined
    1..1 component:coordinate-system = &#39;0-based interval counting&#39;
    1..1 component:genomic-ref-seq
</pre></div>


<p>To this: </p>
<div class="codehilite"><pre><span></span>0..1 Observation (RegionStudied)
    1..* component:ranges-examined
    1..1 component:coordinate-system = &#39;0-based interval counting&#39;
    1..1 component:genomic-ref-seq
</pre></div>


<p>and add more descriptive text, along the lines of:</p>
<p>"RegionStudied can be used to reflect back those studied regions that overlap with the query range. For WGS, the entire region of the request may have been studied and can be represented as a single component:ranges-examined. For WES, each examined exon overlapping the query range can be represented in its own component:ranges-examined. For targeted panels, each examined region overlapping the query range can be represented in its own component:ranges-examined". </p>
<p>In the future, if we wind up adding some type of 'uncallable-region' component to the RegionStudied profile, we might consider adding an optional IN parameter, something like 'IncludeUncallableRegions' (boolean), to enable finer granularity.</p>
<p>Thoughts?</p>



<a name="186453364"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186453364" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186453364">(Jan 24 2020 at 00:56)</a>:</h4>
<p>That makes sense to me and I think aligns well with what we are using RegionStudied for.</p>



<a name="186506900"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186506900" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186506900">(Jan 24 2020 at 16:23)</a>:</h4>
<p>I think this is a great addition, and would love to see the uncallable regions included as well if we can get a vote on that soon.</p>



<a name="186534523"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186534523" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186534523">(Jan 24 2020 at 21:11)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="191913">@Kevin Power</span> , <span class="user-mention" data-user-id="196553">@Jamie Jones</span>  for feedback. I've updated the site. I think it's ready to be presented to committee, but will follow your lead. Jamie, I'm wondering if it might help to let uncallable region move along at it's own pace, and when and if it gets added, we then propose revisions to the operation?</p>



<a name="186701969"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186701969" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186701969">(Jan 27 2020 at 17:01)</a>:</h4>
<p>Seems like we all got kicked out of FCC before finalizing the plan.  I would propose we do NOT vote tomorrow, have some time to review (time boxed please), plan for <span class="user-mention" data-user-id="193751">@Bob Dolin</span> to do some more work as discussed, then plan to vote on it in the next Tuesday call after the WGM?</p>



<a name="186702146"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186702146" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186702146">(Jan 27 2020 at 17:03)</a>:</h4>
<p>Yes, we went back in and highlighted the few feedback points, need to address liftover error and scope for "supporting" this operation. (i.e., make sure we have some text saying servers MAY support this operation, separately from supporting validation of our IG's profiles).</p>



<a name="186812398"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186812398" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186812398">(Jan 28 2020 at 18:34)</a>:</h4>
<p>We may want to put some of the normalization info on <a href="http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html">http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html</a></p>



<a name="186812590"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186812590" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186812590">(Jan 28 2020 at 18:36)</a>:</h4>
<p>That page could be a lot more helpful in delineating the 4 different ways we support representing variants, and highlight manual queries and the operation there.</p>



<a name="186823579"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186823579" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186823579">(Jan 28 2020 at 20:13)</a>:</h4>
<p>I would agree, but don't want to hold up the Operations JIRA by working in that change as well.  Considering the comments from Clem and <span class="user-mention" data-user-id="194730">@Liz Amos</span> today on the call, it seems this should be something we consider more broadly?  I don't know that I can even come up with a JIRA / change request for it now, but it is a topic that comes up enough that we should discuss it in more detail.</p>



<a name="186823890"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186823890" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186823890">(Jan 28 2020 at 20:16)</a>:</h4>
<p>my concern here is how we should present/link to the content Bob is creating. currently he has drafted adding 2 pages and a "quick link". I think a section/link from <a href="http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html">http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html</a> would be helpful</p>



<a name="186823961"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186823961" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186823961">(Jan 28 2020 at 20:17)</a>:</h4>
<p>I think it aligns with the core part of the first item in the "STU2 Themes" document: "Definitional and Knowledge versus Observationl", yes?  It seems that by asking these questions, we are really asking 'how do we appropriately represent a Variant Definition?'</p>



<a name="186824113"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186824113" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186824113">(Jan 28 2020 at 20:19)</a>:</h4>
<blockquote>
<p>my concern here is how we should present/link to the content Bob is creating. currently he has drafted adding 2 pages and a "quick link". I think a section/link from <a href="http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html">http://build.fhir.org/ig/HL7/genomics-reporting/sequencing.html</a> would be helpful</p>
</blockquote>
<p>It is also linked under the Table of Contents and Artifacts?  <span class="user-mention" data-user-id="193751">@Bob Dolin</span> what do you think?</p>



<a name="186824300"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/186824300" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#186824300">(Jan 28 2020 at 20:21)</a>:</h4>
<p>Happy to address the content on the Variants page separately, don't need to bundle it into the proposal</p>



<a name="187730604"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/187730604" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#187730604">(Feb 08 2020 at 20:38)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> I've taken a stab at addressing the issues your raised regarding liftover (<a href="http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/find-variants.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/find-variants.html">http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/find-variants.html</a>).  There's no 100% solution to lifting over a variant or a region that I could find. So, it looks like the easiest approach is to have the server lift over the query region (as opposed to expecting the server to normalize all variants against a single build), and just broaden the region a bit in the rare case that lift over fails.</p>



<a name="187744527"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/187744527" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#187744527">(Feb 09 2020 at 04:44)</a>:</h4>
<p>That's great work.</p>



<a name="188214728"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/188214728" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#188214728">(Feb 14 2020 at 15:17)</a>:</h4>
<p>It is great work, thanks <span class="user-mention" data-user-id="193751">@Bob Dolin</span> .</p>
<p>After review my only comment is we ought to provide some convenience links to <a href="http://hl7.org/fhir/operations.html" target="_blank" title="http://hl7.org/fhir/operations.html">http://hl7.org/fhir/operations.html</a> and <a href="http://hl7.org/fhir/parameters.html" target="_blank" title="http://hl7.org/fhir/parameters.html">http://hl7.org/fhir/parameters.html</a> in the text near the top, and more clearly state that we are having this operation return a single "Parameters" resource and not a bundle.</p>
<p>I do worry that this page is getting a bit long, and think the comments on normalization and liftover (which I admit to have personally requested) may be better suited to be stated more generally on <a href="http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/sequencing.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/sequencing.html">http://build.fhir.org/ig/HL7/genomics-reporting/branches/fhir_operations/sequencing.html</a> at some point. Happy to consider that as a separate tracker.</p>
<p>Lastly, the example is fantastic but the operation definition would be greatly benefited by a link to an open source implementation/test server that supports it. Is there any hope of getting a roadmap to that?</p>



<a name="188384885"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/188384885" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#188384885">(Feb 17 2020 at 15:03)</a>:</h4>
<p>I have updated the JIRA with a proposed resolution, please review.<br>
<a href="https://jira.hl7.org/browse/FHIR-25250" target="_blank" title="https://jira.hl7.org/browse/FHIR-25250">https://jira.hl7.org/browse/FHIR-25250</a></p>
<p>Probably can't vote on it this week (sorry I was tardy in getting this update made), but we should be able to next week.</p>



<a name="188476142"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/188476142" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#188476142">(Feb 18 2020 at 18:15)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="196553">@Jamie Jones</span> , to be sure, did you want additional edits now, or should we wait until voting on pulling this into the main trunk?</p>



<a name="188493754"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/query%20variants%20in%20a%20specific%20region/near/188493754" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/query.20variants.20in.20a.20specific.20region.html#188493754">(Feb 18 2020 at 21:35)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> - Considering the still limited activity in master, I think you are ok waiting and making further changes in the branch.  Looks like yours will be an easy Pull Request/Merge into master.  However, the 'implications' branch will be tougher.  <span aria-label="frown" class="emoji emoji-1f641" role="img" title="frown">:frown:</span></p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
