---
layout: page
title: "FHIR Chat  · Region Studied Implementation · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html">Region Studied Implementation</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="155037904"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155037904" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155037904">(Jan 13 2019 at 17:03)</a>:</h4>
<p>Hey all, looking at using the recently added <a href="http://region%20studied%20profile" target="_blank" title="http://region%20studied%20profile">http://build.fhir.org/ig/HL7/genomics-reporting/obs-region-studied.html</a> for querying on specific genes during the connectathon, some questions came up around the suggested valueset for the code and use of the region-scope component vs Observation.method. Please feel free to elaborate <span class="user-mention" data-user-id="191451">@Patrick Werner</span></p>



<a name="155038676"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155038676" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155038676">(Jan 13 2019 at 17:24)</a>:</h4>
<p>Here is a proposed example meeting the use case of providing the observations for a specific gene and specific patient:     <br>
           {<br>
      "resourceType": "Observation",<br>
      "id": "studReg",<br>
      "meta": {<br>
        "profile": [<br>
          "<a href="http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/obs-region-studied" target="_blank" title="http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/obs-region-studied">http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/obs-region-studied</a>"<br>
        ]<br>
      },<br>
      "status": "final",<br>
      "category": {<br>
        "coding": [<br>
            {"coding":[{"system":"http://loinc.org","code":"TBD-RegionsStudied"}]}<br>
        ]<br>
      },<br>
      "code": {<br>
        "coding": [<br>
          {<br>
            "system": "<a href="http://loinc.org" target="_blank" title="http://loinc.org">http://loinc.org</a>",<br>
            "code": "TBD-Described",<br>
            "display": "Genetic variant assessment"<br>
          }<br>
        ]<br>
      },<br>
      "subject": {<br>
        "reference": "Patient/HG03239"<br>
      },<br>
      "issued": "2019-01-07T17:48:34+05:30",<br>
      "valueCodeableConcept": {<br>
        "coding": [<br>
          {<br>
            "system": "willBeRemoved",<br>
            "code": "None",<br>
            "display": "NotNeeded"<br>
          }<br>
        ]<br>
      },<br>
      "specimen": {<br>
        "reference": "Specimen/specimen1"<br>
      },    <br>
      "component": [<br>
        {<br>
          "code": {<br>
            "coding": [<br>
              {"coding":[{"system":"http://loinc.org","code":"48018-6"}]}<br>
            ]<br>
          },<br>
          "valueCodeableConcept": {<br>
            "coding": [<br>
              {<br>
                "system": "<a href="http://www.genenames.org" target="_blank" title="http://www.genenames.org">http://www.genenames.org</a>",<br>
                "code": "HGNC:2625",<br>
                "display": "CYP2D6"<br>
              }<br>
            ]<br>
          }<br>
        },      <br>
        {<br>
          "code": {<br>
            "coding": [<br>
                {"coding":[{"system":"http://loinc.org","code":"TBD-RegionScope"}]}<br>
            ]<br>
          },<br>
          "valueCodeableConcept": {<br>
            "coding": [<br>
              {<br>
                "system:"CG"<br>
                "code": "SpecificVariants"<br>
              }<br>
            ]<br>
          }<br>
        }<br>
    ] <br>
      "hasMember":[<br>
        {<br>
            "reference":"Observation/descrete1"<br>
        },<br>
        {<br>
            "reference":"Observation/descrete2"<br>
        }     <br>
      ]  <br>
    }</p>



<a name="155038793"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155038793" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155038793">(Jan 13 2019 at 17:27)</a>:</h4>
<p>corresponding query draft: <code>Observation?code=TBD-RegionsStudied&amp;combo-code=http://loinc.org|48018-6&amp;combo-value-concept=http://www.genenames.org|HGNC:2625</code></p>



<a name="155039272"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155039272" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155039272">(Jan 13 2019 at 17:39)</a>:</h4>
<p>So, you are asking about what would be the RegionStudied.method versus RegionStudied.component[region scope].valueCodeableConcept?  <span class="user-mention" data-user-id="196553">@James Jones</span></p>



<a name="155040658"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155040658" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155040658">(Jan 13 2019 at 18:14)</a>:</h4>
<blockquote>
<p>So, you are asking about what would be the RegionStudied.method versus RegionStudied.component[region scope].valueCodeableConcept?  <span class="user-mention" data-user-id="196553">@James Jones</span></p>
</blockquote>
<p>yes. That was the question. I would be in favor to use method and get rid of the component as it duplicates method and could lead to redundant data.</p>



<a name="155040740"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155040740" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155040740">(Jan 13 2019 at 18:16)</a>:</h4>
<p>Not sure I agree, but am OK with a tracker to discuss.  The "scope" was meant to be a simpler way to understand the region, as methods will vary quite a bit (I think?).</p>



<a name="155041165"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155041165" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155041165">(Jan 13 2019 at 18:29)</a>:</h4>
<p>I'll create a tracker now.</p>



<a name="155041436"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155041436" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155041436">(Jan 13 2019 at 18:36)</a>:</h4>
<p>Can you give me some examples of what we will use for method? <span class="user-mention" data-user-id="192123">@Bret H</span></p>



<a name="155041564"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155041564" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155041564">(Jan 13 2019 at 18:41)</a>:</h4>
<p>created <a href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19975" target="_blank" title="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=19975">GF#19975</a> <br>
but i'm now also unsure if it is the same. Depends on what will be included in method.</p>



<a name="155041928"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155041928" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155041928">(Jan 13 2019 at 18:52)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  already present are 'Exome, Genome, SpecificVariant...' these are not really methods but conceptual context of what was done. E.g. 'PCR' is a method but could be whole-gene or specific variant. You'd need the PCR probes to infer OR be told. FYI: <span class="user-mention" data-user-id="196553">@James Jones</span>  suggested context when I was looking for a word.<br>
Scope is not quite the right word and it is not a broad description of the method but related to the methods used to study, in my opinion.</p>



<a name="155041940"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155041940" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155041940">(Jan 13 2019 at 18:53)</a>:</h4>
<p><a href="/user_uploads/10155/sTsAku3JnrenKBvr5L0Mu5Sn/pasted_image.png" target="_blank" title="pasted_image.png">pasted image</a> <br>
We need to add these to the example. <span class="user-mention" data-user-id="193751">@Bob Dolin</span>  sent me the specific described variant profiles that would be used for the 'has member' relationship</p>
<div class="message_inline_image"><a href="/user_uploads/10155/sTsAku3JnrenKBvr5L0Mu5Sn/pasted_image.png" target="_blank" title="pasted image"><img src="/user_uploads/10155/sTsAku3JnrenKBvr5L0Mu5Sn/pasted_image.png"></a></div>



<a name="155042082"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155042082" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155042082">(Jan 13 2019 at 18:57)</a>:</h4>
<p>So you are saying we need something like “scope” but the name isn’t right?  Your example of PCR is why I was just thinking we needed “scope”. I like the name “context” as well if that makes more sense ?</p>



<a name="155042286"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155042286" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155042286">(Jan 13 2019 at 19:03)</a>:</h4>
<p>be interested in Patrick's (and any others comments)...but yeah, 'scope' to me is not quite right. But this could be word smithing as 'context' kinda says the same as 'scope'</p>



<a name="155042461"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155042461" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155042461">(Jan 13 2019 at 19:08)</a>:</h4>
<p>I think the question here is if it's meant as a summary/clarification of the method or trying to say something of its own, perhaps from where the regionstudied observation was created? (we are looking at building these observations from a query, perhaps we should be using GDR for this though...)</p>



<a name="155044862"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155044862" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155044862">(Jan 13 2019 at 20:18)</a>:</h4>
<p>main issue I have with Region-studied profile (ballot comment #19845) is that we need to have the ability to assert arbitrary locations. Main use cases that come to mind are: [1] region studied may be intergenic; [2] differentiating variant calling accuracy across sub-regions of a gene.</p>



<a name="155044956"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155044956" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155044956">(Jan 13 2019 at 20:20)</a>:</h4>
<p>A bit more explanation on [2] - I can't simply say that gene TPMT was studied. It might be that I only studied the exons. It might be that for technical reasons, portions of the gene couldn't be assessed. But I need that level of granularity in order to assess the accuracy of variant calls, determine where there might be false-negatives, etc.</p>



<a name="155045373"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155045373" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155045373">(Jan 13 2019 at 20:32)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> - those use cases make sense - we just can't do it yet.  We can discuss those when we get to that ballot comment.  If you have a proposal, that would help the discussion about that ballot comment significantly.</p>



<a name="155046514"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Region%20Studied%20Implementation/near/155046514" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Region.20Studied.20Implementation.html#155046514">(Jan 13 2019 at 21:08)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> Thanks Kevin.  I stopped short of including a proposal in the  ballot comment, other than to suggest we look at GA4GH's htsget protocol, which supports location-based queries. From there, I feel like we have to think through the design a bit. Location start and end position alone isn't sufficient - it needs to be coupled with chromosome and build, or with reference sequence, along with coordinate system.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
