---
layout: page
title: "FHIR Chat  · Representation of secondary findings · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html">Representation of secondary findings</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="179653789"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179653789" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Mullai Murugan <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179653789">(Nov 01 2019 at 18:52)</a>:</h4>
<p>Hi all - the only element related to secondary findings is the observation-secondaryFinding extension element in the InheritedDiseasePathogenicity resource.  Though the definition for this element states "This extension should be used to denote when a genetic finding is being shared as a secondary finding, and ideally refer to a corresponding guideline or policy statement.", the related valueset seems to be more about the policy statement as the two options available are acmg-version1 and acmg-version2.  Indicating if a finding is secondary is considered somewhat of a first class element for us and establishing a granular distinction between related secondary finding elements will be useful.  Can this expanded further to include something similar to the following -<br>
- isSecondaryFinding  (boolean)<br>
- SecondaryFindingSummaryText ( e.g. A heterozygous pathogenic variant in the PMS2 gene was detected . This finding is considered unrelated to the patient's primary phenotypic indications and is considered an actionable incidental finding)<br>
- categorycode (acmg v1, v2)</p>



<a name="179661798"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179661798" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179661798">(Nov 01 2019 at 20:22)</a>:</h4>
<p>That extension is actually on the base Observation resource as of R4:<br>
<a href="https://hl7.org/fhir/extension-observation-secondaryfinding.html" target="_blank" title="https://hl7.org/fhir/extension-observation-secondaryfinding.html">https://hl7.org/fhir/extension-observation-secondaryfinding.html</a></p>
<p>Regarding the ValueSet, it is extensible, so you can indicate other values as well.  So if you include the extension, then you are indicating this is a secondary finding.</p>
<p>Regarding your text statement, I think most of that data will be included discretely, but you could include it in an InheritedDiseasePathogenity as a RelatedArtifact extension value?</p>



<a name="179671673"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179671673" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Mullai Murugan <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179671673">(Nov 01 2019 at 22:37)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> thanks. Instead of the relatedartifact, I added the secondaryfinding text to the "text" value of the observation-secondaryFinding and included a boolean value to CodeableConcept to indicate if the finding is secondary.</p>



<a name="179672312"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179672312" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179672312">(Nov 01 2019 at 22:48)</a>:</h4>
<p>My only caution to you is that the CodedableConcept.text field is intended to represent a "Plain text representation of the concept" and not necessarily additional information like this.  And I am not sure where you put the boolean value?</p>



<a name="179938036"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179938036" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179938036">(Nov 05 2019 at 14:40)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> To be clear, the way this extension attribute (observation-secondary-finding) is designed to work is to provide a coded value that represents the secondary finding policy or guideline used to find secondary findings with clinical significance to phenotypes not directly indicated by the ordering physician. So, the producer and consumer must understand that if this extended attribute is absent or has no value then the finding is related to the primary indication (or a primary finding) and if there is any value in this field then it is a secondary finding. And if the consumer wants to know the secondary finding policy that the lab used as a part of the assay methodology then they must inspect this value on each secondary finding to figure that out.</p>
<p>It seems to me that this is a bit of an indirect way to indicate secondary findings. It would be much cleaner IMO if we simply had an indicator on each genomicsbase subprofile to state explicitly whether it is a secondary finding or not (if left out then the default is that it is NOT a secondary finding- very similar to how present and absent works). But the producer can explicitly indicate "Yes" this is a secondary finding or "No" this is not a secondary finding.  This could be done in a codeableconcept if desired, but I wouldn't mix it with the method or policy for how secondary findings where determined. The secondary finding policy or method should be part of the assay plan definition or methodology.  </p>
<p>If we open the door to putting "methods" into specific observations which would otherwise be defined at the overall test definition then things could get pretty confusing for the implementers and users. I would reserve adding method-like information to observations only in exceptionally situations where it deviates from overall assay plans.  </p>
<p>If this is not going to be changed then I would clearly state in the extensibility notes for this observation-secondary-finding that implementers should never add a value that represents a "negative" value (e.g. observation-secondary-finding = "Not a secondary finding").  I know that seems silly but folk may add the notion of positive and negative values to this extension in cases where they want to clearly indicate which genomicsbase sub-profiles are secondary findings vs primary findings.</p>



<a name="179965785"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179965785" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179965785">(Nov 05 2019 at 19:11)</a>:</h4>
<p>I made the original request to O&amp;O to get it defined.  Based on feedback I had received, the indication of the 'policy' was an important thing to understand, in addition to the base fact that an Observation is a secondary finding.  </p>
<p>If we want to change the data type of the current extension, that would be a request to O&amp;O since they own Observation. That extension is still in draft so would be changeable if approved.  We could then consider how the 'policy' portion is best modeled in our IG, perhaps as its own extension or, as you hint, perhaps it should be aligned with a broader definition of methodology.</p>
<p>Thoughts from others?</p>



<a name="179968746"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179968746" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179968746">(Nov 05 2019 at 19:39)</a>:</h4>
<p>I think ideally what is in the "SecondaryFindingSummaryText" in the proposal should be generated from the coded components/extensions and included in the Observation's narrative. <span aria-label="smiley" class="emoji emoji-1f603" role="img" title="smiley">:smiley:</span></p>



<a name="179969470"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179969470" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179969470">(Nov 05 2019 at 19:46)</a>:</h4>
<p>If a system is attempting to query for a bundle of Observations based on secondary-finding status, the current setup would require primary findings to not populate this extension or have some coherent way to distinguish themselves.</p>



<a name="179970240"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179970240" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179970240">(Nov 05 2019 at 19:55)</a>:</h4>
<p>Current setup absolutely assumes that only Observations that are 'secondary' would get a value for this extension.  I don't know why a 'primary' finding would ever populate something used to indicate 'secondary' anyway, regardless of its data type.</p>



<a name="179970624"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/179970624" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#179970624">(Nov 05 2019 at 19:59)</a>:</h4>
<p>It does get us dangerously close to one of our favorite spaces however, the "assumed negative by omission" issue. Certainly something implementers should be aware of.</p>



<a name="180029432"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/180029432" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#180029432">(Nov 06 2019 at 13:04)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> or <span class="user-mention" data-user-id="196553">@James Jones</span> can you clarify if the notion of "secondary findings" is an O&amp;O issue or is it a CG issue? Is O&amp;O taking on the analysis and generalization of how secondary findings would be communicated in any kind of observation for all domains?</p>
<p>Maybe we could try to separate the kinds of observations that are happening to determine how to best use the Observation resource to communicate each individual piece of information...</p>
<p>The tuple of Observation code, value pairs as we know is the foundation of the observation. The <em>code</em> is typically a LOINC code which describes the question in a precise enough manner to provide significant context to the <em>value</em> that can and should be provided.  The LOINC system has taken on an significant amount of complexity to describe the question and value in a context that makes this code-value pairing useful in computational healthcare systems.</p>
<p>So what are the questions, context and answers that we really need for clinical genetic testing (just spitballing here)</p>
<ol>
<li>
<p>Code: What region of the specimens germline genome did I interrogate? <br>
  Value: A structure that represents a reference-based sequence location.</p>
</li>
<li>
<p>Code: What genotype of the specimens germline genome did I interrogate? <br>
 Value: A structure that represents a small SNP InDel genotype.</p>
</li>
<li>
<p>Code: What genotype (small SNP or InDel) was PRESENT in the germline genome of the specimen? <br>
 Value: a construct / data type that represents the precise sequence variants and its zygosity  or allelic state?</p>
</li>
<li>
<p>Code: What sequence variant genotype (small SNP or InDel) was ABSENT in the germline genome of the specimen?<br>
 Value: a construct / data type that represents the precise sequence variants and its zygosity  or allelic state?</p>
</li>
<li>
<p>Code: What is the primary clinical significance assessment of the genotype found in the germline specimen?<br>
  Value: Maybe some type of structure that holds both the level of pathogenicity coupled with the disease/phenoteyp associated. (NOTE: this particular observation would be required to have a "DerivedFrom" observation of the type of #3 above.)</p>
</li>
<li>
<p>Code: What is the secondary clinical significance assessment of the genotype found in the germline specimen?<br>
  Value: same as #5 above but clearly indicating that this is a secondary finding type of assessment.</p>
</li>
</ol>
<p>... I could go on and on...</p>
<p>This is a brand new idea that hit me this morning. I've been working for a long time to try to find ways to organize and simplify the way genetic observations are reported. This approach may be something worth exploring at the least. LOINC can be extended to support the class of codes we would need to annotate things like "germline vs somatic" genomes, types of variation (sequence variant genotypes, alleles, complex hets, structurcal variants, copy number variants, etc...).  </p>
<p>While I was never too keen on LOINC due to the fact that it externalized the context of the question, I think I understand the value in it now in that it removes the need to include that complexity directly in the HL7 message itself.  Let's let LOINC help us shape the world of questions (observation codes) in a thoughtful way that works for genetic testing results.  Then we can focus on providing the new data structures to put in the values that align with those newly crafted LOINC observation codes for genetics.</p>
<p>If anyone is up for discussing this let me know. It's a pretty raw idea, but I think it opens the door to a systematic approach to sharing these results.</p>



<a name="180040817"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/180040817" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#180040817">(Nov 06 2019 at 15:05)</a>:</h4>
<p><span class="user-mention" data-user-id="209510">@Larry Babb</span> - The current <a href="http://build.fhir.org/extension-observation-secondaryfinding.html" target="_blank" title="http://build.fhir.org/extension-observation-secondaryfinding.html">secondary findings</a> extension is owned by O&amp;O, so they would have to approve changes.  It was introduced at CG's request.</p>
<p>Regarding your other points - I think  you just described how we approached this problem.  Just a couple of other points/clarifications on our approach:</p>
<ul>
<li>The questions we are asking are different.  Your proposals above are much more specific than the ones we have.</li>
<li>The 'structure that represents X' is not a structure specific to each question, but instead is mapped to the value[x] + the components we have defined for each profile.</li>
</ul>
<p>I would map your questions to our profiles roughly like this:</p>
<ol>
<li>RegionStudied</li>
<li>RegionStudied</li>
<li>Variant (with a .value = Present and components to define the 'genotype'</li>
<li>Variant (with a .value  = Absent and components to define the 'genotype'</li>
<li>InheritedDiseasePathogenicity</li>
<li>Use the profiles above, but add a value to 'secondaryFindings' extension on the appropriate observations.</li>
</ol>
<p>Does that make sense, or did I completely miss something?</p>



<a name="180047382"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/180047382" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Larry Babb <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#180047382">(Nov 06 2019 at 16:07)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> i do completely get what the CG group did to define more generalized questions with more complex component-based results. I think there's a better balance to avoid the structure by component model as is being done currently. This would also offer the opportunity to define the more computable, reliable and standardized structures to use in the "value" of the observation.</p>



<a name="180062108"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/180062108" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#180062108">(Nov 06 2019 at 18:27)</a>:</h4>
<p>I hope that the IM work will inform what our model should ideally look like, but once that is complete we will still be faced with aligning that with FHIR - and that is where introducing new structures requires someone to build firm proposals and clearly describe why what is in place today does not meet our requirements.</p>



<a name="180062248"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20secondary%20findings/near/180062248" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20secondary.20findings.html#180062248">(Nov 06 2019 at 18:28)</a>:</h4>
<p>If you are willing to build on your ideas, I for one would be happy to help work through it.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
