---
layout: page
title: "FHIR Chat  · Representation of Repeat Expansions in FHIR · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html">Representation of Repeat Expansions in FHIR</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="263635055"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263635055" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Rachel Kutner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263635055">(Dec 03 2021 at 20:39)</a>:</h4>
<p>Hey all, I just submitted the following tracker in Jira: <a href="https://jira.hl7.org/browse/FHIR-34418">https://jira.hl7.org/browse/FHIR-34418</a> </p>
<p>There's a proposal attached with extra information, I gave a short summary in the main JIRA ticket. </p>
<p>Looking forward to discussing this further on the 20th.</p>



<a name="263638426"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263638426" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263638426">(Dec 03 2021 at 21:05)</a>:</h4>
<p>Awesome. I commented on the JIRA. The documentation is extremely informative and well done. I would be particularly interested in why the current mechanisms for phasing do not work. BTW While I do believe the current components meet the use case, I would like to understand better why/if your team sees a gap. </p>
<p>It should be more than a desire to have a named component called 'repeat' versus something called 'copy-number' on a molecular level they are the same (and the component names are not meant to be computable). At a higher-level there are differences, but this is similar to the differences at a high-level for an HLA-allele and a Star-allele. For these two the same underlying components are used - i.e. we do not have a component called HLA-allele and one called Start-allele. </p>
<p>Be good to learn if it is the high-level category and classification that is desired to be communicated. (might be worth exploring component:variation-code for that. Or perhaps,component:molecular-consequence or component:coding-change-type ).</p>
<p>Here's the Sequence Ontology term to look at <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0002165">http://www.sequenceontology.org/browser/current_release/term/SO:0002165</a> (it fits in the extensible value set for component: molecular consequence).<br>
<span class="user-mention" data-user-id="249407">@Rachel Kutner</span></p>
<p>The sender could hard-code the value in when sending a triplet repeat finding.</p>



<a name="263655593"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263655593" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Arthur Hermann <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263655593">(Dec 03 2021 at 21:30)</a>:</h4>
<p><span class="user-mention silent" data-user-id="249407">Rachel Kutner</span> <a href="#narrow/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR/near/263635055">said</a>:</p>
<blockquote>
<p>Hey all, I just submitted the following tracker in Jira: <a href="https://jira.hl7.org/browse/FHIR-34418">https://jira.hl7.org/browse/FHIR-34418</a> </p>
<p>There's a proposal attached with extra information, I gave a short summary in the main JIRA ticket. </p>
<p>Looking forward to discussing this further on the 20th.</p>
</blockquote>
<p><span class="user-mention" data-user-id="192198">@Bob Freimuth</span>  - please take a look at well... Bret H has been responding quite a bit - but I would love to hear what you think from a modeling perspecitve (or if Bret H has that covered already)</p>



<a name="263669705"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263669705" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263669705">(Dec 04 2021 at 00:06)</a>:</h4>
<p>if the nested tandem repeat structures are added. We might not be able to align with the WG IM unless the IM model decides to adopt something like a 'structural' repeat based on repeats of alleles (in the IM sense and for CNV) versus a 'local sequence tandem repeat' where sequence repeats exist that do not encompass an entire gene (current arbitrary limits for CNVs tend to be 1kb in size but it's arbitrary).</p>
<p>As long as the start/end of the repeats are defined relative to a reference, then you're good to go. With the nested structure suggested it can simplify the representation of a tandem repeat (won't need to use haplotype, for example, when two types of trinucleotide repeats are occurrent).</p>



<a name="263670318"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263670318" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263670318">(Dec 04 2021 at 00:15)</a>:</h4>
<p>and just to say it - from a sequence level perspective sending the sequence change in Alt and a reference and a position would communicate the change. However, the desire to communicate the number of each type of repeat is what forces a more complex representation (either multiple variant profiles with haplotype or the nested structure proposed)</p>



<a name="263670397"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263670397" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263670397">(Dec 04 2021 at 00:16)</a>:</h4>
<p>I still like the idea of sending an SO code for tandem repeat to give the consumer an opportunity to know more clearly what they've just received.</p>



<a name="263677058"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263677058" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263677058">(Dec 04 2021 at 02:23)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> If the ref-allele field were changed to 0..*, and you include multiple ref-alleles, along with multiple locations and multiple copy numbers, does FHIR or JSON guarantee that you can determine which location and copy number goes with which ref-allele?</p>



<a name="263714584"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263714584" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263714584">(Dec 04 2021 at 15:39)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span>  It depends on what you are thinking. If we are talking about using multiple Variant profiles they would each have there own references. </p>
<p>Not sure what your thinking.</p>



<a name="263714733"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263714733" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263714733">(Dec 04 2021 at 15:42)</a>:</h4>
<p>I assumed for repeats composed of multiple triplets that each triplet would have it's reference included, or positional information provided. This would be in each repeat-pair if using the nested extension type of communication.</p>



<a name="263714942"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263714942" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263714942">(Dec 04 2021 at 15:46)</a>:</h4>
<p>The use of component:molecular-consequence can tip off the receiving system that it's got a triplet to deal with. </p>
<p>Here's the Sequence Ontology term to look at <a href="http://www.sequenceontology.org/browser/current_release/term/SO:0002165">http://www.sequenceontology.org/browser/current_release/term/SO:0002165</a> (it fits in the extensible value set for component: molecular consequence).</p>
<p>Using multiple Variants would be more consistent in some ways with how other Variants arrive - meaning that the system can use the data without special programming to handle tandem repeats.</p>



<a name="263715232"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263715232" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263715232">(Dec 04 2021 at 15:53)</a>:</h4>
<p>a third way to look at it, just provide one Variant which covers the whole region, don't call out the repeat units with an element but have them calculated from sequence data or parse from a provided HGVS string. </p>
<p>A fourth way is to have the repeat numbers provided as an implications (with the sequence region provided in a Variant profile).</p>
<p>Looks like it is an opportunity to rethink CNV and other structural variation. </p>
<p>BTW In the IM group we've been talking about multi-loci and alleles - that would map nicely for tandem repeats, nearly 1 to 1, using multiple variant profiles and haplotype. But not so much to a nested tandem repeat extension. However, there might be a compelling reason to use the nested tandem repeat extension.</p>



<a name="263857810"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263857810" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263857810">(Dec 06 2021 at 14:33)</a>:</h4>
<p><span class="user-mention silent" data-user-id="193751">Bob Dolin</span> <a href="#narrow/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR/near/263677058">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="192123">Bret H</span> If the ref-allele field were changed to 0..*, and you include multiple ref-alleles, along with multiple locations and multiple copy numbers, does FHIR or JSON guarantee that you can determine which location and copy number goes with which ref-allele?</p>
</blockquote>
<p>Just to clarify Bob, simple example:</p>
<blockquote>
<p>component[exact-start-end] = {10, 18}<br>
component[ref-allele] = CAGCAGCAG<br>
component[copy-number] = 3</p>
<p>component[exact-start-end] = {19, 30}<br>
component[ref-allele] = GCAGCAGCAGCA<br>
component[copy-number] = 4</p>
</blockquote>
<p>If you are asking if we can assume the first three are related, and the second there are related - today, I think the answer is 'no' other than to document guidance to recommend <code>when you send these, please make sure they are in the correct sequence</code></p>



<a name="263866636"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263866636" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263866636">(Dec 06 2021 at 15:30)</a>:</h4>
<p>That's not quite what I meant. Obviously with two variant profiles the position and sequence reference for each repeat is explicit. I was saying that with the nested structure to have the same level of precision you would need to add the sequence reference and position in each 'repeat pair' of the proposed.</p>
<p>sequence reference being an accessioned reference.</p>



<a name="263867066"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263867066" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263867066">(Dec 06 2021 at 15:32)</a>:</h4>
<p>Understood <span class="user-mention" data-user-id="192123">@Bret H</span> - I was trying to clarify if that is what <span class="user-mention" data-user-id="193751">@Bob Dolin</span> was asking.</p>



<a name="263892775"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263892775" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263892775">(Dec 06 2021 at 18:18)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> - Did your thumbs up mean that what I described is what you were asking?</p>



<a name="263892868"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263892868" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263892868">(Dec 06 2021 at 18:19)</a>:</h4>
<p>yes <span class="user-mention" data-user-id="191913">@Kevin Power</span> , thanks.</p>



<a name="263899314"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/263899314" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#263899314">(Dec 06 2021 at 19:00)</a>:</h4>
<p>I like to try and fit things into current models, but I don't want to force it either. At minimum, I want to limit the number of extensions we add.</p>
<p>To that end, another option would be to introduce some sort of 'component association' extension at the Observation.component level that would allow implementers to indicate when components should be treated as a set. So the two sets of components I said above could have an extension something like:</p>
<blockquote>
<p>component[exact-start-end] = {10, 18}<br>
component[exact-start-end] .extension[component-set] = "repeat-expansion-set1"<br>
component[ref-allele] = CAGCAGCAG<br>
component[ref-allele] .extension[component-set] = "repeat-expansion-set1"<br>
component[copy-number] = 3<br>
component[copy-number] .extension[component-set] = "repeat-expansion-set1"</p>
<p>component[exact-start-end] = {19, 30}<br>
component[exact-start-end] .extension[component-set] = "repeat-expansion-set2"<br>
component[ref-allele] = GCAGCAGCAGCA<br>
component[ref-allele] .extension[component-set] = "repeat-expansion-set2"<br>
component[copy-number] = 4<br>
component[copy-number].extension[component-set] = "repeat-expansion-set2"</p>
</blockquote>
<p>It is still an extension, but only a single (and therefore a simple) extension. I admit its intention isn't as clear as the proposed complex extension from <span class="user-mention" data-user-id="249407">@Rachel Kutner</span>  and would require some very clear guidance when and when not to use this extension. If this is the only use case for it, perhaps we name this extension very narrowly, calling the extension "repeat-expansion" to be clear that is only should be used for this use case. Or, we keep it more open/generic and rely on implementers to use it correctly.</p>



<a name="264019928"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264019928" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264019928">(Dec 07 2021 at 15:50)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  what do the positions refer to? Assuming you have a reference sequence with an accession id in the Variant profile.</p>



<a name="264020627"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264020627" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264020627">(Dec 07 2021 at 15:55)</a>:</h4>
<p>Sorry for my very narrow example, just trying to keep it focused on the specific attributes needed for the problem at hand. I proposed the positions would be the start and optionally the end of that specific repeat expansion.</p>



<a name="264046947"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264046947" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264046947">(Dec 07 2021 at 18:16)</a>:</h4>
<p>within?</p>



<a name="264047701"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264047701" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264047701">(Dec 07 2021 at 18:20)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> what are your thoughts around position? Imagine for instance we want to represent CAG[20]CTAG[15]. Let's say we were to represent it as two variants: CAG[20], CTAG[15]. If we include positional information for each variant, then we know 5' vs. 3'. But what exactly do we mean by 'position'? Taking CAG[20], if position is the location of the 'C', and ref-allele is 'CAG', and copy-number is 20, this is not the same as CAG[20]. Rather, it's saying that we are replacing one CAG with 20 CAGs.</p>



<a name="264048827"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264048827" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264048827">(Dec 07 2021 at 18:27)</a>:</h4>
<p>DOH! Just realized I totally got that part of my example wrong. I should have been 'alt-allele' not 'ref-allele' sorry for the confusion. So maybe something like this:</p>
<blockquote>
<p>component[exact-start-end] = {10, 18}<br>
component[exact-start-end] .extension[component-set] = "repeat-expansion-set1"<br>
component[ref-allele] = CAG<br>
component[ref-allele] .extension[component-set] = "repeat-expansion-set1"<br>
component[alt-allele] = CAGCAGCAG<br>
component[alt-allele] .extension[component-set] = "repeat-expansion-set1"<br>
component[copy-number] = 3<br>
component[copy-number] .extension[component-set] = "repeat-expansion-set1"</p>
<p>component[exact-start-end] = {19, 30}<br>
component[exact-start-end] .extension[component-set] = "repeat-expansion-set2"<br>
component[ref-allele] = GCA<br>
component[ref-allele] .extension[component-set] = "repeat-expansion-set2"<br>
component[alt-allele] = GCAGCAGCAGCA<br>
component[alt-allele] .extension[component-set] = "repeat-expansion-set2"<br>
component[copy-number] = 4<br>
component[copy-number].extension[component-set] = "repeat-expansion-set2"</p>
</blockquote>



<a name="264060980"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264060980" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264060980">(Dec 07 2021 at 19:46)</a>:</h4>
<p><span class="user-mention silent" data-user-id="193751">Bob Dolin</span> <a href="#narrow/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR/near/264047701">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="192123">Bret H</span> what are your thoughts around position? Imagine for instance we want to represent CAG[20]CTAG[15]. Let's say we were to represent it as two variants: CAG[20], CTAG[15]. If we include positional information for each variant, then we know 5' vs. 3'. But what exactly do we mean by 'position'? Taking CAG[20], if position is the location of the 'C', and ref-allele is 'CAG', and copy-number is 20, this is not the same as CAG[20]. Rather, it's saying that we are replacing one CAG with 20 CAGs.</p>
</blockquote>
<p>Not sure what you mean. CAG[20] as start with position 20 or did you want to say CAGCAGCAG..., that is CAG with 20 repeats?</p>



<a name="264063119"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264063119" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264063119">(Dec 07 2021 at 20:00)</a>:</h4>
<p>sorry <span class="user-mention" data-user-id="192123">@Bret H</span> , a poorly worded question that I'm still thinking through. But take for example  CAG[20]CTAG[15]. Let's say that we have a reference sequence, CAG[3], starting at position 4, followed by CTAG[5], starting at position 13:</p>



<a name="264063157"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264063157" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264063157">(Dec 07 2021 at 20:01)</a>:</h4>
<p>...CAGCAGCAGCTAGCTAGCTAGCTAG...'</p>



<a name="264063386"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264063386" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264063386">(Dec 07 2021 at 20:02)</a>:</h4>
<p>And let's say we want to treat CAG[20]CTAG[15] as two variants. The first variant, CAG[20], might look like this:<br>
component start-end=4..12<br>
ref-allele=CAG<br>
copy-number=20<br>
It's a bit different from CNVs in that what we're saying here is to replace 4..12 with 20 copies of the reference allele, whereas for CNVs, we're saying that there are 20 copies of the bases in 4..12.</p>



<a name="264074358"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264074358" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264074358">(Dec 07 2021 at 21:25)</a>:</h4>
<p>Maybe a basic question here - if we are thinking like a biologist/bioinformatician/clinician, is it Ok to think about something like CAG[20]CTAG[15] as two variants?</p>



<a name="264080609"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264080609" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264080609">(Dec 07 2021 at 22:09)</a>:</h4>
<p>I'm not sure <span class="user-mention" data-user-id="191913">@Kevin Power</span> , I was just thinking it would be easier - similar to where there are, say, two SNVs in a gene (that could potentially be grouped into a haplotype)</p>



<a name="264081955"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264081955" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264081955">(Dec 07 2021 at 22:19)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> - Yea, I do think our guidance for how to do this would be easier if we are OK with expressing it as &gt;1 Variant observation with some way to tie them together (I almost said "group" them, but figured that has too much baggage right now <span aria-label="wink" class="emoji emoji-1f609" role="img" title="wink">:wink:</span> )</p>



<a name="264082752"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264082752" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264082752">(Dec 07 2021 at 22:26)</a>:</h4>
<p>And, there is nothing saying that a biologically approved definition must exactly match our computational representation anyway. The closer we get to matching the better, but I would argue that even if every biologist in the world would agree that CAG[20]CTAG[15] is a single 'variant' that doesn't mean it has to fit into one of our Variant observations.</p>
<p>Before we go much farther though, <span class="user-mention" data-user-id="249407">@Rachel Kutner</span> -- what we are discussing here is certainly different that what you proposed. I think with the right guidance it could work, but do you agree? Perhaps we would need to work up (at least) a specific example for review.</p>



<a name="264084056"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264084056" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264084056">(Dec 07 2021 at 22:38)</a>:</h4>
<p>It's too bad we can't just use the bracket notation as an alt allele...</p>



<a name="264084305"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264084305" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264084305">(Dec 07 2021 at 22:41)</a>:</h4>
<p>We certainly could, it is just a question of how computable and structured that is.</p>



<a name="264084471"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264084471" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264084471">(Dec 07 2021 at 22:43)</a>:</h4>
<p>I would say that Haplotype can tie together but sequence position would be needed to infer 5' to 3' relationship (with current constructs). Haplotype has a coded value allowing one to communicate the haplotype if there is a concept of the specific composed repeat one would like to use.</p>
<p>Looking at the VRS as an example,<br>
<a href="https://vrs.ga4gh.org/en/stable/terms_and_model.html?highlight=repeated#repeatedsequenceexpression">https://vrs.ga4gh.org/en/stable/terms_and_model.html?highlight=repeated#repeatedsequenceexpression</a><br>
the model use the starting base of where the repeat is in a reference.</p>
<p>At any rate, positional information is communicated relative to a reference sequence not the alt sequence. <span class="user-mention" data-user-id="191913">@Kevin Power</span> so there's no conflict between variants and numeric position. </p>
<p>If a complete patient composite sequence is needed Sequence could be used. OR send the whole repeat (with  reference sequence and location) and perhaps communicate repeat subunit counts with something like the nested structure.</p>
<p>The goal of sending the repeat counts is to keep the receiving system from having to do a calculation. It's a display feature. But if sent in a structured way it'll get used.</p>
<p>Also, be good to have a consensus on composing variants. Multiple Variant profiles with Haplotype pulling together OR something new. </p>
<p>VRS is adding a something new but it still uses their model of Location (like the position elements in Variant profile) and sequence (actual nucleotides, like our alt element).</p>
<p>Tricky part is deciding if receivers are going to infer the order or not. Haplotype can put them together, but the position relative to a reference sequence (hopefully the same one) would be needed to infer the order. </p>
<p>Using the nested form, like what VRS is contemplating, gives you a mechanism for declaring an order (either as an ordered list or with a numeric order number).</p>



<a name="264086325"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264086325" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264086325">(Dec 07 2021 at 23:01)</a>:</h4>
<p>You know, I didn't like it at first, but just simply using the bracket notation in the ref and alt allele attributes is starting to grow on me. That is a pretty simple grammar.</p>



<a name="264087035"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264087035" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264087035">(Dec 07 2021 at 23:08)</a>:</h4>
<p>Would it be able to alleviate the need for a count attribute? yes<br>
Would one still be able to provide position relative to a reference? yes<br>
Would it communicate order? yes<br>
Would it reduce the number of Variant profiles, which seems to be a goal? yes<br>
Does it mean our data type needs to change from string? no...maybe...<br>
Would we rename the CNV count field to make sure it is understood as being for largish structural copy communication? yes</p>
<p>I'm  not a fan of overloading the string data type, but....probably worth considering more closely. It's possible that the string could be malformed or not be parsed correctly. if the count data is vital to clinical interpretation or communication for computable CDS, need to consider how to protect users.</p>



<a name="264087123"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264087123" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264087123">(Dec 07 2021 at 23:08)</a>:</h4>
<p>But if you're doing that why not only send HGVS and ask receivers to rely on parsing HGVS? (we'd be inventing a bit another syntactical format)</p>



<a name="264087368"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264087368" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264087368">(Dec 07 2021 at 23:11)</a>:</h4>
<p>HGVS syntax has a number of ways to represent composition. including where the components are not continuous.</p>



<a name="264183476"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264183476" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264183476">(Dec 08 2021 at 16:47)</a>:</h4>
<p>FYI from the VRS G4GH group:  <a href="https://github.com/ga4gh/vrs/issues/363#issuecomment-988334323">https://github.com/ga4gh/vrs/issues/363#issuecomment-988334323</a></p>
<p>They use the location in the reference and do not use the repeats in the alt sequence to number location. The location is encompassing the entire affected space in the reference. It is implied to be a Substitution for the reference sequence data. (so if the reference had AAACTGCTGCTGAAA, 3, and the alt had CTGCTGCTGCTGCTG, 5, then the correct region start and stop for position in the reference sequence would be start at 4 and stop at 12 - which would give the correct 5 repeats only in the alt sequence if constructed).</p>



<a name="264188430"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264188430" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264188430">(Dec 08 2021 at 17:19)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span> - With all this input, any chance you could mock up a couple of different examples?</p>



<a name="264190075"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264190075" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264190075">(Dec 08 2021 at 17:30)</a>:</h4>
<p>Yes. You thinking:  like EPIC's example with<br>
version 1 using couple Variant profiles and Haplotype<br>
version 2 with components set out like you describe<br>
version3 EPIC proposal-ish (promise only slight modification)<br>
version with brackets?</p>
<p>should we put in a version with full HGVS or would that be too much, seeing as that would be one Variant profile - no specific repeat information added in separate fields. i.e. obvious and not needed.</p>



<a name="264192321"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264192321" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264192321">(Dec 08 2021 at 17:45)</a>:</h4>
<p>Sounds about right to me <span class="user-mention" data-user-id="192123">@Bret H</span> </p>
<p>Two things:<br>
I think we will always have the full HGVS, but it should always be seen as something for visual display and (ideally) we would never require someone to parse it.<br>
And yea, let's do a version with brackets - that WOULD require parsing, but I think we can keep it really simple and easy to parse. Still not sure if it is a good idea or not, but worth keeping on the list for now.</p>



<a name="264486565"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/264486565" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#264486565">(Dec 10 2021 at 18:14)</a>:</h4>
<p>FYI I am planning to have the mentioned examples up mid-next week.</p>



<a name="265172521"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265172521" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Rachel Kutner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265172521">(Dec 16 2021 at 15:31)</a>:</h4>
<p>Sorry all, I've been busy and haven't had a chance to check back on this thread. </p>
<p>I think my main concern is the assumption that the clinical results will have exact molecular coordinates in order to indicate the order in which the tandem repeats occur. I haven't seen many examples of exact genomic coordinates in the lab results I've looked at, and this isn't necessarily considered "clinically relevant" information for these results, so a spec for sending these results requiring that information might not be practical. </p>
<p>We based our structure off of existing clinical reports and the HGVS definition for repeat expansion, but we rarely see the actual HGVS name represented in clinical reports (at least the several we have) and it isn't that common in research either. </p>
<p>Per the string solution - this is my main concern as well (from Brett): <br>
"I'm not a fan of overloading the string data type, but....probably worth considering more closely. It's possible that the string could be malformed or not be parsed correctly. if the count data is vital to clinical interpretation or communication for computable CDS, need to consider how to protect users."</p>
<p>I'll bring this back to my coworker who is working on the proposal for Epic and get back on this.</p>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span>  - are we set up to discuss this during next Monday's meeting?</p>



<a name="265175664"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265175664" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265175664">(Dec 16 2021 at 15:52)</a>:</h4>
<p><span class="user-mention" data-user-id="249407">@Rachel Kutner</span>  Yep it is on the agenda--along with a 5-10 minute update on genomics operations from <span class="user-mention" data-user-id="193751">@Bob Dolin</span></p>



<a name="265209584"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265209584" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265209584">(Dec 16 2021 at 19:46)</a>:</h4>
<p><span class="user-mention" data-user-id="249407">@Rachel Kutner</span> it's a been a long-standing goal to improve the quality of genetic data labs deliver. Genetic data, in this case the positional information, serves as a very valuable source of information for the recipient organization. It has uses in clinical decision support, identifying all instances of data on a patient or population basis, utility in business intelligence (e.g. number of tests which are reporting on overlapping positions - needed for a Chief Data Officer to help guide elimination of waste), population health assessment, research and the positional information is a valuable asset in it's own right. Germline data is hyper-valuable as it can live as long as the patient does. Your clients would be well served to have the capacity to receive it. If the standard is built to accommodate positional information then they can ask the Labs for it and point to a specification. If we don't accommodate positional data then cost and waste can be higher....that's my ten cents.</p>



<a name="265210152"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265210152" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265210152">(Dec 16 2021 at 19:51)</a>:</h4>
<p>the labs don't necessarily have to send it, but building the capability into the spec I believe will pay off huge returns!</p>



<a name="265585436"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265585436" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265585436">(Dec 20 2021 at 16:28)</a>:</h4>
<p>example from FMR1 with 10 GCC repeats and 1 AGG repeat<br>
reference has GCC[9]AGG[1] and the variation has GCC[10]AGG[1]</p>
<ul>
<li>component[molecular-consequence] = SEQONT#SO:0002165</li>
</ul>
<p>component[coding-hgvs] = "NM_002024.5:c.-128_-69GGC[10]GGA[1]"<br>
component[genomic-ref-seq] = NC_000023.11</p>
<p>component[exact-start-end] = {147912051, 147912078}<br>
component[exact-start-end] .extension[component-set] = 1 "repeat-expansion-set1"<br>
component[ref-allele] = GGC GGC GGC GGC GGC GGC GGC GGC GGC<br>
component[ref-allele] .extension[component-set] = 1 "repeat-expansion-set1"<br>
component[alt-allele] = GGC GGC GGC GGC GGC GGC GGC GGC GGC GCC<br>
component[alt-allele] .extension[component-set] = 1 "repeat-expansion-set1"<br>
component[copy-number] = 10<br>
component[copy-number] .extension[component-set] = 1 "repeat-expansion-set1"</p>
<p>component[exact-start-end] = {147912079, 147912081}<br>
component[exact-start-end] .extension[component-set] = 2 "repeat-expansion-set2"<br>
component[ref-allele] = AGG<br>
component[ref-allele] .extension[component-set] = 2 "repeat-expansion-set2"<br>
component[alt-allele] = AGG<br>
component[alt-allele] .extension[component-set] = 2 "repeat-expansion-set2"<br>
component[copy-number] = 1<br>
component[copy-number] .extension[component-set] = 2 "repeat-expansion-set2"</p>



<a name="265588936"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265588936" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265588936">(Dec 20 2021 at 16:57)</a>:</h4>
<p>The above is not he EPIC like with a new nested component. btw, it's a little more conservative.</p>



<a name="265590793"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265590793" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265590793">(Dec 20 2021 at 17:13)</a>:</h4>
<p>summary from today's call - suggestions came in against overloading component[copy-number] for this, and the group wants to pursue testing the viability of using HGVS in the short term via guidance on <a href="https://varnomen.hgvs.org/recommendations/DNA/variant/repeated/">https://varnomen.hgvs.org/recommendations/DNA/variant/repeated/</a>.</p>



<a name="265601707"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265601707" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265601707">(Dec 20 2021 at 18:42)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR/near/265590793">said</a>:</p>
<blockquote>
<p>summary from today's call - suggestions came in against overloading component[copy-number] for this, and the group wants to pursue testing the viability of using HGVS in the short term via guidance on <a href="https://varnomen.hgvs.org/recommendations/DNA/variant/repeated/">https://varnomen.hgvs.org/recommendations/DNA/variant/repeated/</a>.</p>
</blockquote>
<p>Any chance someone might have a code to represent this slice, or do we just create another TBD code for it?</p>
<p>Who was going to be testing the viability of using HGVS? (sorry, I was multi-tasking during the call, perhaps I missed it)</p>



<a name="265602147"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265602147" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> John Li <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265602147">(Dec 20 2021 at 18:46)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> <br>
I don't think there is an universal code to represent repeat expansion. We are using string instead of code for that in Epic.</p>



<a name="265610770"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265610770" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265610770">(Dec 20 2021 at 20:14)</a>:</h4>
<p>Use  a Sequence Ontology term. The ontology is a better set of concepts and essentially this will be hard-coded by systems, as long as it is consistent.</p>



<a name="265625822"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265625822" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265625822">(Dec 20 2021 at 22:57)</a>:</h4>
<p>Is there a good one <span class="user-mention" data-user-id="192123">@Bret H</span> ?</p>



<a name="265629726"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265629726" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265629726">(Dec 20 2021 at 23:46)</a>:</h4>
<p>Yep. The same as I would suggest for molecular consequence - but open for discussion (here's the concept - <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0002165">http://www.sequenceontology.org/browser/current_svn/term/SO:0002165</a> ). BTW we don't talk about the power that molecular consequence and DNA change type can give computational (or with invariants), enough ; ^ ) They can be a means to clue the recipient system as to how it wants to handle the data without the system needing to calculate that the information is a tandem repeat (for example)</p>



<a name="265629826"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265629826" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265629826">(Dec 20 2021 at 23:48)</a>:</h4>
<p>but to be fair the standard is still in the early adoption phase.</p>



<a name="265630884"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265630884" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265630884">(Dec 21 2021 at 00:03)</a>:</h4>
<p>Hmm, does anyone have a concern with using a code for the 'answer' in one context, and the 'question' in another ? I suppose that should be OK, but seems a little weird to me.</p>



<a name="265631042"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631042" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631042">(Dec 21 2021 at 00:05)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> Can you give an example?</p>



<a name="265631132"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631132" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631132">(Dec 21 2021 at 00:06)</a>:</h4>
<p>Could run it past the curators of SO. But I get ya, it's been a nice rule of thumb in observations to keep LOINC for the 'question' and SNOMEDCT for the 'answer.' But in our case one could argue that the context of the use provides that. E.g. you would use a different query and not just be relying on the code and code system. The answer would be in answer slot and the question level in a question slot.</p>



<a name="265631161"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631161" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631161">(Dec 21 2021 at 00:07)</a>:</h4>
<p>query for component.code versus component.value+component.code</p>



<a name="265631225"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631225" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631225">(Dec 21 2021 at 00:08)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> - I don't have an example before this one, so that is partly why I wanted to ask.</p>



<a name="265631404"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631404" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631404">(Dec 21 2021 at 00:10)</a>:</h4>
<p>As long as we document it in the IG, it would work. One risk would be that if someone did a query wrong, they might be a result (as opposed to finding nothing) that might lead to a misinterpretation of what they received.</p>



<a name="265631427"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631427" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631427">(Dec 21 2021 at 00:11)</a>:</h4>
<p>not sure you can do the query wrong, unless you're really super-confused. but guidance can help</p>



<a name="265631605"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631605" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631605">(Dec 21 2021 at 00:13)</a>:</h4>
<p>Maybe I am worried someone will be playing Jeopardy and phrase the answer in the form of a question?  <span aria-label="smile" class="emoji emoji-1f642" role="img" title="smile">:smile:</span> <span aria-label="smile" class="emoji emoji-1f642" role="img" title="smile">:smile:</span></p>



<a name="265631702"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631702" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631702">(Dec 21 2021 at 00:14)</a>:</h4>
<p>But either way, appropriate guidance should clear up possible confusion.</p>



<a name="265631832"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representation%20of%20Repeat%20Expansions%20in%20FHIR/near/265631832" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representation.20of.20Repeat.20Expansions.20in.20FHIR.html#265631832">(Dec 21 2021 at 00:16)</a>:</h4>
<p>good question. Hope others will comment soon : ^ )</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
