---
layout: page
title: "FHIR Chat  · Representing tumor-normal sequencing tests? · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html">Representing tumor-normal sequencing tests?</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="188950435"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188950435" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188950435">(Feb 24 2020 at 18:07)</a>:</h4>
<p>Have we considered how to represent tumor-normal testing with the Genomics Reporting IG?  I'm currently investigating how we would represent this with our mCODE FHIR IG and I was wondering if there was any insight to share while I do this.</p>



<a name="188950890"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188950890" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188950890">(Feb 24 2020 at 18:12)</a>:</h4>
<p>We touched on this briefly at a connectathon last year. For variant, I believe we identified the lab would likely need to link to additional specimens through an extension (that we haven't defined anywhere yet). The 0..1 cardinality is fixed on Observation.</p>



<a name="188951182"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188951182" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188951182">(Feb 24 2020 at 18:16)</a>:</h4>
<p>Genomic source class seems to report the result but we haven't modeled the methodology anywhere yet to my knowledge.</p>



<a name="188951414"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188951414" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188951414">(Feb 24 2020 at 18:20)</a>:</h4>
<p>Thanks Jamie. So for the most part, it's pretty greenfield in our WG.  <br>
Happy to explore this further for both mCODE and the overall CG Reporting IG and present/solicit feedback and guidance from the team when I complete a first pass.</p>



<a name="188952950"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188952950" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188952950">(Feb 24 2020 at 18:38)</a>:</h4>
<p>Very much greenfield / not considered much <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>
<p>DiagnosticReport can reference multiple specimens, so we are good there.  </p>
<p>Would a particular Observation need to be about both specimens?  Simple example - if the same variant being found in both tumor and normal, can that be two Observations?  I think two makes sense to me.  Just trying to see if the 0..1 -&gt; Specimen on Observation is problem or not.  </p>
<p>I do think exploring this is a fine idea <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="188955290"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188955290" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188955290">(Feb 24 2020 at 19:03)</a>:</h4>
<p>Just to toss in another consideration - some of the somatic variant callers compute calls based on the comparison to normal, and therefore only report what are felt to be tumor variants. In this case, it might be that we could use the Variant profile, note that these are somatic variants, and just have something in the methodology</p>



<a name="188956337"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188956337" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188956337">(Feb 24 2020 at 19:14)</a>:</h4>
<p>Thanks for articulating the point I attempted to make, Bob--we haven't modeled the "testing" or explicitly given guidance on reporting the methodology required, but marking a variant as "somatic" from <a href="https://loinc.org/LL378-1/" target="_blank" title="https://loinc.org/LL378-1/">https://loinc.org/LL378-1/</a> as opposed to "likely somatic" or "unknown ..." implies the lab confirmed it</p>



<a name="188957088"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188957088" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188957088">(Feb 24 2020 at 19:22)</a>:</h4>
<p>This is a good example of how 'methodology' is a very layered concept - lots of different levels at which we might need to express these things.  Lots to figure out if someone is willing to dig in and start documenting ideas? <span aria-label="construction zone" class="emoji emoji-1f6a7" role="img" title="construction zone">:construction_zone:</span></p>
<p>However, I don't know that 'representing tumor-normal' makes clearly defining our approach to methodology a "must have" requirement?  Sounds like another consideration that needs to be accounted for when someone figures out methodology.  Does anyone disagree?</p>



<a name="188957288"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188957288" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188957288">(Feb 24 2020 at 19:25)</a>:</h4>
<p>An extreme approach that has been gaining popularity: allow a relatedArtifact extension to be used on each Observation.component to provide references in context. <span aria-label="grinning" class="emoji emoji-1f600" role="img" title="grinning">:grinning:</span></p>



<a name="188957399"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188957399" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188957399">(Feb 24 2020 at 19:26)</a>:</h4>
<p>Agreed. I find this is an issue that is cropping up a lot more for us in the CodeX FHIR accelerator.  I'm not enough of a genomics expert to be leading such a task on a methodology and best practices but happy to help where I can. Perhaps we can at least use some of the tumor-normal research I'll be doing as a conversation starter?</p>



<a name="188957526"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188957526" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188957526">(Feb 24 2020 at 19:28)</a>:</h4>
<p>I agree <span class="user-mention" data-user-id="191913">@Kevin Power</span> , and I think there is sufficient variability out there in how folks name their pipelines that it may be tricky to standardize.  <span class="user-mention" data-user-id="192198">@Bob Freimuth</span> has done some nice 'methodology' work in the IM model, and that might be good to review.</p>



<a name="188957609"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188957609" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188957609">(Feb 24 2020 at 19:29)</a>:</h4>
<p>and on a related note, <span class="user-mention" data-user-id="193751">@Bob Dolin</span> - I like the idea of leveraging Variant with genomic source class and also a tie to specimen and rolling both variants with their respective tumor and normal specimens up to DiagnosticReport.  I'll give that try with examples.</p>



<a name="188957982"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188957982" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188957982">(Feb 24 2020 at 19:33)</a>:</h4>
<p>For the curious, below is the clinical use case provided to me for analysis/prototyping:</p>
<p>A 50 year old woman presented to her local emergency department with back and shoulder pain. Initial imaging revealed diffuse lytic lesions through her axial and appendicular skeleton along with a presumed primary, 3cm, breast mass. In addition to addressing her pain, she receives a core needle biopsy which confirmed the diagnosis of ER/PR positive, HER2 negative, breast cancer. She has no family history of cancer. She has two children, a daughter in college and son about to finish high school. She is seen by medical oncology and in additional to other recommendations, she receives a recommendation to undergo genomic testing of her tumor as it may identify additional treatment options including clinical trials.</p>
<p>Branch #1Her tumor tissue is sent to vendor #1 which performs analysis and calls mutations in TP53, PIK3CA, and BRCA1. Vendor #1 notes that BRCA1 may be germline and follow up dedicated germline testing is appropriate.</p>
<p>Branch #2Her tumor tissue and peripheral blood sample are sent to vendor #2 which performs analysis and calls mutations in TP53, PIK3CA, and BRCA1. Vendor #2 notes that BRCA1 is somatic (not found in the peripheral blood, germline cell).</p>
<p>Use cases impacted from the above scenario: A researcher is interested in using mCODE to query for breast cancer patients with somatic BRCA alterations. An implementer wanted to create decision support to identify patients that have been recommended for germline testing but have not yet received it.</p>



<a name="188958339"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188958339" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188958339">(Feb 24 2020 at 19:37)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="192302">@May Terry</span> . Just out of curiosity, how often would a patient have tumor testing, and not also have germline testing in parallel (or at least prior germline testing for comparison)?</p>



<a name="188959542"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188959542" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188959542">(Feb 24 2020 at 19:51)</a>:</h4>
<p>Querying for the first use case is easily supported by our model and should be described on <a href="http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html">http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html</a>.</p>
<p>The second one is tricky because it naively involves logging and querying for <a href="http://build.fhir.org/ig/HL7/genomics-reporting/task-rec-followup.html" target="_blank" title="http://build.fhir.org/ig/HL7/genomics-reporting/task-rec-followup.html">http://build.fhir.org/ig/HL7/genomics-reporting/task-rec-followup.html</a>.<br>
Tasks can be uploaded in the same bundle as the report but aren't necessarily stored together. You could ask a server for all Tasks with code LA14022-0 "Additional testing recommended" and they will hopefully have used "reasonReference" to point to a genomic finding but just getting the patients from the "Task.for" field and then querying for their full genomic reports may be easier and safer.</p>
<p>Side note: it sounds like the result of the report is still a variant that was observed in one sample, so that is the one I'd recommend referencing on the Observation. The "normal" sample(s) could go on the DR if needed, or we may find a better place to slot them.</p>



<a name="188959595"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188959595" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188959595">(Feb 24 2020 at 19:52)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span> -  I'll need to verify, but afaik, the majority of cancer patients would not have germline testing in parallel or done prior for comparison. My understanding from the 2 ASCO oncologists who provided this use case are that tumor-normal sequencing, while not common yet, is emerging.  A recently published paper I read on Tumor-Normal Sequencing seems to confirm their comments: "Evolving Significance of Tumor-Normal Sequencing in Cancer Care" <a href="https://doi.org/10.1016/j.trecan.2019.11.006" target="_blank" title="https://doi.org/10.1016/j.trecan.2019.11.006">https://doi.org/10.1016/j.trecan.2019.11.006</a>.</p>



<a name="188960971"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188960971" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188960971">(Feb 24 2020 at 20:07)</a>:</h4>
<p>Interesting thread.  Thanks, <span class="user-mention" data-user-id="192302">@May Terry</span> </p>
<p>I think one of the first questions we'll need to answer is which of the genomic results are in which report(s).  It sounds like there could be several reports, some of which test only a single biospecimen while others test both tumor and normal specimens.  We'll need to support a single report that has both tumor and normal results, but I suspect we'll also need a way to make higher-level statements about tests that are reported separately.  I presume those higher-level statements would not need to model the result, provenance, and specimen info directly, but could simply refer to another Observation that contains that info.</p>



<a name="188999257"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188999257" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188999257">(Feb 25 2020 at 08:15)</a>:</h4>
<p><span class="user-mention silent" data-user-id="192302">May Terry</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/188959595" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/188959595">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="193751">Bob Dolin</span> -  I'll need to verify, but afaik, the majority of cancer patients would not have germline testing in parallel or done prior for comparison. My understanding from the 2 ASCO oncologists who provided this use case are that tumor-normal sequencing, while not common yet, is emerging.  A recently published paper I read on Tumor-Normal Sequencing seems to confirm their comments: "Evolving Significance of Tumor-Normal Sequencing in Cancer Care" <a href="https://doi.org/10.1016/j.trecan.2019.11.006" target="_blank" title="https://doi.org/10.1016/j.trecan.2019.11.006">https://doi.org/10.1016/j.trecan.2019.11.006</a>.</p>
</blockquote>
<p>Our partner lab only does tumor-normal sequencing. Currently we are pointing out observations.specimen to the tumor tissue, we don't create observations for the normal specimen. <br>
To mark the origin of a variant we are using the component: <a href="https://loinc.org/48002-0/" target="_blank" title="https://loinc.org/48002-0/">https://loinc.org/48002-0/</a></p>



<a name="188999425"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188999425" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188999425">(Feb 25 2020 at 08:18)</a>:</h4>
<p>I think to model this correctly you would have two reports: one for the tumor tissue, one for the normal sample. I then would create a third report with variant observations pointing to the normal sample and the the tumor sample via derived from and will populate the <a href="https://loinc.org/48002-0/" target="_blank" title="https://loinc.org/48002-0/">https://loinc.org/48002-0/</a>  component.</p>



<a name="188999533"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/188999533" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#188999533">(Feb 25 2020 at 08:20)</a>:</h4>
<p>Which seemed to be to much work/too complicated. What about an extension on Observation.specimen to point to the normal sample? So we can use one Variant per Variant instead of three, germline, non-germline goes into the source class, and we still can point to the correct normal sample (currently we are only pointing to the tumor sample, which isn't quite right)</p>



<a name="189015275"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189015275" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189015275">(Feb 25 2020 at 12:33)</a>:</h4>
<p>We should probably determine whether or not our report structure is intended to mirror what is signed out by the lab, or if we have the flexibility to re-package reports into structures that match FHIR.</p>



<a name="189020470"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189020470" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189020470">(Feb 25 2020 at 13:48)</a>:</h4>
<p>in our case the lab report never mentions the normal sample, it is only included as germline/somatic annotations in variants</p>



<a name="189020927"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189020927" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189020927">(Feb 25 2020 at 13:55)</a>:</h4>
<p>If a lab wanted to report every normal (same as every wild type) they could, I don't see anything in the spec stopping them, but agree we could increase our guidance on genomic source class surrounding somatic</p>



<a name="189021530"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189021530" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189021530">(Feb 25 2020 at 14:02)</a>:</h4>
<p>The tricky part seems to be making sure the lab can describe uncertainty in the source class with a request/note for additional testing. Would love to get someone to draft an example report/bundle for that out of this discussion, if folks are interested in the use case</p>



<a name="189030717"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189030717" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189030717">(Feb 25 2020 at 15:42)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189020927" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189020927">said</a>:</p>
<blockquote>
<p>If a lab wanted to report every normal (same as every wild type) they could, I don't see anything in the spec stopping them, but agree we could increase our guidance on genomic source class surrounding somatic</p>
</blockquote>
<p>what's missing is an extension on Observation Specimen to point to the normal sample.</p>



<a name="189031332"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189031332" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189031332">(Feb 25 2020 at 15:48)</a>:</h4>
<p>One could report the wildtype variant from the normal sample if they wanted to explicitly include it. I think attaching extra samples opens up a lot of room for error unless we do it very carefully.</p>



<a name="189031771"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189031771" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189031771">(Feb 25 2020 at 15:53)</a>:</h4>
<p>I agree with Jamie.  The Observation (Variant) is really about the tumor sample.  Why does it need to also refer to the normal sample as well?  At this point, this feels more 'methodology' related to me (to indicated the method they used was to compare to the germline and report tumor variants).  Maybe?</p>
<p>At the DiagnosticReport level, we can have multiple samples referenced (or multiple nested reports, which ever way we go).  So at least at the report level, it would be clear there were two samples.</p>
<p>Lastly, we could also consider using our 'Grouper' profile to group the observations into tumor and normal.</p>



<a name="189053518"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189053518" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189053518">(Feb 25 2020 at 19:45)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> - yes, I think we need to refer to the normal tumor sample as well.  I also agree with Patrick that per the tumor-normal description, it would ideally be 2 tests, but this would indeed be too complex a model. <br>
Looks like there are number of options available based on this thread. Lemme see if I can summarize it for discussion at next Monday's modeling meeting.  I can also create a simple starter example based on the initial assumption referencing multiple specimens, multiple variants with different genomic source classes.  Can we add it to the next Monday agenda?</p>



<a name="189055578"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189055578" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189055578">(Feb 25 2020 at 20:05)</a>:</h4>
<p>Sure. I also want to talk about it. <span class="user-mention" data-user-id="196553">@Jamie Jones</span> ?</p>



<a name="189055972"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189055972" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189055972">(Feb 25 2020 at 20:09)</a>:</h4>
<p>I also really don't want to create normal Observations. And Kevin almost convinced me with pointing to both samples in the DR, and in the Observation just to the tumor and filling out the source class. I now think this could be sufficient.</p>



<a name="189057820"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189057820" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189057820">(Feb 25 2020 at 20:29)</a>:</h4>
<p>The only concern I have with attaching unreferenced specimens on the DR is that I don't have a good suggestion for how to annotate them to retain meaning.</p>



<a name="189058003"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189058003" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189058003">(Feb 25 2020 at 20:30)</a>:</h4>
<p>Specimen.note seems to be the place, but how would one describe the role the specimen had in the tests other than by pointing an Observation at it?</p>



<a name="189058051"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189058051" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189058051">(Feb 25 2020 at 20:31)</a>:</h4>
<p>Seems like a question for O&amp;O</p>



<a name="189058438"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189058438" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189058438">(Feb 25 2020 at 20:35)</a>:</h4>
<p>I think <a href="http://build.fhir.org/specimen-definitions.html#Specimen.type" target="_blank" title="http://build.fhir.org/specimen-definitions.html#Specimen.type">http://build.fhir.org/specimen-definitions.html#Specimen.type</a> and/or <a href="http://build.fhir.org/specimen-definitions.html#Specimen.collection.bodySite" target="_blank" title="http://build.fhir.org/specimen-definitions.html#Specimen.collection.bodySite">http://build.fhir.org/specimen-definitions.html#Specimen.collection.bodySite</a> ?</p>



<a name="189059612"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189059612" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189059612">(Feb 25 2020 at 20:48)</a>:</h4>
<p>Yeah, I'm not convinced we need to add complexity to our structures to accommodate this use case (or even to streamline it).</p>
<p>I do definitely look forward to seeing the example report though--and would love to get one up in the IG--so we could give it a small time box on Monday in my opinion.</p>
<p>I'd also love to see someone extend this example to the "trio" testing picture where we have 3 samples in play and are explicitly identifying de novo variants. <span aria-label="grinning face with smiling eyes" class="emoji emoji-1f601" role="img" title="grinning face with smiling eyes">:grinning_face_with_smiling_eyes:</span></p>



<a name="189063150"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189063150" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189063150">(Feb 25 2020 at 21:28)</a>:</h4>
<p>Do I hear 4 samples?!?  <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="189064477"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189064477" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189064477">(Feb 25 2020 at 21:44)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> I have some examples of trio testing and somatic-normal testing VCF files if that would help.  It may not answer the question about how best to represent the pipeline or methodology, but might be useful to look at the variant calls.</p>



<a name="189065763"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189065763" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189065763">(Feb 25 2020 at 21:59)</a>:</h4>
<p><span class="user-mention silent" data-user-id="191913">Kevin Power</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189063150" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189063150">said</a>:</p>
<blockquote>
<p>Do I hear 4 samples?!?  :)</p>
</blockquote>
<p>this brings me back to having a (complex) extension on specimen. code: kind of specimen, e.g. normal specimen,  maternal, paternal and a reference to a Specimen Resource. The Extension is 0..* and then you can hear 40 samples :-)</p>



<a name="189065888"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189065888" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189065888">(Feb 25 2020 at 22:00)</a>:</h4>
<p>I've seen worse extensions.</p>



<a name="189065897"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189065897" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189065897">(Feb 25 2020 at 22:00)</a>:</h4>
<p>if you don't like it, don't use it. Just use method.</p>



<a name="189065954"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189065954" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189065954">(Feb 25 2020 at 22:01)</a>:</h4>
<p>But it enables an easier use than having 3 variants to describe a germline variant if you want to be precise</p>



<a name="189066316"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189066316" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189066316">(Feb 25 2020 at 22:06)</a>:</h4>
<p>I think specimen.note may be ok actually... "Arrived with patient" and "frozen" are examples--don't see how "germline reference" is any different</p>



<a name="189066396"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189066396" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189066396">(Feb 25 2020 at 22:07)</a>:</h4>
<p>So we would allow a variant found in different samples to be represented as a single observation?  That seems so easy to misinterpret on the receiver side.</p>



<a name="189066518"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189066518" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189066518">(Feb 25 2020 at 22:09)</a>:</h4>
<p>I don't think so, the method will have to be coded "tumor-normal-test" which involves 2 samples</p>



<a name="189066572"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189066572" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189066572">(Feb 25 2020 at 22:10)</a>:</h4>
<p>in reality we are currently using a single variant, pointing to the tumor specimen and having the source class germline.</p>



<a name="189066617"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189066617" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189066617">(Feb 25 2020 at 22:10)</a>:</h4>
<p>Also if the receiver doesn't care about the Extension, it will be germline Variant pointing to the tumor sample</p>



<a name="189066872"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189066872" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189066872">(Feb 25 2020 at 22:14)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189066316" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189066316">said</a>:</p>
<blockquote>
<p>I think specimen.note may be ok actually... "Arrived with patient" and "frozen" are examples--don't see how "germline reference" is any different</p>
</blockquote>
<p>i don't like having this information as a non coded note. A possible alternative would be another coded field in specimen, but there is none usable at the moment</p>



<a name="189066987"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189066987" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189066987">(Feb 25 2020 at 22:16)</a>:</h4>
<p>I don't think the extension really adds that much... We should just be explicit when we talk about source class. The use case asks to determine if follow-up testing is beneficial, not to capture every possible step of the lab's workflow</p>



<a name="189067062"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067062" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067062">(Feb 25 2020 at 22:17)</a>:</h4>
<p><span class="user-mention silent" data-user-id="191451">Patrick Werner</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189066572" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189066572">said</a>:</p>
<blockquote>
<p>in reality we are currently using a single variant, pointing to the tumor specimen and having the source class germline.</p>
</blockquote>
<p>So, if we create two new operations in the future: 'find-somatic-variants' and 'find-germline-variants' , would you expect this single variant to be returned for both?</p>



<a name="189067081"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067081" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067081">(Feb 25 2020 at 22:17)</a>:</h4>
<p>(which is still possible by blowing up that component with an extra Observation)</p>



<a name="189067383"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067383" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067383">(Feb 25 2020 at 22:21)</a>:</h4>
<p>Or even, even easier and scarier, if some did a query by component value of 'germline', this variant in the tumor would not be returned, right?</p>



<a name="189067455"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067455" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067455">(Feb 25 2020 at 22:22)</a>:</h4>
<p>I wouldn't think a query for somatic variants would go by specimen.type so this would show up as germline</p>



<a name="189067477"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067477" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067477">(Feb 25 2020 at 22:22)</a>:</h4>
<p>If the lab flagged it as germline, I think that is intended</p>



<a name="189067602"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067602" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067602">(Feb 25 2020 at 22:24)</a>:</h4>
<p>Oh, sorry, my second example was not right:</p>
<p>Or even, even easier and scarier, if someone did a query by component value of '<strong>somatic</strong>', this variant that was found in a tumor would not be returned, right?</p>
<p>Or, maybe I am not getting it <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="189067671"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067671" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067671">(Feb 25 2020 at 22:25)</a>:</h4>
<p>I think this latest version is right too, a "tumor sample" can have germline variants to my knowledge</p>



<a name="189067737"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067737" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067737">(Feb 25 2020 at 22:26)</a>:</h4>
<p>So, anything that queries by 'somatic' should return variants ONLY found in the tumor?</p>



<a name="189067747"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067747" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067747">(Feb 25 2020 at 22:26)</a>:</h4>
<p>Wait, are we confusing source class and variant origin??</p>



<a name="189067818"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189067818" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189067818">(Feb 25 2020 at 22:27)</a>:</h4>
<p>There are overlapping terms.</p>



<a name="189068236"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068236" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068236">(Feb 25 2020 at 22:32)</a>:</h4>
<p>I was talking about 'source class', but you were talking about 'Variant Inheritence', which SO calls variant origin?</p>



<a name="189068563"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068563" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068563">(Feb 25 2020 at 22:37)</a>:</h4>
<p>A tumor sample presents a variant that is found to be inherited</p>



<a name="189068648"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068648" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068648">(Feb 25 2020 at 22:38)</a>:</h4>
<p>source class=somatic, <br>
variant inheritance=maternal (eg)??</p>



<a name="189068666"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068666" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068666">(Feb 25 2020 at 22:38)</a>:</h4>
<p>Yes, we do have overlapping terms - so we need to include those in our examples/thinking we are doing.</p>



<a name="189068724"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068724" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068724">(Feb 25 2020 at 22:39)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189068648" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189068648">said</a>:</p>
<blockquote>
<p>source class=somatic, <br>
variant inheritance=maternal (eg)??</p>
</blockquote>
<p>or<br>
variant inheritance=germline_variant (SO:0001778) (if you don't know parental origin)</p>



<a name="189068844"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068844" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068844">(Feb 25 2020 at 22:41)</a>:</h4>
<p>I was hasty to claim everything depended on source class, we broke this up into 2 concepts right before publication for a reason!</p>



<a name="189068863"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068863" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068863">(Feb 25 2020 at 22:41)</a>:</h4>
<p>But yea, dang - potential areas of misuse between those two components.  urggg.  We need to AT LEAST make some good documentation on that one.  But, I say we let <span class="user-mention" data-user-id="192302">@May Terry</span> wow us with her write up and examples <span aria-label="slight smile" class="emoji emoji-1f642" role="img" title="slight smile">:slight_smile:</span></p>



<a name="189068987"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189068987" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189068987">(Feb 25 2020 at 22:43)</a>:</h4>
<p>Unknown Variant inheritance is also a good indicator for additional testing</p>



<a name="189072984"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189072984" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189072984">(Feb 25 2020 at 23:44)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> et al - I'll give it a whirl to present findings and options this Monday but hoping to rely on the team's genomics expertise (exponentially way beyond mine) to help guide it.  :-)</p>



<a name="189073700"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189073700" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189073700">(Feb 25 2020 at 23:55)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="192302">@May Terry</span> !!</p>



<a name="189095454"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189095454" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189095454">(Feb 26 2020 at 08:51)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189068844" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189068844">said</a>:</p>
<blockquote>
<p>I was hasty to claim everything depended on source class, we broke this up into 2 concepts right before publication for a reason!</p>
</blockquote>
<p>looking at the two ValueSets and concepts i get the feeling they are the same. If we would extend the Loinc AL with the missing terms from variant origin we could merge them into one. Or am i missing something here?</p>



<a name="189095578"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189095578" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189095578">(Feb 26 2020 at 08:53)</a>:</h4>
<p><span class="user-mention silent" data-user-id="191913">Kevin Power</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189067062" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189067062">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="191451">Patrick Werner</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189066572" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189066572">said</a>:</p>
<blockquote>
<p>in reality we are currently using a single variant, pointing to the tumor specimen and having the source class germline.</p>
</blockquote>
<p>So, if we create two new operations in the future: 'find-somatic-variants' and 'find-germline-variants' , would you expect this single variant to be returned for both?</p>
</blockquote>
<p>only for find-germline-variants</p>



<a name="189095650"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189095650" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189095650">(Feb 26 2020 at 08:54)</a>:</h4>
<p><span class="user-mention silent" data-user-id="191913">Kevin Power</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189067602" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189067602">said</a>:</p>
<blockquote>
<p>Or even, even easier and scarier, if someone did a query by component value of '<strong>somatic</strong>', this variant that was found in a tumor would not be returned, right?</p>
</blockquote>
<p>(if it was a germline mutation -&gt; found in normal and tumor tissue: yes you are right. And i think thats correct. <span class="user-mention" data-user-id="192302">@May Terry</span>  what do you think?</p>



<a name="189095813"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189095813" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189095813">(Feb 26 2020 at 08:57)</a>:</h4>
<p>Here is our example report: <br>
<a href="https://drive.google.com/open?id=1y4E5lzsxLJ644kHpR7fTYN-xoGgPQVmt" target="_blank" title="https://drive.google.com/open?id=1y4E5lzsxLJ644kHpR7fTYN-xoGgPQVmt">https://drive.google.com/open?id=1y4E5lzsxLJ644kHpR7fTYN-xoGgPQVmt</a></p>



<a name="189095843"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189095843" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189095843">(Feb 26 2020 at 08:57)</a>:</h4>
<p><a href="/user_uploads/10155/-FjvUJr23PlxrwuHd879KHdw/image.png" target="_blank" title="image.png">image.png</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/-FjvUJr23PlxrwuHd879KHdw/image.png" target="_blank" title="image.png"><img src="/user_uploads/10155/-FjvUJr23PlxrwuHd879KHdw/image.png"></a></div>



<a name="189095904"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189095904" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189095904">(Feb 26 2020 at 08:58)</a>:</h4>
<p>the first variant is a germline one.</p>



<a name="189111310"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189111310" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189111310">(Feb 26 2020 at 12:58)</a>:</h4>
<p>Regarding the two value sets, I was thinking it would be:<br>
Source Class- what was I testing<br>
Variant Inheritance- where did the variant originate</p>
<p>But not sure we need that level of distinction</p>



<a name="189113202"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189113202" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189113202">(Feb 26 2020 at 13:27)</a>:</h4>
<p>Source class is captured in specimen.type it seems... Ought to double check the DAM</p>



<a name="189114507"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189114507" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189114507">(Feb 26 2020 at 13:44)</a>:</h4>
<p>Specimen Identification is Scenario 1 in the DAM and states a lot of what we have seen in this thread. It mentions the two testing methods and outlines what variants should be marked as somatic either with normal testing or through lab's calculation</p>



<a name="189114590"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189114590" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189114590">(Feb 26 2020 at 13:45)</a>:</h4>
<p>I don't see a reason for us to have 2 places to call a variant somatic and think we should use variant inheritance specifically to cover the full logic table for germline inheritance only</p>



<a name="189114700"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189114700" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189114700">(Feb 26 2020 at 13:47)</a>:</h4>
<p>For the tumor/normal use case, source class should be appropriate</p>



<a name="189116325"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189116325" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> May Terry <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189116325">(Feb 26 2020 at 14:06)</a>:</h4>
<p>The oncologist who made the initial inquiry provided this sample report (fictitious patient and approved for distribution) for review by the team.  My example will probably leverage some of the example data noted here. <a href="/user_uploads/10155/-ltSSSyB_EcbV4rHkvqZGcyY/sample-tempus-ngs.pdf" target="_blank" title="sample-tempus-ngs.pdf">sample-tempus-ngs.pdf</a></p>



<a name="189118639"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189118639" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189118639">(Feb 26 2020 at 14:30)</a>:</h4>
<p><span class="user-mention silent" data-user-id="196553">Jamie Jones</span> <a href="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189114590" title="#narrow/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F/near/189114590">said</a>:</p>
<blockquote>
<p>I don't see a reason for us to have 2 places to call a variant somatic and think we should use variant inheritance specifically to cover the full logic table for germline inheritance only</p>
</blockquote>
<p>sounds great. and would remove the overlap</p>



<a name="189118901"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189118901" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189118901">(Feb 26 2020 at 14:33)</a>:</h4>
<p>i will also try to create an example of our current format for the monday call</p>



<a name="189121716"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189121716" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189121716">(Feb 26 2020 at 15:01)</a>:</h4>
<p>Thanks both!</p>
<p>It seems the remaining steps here are to:</p>
<ol>
<li>confirm May's 2 use cases are represented in the next version of the DAM</li>
<li>log a tracker to provide guidance on source class terms "somatic" vs "likely somatic" surrounding this use case (or whatever solution we decide)</li>
<li>log another to reconcile the concepts <a href="https://loinc.org/LL378-1/" target="_blank" title="https://loinc.org/LL378-1/">https://loinc.org/LL378-1/</a> and <a href="http://www.sequenceontology.org/browser/release_2.5/term/SO:0001762" target="_blank" title="http://www.sequenceontology.org/browser/release_2.5/term/SO:0001762">http://www.sequenceontology.org/browser/release_2.5/term/SO:0001762</a>.</li>
<li>get these example reports FHIR-ized and in the IG</li>
</ol>



<a name="189514109"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189514109" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189514109">(Mar 02 2020 at 17:27)</a>:</h4>
<p>One thing hit me just now - Do we have a JIRA (tracker) for this?  If we are going to spend much time on this, it seems we need one.</p>



<a name="189514507"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189514507" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189514507">(Mar 02 2020 at 17:30)</a>:</h4>
<p>None yet, I outlined 2 to log here but could consolidate and wanted to give it a bit of air time to gauge likely scope of changes.</p>



<a name="189514648"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/189514648" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#189514648">(Mar 02 2020 at 17:31)</a>:</h4>
<p>Oh geesh, sorry <span class="user-mention" data-user-id="196553">@Jamie Jones</span> -- I missed your points 2 and 3 above <span aria-label="frown" class="emoji emoji-1f641" role="img" title="frown">:frown:</span></p>



<a name="190087261"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/190087261" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#190087261">(Mar 09 2020 at 16:17)</a>:</h4>
<p>asked O&amp;O on their opinion. <a href="#narrow/stream/179256-Orders-and.20Observation.20WG/topic/tumor-normal.20testing" title="#narrow/stream/179256-Orders-and.20Observation.20WG/topic/tumor-normal.20testing">https://chat.fhir.org/#narrow/stream/179256-Orders-and.20Observation.20WG/topic/tumor-normal.20testing</a></p>



<a name="190654595"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/190654595" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#190654595">(Mar 15 2020 at 16:31)</a>:</h4>
<p>So, a question for labs and oncology is: Is the TUMOR the same patient as the patient? The specimen identity should be attached to the variant data (we're all agreeing there),  but the biggest problem is that we also identify the TUMOR as part of the patient. With a microorganism it is easy to determine who is who. With TRIO analysis, the clinical lab community is in a similar state - consider that the parents genetic sequence may only be recorded in the probands electronic record. But it would be obvious that mom and dad are not the same person as the proband - a smart system would give mom and dad unique identifiers (i.e. unique patient records in the system - yep, even if they are not ever going to be 'legally' treated or seen in the system) and use a family linkage to the proband....find a system that does that!</p>



<a name="190654636"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/190654636" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#190654636">(Mar 15 2020 at 16:32)</a>:</h4>
<p>that will be the group to talk too and get input from</p>



<a name="190654679"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/190654679" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#190654679">(Mar 15 2020 at 16:33)</a>:</h4>
<p>most other places are using the data ephemerally, leaving disambiguation to non-real-time data analysis efforts that are typically research related.</p>



<a name="190654860"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/190654860" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#190654860">(Mar 15 2020 at 16:36)</a>:</h4>
<p>How's it work in Germany <span class="user-mention" data-user-id="191451">@Patrick Werner</span>?  where I understand employers have been cited for using genetic data to disqualify applicants. what is done with trio data where one participant does not want their genetic data exposed? is it protected? obfuscated?</p>



<a name="190739748"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/190739748" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#190739748">(Mar 16 2020 at 15:39)</a>:</h4>
<p>great thoughts. I will finalize my example this week. So far my thougts are:</p>
<ul>
<li>Observation.specimen always points to the specimen which was in focus (in tumor-normal testing this would be the tumor). All other specimens can be linked through an complex extension which includes:</li>
<li>type of specimen relationship (coded): normal-specmine|normal-specimen-maternal|normal-specimen-paternal </li>
<li>reference to the Specimen</li>
</ul>



<a name="190740132"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/190740132" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#190740132">(Mar 16 2020 at 15:42)</a>:</h4>
<p>So we don't need mum &amp; dad as instances, but we can describe which specimen played which role in the test. If we have Instances of mum&amp;dad we can point to them in Specimen.subject.</p>



<a name="191640717"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/191640717" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#191640717">(Mar 24 2020 at 16:54)</a>:</h4>
<p>Hey <span class="user-mention" data-user-id="192302">@May Terry</span> here is an example on how the multiple specimen extension could look like: <a href="https://gist.github.com/patrick-werner/5d715f00c4c0e096be35debcb0962267#file-first-demo" target="_blank" title="https://gist.github.com/patrick-werner/5d715f00c4c0e096be35debcb0962267#file-first-demo">https://gist.github.com/patrick-werner/5d715f00c4c0e096be35debcb0962267#file-first-demo</a></p>



<a name="191640787"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/191640787" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#191640787">(Mar 24 2020 at 16:55)</a>:</h4>
<p>not valid FHIR, tried to condense it to the relevant parts as much as possible. What do you think?</p>



<a name="191641067"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/191641067" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#191641067">(Mar 24 2020 at 16:57)</a>:</h4>
<p>This Extension then has to be accompanied by exact guidance on how to use it; or even better guidance on tumor-normal testing. This guidance also should show how to the same thing with 3 Observations an no Extension: Variants(tumor/normal) and a derived Observation for somatic/germline</p>



<a name="192381683"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192381683" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192381683">(Mar 31 2020 at 12:30)</a>:</h4>
<p>You've got the option of using specimen.type = 'tumor -normal' when appropriate</p>



<a name="192382415"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192382415" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192382415">(Mar 31 2020 at 12:37)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span>  sorry to ask for this. but can you use a more specific example of the statement you are trying to make with the multi-specimen? <span class="user-mention" data-user-id="192302">@May Terry</span>  there is specimen.type which is extensible and can be given the type tumor-normal. However, I understand the goal is to link together somehow the tumor-normal results and tumor results.  Why would it not be sufficient for the 'blood' specimen in the example to appear as a specimen in the diagnostic report? In the example are there any observations made specifically on the blood sample?</p>



<a name="192383133"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192383133" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192383133">(Mar 31 2020 at 12:42)</a>:</h4>
<p>I also thought about using the specimen type already..</p>



<a name="192383332"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192383332" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192383332">(Mar 31 2020 at 12:44)</a>:</h4>
<p>argument against: if you loose the specimen you loose the type info. On the other hand: you don't need the extension if you don't have specimens.</p>



<a name="192383515"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192383515" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192383515">(Mar 31 2020 at 12:45)</a>:</h4>
<p>But: speciment.type is also 0..1. So if i want to use this to code: blood, tissue, liquidBiopsy, .. i can't additionally code: normal tissue, or paternal blood, etc.</p>



<a name="192383688"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192383688" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192383688">(Mar 31 2020 at 12:47)</a>:</h4>
<p>Specimen also has a 'note' field for those who don't want to employ an extension</p>



<a name="192383864"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192383864" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192383864">(Mar 31 2020 at 12:48)</a>:</h4>
<p>Yes, but note isn't coded. Or did i understood your comment wrong?</p>



<a name="192384167"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192384167" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192384167">(Mar 31 2020 at 12:51)</a>:</h4>
<p>I see attaching the additional specimens to the report with decent notes as a fine solution for these use cases. That said, I also like the extension for systems that are expecting to do a lot of these tests</p>



<a name="192384835"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192384835" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192384835">(Mar 31 2020 at 12:56)</a>:</h4>
<p><span class="user-mention" data-user-id="192123">@Bret H</span>  I can supply a potential use case where the extension may be helpful--biobanking samples beyond a deidentification workflow</p>



<a name="192385216"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385216" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385216">(Mar 31 2020 at 12:59)</a>:</h4>
<p>ah now i got your comment. Right, systems that don't need that specific link can ignore the extension and only Link it through DR with some notes. If you need/want to be specific you just use the extension. Nobody will be forced to use-it or incorporate it into their use-case specific ig.</p>



<a name="192385219"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385219" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385219">(Mar 31 2020 at 12:59)</a>:</h4>
<p>notes can work but if sample tracking is key to the system they will want something more structured (80/20 but that 20 will want an extension and we can centrally provide the definition+url)</p>



<a name="192385228"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385228" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385228">(Mar 31 2020 at 13:00)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span>  not sure. Why would the new  deidentified  sample not have a new identification.</p>



<a name="192385381"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385381" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385381">(Mar 31 2020 at 13:01)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span>  why are you losing specimen?</p>



<a name="192385614"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385614" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385614">(Mar 31 2020 at 13:03)</a>:</h4>
<p><span class="user-mention" data-user-id="195901">@patrick</span>  <span class="user-mention" data-user-id="196553">@Jamie Jones</span>  have you reviewed <a href="https://www.hl7.org/fhir/family-member-history-genetics.html" title="https://www.hl7.org/fhir/family-member-history-genetics.html">https://www.hl7.org/fhir/family-member-history-genetics.html</a> <span class="user-mention" data-user-id="196282">@May Terry</span>  when considering trios?</p>



<a name="192385801"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385801" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385801">(Mar 31 2020 at 13:04)</a>:</h4>
<p>isn't this pushing complex extensions from resource to resource?</p>



<a name="192385904"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385904" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385904">(Mar 31 2020 at 13:05)</a>:</h4>
<p>or perhaps that's what we're proposing now haha. Certainly multiple structures are valid for communicating this info</p>



<a name="192385929"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385929" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385929">(Mar 31 2020 at 13:05)</a>:</h4>
<p>For trio analysis it was handled via family history. Gil has slides circa 2017. But let's leave off Trio analysis here</p>



<a name="192385983"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385983" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385983">(Mar 31 2020 at 13:06)</a>:</h4>
<p>Trio analysis is my use case...</p>



<a name="192385989"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385989" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385989">(Mar 31 2020 at 13:06)</a>:</h4>
<p>the topic is tumor-normal sequencing tests.</p>



<a name="192385998"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192385998" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192385998">(Mar 31 2020 at 13:06)</a>:</h4>
<p>just saying</p>



<a name="192386026"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Representing%20tumor-normal%20sequencing%20tests%3F/near/192386026" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Representing.20tumor-normal.20sequencing.20tests.3F.html#192386026">(Mar 31 2020 at 13:06)</a>:</h4>
<p>and  GIl and Grant's past work should be examined.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
