---
layout: page
title: "FHIR Chat  · Sequence / Observation comparison · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html">Sequence / Observation comparison</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="153982829"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153982829" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153982829">(Aug 06 2018 at 22:30)</a>:</h4>
<p>CG'ers - Since it is a constant topic now days, and we  have the R4 dates looming  to hit with any changes to the Sequence resource - I have put together a comparison document.  We can start coming up with some alternatives and discussions, but wanted to see if this format made sense.  Of course, welcome any and all feedback:<br>
<a href="https://docs.google.com/spreadsheets/d/1z4DodoLYawW-s0jbFKQg_xpwir8rEORkNjMfemvqxE0/edit#gid=0" target="_blank" title="https://docs.google.com/spreadsheets/d/1z4DodoLYawW-s0jbFKQg_xpwir8rEORkNjMfemvqxE0/edit#gid=0">https://docs.google.com/spreadsheets/d/1z4DodoLYawW-s0jbFKQg_xpwir8rEORkNjMfemvqxE0/edit#gid=0</a></p>



<a name="153995779"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153995779" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Gideon Giacomelli <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153995779">(Sep 11 2018 at 16:07)</a>:</h4>
<p>we just discussed this topic in the weekly call and there are quite some open issues how to proceed and come to a common understanding. For me the purpose of the sequence resource is not clearly understood. Should it represent sequence data or also mutations which is also covered quite well by observations right now. Anyway it looks like this will be a topic in baltimore to be discussed</p>



<a name="153995799"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153995799" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153995799">(Sep 11 2018 at 17:30)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="196701">@Gideon Giacomelli</span>  - Sorry I missed the call today, but good to hear some conversation was started.  I think of Sequence like this: It represents the Sequence data, but it has a variety of ways that it can represent the sequence such that a consumer should be able to create it.  To answer your question about "mutations" - given a reference sequence, and list of a variants, someone should be able to take that reference, apply the variants, and come up with the sequence.  So, the list of 'variants' on the Sequence resource is really a short cut to sending the full sequence.</p>
<p>I am fairly sure that is right, but welcome <span class="user-mention" data-user-id="191719">@Gil Alterovitz</span> or <span class="user-mention" data-user-id="196553">@James Jones</span>  (or I suppose anyone listening on <a class="stream" data-stream-id="179197" href="/#narrow/stream/179197-genomics">#genomics</a>) to weigh in.</p>



<a name="153995824"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153995824" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153995824">(Sep 11 2018 at 18:06)</a>:</h4>
<p>I totally agree, as a consumer of a Genetic Diagnostic Report (e.g. Practicioner in a Hospital) i don't care about the Sequence, i only want to know the variants compared to the reference sequence. These will be described variants.<br>
Sequences would be needed in Bioinformatics or inside a Lab i suppose.</p>



<a name="153995853"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153995853" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153995853">(Sep 11 2018 at 19:00)</a>:</h4>
<p>That is true, but we do need to consider other use cases where the sequence data is important.</p>



<a name="153995881"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153995881" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153995881">(Sep 11 2018 at 19:44)</a>:</h4>
<p>i know, i just wanted to explain the clinical use case. I‘m super interested in other usecases which have the need for the actual sequence itself.</p>



<a name="153997073"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997073" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997073">(Sep 14 2018 at 17:04)</a>:</h4>
<p>Kevin is absolutely right here, sorry for the delayed response. The variant information on sequence was included as an alternate method to describe the sequence. Rather than building it from scratch you could say "It's <em>this</em> reference/other sequence, but with <em>these</em> variants". I do think this should be made clearer on the resource documentation itself, particularly now that there are dedicated variant profiles, etc.</p>



<a name="153997644"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997644" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997644">(Sep 18 2018 at 16:11)</a>:</h4>
<p>When considering the desire to be able to encapsulate and send VCF file data to certain places in FHIR, I wonder if we should be looking to extend the sequence resource here rather than restrict it further and use the heavier observations and clunky attachments from other specs.</p>
<p>That may be totally out of scope for now, but we have the opportunity to keep all of the "specialized" genomics data we want to send in the custom resource in a concise way, rather than in bulky observations to users who won't need to look at it.</p>



<a name="153997652"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997652" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997652">(Sep 18 2018 at 16:40)</a>:</h4>
<p>I would like to propose we separate those two things - As we talked, I could see value in having a home for sending the VCF file basically as something simple like an attachment.  Like <span class="user-mention" data-user-id="192123">@Bret H</span> stated, the VCF is such a "common currency" in this field that I can see value in us supporting it in a very basic (and optional) way.</p>
<p>While the conversation started as "could a VCF replace Sequence?", I feel like we moved away from that being a possibility.  There are still things that Sequence can do today that would not be covered by a VCF.  So, we still need to grapple with the Sequence / Observation overlap discussion.</p>
<p>Is it worth our time to evaluate what that "bulky observation" approach might look like? Can we say for sure it is bulkier?  I will have to admit I have let myself believe it is, for what seem like good reasons.  But perhaps it is worth a determination of how much bulkier is it? And "bulkiness" wouldn't be the only criteria we should consider.</p>



<a name="153997655"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997655" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997655">(Sep 18 2018 at 16:51)</a>:</h4>
<p>Observation is absolutely bulkier and less elegant than Sequence.  If that were the sole consideration, Observation profiles would never have been on the table.  The issue is our ability to propagate data to (and more importantly through) systems that are not going to be designed with genetics in mind.  If you imagine data coming from genomic specific software and then propagating information into a lab's generic reporting software, then sent as a diagnostic report to an EHR, then uploaded into a patient's Apple phone, then downloaded into another EHR where it'll be looked at by someone like Bob Dollin, then made available to a genetics-aware decision support system, what are the chances the decision support system is going to have the data it needs in each of the two scenarios: all data is represented using (bulkier) Observations or all data is represented using the more concise Sequence?  </p>
<p>I think the answer is clear that data propagation is much more likely to be successful using Observations.  We <em>can</em> choose to go with both approaches, where "key" information is made available through Observation and more detailed information is only shared using Sequence.  The question then is "What information are Bob and the CDS system ok with not having access to?"</p>



<a name="153997671"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997671" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997671">(Sep 18 2018 at 17:19)</a>:</h4>
<blockquote>
<p>The question then is "What information are Bob and the CDS system ok with not having access to?"</p>
</blockquote>
<p>Or, is the appropriate framing - "What information are Bob and the CDS system ok with not have access to <em>via Observation and take extra steps to get from Sequence?</em>"</p>
<p>I think it is clear that to get <em>and utilize</em> this data (even our Observation profiles) will take a genomics aware system.</p>



<a name="153997698"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997698" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997698">(Sep 18 2018 at 17:41)</a>:</h4>
<p>I am confused as to where the breakdown in propagating data stored in a sequence resource happens in the Lloyd's example. We seem to be assuming that all of the multiple (arbitrarily deeply) nested observation profiles hanging off a diagnosticReport will make it through okay, but not our custom FHIR resources they reference? What about references to Device profiles and other lower-maturity resources?</p>



<a name="153997702"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997702" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997702">(Sep 18 2018 at 17:54)</a>:</h4>
<p>Device will be more common than Sequence as many Observations are specifying the used Device.<br>
The DiagnosticReport and the Observations can be handled by most of the systems, the referenced Sequence wont make it through.<br>
The question to me is if information from Sequence will be important at this level (HIS, iPhone, etc.) or would they work with only describedVariants and other Observation profiles anyway.</p>



<a name="153997703"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997703" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997703">(Sep 18 2018 at 18:03)</a>:</h4>
<p>I think in general, the "propagation" of data is at risk in the short term until other resources become normative or at least more mature.  Since Observation is close to normative, it has a head start.  And I agree with <span class="user-mention" data-user-id="191451">@Patrick Werner</span> - things like Device are more likely to be more broadly implemented.</p>
<p>I think the comment by <span class="user-mention" data-user-id="196553">@James Jones</span>  about "arbitrarily deep" is really asking - Will systems support the relationships + all sorts of components in their "standard" implementation of Observation?  If that answer to that as yes, then that system would meet the "pass through" test.  It could at least store and maintain everything it is given, but likely not make much use of that data.</p>



<a name="153997714"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997714" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997714">(Sep 18 2018 at 18:42)</a>:</h4>
<p>Displaying and navigating through them in a pleasant way is certainly going to benefit from further support, but I'm very surprised we are worried about any FHIR resources failing the "pass through" test, with their inherited base structure and standardized data types. </p>
<p>Either way, being able to pass the sequence/quality informatics data downstream for further analysis by researchers and regulators seems very valuable, even if it won't all be directly viewed in the clinic or by the patient.</p>



<a name="153997715"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997715" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997715">(Sep 18 2018 at 18:56)</a>:</h4>
<blockquote>
<p>Displaying and navigating through them in a pleasant way is certainly going to benefit from further support, but I'm very surprised we are worried about any FHIR resources failing the "pass through" test, with their inherited base structure and standardized data types.</p>
</blockquote>
<p>I might have misunderstood your earlier statement.  In short, I don't know that our group needs to worry about this level of detail.  We should base any of our thinking on the path to normative for the resources we are using and considering.</p>
<blockquote>
<p>Either way, being able to pass the sequence/quality informatics data downstream for further analysis by researchers and regulators seems very valuable, even if it won't all be directly viewed in the clinic or by the patient.</p>
</blockquote>
<p>There is value, but I still wonder where we draw the line with FHIR?  What level of detail do we model in FHIR, versus when should something like GA4GH specs take over?  I honestly don't know, but that boundary doesn't feel completely clear to me (yet).</p>



<a name="153997716"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997716" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997716">(Sep 18 2018 at 19:08)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> Your interpretation of what I said was certainly a follow-up I meant to imply, and I agree on both points--thanks for the discussion!</p>



<a name="153997717"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997717" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997717">(Sep 18 2018 at 19:10)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span>  I'm glad to hear you say that ('boundary doesn't feel completely clear to me'), because I feel the same way, and I feel that the boundary is likely to change as we learn to leverage more detailed genomic data. I'm beginning to think that we should also be considering a set of FHIR GACS Operations, because knowing what we need returned to us from an GACS can be thought of as requirements for Sequence resource or our Observation profiles.</p>



<a name="153997737"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997737" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997737">(Sep 18 2018 at 19:36)</a>:</h4>
<blockquote>
<p>I'm beginning to think that we should also be considering a set of FHIR GACS Operations, because knowing what we need returned to us from an GACS can be thought of as requirements for Sequence resource or our Observation profiles</p>
</blockquote>
<p>I would think a GACS would more likely want to implement something from GA4GH or other relevant spec, not FHIR?  I would expect them to do something like <a href="https://samtools.github.io/hts-specs/htsget.html" target="_blank" title="https://samtools.github.io/hts-specs/htsget.html">htsget</a>  or other things found in the <a href="https://www.ga4gh.org/ga4ghtoolkit/genomicdatatoolkit/" target="_blank" title="https://www.ga4gh.org/ga4ghtoolkit/genomicdatatoolkit/">GA4GH Genomic Data Toolkit</a>.  I am really concerned about FHIR taking on that level of detail.<br>
Perhaps some of our CG / GA4GH folks could weigh in?  <span class="user-mention" data-user-id="192198">@Bob Freimuth</span>  <span class="user-mention" data-user-id="196622">@Larry Babb</span></p>



<a name="153997770"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997770" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997770">(Sep 18 2018 at 20:12)</a>:</h4>
<p>That's a good question, and it may be that a clinical decision support application will need to support both FHIR and htsget. From a GACS, I anticipate (at least) needing information on variants, haplotypes, genotypes, and regions studied. I'm finding that it's a bit of a learning curve, to figure out how best to extract that information from BAM/VCF, whereas FHIR GACS Operators might be more intuitive and consumable by the clinical informatics community.</p>



<a name="153997777"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997777" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997777">(Sep 18 2018 at 20:20)</a>:</h4>
<p>I don't know that "extra steps" will be an option.  The chances of you having access to the original lab or system where the sequence data existed if you're the decision-making system or the secondary EHR is pretty low.  I know in theory you could have the URL and query it RESTfully - but for that to work, you'd also have to have authorization on the original server, and that's typically hard to come by.</p>



<a name="153997778"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997778" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997778">(Sep 18 2018 at 20:20)</a>:</h4>
<p>My expectation is that if the Sequence doesn't propagate through a system, then nothing downstream will have any mechanism to access that discrete data.</p>



<a name="153997780"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997780" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997780">(Sep 18 2018 at 20:23)</a>:</h4>
<p>I agree that "normative" is one of the considerations for adoption.  But even if Sequence were going normative this round too, I think its adoption would suffer because it's such a specialized space.  <em>Everyone</em> needs Observations - and the base structure is pretty easy to manage and maps well to persistence structures almost every system has.  Sequence is a lot tougher for folks to wrap their heads around.  And few, if any, systems will have dedicated persistence layers that know how to manage that sort of structure.</p>



<a name="153997817"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997817" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997817">(Sep 18 2018 at 21:29)</a>:</h4>
<blockquote>
<p>whereas FHIR GACS Operators might be more intuitive and consumable by the clinical informatics community.</p>
</blockquote>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span>  - They will only be intuitive and consumable if we have the time to put into them to make them that way.</p>



<a name="153997822"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997822" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997822">(Sep 18 2018 at 21:46)</a>:</h4>
<blockquote>
<p><em>Everyone</em> needs Observations - and the base structure is pretty easy to manage and maps well to persistence structures almost every system has.  Sequence is a lot tougher for folks to wrap their heads around.  And few, if any, systems will have dedicated persistence layers that know how to manage that sort of structure.</p>
</blockquote>
<p><span class="user-mention" data-user-id="191320">@Lloyd McKenzie</span>  - While I agree with the premise that everyone needs Observations, I won't say Sequence is a lot tougher for folks to wrap their heads around.  Our current IG is not for the faint of heart, and it will take a great deal of time and research to really understand and appropriately implement.  It could be argued that if we took an approach of building a first class resources for our data, it could be significantly easier for people to understand.  That of course comes at the cost of needing every system to, from scratch, implement resources  that are different.</p>
<p>It still feels like we can find something in between.  I still don't have a feel for where that balance is.</p>



<a name="153997825"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997825" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997825">(Sep 18 2018 at 21:50)</a>:</h4>
<p>The IG only needs to be understood by those producing the content and those doing the decision support.  To all of the intervening systems, it's just a bunch of Observations like any other.  They don't necessarily need to understand the details.  And a generic Observation rendering tool should still get it right-enough for anyone who just wants to look at it.</p>



<a name="153997826"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997826" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997826">(Sep 18 2018 at 21:50)</a>:</h4>
<p>BTW - A <a href="#narrow/stream/4-implementers/subject/Observation.20value.20attachment" title="#narrow/stream/4-implementers/subject/Observation.20value.20attachment">conversation</a> that is related to some of our discussion for those that want to follow along.</p>



<a name="153997827"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997827" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997827">(Sep 18 2018 at 21:51)</a>:</h4>
<p>Sequence is tougher because it requires domain-specific knowledge.  Observation is just a name-value pair (or a collection of them) which tends to be easier for people to "get".</p>



<a name="153997828"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997828" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997828">(Sep 18 2018 at 21:52)</a>:</h4>
<p>Also, an investment in Observation can meet lots of use-cases.  An investment in Sequence only makes sense if you're doing lots of genetic stuff.</p>



<a name="153997832"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997832" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997832">(Sep 18 2018 at 21:53)</a>:</h4>
<blockquote>
<p>And a generic Observation rendering tool should still get it right-enough for anyone who just wants to look at it.</p>
</blockquote>
<p>This probably should be the case now that I read this, but I am not sure I agree with this (specifically the "right-enough" comment).   I would have to think about how a generic viewing tool looking at our observations would work.</p>



<a name="153997845"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997845" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997845">(Sep 18 2018 at 22:18)</a>:</h4>
<p>If our IG requires that a lab who wants to share <em>all</em> variants as Observations instances (WES ~ 4-5k, WGS ~ 4-5m) - I see implementors saying "nope, sorry, my underlying data storage for observations was never intended for that" - also receiving docs may say "nope, sorry, my observation viewer will blow up" and might just go back to "where is my PDF please?"</p>
<p>I will admit it - I say that as an implementor whose underlying "observation infrastructure" won't handle that volume.  Can it handle small(ish) subsets of that?  Sure, with some tweaks, it would be fine.  However, I do <em>not</em> feel using Observations to deliver 4k (or 4m) VUSs will ever work.</p>
<p>Hence why I go back to framing like this:<br>
1) Observations as used for genomic data that is clinically (actionable/relevant/important/???)  <br>
2) The CG workgroup should not define standards for bioinformatics in FHIR (or any product for that matter)<br>
3) Sequence (or something else) (perhaps in combination of an optional VCF as an attachment) can express something in-between</p>
<p>The lines between those groupings are still too fuzzy to me, but should ask - Does this group agree / disagree with those statements?</p>



<a name="153997849"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997849" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997849">(Sep 18 2018 at 22:24)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> I think I like your heuristics, but can you expand on number 2? I do think there are opportunities for GA4GH and HL7 CG to work together,  to define, say, a translation from (relevant portions of) SAM/VCF into FHIR objects.</p>



<a name="153997850"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997850" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997850">(Sep 18 2018 at 22:45)</a>:</h4>
<p>I don't feel our group should be in the business the standards like BAM, FASTQ, htsget, BED, etc... anything that already has a home somewhere else.  Could we take a page from Imaging and learn how <a href="http://build.fhir.org/imagingstudy.html" target="_blank" title="http://build.fhir.org/imagingstudy.html">ImagingStudy</a> wraps around DICOM?  Does Sequence ~= ImagingStudy, and DICOM ~= bioinformatics standards?  Maybe, I think we even mention that in previous discussions.  If that is what you mean <span class="user-mention" data-user-id="193751">@Bob Dolin</span> then perhaps there is something there.</p>
<p>I just want us to be careful that we don't force FHIR into something where it is not needed.</p>



<a name="153997934"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153997934" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153997934">(Sep 19 2018 at 02:53)</a>:</h4>
<p>I agree with #1 and #2.  I'm not clear on the value of #3.  What is the "in-between" use-case?  Who needs that data?  And how is it going to propagate?</p>



<a name="153998022"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998022" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998022">(Sep 19 2018 at 08:29)</a>:</h4>
<p>I agree with #1 and #2. I also see the idea behind #3: capturing metadata of  raw data, similar to Imaging Study. The actual raw-data could then be attached as an Attachment.</p>



<a name="153998078"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998078" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998078">(Sep 19 2018 at 13:30)</a>:</h4>
<p>What metadata is worth capturing that we're fine not propagating?</p>



<a name="153998095"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998095" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998095">(Sep 19 2018 at 13:52)</a>:</h4>
<p>We want to propagate all relevant metadata. I meant that i can see the need of sending Sequence Metadata and embed/link the actual sequence File.</p>



<a name="153998096"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998096" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998096">(Sep 19 2018 at 13:53)</a>:</h4>
<p>As a profiled observation or as a sequence.</p>



<a name="153998097"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998097" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998097">(Sep 19 2018 at 13:53)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span> - We should be careful when considering "raw-data" as an Attachment.  I am fairly OK with the VCF, but assuming you mean things like BAM, FASTQ, I am not OK with that.  I like the approach with Sequence.repository and referring to that raw data, but I don't think we want to use a FHIR Attachment.</p>
<p><span class="user-mention" data-user-id="191320">@Lloyd McKenzie</span> - To be clear, by "not propagating", you mean "not propagating through a system that ONLY implements Observation, DiagnosticReport, RiskAssessment, Device, etc ..." correct?  If so, I don't think that is a fair assessment.  If we held everything to that rule, no new resource would ever be created would it?</p>
<p>Let's start building a list of "in-between" uses cases.  I can provide two:</p>
<p>1) <span class="user-mention" data-user-id="193751">@Bob Dolin</span>'s notion of "Region Coverage and Quality"</p>
<p>2) Delivering all detected variants</p>



<a name="153998098"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998098" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998098">(Sep 19 2018 at 13:57)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> yes i meant VCF Files, not FASTQ etc</p>



<a name="153998099"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998099" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998099">(Sep 19 2018 at 13:59)</a>:</h4>
<p>Oy, and a 3rd "in-between" example (I say Oy because it is the most obvious and the real intention of Sequence today):</p>
<p>3) Represent the actual sequence (ACTG....) that was observed</p>



<a name="153998101"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998101" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> John Moehrke <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998101">(Sep 19 2018 at 14:14)</a>:</h4>
<p>Where an existing file format is desired to use, then use DocumentReference. It can provide the metadata to any arbitrary file that has a mime-type. This is used for some proprietary formats such as WORD. It is also used for generic forms like non-FHIR xml schema, or even CSV.</p>



<a name="153998102"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998102" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> John Moehrke <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998102">(Sep 19 2018 at 14:17)</a>:</h4>
<p>The ImagingStudy resource is a very thin metadata for the deeper knowledge that can be probed and retrieved using DICOM, where the http RESTful flavor of DICOM is prefered. The structure of ImagingStudy is the basic structure of DICOM. Study-&gt;Series-&gt;Instance is a progressively deeper structure. Where all actual data lives at the instance level, but is structured within series which are structured within a study. There is also a degenerate form that just lists the key images for a purpose (like a specific Observation).</p>



<a name="153998111"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998111" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998111">(Sep 19 2018 at 14:34)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> Resources reflect the data elements and data structures systems already deal with and support.  It's certainly true that the existence of resources might entice some system developers to enhance the capabilities of their systems to support a resource.  But the typical pattern is that systems simply expose the data they already have and manage using the resource appropriate to that type of data.  As a result, I expect we'll only see most systems surfacing support for the Sequence (and RiskAssessment and other more essoteric resources) if they already have a specific interest in and support for those concepts as distinct notions in their systems.  Introducing a net new resource is considerably more work - and, at in most cases, will at best happen after the low-hanging fruit of "already supported" concepts has been addressed and will be subject to a financial business case assessment.</p>



<a name="153998115"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998115" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998115">(Sep 19 2018 at 14:36)</a>:</h4>
<p>And, in the worst case, implementers see Sequence, decide against &amp; rebuilt Sequence as a profile on Observation.</p>



<a name="153998116"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998116" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998116">(Sep 19 2018 at 14:37)</a>:</h4>
<p>Now, it may be that the demand for genomics data will be sufficiently high that applications across the whole propagation pathway will prioritize introducing support for Sequence and we'll see wide support for it over a period of a couple of years rather than 10-15+.  Our decision should be informed by a realistic assessment of what labs, EHRs and organizations like Apple are committed to doing in the near-ish term.</p>



<a name="153998117"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998117" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998117">(Sep 19 2018 at 14:37)</a>:</h4>
<p>The worst case is that implementers throw away Sequence data.</p>



<a name="153998118"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998118" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998118">(Sep 19 2018 at 14:38)</a>:</h4>
<p>Converting Sequence to Observation is a much more positive outcome - but is only a likely outcome if we define exactly what that conversion should look like and - ideally - provide tools for doing so.</p>



<a name="153998120"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998120" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> John Moehrke <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998120">(Sep 19 2018 at 14:40)</a>:</h4>
<p>There is just as likely revolt against putting the sequence within Observation as unwelcome bloat... All of these are good reasons why we go through "Standard for Trial Use" to see what works vs what we think might work.</p>



<a name="153998122"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998122" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> John Moehrke <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998122">(Sep 19 2018 at 14:41)</a>:</h4>
<p>so in STU it is good to have alternatives for the market to "Trial Use"</p>



<a name="153998126"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998126" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998126">(Sep 19 2018 at 14:44)</a>:</h4>
<p>What if we do both? Sequence Resource including Mappings to a profiled Observation? Too much work?</p>



<a name="153998147"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998147" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998147">(Sep 19 2018 at 14:55)</a>:</h4>
<p>A full mapping might be one path forward - and perhaps a standard operation that can handle the converstion.  That then gives implementers the choice of how to access and share the data - and it gives us a chance to evaluate what implementers are actually willing to do.</p>



<a name="153998152"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998152" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998152">(Sep 19 2018 at 14:59)</a>:</h4>
<p>I do think it would be interesting to at least sketch out what a sequence profile of Observation would look like.  Perhaps we could do a session on that at the WGM?</p>



<a name="153998162"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998162" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998162">(Sep 19 2018 at 15:10)</a>:</h4>
<p>As a bit of an aside, from what I understand about htsget, it doesn't simply return a VCF, but rather, just the relevant portion of VCF (or SAM, etc) - so, a much smaller package. It may be worth discussing how we envision, say, a clinical decision support application interacting with a GACS, and how it will leverage FHIR and/or htsget.</p>



<a name="153998165"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998165" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998165">(Sep 19 2018 at 15:15)</a>:</h4>
<blockquote>
<p>I do think it would be interesting to at least sketch out what a sequence profile of Observation would look like.  Perhaps we could do a session on that at the WGM?</p>
</blockquote>
<p>Genomics sessions at the WGM is mo-we, so thursday at the wgm would be an option</p>



<a name="153998166"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998166" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998166">(Sep 19 2018 at 15:18)</a>:</h4>
<p>I am leaving Wed evening, so Thurs would leave me out.  I would be OK taking one of our current "FHIR Reconciliation" blocks Tuesday or Wednesday and discussing.  Perhaps we could even have a breakout during the Connectathon to draft something, then review with the broader group Tues or Wed?</p>



<a name="153998177"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998177" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998177">(Sep 19 2018 at 15:25)</a>:</h4>
<p>sounds great</p>



<a name="153998197"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998197" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Freimuth <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998197">(Sep 19 2018 at 15:48)</a>:</h4>
<blockquote>
<p>I would think a GACS would more likely want to implement something from GA4GH or other relevant spec, not FHIR?  I would expect them to do something like <a href="https://samtools.github.io/hts-specs/htsget.html" target="_blank" title="https://samtools.github.io/hts-specs/htsget.html">htsget</a>  or other things found in the <a href="https://www.ga4gh.org/ga4ghtoolkit/genomicdatatoolkit/" target="_blank" title="https://www.ga4gh.org/ga4ghtoolkit/genomicdatatoolkit/">GA4GH Genomic Data Toolkit</a>.  I am really concerned about FHIR taking on that level of detail.</p>
</blockquote>
<p>This is a scope question.  If bioinformatics tooling is implemented as part of a clinical genomics workflow, should the scope of FHIR (which is clinical in nature) expand to include it?  If so, how far upstream should we go?</p>
<p>My current thinking restricts the core of FHIR genomics to clinically reported results (and relevant metadata).  If there is need to support the exchange of file formats used by upstream systems, then we could find a way to connect to and leverage GA4GH (or other) specs to handle the details.</p>



<a name="153998219"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998219" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998219">(Sep 19 2018 at 16:10)</a>:</h4>
<p>I support this notion too, with some caveats. Let's say, for instance, I have an 'in-between' use case, which isn't addressed by an existing bioinformatics API. An advantage of a FHIR GACS Operation is that we define exactly what data we need, and leave it to the GACS to do the translation. My own feeling is that the scope of FHIR Genomics is a bit broader than clinically reported results, and should include vetted and approved clinical genomics use cases. In other words, if there is genomics data that is used by a clinical application, it is in scope for FHIR Genomics.</p>



<a name="153998308"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998308" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998308">(Sep 19 2018 at 17:55)</a>:</h4>
<blockquote>
<p>Where an existing file format is desired to use, then use DocumentReference. It can provide the metadata to any arbitrary file that has a mime-type. This is used for some proprietary formats such as WORD. It is also used for generic forms like non-FHIR xml schema, or even CSV.</p>
</blockquote>
<p><span class="user-mention" data-user-id="191404">@John Moehrke</span>  - If we consider a <a href="https://samtools.github.io/hts-specs/VCFv4.3.pdf" target="_blank" title="https://samtools.github.io/hts-specs/VCFv4.3.pdf">VCF</a>, DocumentReference doesn't feel quite right.  In our case, we will have a DiagnosticReport to track much of the context (subject, encounter ...) and metadata (status, category,  ...), so Attachment still feels right to me.  But I will admit, as discussed in the other thread, the boundaries between some of those options are pretty fuzzy to me.</p>



<a name="153998311"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998311" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998311">(Sep 19 2018 at 18:02)</a>:</h4>
<p>VCF is one of those data types that we would previously have conveyed using Observation.valueAttachment with a reference to a Binary.  Further discussion about whether DocumentReference should/should not be used for such things should happen in the existing <a href="#narrow/stream/4-implementers/subject/Observation.20value.20attachment" title="#narrow/stream/4-implementers/subject/Observation.20value.20attachment">Implementer's thread</a>.</p>



<a name="153998354"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998354" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998354">(Sep 19 2018 at 19:23)</a>:</h4>
<p>We've gotten into some slightly varied topics here and I'm partially at fault for talking about VCF data, but the question of "where we draw the line" with GA4GH and embedding raw bioinformatics data seems to be a concern for future enhancement requests and isn't <em>directly</em> related to the overlap between sequence and observation as it is now.</p>
<p>Toward that end, we accepted a resolution to <a href="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=16252&amp;start=0" target="_blank" title="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=16252&amp;start=0">https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=16252&amp;start=0</a> on July 31st, and stated the committee is "considering options for modifying the resource and anticipates potential changes being brought forward in an upcoming ballot". After reading that and not seeing any other related trackers I'm a bit unsure of the immediate pressure to resolve this overlap and/or reevaluate the domain before balloting again, but here goes:</p>
<p>Apart from the metadata, the overlap seems to be in the ability to describe variants in both places (although it's been said that in Sequence those fields are intended for defining the sequence, and are optional). My suggestion would be to have the IG state that we use observation profiles for describing variants where possible, where they can then be interpreted as parts of complex variants or haplo/genotypes. This would line up with the variant fields being restricted in the previous "definitional" sequence profile that we discontinued.</p>



<a name="153998393"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998393" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998393">(Sep 19 2018 at 22:15)</a>:</h4>
<p>I think this discussion has been good for exploring several things, so short of a concrete comment / proposal from anyone, I could see this very broad discussion being broken down like this:</p>
<p>1 - Log a tracker to update the IG similar to what <span class="user-mention" data-user-id="196553">@James Jones</span> recommended - more clearly documenting when one should use Sequence.variant compared to our IG profiles (perhaps it could include other guidance as well).</p>
<p>2 - Log a tracker for us to consider allowing a VCF to be delivered.</p>
<p>3 - Log a tracker to consider removing Sequence.structuralVariant (I made a comment on the spreadsheet as to why I think we should consider it).</p>
<p>4 - Maybe not a tracker, but have someone volunteer to draft guidance/thoughts on boundaries between FHIR and GA4GH (or other bioinformatic standards).</p>
<p>5 - Not a tracker, but I would still like to workup a draft of what a "sequence profile of Observation" would look like to compare to what we have today and present it to the group.  I had suggested we review it at the WGM, but that might be premature.</p>
<p>Other thoughts?</p>



<a name="153998394"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998394" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Lloyd McKenzie <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998394">(Sep 19 2018 at 22:18)</a>:</h4>
<p>I'm happy to undertake Sequence profile of Observation, but no chance of it being done before (or at) the WGM.</p>



<a name="153998408"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998408" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998408">(Sep 19 2018 at 22:51)</a>:</h4>
<p><span class="user-mention" data-user-id="191913">@Kevin Power</span> I might suggest broadening 3 a bit - there may be other revisions we want to consider for Sequence (such as the representation of quality). I'm happy to help with 4.</p>



<a name="153998422"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/153998422" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#153998422">(Sep 19 2018 at 23:35)</a>:</h4>
<p>Perhaps #3 can stay as is, but we always welcome additional trackers to make things better.  I think your statement also overlaps a bit with #4 anyway, so welcome any drafting you would want to do.</p>



<a name="154022808"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/154022808" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#154022808">(Dec 04 2018 at 17:24)</a>:</h4>
<p>Do we want to reopen some of these discussions re: VCF here or break out a new substream?</p>



<a name="154022810"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence%20/%20Observation%20comparison/near/154022810" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.20.2F.20Observation.20comparison.html#154022810">(Dec 04 2018 at 17:26)</a>:</h4>
<p>New please</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
