---
layout: page
title: "FHIR Chat  · Sequence.readDepth · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html">Sequence.readDepth</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="154006785"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence.readDepth/near/154006785" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html#154006785">(Oct 08 2018 at 18:43)</a>:</h4>
<p>At the WGM, we discussed the following tracker:<br>
<a href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=13834" target="_blank" title="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&amp;tracker_item_id=13834">13834</a> / Add seqeunce.readdepth<br>
We current have <a href="http://build.fhir.org/sequence-definitions.html#Sequence.readCoverage" target="_blank" title="http://build.fhir.org/sequence-definitions.html#Sequence.readCoverage">Sequence.readCoverage</a>, which is defined as "Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence."  It seems that readCoverage is an average depth for the whole test, where perhaps readDepth is usually more at a specific location?  But, we lacked a real expert in the room at the WGM, so anyone else is welcome to weigh in.</p>



<a name="154006959"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence.readDepth/near/154006959" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Dora Walter <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html#154006959">(Oct 09 2018 at 09:07)</a>:</h4>
<p>Quote from our NGS Example Laboratory Findings</p>
<p>Genetic data evaluation:<br>
The sensitivity of the test is depending on the tumor load, the sample quality and <strong>sequencing depth</strong>. A <strong>coverage</strong> of 120 reads per base achieves a theoretical sensitivity of 99% for the detection of variants with a ≥ 10% NAF. In this case, 95.1% of the targeted regions were covered by a minimum of 120 high-quality <strong>sequencing reads per base</strong>. Variants are named according to the HGVS recommendations without any information regarding the cis or trans configuration.</p>
<p>--&gt; It seems to me that both terms are used identically here</p>



<a name="154007292"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence.readDepth/near/154007292" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Joel Schneider <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html#154007292">(Oct 10 2018 at 10:16)</a>:</h4>
<p>The <strong>Sequence.readCoverage</strong> definition says <strong>average</strong> depth, and the <strong>sequencing depth</strong> definition says <strong>minimum</strong> depth.</p>
<p>If average depth is useful, it may be best to specify the type of average (e.g. mean, mode, median) and possibly also a standard deviation.</p>



<a name="154007417"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence.readDepth/near/154007417" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Gideon Giacomelli <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html#154007417">(Oct 10 2018 at 15:03)</a>:</h4>
<p>what about using 82121-5    Allelic read depth as a name for  a coverage for SNPs and small indels (&lt;=30bps) <br>
<a href="https://r.details.loinc.org/LOINC/82121-5.html?sections=Comprehensive" target="_blank" title="https://r.details.loinc.org/LOINC/82121-5.html?sections=Comprehensive">https://r.details.loinc.org/LOINC/82121-5.html?sections=Comprehensive</a></p>



<a name="154007431"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence.readDepth/near/154007431" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Gideon Giacomelli <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html#154007431">(Oct 10 2018 at 15:09)</a>:</h4>
<p>coverage indeed is not precisely defined and there is also an ongoing discussion if the average  or the median should be used. IMO using the median gives a better idea about the quality of the data especially if whole genome sequencing was used</p>



<a name="154008020"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence.readDepth/near/154008020" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Andrea Pitkus, PhD, MLS(ASCP)CM, CSM <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html#154008020">(Oct 11 2018 at 14:38)</a>:</h4>
<p>What are your laboratories actually reporting?  That would best help inform the shape of this item.</p>



<a name="154008073"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/Sequence.readDepth/near/154008073" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Kevin Power <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/Sequence.2EreadDepth.html#154008073">(Oct 11 2018 at 15:54)</a>:</h4>
<p>Here are a few statements from example reports:</p>
<div class="codehilite"><pre><span></span>Exomes are sequenced to an average target coverage of 125X with 90-95% of bases sequenced to at least 20X coverage.

All genes relevant to the indication had adequate coverage (&gt;95% at 20X).

The table below provides the list of these genes (and their coverage at ≥20X).
&lt;list of genes, each with a &quot;20x coverage (%)&quot; value&gt;
</pre></div>


<div class="codehilite"><pre><span></span>&gt;97% of target bases are covered at &gt;20x, mean coverage of target bases &gt;120x.
</pre></div>


<div class="codehilite"><pre><span></span>The Color test has 100% coverage for all regions in our reportable range &gt;20X. Our median cov-
erage across our samples is &gt;250X (can exceed 1000X) and our minimum acceptance criteria

for depth is: &gt;99% at &gt;50X and 100% at 20X. Any exceptions to this are noted in the Limitations
section.
</pre></div>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
