---
layout: page
title: "FHIR Chat  · somatic variant reporting · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html">somatic variant reporting</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="236430933"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/236430933" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#236430933">(Apr 28 2021 at 00:19)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span> , <span class="user-mention" data-user-id="192302">@May Terry</span>  I've probably asked you this before, but I don't understand why some of our example reports only show c.HGVS and p.HGVS. For instance, this example [http://build.fhir.org/ig/HL7/genomics-reporting/Bundle-oncology-report-example.html] includes a variant for FBXW7 (NM_001349798.2:c.1394G&gt;A). Presumably this is a NGS test, where a genomic variant was detected. The FBXW7 gene has multiple transcripts. Why did the lab specifically show this c.HGVS and p.HGVS?</p>



<a name="236431755"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/236431755" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#236431755">(Apr 28 2021 at 00:30)</a>:</h4>
<p>and a corollary question - in this example  [http://build.fhir.org/ig/HL7/genomics-reporting/Bundle-oncology-diagnostic.html], there is a genomic BRAF variant, and a single protein variant. Why, amongst all the BRAF transcripts, was only this protein variant included?</p>



<a name="237634781"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237634781" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237634781">(May 06 2021 at 10:12)</a>:</h4>
<p><span class="user-mention silent" data-user-id="193751">Bob Dolin</span> <a href="#narrow/stream/179197-genomics/topic/somatic.20variant.20reporting/near/236430933">said</a>:</p>
<blockquote>
<p><span class="user-mention silent" data-user-id="191451">Patrick Werner</span> , <span class="user-mention silent" data-user-id="192302">May Terry</span>  I've probably asked you this before, but I don't understand why some of our example reports only show c.HGVS and p.HGVS. For instance, this example [http://build.fhir.org/ig/HL7/genomics-reporting/Bundle-oncology-report-example.html] includes a variant for FBXW7 (NM_001349798.2:c.1394G&gt;A). Presumably this is a NGS test, where a genomic variant was detected. The FBXW7 gene has multiple transcripts. Why did the lab specifically show this c.HGVS and p.HGVS?</p>
</blockquote>
<p>Hi Bob, i'm not 100% if i understood the question correctly.</p>



<a name="237634826"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237634826" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237634826">(May 06 2021 at 10:12)</a>:</h4>
<p>Was the question why they ended up reporting it with this specific transcript?</p>



<a name="237660950"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237660950" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237660950">(May 06 2021 at 13:51)</a>:</h4>
<p>Thanks <span class="user-mention" data-user-id="191451">@Patrick Werner</span> . Yes.  Let's say I do NGS and detect NC_000001.11:g.1014228G&gt;A. I could report ENST00000624697.4:c.224G&gt;A and p.Ser75Asn, or I could report ENST00000649529.1:c.248G&gt;A and p.Ser83Asn. I was curious how a lab decides which to report - is it, for instance, based on transcript prevalence, based on what form is typically annotated, etc?</p>



<a name="237696862"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237696862" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237696862">(May 06 2021 at 17:23)</a>:</h4>
<p>I think this has something todo with <a href="https://www.ncbi.nlm.nih.gov/refseq/MANE/">https://www.ncbi.nlm.nih.gov/refseq/MANE/</a></p>



<a name="237697266"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237697266" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237697266">(May 06 2021 at 17:26)</a>:</h4>
<p><a href="https://www.ncbi.nlm.nih.gov/clinvar/variation/376427/">https://www.ncbi.nlm.nih.gov/clinvar/variation/376427/</a></p>



<a name="237697317"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237697317" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237697317">(May 06 2021 at 17:26)</a>:</h4>
<p>here you can see the different transcripts and which one is the mane transcript</p>



<a name="237697332"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237697332" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237697332">(May 06 2021 at 17:26)</a>:</h4>
<p>but i'm no expert.</p>



<a name="237697393"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237697393" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237697393">(May 06 2021 at 17:27)</a>:</h4>
<p>just my experience, i'll try to ask somebody from the lab</p>



<a name="237753581"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/somatic%20variant%20reporting/near/237753581" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/somatic.20variant.20reporting.html#237753581">(May 07 2021 at 01:10)</a>:</h4>
<p><span class="user-mention" data-user-id="191451">@Patrick Werner</span> Thanks for the pointer to MANE. I hadn't seen that before.</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
