---
layout: page
title: "FHIR Chat  · STU2 Theme: Uncallable subregions in a region studied · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html">STU2 Theme: Uncallable subregions in a region studied</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="183017313"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/183017313" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#183017313">(Dec 10 2019 at 00:51)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="191913">@Kevin Power</span>, <span class="user-mention" data-user-id="196553">@Jamie Jones</span>  , there seemed to be some interest in this topic on the call today, so I thought I'd post something. We've been experimenting with a new component 'no call region', on the region-studied profile. With this component, in response to a query for variants in a given region, a server can provide a report that also enumerates non-callable subregions, potentially enabling one to differentiate non-callable vs. true negatives. The main use case we based this on was the need to include wildtype variant calls for PharmGKB's PharmCAT (<a href="https://github.com/PharmGKB/PharmCAT" target="_blank" title="https://github.com/PharmGKB/PharmCAT">https://github.com/PharmGKB/PharmCAT</a>) software. </p>
<p>The example is a set of variants found for 1000 Genomes patient HG00403, that also include non-callable subregions. (It's based on a slightly outdated pre-publication version of the IG).</p>



<a name="183017357"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/183017357" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#183017357">(Dec 10 2019 at 00:52)</a>:</h4>
<p><a href="/user_uploads/10155/q5-GNiBQp4oT2Kw5dWXBA9Zp/HG00403_NC_000001.10_1000000_1100000_includeNotCallableRegions.json.txt" target="_blank" title="HG00403_NC_000001.10_1000000_1100000_includeNotCallableRegions.json.txt">HG00403_NC_000001.10_1000000_1100000_includeNotCallableRegions.json.txt</a></p>



<a name="183113280"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/183113280" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#183113280">(Dec 10 2019 at 23:23)</a>:</h4>
<p>(entered in Jira as <a href="https://jira.hl7.org/browse/FHIR-25296" target="_blank" title="https://jira.hl7.org/browse/FHIR-25296">https://jira.hl7.org/browse/FHIR-25296</a>)</p>



<a name="183113461"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/183113461" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#183113461">(Dec 10 2019 at 23:26)</a>:</h4>
<p>Linking out of JIRA is a bit funny, looks like it was entered at <a href="https://jira.hl7.org/browse/FHIR-25296" target="_blank" title="https://jira.hl7.org/browse/FHIR-25296">https://jira.hl7.org/browse/FHIR-25296</a>, looks good, thanks!</p>



<a name="183113512"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/183113512" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#183113512">(Dec 10 2019 at 23:27)</a>:</h4>
<p>thanks Jamie</p>



<a name="184750370"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/184750370" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#184750370">(Jan 03 2020 at 17:14)</a>:</h4>
<p>Left a follow-up on the tracker, copying here in case anyone else wants to weigh in:</p>
<p>I love the idea and think this could be a great extension to the region-studied profile as a DiagnosticReport result.</p>
<p>To clarify, the proposal is to add an optional, repeatable component to RegionStudied with a new placeholder code. The value for the component would be a range, using the same coordinate system as the rest of the profile (from LOINC Answer List LL5323-2). Text for this requirement should be added to the element's definition.</p>
<p>To complete the proposal, we will need text for the new concept on <a href="http://hl7.org/fhir/uv/genomics-reporting/tbd-codes.html" target="_blank" title="http://hl7.org/fhir/uv/genomics-reporting/tbd-codes.html">http://hl7.org/fhir/uv/genomics-reporting/tbd-codes.html</a>, including "Display", "Code", and "Definition".</p>
<p>An alternative proposal (that I am not making at this time) would be to represent this information on Variant itself, or in a separate related Observation profile that directly references Variants, as it is related to confirmation status.</p>



<a name="184777287"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/184777287" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#184777287">(Jan 03 2020 at 23:44)</a>:</h4>
<p>Thanks Jamie. Here's a draft: Code='non-callable-region'; Display='non-callable-region';  Definition: </p>
<p>'A non-callable region is a subregion within an observation:region-studied profile that represents a range for which variants are unable to be called. Generally, a region is non-callable due to technical issues surrounding the test. Multiple non-callable regions may be contained within a region studied. </p>
<p>In many cases, a recipient of a FHIR Genomics report may only be interested in identified variants, in which case non-callable regions may not be of use. In some cases however, such as where there is a desire to infer the presence of a wild-type allele at a particular position, it can be useful to know that not only were no variants reported at a position, but also that the position was not deemed non-callable'.</p>



<a name="184778044"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/184778044" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#184778044">(Jan 04 2020 at 00:01)</a>:</h4>
<p>Thanks for the proposal!</p>
<p>It seems you could pretty easily use this approach to approximate the contents of a gVCF file, which may be of interest to some of our research-focused members. </p>
<p>My only reservation with encouraging calculating wild types is that it implies all non-callable regions are represented here, so we may want to draw attention to that, particularly as multiple region-studied instances may be included with a single report and folks may arbitrarily split up region descriptions.</p>



<a name="185639574"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/185639574" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#185639574">(Jan 14 2020 at 20:59)</a>:</h4>
<p>To extend this thought, it may be useful to have a "wildtype block" profile/guidance for Variant. Using the current profile, one could provide a block of contiguous wild type calls with respect to a given reference sequence in a single Observation. You only need to populate the <code>reference sequence</code>, <code>start</code>, <code>end</code>, and set <code>dna-chg-type</code>=SO:0002073/no_sequence_alteration ("WildType"). Mirroring gVCF, you could use using the minimum <code>allelic-read-depth</code> value within the range along with Observation.method to provide more information.</p>



<a name="192389778"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/192389778" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#192389778">(Mar 31 2020 at 13:34)</a>:</h4>
<p>Looks promising.</p>



<a name="207862121"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207862121" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207862121">(Aug 24 2020 at 16:08)</a>:</h4>
<p>A bit of pushback today on call to nail this down to a component. Need more homework to identify and describe use cases for sending callability of specific regions in reports. Modeling this as an extension instead was brought up as the expected stakeholders may be low.</p>



<a name="207919243"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207919243" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207919243">(Aug 25 2020 at 02:23)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> At some point, once we resolve how we want to do this, I'd like to propose adding this component/extension into the $find-subject-variant operation. Here is an example use case:</p>



<a name="207919302"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207919302" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207919302">(Aug 25 2020 at 02:24)</a>:</h4>
<p>Imagine these scenarios, where the top blue line is the region you are querying, the middle purple line is the region studied, and the bottom pink lines are uncallable regions:</p>



<a name="207919309"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207919309" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207919309">(Aug 25 2020 at 02:24)</a>:</h4>
<p><a href="/user_uploads/10155/2MVPaxg3ctrR-qf0AjNewGTD/image.png">image.png</a></p>
<div class="message_inline_image"><a href="/user_uploads/10155/2MVPaxg3ctrR-qf0AjNewGTD/image.png" title="image.png"><img src="/user_uploads/10155/2MVPaxg3ctrR-qf0AjNewGTD/image.png"></a></div>



<a name="207919327"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207919327" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207919327">(Aug 25 2020 at 02:25)</a>:</h4>
<p>My use case is that I want to know the genotype for SNPs 1-5</p>



<a name="207919393"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207919393" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207919393">(Aug 25 2020 at 02:27)</a>:</h4>
<p>If we have an explicit uncallable region (either as a component or extension), I can differentiate, in each of the above cases, whether or not I can infer wildtype (reference base) in the absence of variant call at SNP 1-5</p>



<a name="207964589"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207964589" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Liz Amos <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207964589">(Aug 25 2020 at 13:48)</a>:</h4>
<p><span class="user-mention" data-user-id="193751">@Bob Dolin</span>  I apologize for my ignorance... what is the difference between what we're trying to flesh out with "uncallable region" and genetic sequencing coverage?  I was digging through some old files and realized that Clem/you had proposed a new LOINC code that never made it to being submitted. I've attached the write up as well as an info sheet from Illumnia. With this proposal we are requesting a specific number vs. codifying something as "uncallable" or "low coverage" (if I'm understanding correctly). <a href="/user_uploads/10155/WmfzAORrEP6lAxDZKRszxliF/Sequencing-Coverage-for-NGS-Experiments.pdf">Sequencing-Coverage-for-NGS-Experiments.pdf</a> <a href="/user_uploads/10155/siAlOEQ2GclwjlGQ96IcDm_a/New-LOINC-term-and-observation-component-for-Region-of-interest.docx">New-LOINC-term-and-observation-component-for-Region-of-interest.docx</a></p>



<a name="207967666"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207967666" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207967666">(Aug 25 2020 at 14:12)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="194730">@Liz Amos</span> . I think they are closely related, in that depth of coverage is a major determinant of whether or not a region is deemed callable.  My understanding is that different labs may have different parameters for determining when a region is callable, not solely based on depth of coverage. Also, while, for instance 30x-50x coverage is good for WGS, it might be considered inadequate for WES or targeted gene sequencing - so depth of coverage in and of itself doesn't directly equate with callability.</p>



<a name="207972187"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/207972187" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Liz Amos <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#207972187">(Aug 25 2020 at 14:47)</a>:</h4>
<p>Thanks, <span class="user-mention" data-user-id="193751">@Bob Dolin</span> . I'm assuming we still need the Gene Coverage term/component? I can try to circulate the proposal again so it actually moves forward this time.</p>



<a name="217035703"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/217035703" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#217035703">(Nov 17 2020 at 17:34)</a>:</h4>
<p>Reviewing this older proposal, it still seems hard to rely on the uncallable component for computing wild types. How would we want folks to report 'no uncallable subregions' in their profile? Perhaps the flag structure (or 2 mirrored profiles) would be required... what do you think? Could suggest they enter an uncallable subregion of length 0 but it doesn't seem intuitive.</p>
<p><span class="user-mention silent" data-user-id="193751">Bob Dolin</span> <a href="#narrow/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied/near/184777287">said</a>:</p>
<blockquote>
<p>Thanks Jamie. Here's a draft: Code='non-callable-region'; Display='non-callable-region';  Definition: </p>
<p>'A non-callable region is a subregion within an observation:region-studied profile that represents a range for which variants are unable to be called. Generally, a region is non-callable due to technical issues surrounding the test. Multiple non-callable regions may be contained within a region studied. </p>
<p>In many cases, a recipient of a FHIR Genomics report may only be interested in identified variants, in which case non-callable regions may not be of use. In some cases however, such as where there is a desire to infer the presence of a wild-type allele at a particular position, it can be useful to know that not only were no variants reported at a position, but also that the position was not deemed non-callable'.</p>
</blockquote>



<a name="217052168"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/217052168" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#217052168">(Nov 17 2020 at 19:43)</a>:</h4>
<p>Hmmm, good question. In addition to 'no uncallable subregions' is the case 'there were some, which I'm not bothering to report'. So perhaps some indication that they are / are not included?</p>



<a name="217792687"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/217792687" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#217792687">(Nov 24 2020 at 18:12)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span>  I'm thinking that it might be better to propose this as an extension until we see if it gains any traction. My primary use case is to add it as an optional component into the $find-subject-variants operation. Your point about the need to represent 'no uncallable subregions' could be addressed by another field. </p>
<p>What do you think of something like this, an optional (0..1) extension, added to the region-studied profile:  </p>
<p>0..1 observation-uncallableRegions (complex)<br>
  -- 1..1 includedInReport: boolean<br>
  -- 0..* uncallableRange: integer range</p>



<a name="225583670"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/225583670" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#225583670">(Feb 08 2021 at 18:04)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> Jamie, I added tracker <a href="https://jira.hl7.org/browse/FHIR-31022">https://jira.hl7.org/browse/FHIR-31022</a> to change cardinality of region-studied's ranges-examined and uncallable-regions components. Hopefully I entered the tracker correctly...</p>



<a name="225685443"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/225685443" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Patrick Werner <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#225685443">(Feb 09 2021 at 12:58)</a>:</h4>
<p>looking good!</p>



<a name="228437036"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/228437036" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bret H <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#228437036">(Mar 02 2021 at 16:15)</a>:</h4>
<p>Would it make sense to create branches for trackers like the above that can be merged immediately upon voting?</p>



<a name="230777471"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/230777471" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#230777471">(Mar 17 2021 at 22:14)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span>  Jamie, I added tracker <a href="https://jira.hl7.org/browse/FHIR-31565">https://jira.hl7.org/browse/FHIR-31565</a> to add an uncallableRegion parameter to the find-subject-variants operation.</p>



<a name="235190316"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/STU2%20Theme%3A%20Uncallable%20subregions%20in%20a%20region%20studied/near/235190316" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/STU2.20Theme.3A.20Uncallable.20subregions.20in.20a.20region.20studied.html#235190316">(Apr 19 2021 at 14:32)</a>:</h4>
<p><span class="user-mention" data-user-id="196553">@Jamie Jones</span> Jamie, what do you think about using the uncallable-regions 'dataAbsentReason' field where you want to explicitly say you don't have that information?</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
