---
layout: page
title: "FHIR Chat  · VCF2FHIR: Exomiser results · genomics"
---

{% raw %}<h2>Stream: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/index.html">genomics</a></h2>
<h3>Topic: <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html">VCF2FHIR: Exomiser results</a></h3>

<hr>

<base href="https://chat.fhir.org">

<head><link href="https://fhir.github.io/chat-archive/style.css" rel="stylesheet"></head>

<a name="213939341"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/VCF2FHIR%3A%20Exomiser%20results/near/213939341" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Marta Herranz <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html#213939341">(Oct 20 2020 at 15:54)</a>:</h4>
<p>Hi! I am doing a genetic data mapping and would like to know if anyone has worked with this before.</p>
<p>After running exomiser (<a href="https://exomiser.github.io/Exomiser/">https://exomiser.github.io/Exomiser/</a>) with a set of symptoms and VCF files I get the gene information for a particular case. In this result I have data from:</p>
<ul>
<li>Gene identifier</li>
<li>Mode of inheritance</li>
<li>Some scores: combined, phenotype and variant.</li>
<li>Contributing variants: chromosome, position, ref and alt, frequency data, variant effect, clinVar data (alleleId, interpretations, reviewStatus), etc.</li>
</ul>
<p>From what I have read so far, the genetic information is stored in "Observation" resources: <a href="http://hl7.org/fhir/observation-genetic.html">http://hl7.org/fhir/observation-genetic.html</a>. And that this also has several extensions that I could use.<br>
With all the observation resources created for a patient, then I would have to create a reporting resource that points to all of the above, is that right?</p>
<p>I have been investigating if I could do this mapping automatically since the result files are long enough to try to do it one by one, and I have found the following:<br>
How can I save the genetic information in VCF format I can use this converter to go to FHIR: <a href="https://github.com/openelimu/VCF-2-FHIR">https://github.com/openelimu/VCF-2-FHIR</a></p>
<p>Has anyone ever used it or done this conversion from VCF to FHIR format more or less automatically? </p>
<p>Thank you very much!</p>



<a name="213949227"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/VCF2FHIR%3A%20Exomiser%20results/near/213949227" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Joel Schneider <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html#213949227">(Oct 20 2020 at 17:06)</a>:</h4>
<p>The observation-genetics profiles are deprecated and will be removed in FHIR R5, so those are presently NOT recommended for use.</p>
<p>Instead, the workgroup recommends implementers to utilize the Genomics Reporting Implementation Guide:<br>
<a href="http://hl7.org/fhir/uv/genomics-reporting/">http://hl7.org/fhir/uv/genomics-reporting/</a></p>
<p>Additionally, work on an updated revision to the Genomics Reporting Implementation Guide is currently in progress, and is expected to be ready for publication soon:<br>
<a href="http://build.fhir.org/ig/HL7/genomics-reporting/">http://build.fhir.org/ig/HL7/genomics-reporting/</a></p>



<a name="213950824"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/VCF2FHIR%3A%20Exomiser%20results/near/213950824" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html#213950824">(Oct 20 2020 at 17:18)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="355844">@Marta Herranz</span> . We've released an updated version of the vcf2fhir converter here [https://github.com/elimuinformatics/vcf2fhir]. We plan a future enhancement to include translation of additional annotations (see <a href="https://git.hub.com/elimuinformatics/vcf2fhir/issues/9">https://git.hub.com/elimuinformatics/vcf2fhir/issues/9</a>).</p>



<a name="213979506"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/VCF2FHIR%3A%20Exomiser%20results/near/213979506" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Jamie Jones <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html#213979506">(Oct 20 2020 at 21:07)</a>:</h4>
<p>I'll also add in that the chromosome field will be added in an update to <a href="http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-variant.html">http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-variant.html</a>. We are currently upgrading internal tooling on our implementation guide but further developments will resume soon.</p>



<a name="213995657"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/VCF2FHIR%3A%20Exomiser%20results/near/213995657" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html#213995657">(Oct 21 2020 at 00:27)</a>:</h4>
<p><span class="user-mention" data-user-id="355844">@Marta Herranz</span> I've never used exomiser, but I have, for instance, run a VCF file through snpEff and snpSift, to add ClinVar and other annotations. If I understand what you're trying to, it would be, for instance, to take the annotated VCF, and automatically convert it into FHIR Genomics format? </p>
<p>Here for instance is a single row of a VCF containing annotations. I think it would be pretty straightforward to map this into FHIR format, although would need to make some design / mapping decisions. </p>
<p>chr1    2556714 rs4870;135349   A   G   195.77  PASS    AC=1;AF=0.500;AN=2;BaseQRankSum=-1.079e+00;DB;DP=22;ExcessHet=3.0103;FS=7.282;MLEAC=1;MLEAF=0.500;MQRankSum=0.00;POSITIVE_TRAIN_SITE;QD=8.90;ReadPosRankSum=0.569;SOR=0.452;VQSLOD=0.688;culprit=QD;ANN=G|missense_variant|MODERATE|TNFRSF14|ENSG00000157873|transcript|ENST00000355716.4|protein_coding|1/8|c.50A&gt;G|p.Lys17Arg|349/1707|50/852|17/283||,G|missense_variant|MODERATE|TNFRSF14|ENSG00000157873|transcript|ENST00000435221.6|protein_coding|2/7|c.50A&gt;G|p.Lys17Arg|581/1225|50/694|17/230||WARNING_TRANSCRIPT_INCOMPLETE,G|missense_variant|MODERATE|TNFRSF14|ENSG00000157873|transcript|ENST00000426449.5|protein_coding|2/6|c.50A&gt;G|p.Lys17Arg|159/660|50/551|17/182||WARNING_TRANSCRIPT_INCOMPLETE,G|missense_variant|MODERATE|TNFRSF14|ENSG00000157873|transcript|ENST00000434817.5|protein_coding|2/7|c.50A&gt;G|p.Lys17Arg|190/834|50/694|17/230||WARNING_TRANSCRIPT_INCOMPLETE,G|missense_variant|MODERATE|TNFRSF14|ENSG00000157873|transcript|ENST00000451778.5|protein_coding|2/7|c.50A&gt;G|p.Lys17Arg|183/827|50/694|17/230||WARNING_TRANSCRIPT_INCOMPLETE,G|missense_variant|MODERATE|TNFRSF14|ENSG00000157873|transcript|ENST00000409119.5|protein_coding|2/7|c.50A&gt;G|p.Lys17Arg|170/896|50/582|17/193||,G|upstream_gene_variant|MODIFIER|TNFRSF14-AS1|ENSG00000238164|transcript|ENST00000432521.2|pseudogene||n.-2564T&gt;C|||||2564|,G|upstream_gene_variant|MODIFIER|TNFRSF14-AS1|ENSG00000238164|transcript|ENST00000449660.5|pseudogene||n.-4608T&gt;C|||||4608|,G|upstream_gene_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000466750.5|retained_intron||n.-1099A&gt;G|||||1099|,G|upstream_gene_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000463835.5|retained_intron||n.-3840A&gt;G|||||3840|,G|upstream_gene_variant|MODIFIER|TNFRSF14-AS1|ENSG00000238164|transcript|ENST00000443892.2|pseudogene||n.-2469T&gt;C|||||2469|,G|upstream_gene_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000471768.1|retained_intron||n.-2866A&gt;G|||||2866|,G|upstream_gene_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000482602.5|protein_coding||c.-3969A&gt;G|||||3968|WARNING_TRANSCRIPT_NO_START_CODON,G|intron_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000475523.5|retained_intron|1/5|n.70+259A&gt;G||||||,G|intron_variant|MODIFIER|TNFRSF14-AS1|ENSG00000238164|transcript|ENST00000416860.2|pseudogene|1/5|n.36-18T&gt;C||||||,G|intron_variant|MODIFIER|TNFRSF14-AS1|ENSG00000238164|transcript|ENST00000452793.1|pseudogene|1/3|n.56-18T&gt;C||||||,G|non_coding_transcript_exon_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000442392.6|processed_transcript|1/4|n.310A&gt;G||||||,G|non_coding_transcript_exon_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000482074.6|retained_intron|1/3|n.309A&gt;G||||||,G|non_coding_transcript_exon_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000496064.5|retained_intron|1/5|n.158A&gt;G||||||,G|non_coding_transcript_exon_variant|MODIFIER|TNFRSF14|ENSG00000157873|transcript|ENST00000463471.6|retained_intron|1/5|n.144A&gt;G||||||;AF_ESP=0.55803;AF_EXAC=0.54431;AF_TGP=0.61482;ALLELEID=139088;CLNDISDB=MedGen:CN169374;CLNDN=not_specified;CLNHGVS=NC_000001.11:g.2556714A&gt;G;CLNREVSTAT=no_assertion_provided;CLNSIG=not_provided;CLNVC=single_nucleotide_variant;CLNVCSO=SO:0001483;CLNVI=UniProtKB_(protein):Q92956#VAR_013007;GENEINFO=TNFRSF14:8764|TNFRSF14-AS1:115110;MC=SO:0001583|missense_variant;ORIGIN=1;RS=4870    GT:AD:DP:GQ:PL  0/1:13,9:22:99:224,0,394</p>



<a name="214159244"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/VCF2FHIR%3A%20Exomiser%20results/near/214159244" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Marta Herranz <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html#214159244">(Oct 22 2020 at 08:52)</a>:</h4>
<p>Thank you all very much for your help.</p>
<p>I have tried to install the release of the converter from <a href="https://github.com/elimuinformatics/vcf2fhir">https://github.com/elimuinformatics/vcf2fhir</a> but I am having problems.</p>
<p>I cloned the github project and followed the steps explained at <a href="https://vcf2fhir.readthedocs.io/en/latest/user/install.html">https://vcf2fhir.readthedocs.io/en/latest/user/install.html</a> but it won't let me run the "pip install vcf2fhir" command. It gives me errors with pysam:cython, I read and thought that first I had to run the "python setup.py build" and then the installation, but it still doesn't work. Besides, I can't use vcf2fhir/bin/activate source either because in the vcf2fhir folder when I clone the project there is no bin folder.</p>
<p>I have previously installed <a href="https://github.com/openelimu/VCF-2-FHIR">https://github.com/openelimu/VCF-2-FHIR</a>, can it be that I am having problems with that?</p>
<p>Can someone tell me the steps to install it and be able to use it?</p>



<a name="214190701"></a>
<h4><a href="https://chat.fhir.org#narrow/stream/179197-genomics/topic/VCF2FHIR%3A%20Exomiser%20results/near/214190701" class="zl"><img src="https://fhir.github.io/chat-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Bob Dolin <a href="https://fhir.github.io/chat-archive/stream/179197-genomics/topic/VCF2FHIR.3A.20Exomiser.20results.html#214190701">(Oct 22 2020 at 13:59)</a>:</h4>
<p>Hi <span class="user-mention" data-user-id="355844">@Marta Herranz</span> I'll respond to your PM</p>


{% endraw %}
<hr><p>Last updated: Apr 12 2022 at 19:14 UTC</p>
